<html class="js svg" lang="en"><head>
<title>Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces | Scientific Reports</title>
<link as="font" crossorigin="" href="https://www.nature.com/articles//static/fonts/Lora-Regular.8861b0072d.woff2" rel="preload" type="font/woff2"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=2.5,user-scalable=yes" name="viewport"/>
<script src="https://www.googletagservices.com/activeview/js/current/osd.js?cb=%2Fr20100101"></script><script async="" src="https://connect.facebook.net/en_US/fbevents.js"></script><script async="" src="//static.ads-twitter.com/uwt.js"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_rendering_2019070101.js"></script><script src="https://cdn.krxd.net/userdata/get?pub=bd339c69-af54-4a21-b4f1-654bcfcd83ca&amp;technographics=1&amp;callback=Krux.ns.macmillan.kxjsonp_userdata" type="text/javascript"></script><script src="https://beacon.krxd.net/optout_check?callback=Krux.ns.macmillan.kxjsonp_optOutCheck" type="text/javascript"></script><script src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1" type="text/javascript"></script><script src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0" type="text/javascript"></script><script async="" src="https://www.google-analytics.com/plugins/ua/linkid.js" type="text/javascript"></script><script async="" src="//cdn.krxd.net/ctjs/controltag.js.5c4175ffbc462aa356544918ef5c9285"></script><script async="" src="//www.googletagservices.com/tag/js/gpt.js"></script><script async="" src="https://www.google-analytics.com/analytics.js" type="text/javascript"></script><script async="" src="https://cdn.krxd.net/controltag?confid=Jn7TaPwq" type="text/javascript"></script><script async="" src="https://recommended.springernature.com/latest/entry-point.js" type="text/javascript"></script><script async="" defer="" src="//get.s-onetag.com/c692e01a-b65d-4aa9-aff9-bee34f082a39/tag.min.js" type="text/javascript"></script><script async="" src="https://sb.scorecardresearch.com/beacon.js" type="text/javascript"></script><script async="" src="//cdn.cookielaw.org/consent/792a5a21-3d99-4432-8f70-957d389bf2b9.js" type="text/javascript"></script><script async="" src="//scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-NWDMT9Q"></script><script>
    (function(e){var t=e.documentElement,n=e.implementation;t.className='js';if(n&&n.hasFeature('http://www.w3.org/TR/SVG11/feature#Image','1.1')){t.className+=' svg'}})(document)
</script>
<script>window.abTestSharedArticleRenderer = true;</script>
<script id="mosaic-loader">
        (function (win, doc, bundles) {
            if (!doc.querySelector) { return; }

            var script = function(attrs) {
                var s = doc.createElement('script');
                for (var attr in attrs) {
                    if (attrs.hasOwnProperty(attr)) {
                        s[attr] = attrs[attr];
                    }
                }
                return s;
            };

            var insertAfter = function(node, dest) {
                dest.parentNode.insertBefore(node, dest.nextSibling);
            };

            win.Component = win.Component || {};
            win.Component.Loader = {
                init: function() {
                    this.main = script({src: '/static/js/nature.f855f9d4b4.js', id: 'mosaic', defer: true});
                    insertAfter(this.main, doc.getElementById('mosaic-loader'));
                },
                loadPageBundles: function() {
                    var lastId = this.main.id;
                    for (var i = 0; bundles[i]; ++i) {
                        if (doc.querySelector(bundles[i].match)) {
                            insertAfter(script({src: bundles[i].src, id: 'bundle' + i, defer: true}), doc.getElementById(lastId));
                            lastId = 'bundle' + i;
                        }
                    }
                }
            };
            win.Component.Loader.init();

        })(window, document, [
            {match: 'div[data-pan-container]', src: '/static/js/pan-zoom.3e0716c442.js'},
            {match: 'div.js-header-container2', src: '/static/js/header2.063dc5869e.js'},
            {match: 'math,span.mathjax-tex', src: '/static/js/math.f961232688.js'},
            {match: 'div.c-article-metrics__wrapper', src: '/static/js/graphing.09371cfbe8.js'},
            {match: 'body', src: 'https://verify.nature.com/verify/nature.min.js'}
        ]);
    </script><script defer="" id="mosaic" src="https://www.nature.com/articles//static/js/nature.f855f9d4b4.js"></script><script defer="" id="bundle4" src="https://verify.nature.com/verify/nature.min.js"></script>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-114x114.1431047068.png" rel="apple-touch-icon" sizes="114x114"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-144x144.3e61d1f755.png" rel="apple-touch-icon" sizes="144x144"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-180x180.5789768d66.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-16x16.bfd0e25c74.png" rel="icon" sizes="16x16" type="image/png"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-32x32.1c4a03347a.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-96x96.14ff9c7f48.png" rel="icon" sizes="96x96" type="image/png"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon-194x194.5789768d66.png" rel="icon" sizes="194x194" type="image/png"/>
<link href="https://www.nature.com/articles//static/images/favicons/nature/favicon.14515a9f85.ico" rel="shortcut icon"/>
<meta content="#940720" name="msapplication-TileColor"/>
<meta content="/static/images/favicons/nature/favicon-144x144.3e61d1f755.png" name="msapplication-TileImage"/>
<meta content="#940720" name="theme-color"/>
<link href="https://www.nature.com/articles//static/css/grade-c.429184c4d6.css" rel="stylesheet"/>
<link data-test="open-access-css" href="https://www.nature.com/articles//static/css/nature-open-access.c021180f63.css" id="mustard" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)" rel="stylesheet"/>
<link href="https://www.nature.com/articles//static/css/article.7bc74e7e66.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)" rel="stylesheet" type="text/css"/>
<link href="https://www.nature.com/articles//static/css/article-print.1ed0c342ce.css" media="print" rel="stylesheet" type="text/css"/>
<style>
        .background-brand-primary {
            background-color: #cedde4;
            color: #222;
        }

        .background-brand-secondary {
            background-color: #F1F9FC;
            color: #222;
        }

        .background-brand-secondary-pdf {
            background-color: #036599;
            background-size: 22px auto;
            background-position: 80% 50%;
            color: #ffffff;
        }

        .background-brand-secondary-pdf a {
        color: #ffffff;
        }

        .background-brand-tertiary {
            background-color: #FBFEFF;
            color: #222;
        }

        .background-gradient-brand-primary {
            background-image: linear-gradient(to right, #cedde4, #cedde4 50%, transparent 50%);
        }

        .background-gradient-brand-secondary-tertiary {
            background-image: linear-gradient(to right, #F1F9FC, #F1F9FC 50%, #FBFEFF 50%);
        }

        .background-gradient-brand-gray {
            background-image: linear-gradient(to right, #e0e0e0, #e0e0e0 50%, transparent 50%);
        }

        .background-gradient-brand-gray-light {
            background-image: linear-gradient(to right, #eee, #eee 50%, transparent 50%);
        }


        

        

        


        .nav-border {
            border-bottom: 4px solid #F1F9FC;
        }

        /* less than or equal to 875px */
        @media only screen and (max-width: 54.688em) {
            .banner {
                background-color: #cedde4;
            }

            .nav-border .menu-cell li {
                border-bottom: 1px solid #F1F9FC;
            }
        }

        .ie7 .small-header-main,
        .ie8 .small-header-main {
            background-color: #cedde4;
        }

        .sticky-header .inner-banner,
        .header-primary-color {
            background-color: #cedde4;
        }

        
            .menu-button,
            .menu-button-clone,
            .header-submit-button {
                border: 1px solid #222;
                border: 1px solid rgba(34,34,34,.75);
                color: #222;
            }
            .nature-research-logo {
                background-image: url("/static/images/product-logos/nature-research-logo-dark.2dec4e5439.svg"), none;
            }

            .ie7 .tools-menu-button-icon,
            .ie8 .tools-menu-button-icon,
            .icon-rotate.tools-menu-button-icon:after,
            .menu-button-icon:after {
                background-image: url("/static/images/icons/icon-arrow-down-12x7-gray-dark.d2d673ff2e.svg"), none;
            }
            .js-header-container2 .menu-button-icon:after {
                background-image: url("/static/images/icons/icon-chevron-down-24x24-gray-dark.svg"), none;
            }

            .header-top-bar {
                border-bottom: 1px solid #222;
                border-bottom: 1px solid rgba(34,34,34,.75);
            }

            .header-top-bar a {
                color: #222;
            }
            .header-search-button {
                padding-right: 25px;
                background-image: url("/static/images/icons/icon-search-24x24-gray.d711eca805.svg"), none;
            }
        

    </style>
<script type="text/javascript">
document.addEventListener('accessdetailsloaded', function(e) {
    var dL, doi, pageType, eventObject, i, script, loginEventData;

    for (i = 0; i < window.dataLayer.length; i++) {
        if (window.dataLayer[i]["content"] && window.dataLayer[i]["page"]) {
            dL = window.dataLayer[i];
        }
    }

    if (!dL
        || !dL.content.article
        || !(doi = dL.content.article.doi) && doi.length > 0
        || (pageType = dL.page.category.pageType.toLowerCase()) !== "article"
    ) {return false;}

    pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);

    eventObject = {'content_type': pageType, 'doi': doi};

    loginEventData = e.detail;
    if (loginEventData && loginEventData.business_partner_id) {
        eventObject['business_partner_ids'] = loginEventData.business_partner_id;
    }

    script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function() {
        new EventTracker({'platform': 'Nature'}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>
<script>
    dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"genomics;population-genetics","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Scientific Reports","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41598-018-20427-9"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Kenneth L. Chiou","Christina M. Bergey"],"publishedAt":1517356800,"publishedAtString":"2018-01-31","title":"Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"srep","title":"scientific reports","volume":"8","issue":"1"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":false}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"ab_test_header","active":false},{"name":"ab_test_subscribe_button","active":true},{"name":"ab_test_briefing_new_signup_box","active":true},{"name":"ab_test_best_available_version","active":false}]},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null}];
</script>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-NWDMT9Q');</script>
<meta content="Article" name="description"/>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<meta content="Article" name="WT.cg_s"/>
<meta content="a" name="WT.z_bandiera_abtest"/>
<meta content="Article_HTML" name="WT.page_categorisation"/>
<meta content="oscar" name="WT.template"/>
<meta content="Nature Research Journals" name="WT.z_cg_type"/>
<meta content="Scientific Reports" name="WT.cg_n"/>
<meta content="©2019 Macmillan Publishers Limited. All Rights Reserved." name="dc.rights"/>
<meta content="2045-2322" name="prism.issn"/>
<link href="http://www.nature.com/search" rel="search"/>
<link href="http://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="http://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<meta content="41598" name="journal_id"/>
<meta content="Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces" name="dc.title"/>
<meta content="Scientific Reports 2018 8:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2018-01-31" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2018 The Author(s)" name="dc.copyright"/>
<meta content="permissions@nature.com" name="dc.rightsAgent"/>
<meta content="Obtaining high-quality samples from wild animals is a major obstacle for genomic studies of many taxa, particularly at the population level, as collection methods for such samples are typically invasive. DNA from feces is easy to obtain noninvasively, but is dominated by bacterial and other non-host DNA. The high proportion of non-host DNA drastically reduces the efficiency of high-throughput sequencing for host animal genomics. To address this issue, we developed an inexpensive capture method for enriching host DNA from noninvasive fecal samples. Our method exploits natural differences in CpG-methylation density between vertebrate and bacterial genomes to preferentially bind and isolate host DNA from majority-bacterial samples. We demonstrate that the enrichment is robust, efficient, and compatible with downstream library preparation methods useful for population studies (e.g., RADseq). Compared to other enrichment strategies, our method is quick and inexpensive, adding only a negligible cost to sample preparation. In combination with downstream methods such as RADseq, our approach allows for cost-effective and customizable genomic-scale genotyping that was previously feasible in practice only with invasive samples. Because feces are widely available and convenient to collect, our method empowers researchers to explore genomic-scale population-level questions in organisms for which invasive sampling is challenging or undesirable." name="dc.description"/>
<meta content="2045-2322" name="prism.issn"/>
<meta content="Scientific Reports" name="prism.publicationName"/>
<meta content="2018-01-31" name="prism.publicationDate"/>
<meta content="8" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1975" name="prism.startingPage"/>
<meta content="2018 The Author(s)" name="prism.copyright"/>
<meta content="permissions@nature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41598-018-20427-9" name="prism.url"/>
<meta content="doi:10.1038/s41598-018-20427-9" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41598-018-20427-9.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41598-018-20427-9" name="citation_fulltext_html_url"/>
<meta content="Scientific Reports" name="citation_journal_title"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2045-2322" name="citation_issn"/>
<meta content="Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces" name="citation_title"/>
<meta content="8" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2018/01/31" name="citation_online_date"/>
<meta content="1975" name="citation_firstpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="doi:10.1038/s41598-018-20427-9" name="dc.identifier"/>
<meta content="10.1038/s41598-018-20427-9" name="DOI"/>
<meta content="10.1038/s41598-018-20427-9" name="citation_doi"/>
<meta content="Kenneth L. Chiou" name="dc.creator"/>
<meta content="Christina M. Bergey" name="dc.creator"/>
<meta content="Genomics" name="dc.subject"/>
<meta content="Population genetics" name="dc.subject"/>
<meta content="citation_journal_title=Trends Ecol. Evol.; citation_title=Facts from feces revisited; citation_author=MH Kohn, RK Wayne; citation_volume=12; citation_publication_date=1997; citation_pages=223-227; citation_doi=10.1016/S0169-5347(97)01050-1; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=Mamm. Rev.; citation_title=Facts from faeces; citation_author=RJ Putman; citation_volume=14; citation_publication_date=1984; citation_pages=79-97; citation_doi=10.1111/j.1365-2907.1984.tb00341.x; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Excrement analysis by PCR; citation_author=M Höss, M Kohn, S Pääbo, F Knauer, W Schröder; citation_volume=359; citation_publication_date=1992; citation_pages=199; citation_doi=10.1038/359199a0; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=A human gut microbial gene catalogue established by metagenomic sequencing; citation_author=J Qin; citation_volume=464; citation_publication_date=2010; citation_pages=59-65; citation_doi=10.1038/nature08821; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol.; citation_title=Genomic-scale capture and sequencing of endogenous DNA from feces; citation_author=GH Perry, JC Marioni, P Melsted, Y Gilad; citation_volume=19; citation_publication_date=2010; citation_pages=5332-5344; citation_doi=10.1111/j.1365-294X.2010.04888.x; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=Genetics; citation_title=Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples; citation_author=N Snyder-Mackler; citation_volume=203; citation_publication_date=2016; citation_pages=699-714; citation_doi=10.1534/genetics.116.187492; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Diagn.; citation_title=Enhanced retrieval of DNA from human fecal samples results in improved performance of colorectal cancer screening test; citation_author=D Whitney; citation_volume=6; citation_publication_date=2004; citation_pages=386-395; citation_doi=10.1016/S1525-1578(10)60536-3; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=Hybrid selection for sequencing pathogen genomes from clinical samples; citation_author=A Melnikov; citation_volume=12; citation_publication_date=2011; citation_doi=10.1186/gb-2011-12-8-r73; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries; citation_author=ML Carpenter; citation_volume=93; citation_publication_date=2013; citation_pages=852-864; citation_doi=10.1016/j.ajhg.2013.10.002; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Direct selection of human genomic loci by microarray hybridization; citation_author=TJ Albert; citation_volume=4; citation_publication_date=2007; citation_pages=903-905; citation_doi=10.1038/nmeth1111; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Microarray-based genomic selection for high-throughput resequencing; citation_author=DT Okou; citation_volume=4; citation_publication_date=2007; citation_pages=907-909; citation_doi=10.1038/nmeth1109; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Biotechnol.; citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing; citation_author=A Gnirke; citation_volume=27; citation_publication_date=2009; citation_pages=182-189; citation_doi=10.1038/nbt.1523; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Targeted capture and massively parallel sequencing of 12 human exomes; citation_author=SB Ng; citation_volume=461; citation_publication_date=2009; citation_pages=272-276; citation_doi=10.1038/nature08250; citation_id=CR13" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=Hybridization capture using RAD probes (hyRAD), a new tool for performing genomic analyses on museum collection specimens; citation_author=T Suchan; citation_volume=11; citation_publication_date=2016; citation_pages=e0151651; citation_doi=10.1371/journal.pone.0151651; citation_id=CR14" name="citation_reference"/>
<meta content="citation_journal_title=Genetics; citation_title=RAD Capture (Rapture): flexible and efficient sequence-based genotyping; citation_author=OA Ali; citation_volume=202; citation_publication_date=2016; citation_pages=389-400; citation_doi=10.1534/genetics.115.183665; citation_id=CR15" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol. Resour.; citation_title=RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data; citation_author=SL Hoffberg; citation_volume=16; citation_publication_date=2016; citation_pages=1264-1278; citation_doi=10.1111/1755-0998.12566; citation_id=CR16" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=Multiplexed DNA sequence capture of mitochondrial genomes using PCR products; citation_author=T Maricic, M Whitten, S Pääbo; citation_volume=5; citation_publication_date=2010; citation_pages=e14004; citation_doi=10.1371/journal.pone.0014004; citation_id=CR17" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Efficient cross-species capture hybridization and next-generation sequencing of mitochondrial genomes from noninvasively sampled museum specimens; citation_author=VC Mason, G Li, KM Helgen, WJ Murphy; citation_volume=21; citation_publication_date=2011; citation_pages=1695-1704; citation_doi=10.1101/gr.120196.111; citation_id=CR18" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol.; citation_title=Unlocking the vault: next-generation museum population genomics; citation_author=K Bi; citation_volume=22; citation_publication_date=2013; citation_pages=6018-6032; citation_doi=10.1111/mec.12516; citation_id=CR19" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection; citation_author=RD George; citation_volume=21; citation_publication_date=2011; citation_pages=1686-1694; citation_doi=10.1101/gr.121327.111; citation_id=CR20" name="citation_reference"/>
<meta content="citation_journal_title=Trends Genet.; citation_title=Finding the lost treasures in exome sequencing data; citation_author=DC Samuels; citation_volume=29; citation_publication_date=2013; citation_pages=593-599; citation_doi=10.1016/j.tig.2013.07.006; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=A method for selectively enriching microbial DNA from contaminating vertebrate host DNA; citation_author=GR Feehery; citation_volume=8; citation_publication_date=2013; citation_pages=e76096; citation_doi=10.1371/journal.pone.0076096; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=Trends Genet.; citation_title=The methyl-CpG binding domain and the evolving role of DNA methylation in animals; citation_author=B Hendrich, S Tweedie; citation_volume=19; citation_publication_date=2003; citation_pages=269-277; citation_doi=10.1016/S0168-9525(03)00080-5; citation_id=CR23" name="citation_reference"/>
<meta content="citation_journal_title=Gene; citation_title=Cytosine methylation and CpG, TpG (CpA) and TpA frequencies; citation_author=K Jabbari, G Bernardi; citation_volume=333; citation_publication_date=2004; citation_pages=143-149; citation_doi=10.1016/j.gene.2004.02.043; citation_id=CR24" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Biotechnol.; citation_title=Genome-wide mapping of methylated adenine residues in pathogenic Escherichia coli using single-molecule real-time sequencing; citation_author=G Fang; citation_volume=30; citation_publication_date=2012; citation_pages=1232-1239; citation_doi=10.1038/nbt.2432; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=The methylomes of six bacteria; citation_author=IA Murray; citation_volume=40; citation_publication_date=2012; citation_pages=11450-11462; citation_doi=10.1093/nar/gks891; citation_id=CR26" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species; citation_author=BK Peterson, JN Weber, EH Kay, HS Fisher, HE Hoekstra; citation_volume=7; citation_publication_date=2012; citation_pages=e37135; citation_doi=10.1371/journal.pone.0037135; citation_id=CR27" name="citation_reference"/>
<meta content="citation_journal_title=Am. J. Hum. Genet.; citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S Purcell; citation_volume=81; citation_publication_date=2007; citation_pages=559-575; citation_doi=10.1086/519795; citation_id=CR28" name="citation_reference"/>
<meta content="citation_journal_title=Gigascience; citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets; citation_author=CC Chang; citation_volume=4; citation_publication_date=2015; citation_doi=10.1186/s13742-015-0047-8; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol.; citation_title=The effect of RAD allele dropout on the estimation of genetic variation within and between populations; citation_author=M Gautier; citation_volume=22; citation_publication_date=2013; citation_pages=3165-3178; citation_doi=10.1111/mec.12089; citation_id=CR30" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture; citation_author=N Rohland, D Reich; citation_volume=22; citation_publication_date=2012; citation_pages=939-946; citation_doi=10.1101/gr.128124.111; citation_id=CR31" name="citation_reference"/>
<meta content="citation_journal_title=Wildlife Biol.; citation_title=A simple and effective method for obtaining mammal DNA from faeces; citation_author=A Ramón-Laca, L Soriano, D Gleeson, JA Godoy; citation_volume=21; citation_publication_date=2015; citation_pages=195-203; citation_doi=10.2981/wlb.00096; citation_id=CR32" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=In vivo methylation of mtDNA reveals the dynamics of protein-mtDNA interactions; citation_author=AP Rebelo, SL Williams, CT Moraes; citation_volume=37; citation_publication_date=2009; citation_pages=6701-6715; citation_doi=10.1093/nar/gkp727; citation_id=CR33" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Nuclear DNA from primate dung; citation_author=JJ Constable, C Packer, DA Collins, AE Pusey; citation_volume=373; citation_publication_date=1995; citation_pages=393-393; citation_doi=10.1038/373393a0; citation_id=CR34" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol.; citation_title=Amplification of hypervariable simple sequence repeats (microsatellites) from excremental DNA of wild living bonobos (Pan paniscus); citation_author=U Gerloff; citation_volume=4; citation_publication_date=1995; citation_pages=515-518; citation_doi=10.1111/j.1365-294X.1995.tb00247.x; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Ecol.; citation_title=&amp; Vigilant, L. Quantitative polymerase chain reaction analysis of DNA from noninvasive samples for accurate microsatellite genotyping of wild chimpanzees (Pan troglodytes verus); citation_author=PA Morin, KE Chambers, C Boesch; citation_volume=10; citation_publication_date=2001; citation_pages=1835-1844; citation_doi=10.1046/j.0962-1083.2001.01308.x; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=J. Clin. Microbiol.; citation_title=Contamination and sensitivity issues with a real-time universal 16S rRNA PCR; citation_author=CE Corless; citation_volume=38; citation_publication_date=2000; citation_pages=1747-1752; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=1754-1760; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Genet.; citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data; citation_author=MA DePristo; citation_volume=43; citation_publication_date=2011; citation_pages=491-498; citation_doi=10.1038/ng.806; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=The variant call format and VCFtools; citation_author=P Danecek; citation_volume=27; citation_publication_date=2011; citation_pages=2156-2158; citation_doi=10.1093/bioinformatics/btr330; citation_id=CR40" name="citation_reference"/>
<meta content="Kenneth L. Chiou" name="citation_author"/>
<meta content="Department of Anthropology, Washington University, St. Louis, USA" name="citation_author_institution"/>
<meta content="Department of Psychology, University of Washington, Seattle, USA" name="citation_author_institution"/>
<meta content="Christina M. Bergey" name="citation_author"/>
<meta content="Department of Anthropology, New York University, New York, USA" name="citation_author_institution"/>
<meta content="New York Consortium in Evolutionary Primatology, New York, USA" name="citation_author_institution"/>
<meta content="Department of Anthropology, Pennsylvania State University, University Park, USA" name="citation_author_institution"/>
<meta content="Department of Biology, Pennsylvania State University, University Park, USA" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="summary" name="twitter:card"/>
<meta content="Methylation-based enrichment facilitates low-cost, noninvasive genomic" name="twitter:title"/>
<meta content="@SciReports" name="twitter:site"/>
<meta content="Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces" name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png" name="twitter:image"/>
<meta content="cc_by" name="WT.z_cc_license_type"/>
<meta content="Research" name="WT.z_primary_atype"/>
<meta content="Genomics;Population genetics" name="WT.z_subject_term"/>
<meta content="genomics;population-genetics" name="WT.z_subject_term_id"/>
<meta content="https://www.nature.com/articles/s41598-018-20427-9" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Scientific Reports" property="og:site_name"/>
<meta content="Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces" property="og:title"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png" property="og:image"/>
<script async="" src="https://injections.readcube.com/nature/inject.46925abf.js" type="text/javascript"></script><script async="" src="https://api-gateway.readcube.com/injection/nature_status?doi=10.1038/s41598-018-20427-9&amp;format=js&amp;last_modified=2018-12-19" type="text/javascript"></script><link href="https://injections.readcube.com/styles/nature_checkout.d4ed8d87.css" rel="stylesheet" type="text/css"/><script id="springerlink-event-tracker" src="https://event-tracker.springernature.com/dist/eventTracker.js"></script><script class="kxct" data-id="Jn7TaPwq" data-timing="async" data-version="3.0">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var k=document.createElement("script");k.type="text/javascript";k.async=true;var m,src=(m=location.href.match(/\bkxsrc=([^&]+)/))&&decodeURIComponent(m[1]);k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test(src) ? src : src === "disable" ? "" :(location.protocol==="https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid=Jn7TaPwq";var s=document.getElementsByTagName("script")[0];s.parentNode.insertBefore(k,s);}());</script><link href="https://cdn.cookielaw.org/skins/4.1.0/default_flat_top_two_button_black/v2/css/optanon.css" rel="stylesheet" type="text/css"/><style>#optanon ul#optanon-menu li { background-color:  !important }#optanon ul#optanon-menu li.menu-item-selected { background-color:  !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #006699 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #006699 !important; border-color: #006699 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle button { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle button { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #F2F2F2 !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #F2F2F2 !important }.optanon-alert-box-wrapper { background-color:#222222 !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#FFFFFF !important }#optanon-popup-body-left-shading {
background-color:red!important;
}</style><script src="https://beacon.s-onetag.com/beacon.min.js"></script><link as="script" href="https://adservice.google.com.ua/adsid/integrator.js?domain=www.nature.com" rel="preload"/><script src="https://adservice.google.com.ua/adsid/integrator.js?domain=www.nature.com" type="text/javascript"></script><link as="script" href="https://adservice.google.com/adsid/integrator.js?domain=www.nature.com" rel="preload"/><script src="https://adservice.google.com/adsid/integrator.js?domain=www.nature.com" type="text/javascript"></script><script async="" src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2019070101.js"></script><link href="https://tpc.googlesyndication.com/safeframe/1-0-35/html/container.html" rel="prefetch"/></head>
<body class="article-page"><div aria-describedby="alert-box-message" aria-labelledby="alert-box-title" class="optanon-alert-box-wrapper" role="alertdialog" style="top: -121.629px;"><div class="optanon-alert-box-bottom-top"></div><div class="optanon-alert-box-bg"><div class="optanon-alert-box-logo"> </div><div class="optanon-alert-box-body"><p aria-level="2" class="optanon-alert-box-title legacy-banner-title sr-only" id="alert-box-title" role="heading">Cookie Notice</p><p class="banner-content" id="alert-box-message">We use cookies to personalise content and ads, to provide social media features and to analyse our traffic. We also share information about your use of our site with our social media, advertising and analytics partners in accordance with our <a aria-label="Privacy Policy" class="banner-policy-link" href="https://www.nature.com/info/privacy.html" tabindex="1">Privacy Policy</a>. You can manage your preferences in 'Manage Cookies'.</p></div><div class="optanon-clearfix"></div><div class="optanon-alert-box-button-container"><div class="optanon-alert-box-button optanon-button-close"><div class="optanon-alert-box-button-middle"><button aria-label="Close" class="optanon-alert-box-close">Close</button></div></div><div class="optanon-alert-box-button optanon-button-allow"><div class="optanon-alert-box-button-middle accept-cookie-container"><button aria-label="OK" class="optanon-allow-all accept-cookies-button" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Accept Cookies');" tabindex="3" title="OK">OK</button></div></div><div class="optanon-alert-box-button optanon-button-more"><div class="optanon-alert-box-button-middle"><button aria-label="Manage Cookies" class="optanon-toggle-display cookie-settings-button" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Open Preferences');" tabindex="2" title="Manage Cookies">Manage Cookies</button></div></div></div><div class="optanon-clearfix optanon-alert-box-bottom-padding"></div></div></div><div class="modern" id="optanon"><div id="optanon-popup-bg"></div><div aria-modal="true" id="optanon-popup-wrapper" lang="en-GB" role="dialog" tabindex="-1"><div id="optanon-popup-top"><button aria-label="Close" class="optanon-close-link optanon-close optanon-close-ui" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Close Button');" title="Close"><div id="optanon-close" style="background: url(https://cdn.cookielaw.org/skins/4.1.0/default_flat_top_two_button_black/v2/images/optanon-pop-up-close.png);width:34px;height:34px;"></div></button></div><div id="optanon-popup-body"><div id="optanon-popup-body-left"><div id="optanon-popup-body-left-shading"></div><div id="optanon-branding-top-logo" style="background-image: url('https://cdn.cookielaw.org/logos/5138/5138:nature.com/nature-red.png') !important;"></div><ul aria-label="Navigation Menu" id="optanon-menu" role="tablist"><li class="menu-item-on menu-item-about" title="Your Privacy"><p class="preference-menu-item"><button aria-controls="tab-0" aria-selected="true" id="Your-Privacy" role="tab">Your Privacy</button></p></li><li class="menu-item-necessary menu-item-on" title="Strictly Necessary Cookies"><p class="preference-menu-item"><button aria-controls="tab-1" aria-selected="false" id="Strictly-Necessary" role="tab">Strictly Necessary Cookies</button></p></li><li class="menu-item-on menu-item-performance" title="Performance &amp; Analytics Cookies"><p class="preference-menu-item"><button aria-controls="tab-2" aria-selected="false" id="Performance-&amp;" role="tab">Performance &amp; Analytics Cookies</button></p></li><li class="menu-item-on menu-item-functional" title="Functional Cookies"><p class="preference-menu-item"><button aria-controls="tab-3" aria-selected="false" id="Functional-Cookies" role="tab">Functional Cookies</button></p></li><li class="menu-item-on menu-item-advertising" title="Targeting Cookies"><p class="preference-menu-item"><button aria-controls="tab-4" aria-selected="false" id="Targeting-Cookies" role="tab">Targeting Cookies</button></p></li><li class="menu-item-moreinfo menu-item-off" title="More Information"><p class="preference-menu-item"><a aria-label="More Information" href="http://www.nature.com/info/cookies.html" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Cookie Policy');" target="_blank">More Information</a></p></li></ul></div><div id="optanon-popup-body-right"><p aria-label="Privacy Preference Centre" aria-level="2" class="legacy-preference-banner-title h2" role="heading">Privacy Preference Centre</p><div class="vendor-header-container"><p aria-level="3" class="header-3" role="heading"></p><div id="optanon-popup-more-info-bar"><div class="optanon-status"><div class="optanon-status-editable"><form><span class="fieldset"><p><input checked="" class="legacy-group-status optanon-status-checkbox" id="chkMain" type="checkbox" value="check"/><label for="chkMain">Active</label></p></span></form></div><div class="optanon-status-always-active optanon-status-on"><p>Always Active</p></div></div></div></div><div class="optanon-main-info-text" role="tabpanel"></div></div><div class="optanon-bottom-spacer"></div></div><div id="optanon-popup-bottom"> <a href="https://onetrust.com/poweredbyonetrust" target="_blank"><div id="optanon-popup-bottom-logo" style="background: url(https://cdn.cookielaw.org/skins/4.1.0/default_flat_top_two_button_black/v2/images/cookie-collective-top-bottom.png);width:155px;height:35px;" title="powered by OneTrust"></div></a><div class="optanon-button-wrapper optanon-save-settings-button optanon-close optanon-close-consent"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button aria-label="Save Settings" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Save Settings');" title="Save Settings">Save Settings</button></div><div class="optanon-white-button-right"></div></div><div class="optanon-button-wrapper optanon-allow-all-button optanon-allow-all" style="display: none;"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button aria-label="Allow All" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Allow All');" title="Allow All">Allow All</button></div><div class="optanon-white-button-right"></div></div></div></div></div>
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-NWDMT9Q" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50" data-test="top-containers" role="banner">
<div class="hide-print container background-gray-dark">
<div class="content mq1200-padded">
<a class="skiplink contrast-text js-no-scroll" data-track="click" data-track-action="skip to content" data-track-category="header" data-track-label="link" href="#content" id="skiplink">Skip to main content</a>
</div>
</div>
<div class="container cleared container-type-banner-advert inline-group" data-container-type="banner-advert">
<div class="hide-print pt10 pb10 mq875-pa0 position-relative z-index-100 clear-float cleared leaderboard-or-billboard-container">
<div class="leaderboard-or-billboard-inner ad-with-label">
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41598-018-20427-9;doi=10.1038/s41598-018-20427-9;techmeta=22,23,45;subjmeta=208,212,457,631;kwrd=Genomics,Population genetics" data-gpt-unitpath="/285/scientific_reports/article" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-850755231&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-018-20427-9%26doi%3D10.1038/s41598-018-20427-9%26techmeta%3D22,23,45%26subjmeta%3D208,212,457,631%26kwrd%3DGenomics,Population genetics">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-850755231&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-018-20427-9%26doi%3D10.1038/s41598-018-20427-9%26techmeta%3D22,23,45%26subjmeta%3D208,212,457,631%26kwrd%3DGenomics,Population genetics" width="728"/></a>
</noscript>
<div id="google_ads_iframe_/270604982/nature/srep/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
</div>
</div>
</div>
<div class="hide-print text-orange content grade-c-show"><p>Thank you for visiting nature.com. You are using a browser version with
    limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off
    compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site
    without styles and JavaScript.</p></div>
<div class="hide-print container background-bluegray contrast-text text14 strong lower mq640-hide position-relative z-index-100" data-test="header-breadcrumbs">
<div class="content cleared mq1200-padded">
<div class="breadcrumbs pin-left">
<ol class="ma0 cleared clean-list inline-list"><li id="breadcrumb0" itemref="breadcrumb1" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="url"><span itemprop="title">nature</span></a></li><li class="ml6" id="breadcrumb1" itemref="breadcrumb2" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:scientific reports" href="/srep" itemprop="url"><span itemprop="title">scientific reports</span></a></li><li class="ml6" id="breadcrumb2" itemref="breadcrumb3" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/srep/articles?type=article" itemprop="url"><span itemprop="title">articles</span></a></li><li id="breadcrumb3" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a class="ml6" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:article" href="/articles/s41598-018-20427-9" itemprop="url"><span itemprop="title">article</span></a></li></ol>
</div>
<p class="pin-right box-sizing header-primary-color nature-research-logo ma0 pa6 hide-text grade-c-hide">A Nature Research Journal</p>
</div>
</div>
<div class="hide-print container js-header-container mb20 position-relative z-index-50" data-container-type="header" data-ui="header-container">
<div class="cleared clear hide-print position-relative background-gradient-brand-primary js-header default-header z-index-100 composite-layer tighten-line-height">
<div class="banner js-banner content mq1200-padded position-relative">
<div class="inner-banner cleared">
<div class="main-column small-header-main pin-left">
<div class="header-logo-container header-primary-color">
<a aria-controls="menu" aria-haspopup="true" class="js-header-menu-button menu-button js-no-scroll role-button" data-test="menu-button" data-track="click" data-track-action="open menu" data-track-category="header" data-track-label="link" href="#menu" id="menu-button" role="button" tabindex="0">
<span class="menu-button-icon icon-rotate">
<span class="menu-button-label">Menu</span>
</span>
</a>
<h1 class="inline-block"><a class="header-logo inline-block" data-track="click" data-track-action="home" data-track-category="header" data-track-label="image" href="/srep"><img alt="Scientific Reports" class="grade-c-invisible header-logo-primary js-svg" data-png="//media.springernature.com/full/nature-cms/uploads/product/srep/header-e3d497587e4888a8b21427cbb105e528.png" src="//media.springernature.com/full/nature-cms/uploads/product/srep/header-2c7da0708d3d7539cf612bcb0d72c930.svg"/><img alt="Scientific Reports" class="grade-c-invisible header-logo-secondary js-svg" data-png="//media.springernature.com/full/nature-cms/uploads/product/srep/header-63663207242d718880707605d7ea7eee.png" src="//media.springernature.com/full/nature-cms/uploads/product/srep/header-2091eea0f18dbb1fee97bebe766793c1.svg"/></a></h1>
</div>
</div>
<div class="position-absolute position-right small-header-side">
<div class="pin-right">
<a aria-controls="search-menu" aria-haspopup="true" class="pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text mr1 role-button" data-component="tray-button" data-test="search-link" data-track="click" data-track-action="open tray" data-track-category="header" data-track-label="button" href="#search-menu" role="button" tabindex="0">
<span class="icon icon-above icon-search-25x25-white icon-search-25x25-gray block text-center">
<span class="block mt6 mq875-hide small-header-hide">Search</span>
</span>
</a>
<a class="mq640-hide pin-left inline-group-top pa10 small-header-icons background-bluegray text11 contrast-text mr1" data-test="ealert-link" data-track="click" data-track-action="ealert" data-track-category="header" data-track-label="link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=288">
<span class="icon icon-above icon-ealert-25x25-white icon-ealert-25x25-gray block text-center">
<span class="block mt6 mq875-hide small-header-hide">E-alert</span>
</span>
</a>
<a class="mq640-hide inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text mr1" data-test="submit-link" data-track="click" data-track-action="manuscript submission" data-track-category="header" data-track-label="link" href="https://author-welcome.nature.com">
<span class="icon icon-above icon-submit-25x25-white icon-submit-25x25-gray block text-center">
<span class="block mt6 mq875-hide small-header-hide">Submit</span>
</span>
</a>
<a class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text" data-test="login-link" data-track="click" data-track-action="my account" data-track-category="header" data-track-label="link" href="/nams/svc/myaccount" id="my-account" style="display: none;">
<span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
<span class="block mt6 mq875-hide small-header-hide">My Account</span>
</span>
</a>
<a class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text" data-test="login-link" data-track="click" data-track-action="login" data-track-category="header" data-track-label="link" href="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-20427-9" id="login-button" style="display: block;">
<span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
<span class="block mt6 mq875-hide small-header-hide">Login</span>
</span>
</a>
</div>
</div><ul class="background-brand-secondary-pdf ma0 clean-list text14 pin-right icon icon-right hide icon-pdf-download"><li class="pin-left position-relative"><a class="pl20 pr30 pt10 inline-block block-link header-tools-link position-relative z-index-1" data-track="click" data-track-action="download pdf" data-track-category="header" data-track-label="link" href="/articles/s41598-018-20427-9.pdf"><span>PDF</span></a></li></ul>
<h2 class="pt10 pr20 mq640-pr5 background-brand-primary text14 unstyled nowrap overflow-ellipsis hide-overflow hide">Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces</h2></div>
</div>
</div>
<button class="visually-hidden js-hide" data-action="close-on-focus" tabindex="-1" type="button">Close menu</button><div aria-hidden="true" class="menu pt20 mb20 z-index-50 composite-layer background-white border-gray-medium border-bottom-2" data-component="tray" data-track-component="header" id="search-menu" role="navigation">
<div class="menu-inner js-hide">
<section>
<h2 class="visually-hidden">Search</h2>
<div class="hide-print content mb40 mq1200-padded position-relative" data-test="inline-search">
<div class="pt30 cleared background-white sans-serif">
<div class="grid grid-8 grid-left-2 mq875-grid-12 mq875-ml0 mq640-pb30">
<form action="/search" autocomplete="off" class="standard-space-below" data-track="submit" data-track-action="search" data-track-label="form" method="get" role="search">
<label class="block strong" for="keywords">Article search</label>
<div class="position-relative">
<input class="border-gray border-all-1 equalize-line-height pa10 pr40 box-sizing grid-12" data-test="search-keywords" id="keywords" name="q" placeholder="Search by keywords or author" type="search" value=""/>
<div class="position-absolute position-right position-top mt1 mr1">
<button class="icon icon-center search-btn kill-border" data-test="search-submit" type="submit">
                                        Search
                                    </button>
</div>
<p class="mb0 mt4 text14"><a data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">Advanced search</a></p>
</div>
</form>
<h3 class="h3 strong mb4 border-gray-medium border-bottom-1 sans-serif">Quick links</h3>
<ul class="clean-list mb0 text14">
<li class="grid-ng grid-1of2 mb6 mt6"><a data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li class="grid-ng grid-1of2 mb6 mt6"><a data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li class="grid-ng grid-1of2"><a data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li class="grid-ng grid-1of2"><a data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
</div>
</section>
</div>
</div><button class="visually-hidden js-hide" data-action="close-on-focus" tabindex="-1" type="button">Close menu</button><button class="visually-hidden js-hide" data-action="close-on-focus" tabindex="-1" type="button">Close menu</button><div aria-hidden="true" aria-labelledby="menu-button" class="hide-print js-header-menu menu menu2 mb20 z-index-50 composite-layer background-white border-bottom-2 border-gray-medium tighten-line-height" data-track-component="menu" id="menu" role="navigation">
<div class="menu-inner content js-hide">
<nav>
<div class="cleared border-bottom-1 border-gray-medium ml20 mr20">
<div class="grid-ng grid-1of4 mq875-grid-12">
<h2 class="h3 pa20 mq875-pa0 mq875-mt20">
                       Scientific Reports<span class="visually-hidden"> menu</span>
</h2>
</div>
<div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
<ul class="clean-list mb0">
<li class="mb4 pr10"><a data-track="click" data-track-action="browse articles" data-track-label="link" href="/srep/articles">Browse Articles</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="browse collections" data-track-label="link" href="/srep/collections">Browse Collections</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="browse subjects" data-track-label="link" href="/srep/browse-subjects">Browse Subjects</a></li>
</ul>
</div>
<div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
<ul class="clean-list mb0">
<li class="mb4 pr10"><a data-track="click" data-track-action="about the journal" data-track-label="link" href="/srep/about">About the journal</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="publish" data-track-label="link" href="/srep/publish">Publish</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="journal policies" data-track-label="link" href="/srep/journal-policies">Journal policies</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="contact" data-track-label="link" href="/srep/contact">Contact</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific reports top 100 2017" data-track-label="link" href="/srep/top100">Scientific Reports Top 100 2017</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="editor's choice" data-track-label="link" href="/srep/editorschoice">Editor's Choice</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific reports top 10 2018" data-track-label="link" href="/srep/top10">Scientific Reports Top 10 2018</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="guest edited collections" data-track-label="link" href="/srep/guestedited">Guest Edited Collections</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="editorial board highlights" data-track-label="link" href="/srep/ebm">Editorial Board Highlights</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific reports top 100 2018" data-track-label="link" href="/srep/top100-2018">Scientific Reports Top 100 2018</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="author highlights" data-track-label="link" href="/srep/author-highlights">Author Highlights</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific reports rigorous editorial process" data-track-label="link" href="/srep/rigorous-editorial-process">Scientific Reports rigorous editorial process</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="open access funding support" data-track-label="link" href="/srep/open-access-funding-support">Open Access Funding Support</a></li>
</ul>
</div>
</div>
<div class="cleared ml20 mr20">
<div class="cleared pb10">
<div class="grid-ng grid-1of4 mq875-grid-12">
<h3 class="serif pa20 mq875-pa0 mq875-mt20">Nature Research<span class="visually-hidden"> menu</span></h3>
</div>
<div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
<h4 class="mb4">Our Journals</h4>
<ul class="clean-list mb0">
<li class="mb4 pr10"><a data-track="click" data-track-action="nature" data-track-label="link" href="/nature">Nature</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="nature communications" data-track-label="link" href="/ncomms">Nature Communications</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="nature protocols" data-track-label="link" href="/nprot">Nature Protocols</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific reports" data-track-label="link" href="/srep">Scientific Reports</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="journal a-z view all" data-track-label="link" href="/siteindex">View all journals</a></li>
</ul>
</div>
<div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
<h4 class="mb4">Subjects</h4>
<ul class="clean-list mb0">
<li class="mb4 pr10"><a data-track="click" data-track-action="biological sciences" data-track-label="link" href="/subjects/biological-sciences">Biological Sciences</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="scientific community and society" data-track-label="link" href="/subjects/scientific-community-and-society">Scientific Community &amp; Society</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="earth and environmental sciences" data-track-label="link" href="/subjects/earth-and-environmental-sciences">Earth &amp; Environmental Sciences</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="health sciences" data-track-label="link" href="/subjects/health-sciences">Health Sciences</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="physical sciences" data-track-label="link" href="/subjects/physical-sciences">Physical Sciences</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="subjects view all" data-track-label="link" href="/subjects">View all subjects</a></li>
</ul>
</div>
<div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
<h4 class="mb4">More</h4>
<ul class="clean-list mb0">
<li class="mb4 pr10"><a data-track="click" data-track-action="subscriptions" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="authors and referees" data-track-label="link" href="/authors/index.html">Authors &amp; Referees</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="librarians" data-track-label="link" href="/libraries/index.html">Librarians</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="advertisers" data-track-label="link" href="/advertising/index.html">Advertisers</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="press" data-track-label="link" href="/npg_/press_room/index.html">Press</a></li>
<li class="mb4 pr10"><a data-track="click" data-track-action="about" data-track-label="link" href="/npg_/index_npg.html">About Nature Research</a></li>
</ul>
</div>
</div>
</div>
</nav>
</div>
</div><button class="visually-hidden js-hide" data-action="close-on-focus" tabindex="-1" type="button">Close menu</button></div>
</div>
<div class="position-relative z-index-1" id="content">
<div class="container cleared container-type-article" data-container-type="article">
<div class="content position-relative cleared clear mq1200-padded" data-component="article-container">
<article itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle">
<div class="full-width-print main-column pin-left js-main-column highlighter" role="main">
<header>
<p class="c-article-publication-details" data-test="article-identifier">
                            Article
                            
                                <span class="c-article-publication-details__divider"> | </span>
<abbr class="c-article-publication-details--open-access" title="Open Access">Open</abbr>
<span class="c-article-publication-details__divider"> | </span>
<a href="#article-info">Published: <time datetime="2018-01-31" itemprop="datePublished">31 January 2018</time></a>
</p>
<h1 class="c-article-title" data-article-title="" data-test="article-title" itemprop="name headline">Methylation-based enrichment facilitates low-cost, noninvasive genomic scale sequencing of populations from feces</h1>
<ul class="c-author-list js-list-authors js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a class="c-author-list__item-icon js-no-scroll" data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-1">Kenneth L. Chiou</a></span><span class="u-js-hide"> 
            <a class="js-orcid js-no-scroll" href="http://orcid.org/0000-0001-7247-4107" itemprop="url"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7247-4107</a></span><sup class="u-js-hide"><a class="js-no-scroll" href="#Aff1" tabindex="-1">1</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="Department of Anthropology, Washington University" itemprop="name"/><meta content="0000 0001 2355 7002, grid.4367.6, Department of Anthropology, Washington University, St. Louis, MO, 63130, USA" itemprop="address"/></span>,<a class="js-no-scroll" href="#Aff2" tabindex="-1">2</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="Department of Psychology, University of Washington" itemprop="name"/><meta content="0000000122986657, grid.34477.33, Department of Psychology, University of Washington, Seattle, WA, 98195, USA" itemprop="address"/></span></sup><sup class="u-js-hide"> <a class="js-no-scroll" href="#na1" tabindex="-1">na1</a></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a class="c-author-list__item-icon js-no-scroll" data-corresp-id="c2" data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-2">Christina M. Bergey</a></span><sup class="u-js-hide"><a class="js-no-scroll" href="#Aff3" tabindex="-1">3</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="Department of Anthropology, New York University" itemprop="name"/><meta content="0000 0004 1936 8753, grid.137628.9, Department of Anthropology, New York University, New York, NY, 10003, USA" itemprop="address"/></span>,<a class="js-no-scroll" href="#Aff4" tabindex="-1">4</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="New York Consortium in Evolutionary Primatology" itemprop="name"/><meta content="grid.452706.2, New York Consortium in Evolutionary Primatology, New York, NY, USA" itemprop="address"/></span>,<a class="js-no-scroll" href="#Aff5" tabindex="-1">5</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="Department of Anthropology, Pennsylvania State University" itemprop="name"/><meta content="0000 0001 2097 4281, grid.29857.31, Department of Anthropology, Pennsylvania State University, University Park, PA, 16802, USA" itemprop="address"/></span>,<a class="js-no-scroll" href="#Aff6" tabindex="-1">6</a><span class="u-visually-hidden" itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization"><meta content="Pennsylvania State University" itemprop="name"/><meta content="0000 0001 2097 4281, grid.29857.31, Department of Biology, Pennsylvania State University, University Park, PA, 16802, USA" itemprop="address"/></span></sup><sup class="u-js-hide"> <a class="js-no-scroll" href="#na1" tabindex="-1">na1</a></sup> </li></ul>
<div class="c-article-info" data-container-section="info"><p class="c-article-info-details"><i data-test="journal-title">Scientific Reports</i><b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 8</b>, Article number: <span data-test="article-number">1975</span> (<span data-test="article-publication-year">2018</span>)
                    <span class="c-article-info-details__divider u-hide-print"> | </span><a class="c-article-info-details__download-citation u-hide-print" data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-category="article body" data-track-label="link" href="/articles/s41598-018-20427-9.ris">
                    Download Citation
                </a></p></div>
</header>
<div class="c-article-metrics-bar">
<h3 class="u-visually-hidden">Article metrics</h3>
<div class="c-article-metrics-bar__item">
<h4 class="c-article-metrics-bar__label">Citations</h4>
<p class="c-article-metrics-bar__count">8</p>
</div>
<div class="c-article-metrics-bar__item">
<h4 class="c-article-metrics-bar__label">Altmetric</h4>
<p class="c-article-metrics-bar__count">1548</p>
</div>
<div class="c-article-metrics-bar__item">
<h4 class="u-visually-hidden">More details</h4>
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-category="article body" data-track-label="link" href="/articles/s41598-018-20427-9/metrics" rel="nofollow">Article metrics</a></p>
</div>
</div>
<div class="c-pdf-download">
<a class="c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="pdf-link" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link" href="/articles/s41598-018-20427-9.pdf">
<span>Download PDF</span>
</a>
</div>
<ul class="c-article-events">
<li class="c-article-events__item"><span><a href="#change-history">This article has been updated</a></span></li>
</ul>
<div class="c-article-body" data-article-body="true" data-track-component="article body">
<section aria-labelledby="Abs1"><div class="c-article-section js-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content js-collapsible-section" id="Abs1-content"><p>Obtaining high-quality samples from wild animals is a major obstacle for genomic studies of many taxa, particularly at the population level, as collection methods for such samples are typically invasive. DNA from feces is easy to obtain noninvasively, but is dominated by bacterial and other non-host DNA. The high proportion of non-host DNA drastically reduces the efficiency of high-throughput sequencing for host animal genomics. To address this issue, we developed an inexpensive capture method for enriching host DNA from noninvasive fecal samples. Our method exploits natural differences in CpG-methylation density between vertebrate and bacterial genomes to preferentially bind and isolate host DNA from majority-bacterial samples. We demonstrate that the enrichment is robust, efficient, and compatible with downstream library preparation methods useful for population studies (e.g., RADseq). Compared to other enrichment strategies, our method is quick and inexpensive, adding only a negligible cost to sample preparation. In combination with downstream methods such as RADseq, our approach allows for cost-effective and customizable genomic-scale genotyping that was previously feasible in practice only with invasive samples. Because feces are widely available and convenient to collect, our method empowers researchers to explore genomic-scale population-level questions in organisms for which invasive sampling is challenging or undesirable.</p></div></div></section>
<section aria-labelledby="Sec1"><div class="c-article-section js-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content js-collapsible-section" id="Sec1-content"><p>The past decade has witnessed a rapid transformation of biological studies with the continuing development and adoption of massively parallel sequencing technology. This sequencing revolution, however, has thus far had a relatively muted impact on studies of wild nonmodel organisms due largely to the difficulty of obtaining high-quality samples. This problem is particularly salient for endangered animals, cryptic animals, or animals for which it is otherwise difficult, undesirable, or unethical to obtain samples invasively.</p><p>Field researchers working with wild animals have explored several noninvasive sample types for DNA analysis including feces, hair, urine, saliva, feathers, skin, and nails<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR1" id="ref-link-section-d28261e480" title="Kohn, M. H. &amp; Wayne, R. K. Facts from feces revisited. Trends Ecol. Evol. 12, 223–227 (1997).">1</a></sup>. Of these, feces may be the most readily available in many taxa<sup><a aria-label="Reference 2" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR2" id="ref-link-section-d28261e484" title="Putman, R. J. Facts from faeces. Mamm. Rev. 14, 79–97 (1984).">2</a></sup>. Indeed, since PCR amplification of DNA from feces was first demonstrated in the 1990s<sup><a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR3" id="ref-link-section-d28261e488" title="Höss, M., Kohn, M., Pääbo, S., Knauer, F. &amp; Schröder, W. Excrement analysis by PCR. Nature 359, 199 (1992).">3</a></sup>, noninvasive genetic studies from feces have revolutionized our understanding of the evolution, population structure, phylogeography, and behavior of nonmodel organisms. PCR amplification, however, is effective only for short sequences of DNA. The ability to generate cost-effective genomic-scale data of animals from feces using massively parallel sequencing would therefore constitute an important methodological advance towards bringing a greater number of wild organism studies into the genomic age.</p><p>Feces presents significant challenges for genetic analysis. DNA in feces is often fragmented and low in quantity. Fecal DNA extractions are further characterized by a frequent presence of co-extracted PCR inhibitors, sometimes complicating PCR detection of genotypes<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR1" id="ref-link-section-d28261e495" title="Kohn, M. H. &amp; Wayne, R. K. Facts from feces revisited. Trends Ecol. Evol. 12, 223–227 (1997).">1</a></sup>, particularly with long amplicons. Finally, endogenous (host) DNA in feces constitutes a very low proportion, typically less than 5%<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR4" id="ref-link-section-d28261e499" title="Qin, J. et al. A human gut microbial gene catalogue established by metagenomic sequencing. Nature 464, 59–65 (2010).">4</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR5" id="ref-link-section-d28261e499_1" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a>,<a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e502" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup>, of total fecal DNA. Instead, fecal DNA contains a preponderance of DNA from exogenous (non-host) sources such as gut microbes, digesta, intestinal parasites, and environmental organisms. Gut bacteria pose a particular challenge as they account for the highest proportion of DNA in feces<sup><a aria-label="Reference 4" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR4" id="ref-link-section-d28261e506" title="Qin, J. et al. A human gut microbial gene catalogue established by metagenomic sequencing. Nature 464, 59–65 (2010).">4</a>,<a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR5" id="ref-link-section-d28261e509" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a></sup>.</p><p>Because of the high representation of exogenous DNA in feces, shotgun sequencing of fecal DNA would yield only a small proportion of reads matching the host genome. For genomic studies of host organisms, particularly those targeting populations, this represents a crippling obstacle in the presence of typical financial constraints. Without an effective enrichment procedure, sequencing of fecal DNA would be less efficient than that of invasively obtained “high-quality” DNA by at least one order of magnitude regardless of improvements in sequencing throughput or cost.</p><p>Attempts to enrich host DNA from feces for genomic analysis<sup><a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR5" id="ref-link-section-d28261e520" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a>,<a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e523" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup> have thus far employed targeted sequence capture methodologies. Sequence capture, like PCR, enriches DNA based on sequence specificity but unlike traditional PCR can work at any scale from a single locus<sup><a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR7" id="ref-link-section-d28261e527" title="Whitney, D. et al. Enhanced retrieval of DNA from human fecal samples results in improved performance of colorectal cancer screening test. J. Mol. Diagn. 6, 386–395 (2004).">7</a></sup> to a whole genome<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e531" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a>,<a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR8" id="ref-link-section-d28261e534" title="Melnikov, A. et al. Hybrid selection for sequencing pathogen genomes from clinical samples. Genome Biol. 12, R73 (2011).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR9" id="ref-link-section-d28261e537" title="Carpenter, M. L. et al. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. Am. J. Hum. Genet. 93, 852–864 (2013).">9</a></sup>. This method involves hybridizing DNA or RNA “baits,” either affixed to an array<sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR10" id="ref-link-section-d28261e541" title="Albert, T. J. et al. Direct selection of human genomic loci by microarray hybridization. Nat. Methods 4, 903–905 (2007).">10</a>,<a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR11" id="ref-link-section-d28261e544" title="Okou, D. T. et al. Microarray-based genomic selection for high-throughput resequencing. Nat. Methods 4, 907–909 (2007).">11</a></sup> or to magnetic beads in solution<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR12" id="ref-link-section-d28261e548" title="Gnirke, A. et al. Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. Nat. Biotechnol. 27, 182–189 (2009).">12</a></sup>, to a mixture of target and nontarget sequences, thereby capturing targeted DNA from the mixture. Sequence capture has been used for instance to enrich human exomes<sup><a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR13" id="ref-link-section-d28261e553" title="Ng, S. B. et al. Targeted capture and massively parallel sequencing of 12 human exomes. Nature 461, 272–276 (2009).">13</a></sup>, reduced-representation genomes<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR14" id="ref-link-section-d28261e557" title="Suchan, T. et al. Hybridization capture using RAD probes (hyRAD), a new tool for performing genomic analyses on museum collection specimens. PLoS One 11, e0151651 (2016).">14</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR15" id="ref-link-section-d28261e557_1" title="Ali, O. A. et al. RAD Capture (Rapture): flexible and efficient sequence-based genotyping. Genetics 202, 389–400 (2016).">15</a>,<a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR16" id="ref-link-section-d28261e560" title="Hoffberg, S. L. et al. RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data. Mol. Ecol. Resour. 16, 1264–1278 (2016).">16</a></sup>, host DNA from ancient or museum specimens<sup><a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR9" id="ref-link-section-d28261e564" title="Carpenter, M. L. et al. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. Am. J. Hum. Genet. 93, 852–864 (2013).">9</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d28261e567" title="Maricic, T., Whitten, M. &amp; Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. PLoS One 5, e14004 (2010).">17</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR18" id="ref-link-section-d28261e567_1" title="Mason, V. C., Li, G., Helgen, K. M. &amp; Murphy, W. J. Efficient cross-species capture hybridization and next-generation sequencing of mitochondrial genomes from noninvasively sampled museum specimens. Genome Res. 21, 1695–1704 (2011).">18</a>,<a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR19" id="ref-link-section-d28261e570" title="Bi, K. et al. Unlocking the vault: next-generation museum population genomics. Mol. Ecol. 22, 6018–6032 (2013).">19</a></sup>, and pathogen genomes from human clinical samples<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR8" id="ref-link-section-d28261e574" title="Melnikov, A. et al. Hybrid selection for sequencing pathogen genomes from clinical samples. Genome Biol. 12, R73 (2011).">8</a></sup>. While the cost of custom oligonucleotide bait synthesis remains high, methods for transcribing custom baits from existing DNA templates<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR8" id="ref-link-section-d28261e578" title="Melnikov, A. et al. Hybrid selection for sequencing pathogen genomes from clinical samples. Genome Biol. 12, R73 (2011).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR9" id="ref-link-section-d28261e581" title="Carpenter, M. L. et al. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. Am. J. Hum. Genet. 93, 852–864 (2013).">9</a></sup> have driven costs significantly down, increasing sequence capture’s appeal.</p><p>Perry <i>et al</i>.<sup><a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR5" id="ref-link-section-d28261e591" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a></sup> first successfully enriched host DNA from feces at the genomic scale. Using a modified sequence capture employing custom-synthesized baits, they were able to highly enrich 1.5 megabases of chromosome 21, the X chromosome, and the mitochondrial genome from fecal samples of 6 captive chimpanzees. Their protocol, however, remains prohibitively expensive for population-level analysis due to the high cost of bait synthesis. More recently, Snyder-Mackler <i>et al</i>.<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e598" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup> performed whole-genome capture on fecal DNA, using RNA baits transcribed <i>in vitro</i> from high quality baboon samples to enrich host genomes from 62 wild baboons. Resulting libraries were sequenced to low coverage (mean 0.49×), but nevertheless provided sufficient information for reconstructing pedigree relationships.</p><p>Despite these methodological advances, targeted sequence capture has distinct drawbacks. To avoid the high cost of bait synthesis, RNA baits must first be transcribed from high-quality genomic DNA that is consumed by the process, limiting its appeal when working with species for which high-quality DNA is difficult to obtain or in short supply. The processes of both bait generation and hybridization with fecal DNA are labor-intensive and time-consuming, with the hybridization including an incubation step that alone takes 1–3 days<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e608" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup>. Because both RNA baits and the gDNA used to transcribe them are eventually depleted, the composition of RNA baits varies between bait sets, potentially impeding comparison of samples sequenced using different RNA baits and gDNA templates. <i>Trans</i> genomic captures (<i>i</i>.<i>e</i>. capturing DNA using baits from a different species) may complicate enrichment and introduce at least some capture biases<sup><a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR20" id="ref-link-section-d28261e621" title="George, R. D. et al. Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection. Genome Res. 21, 1686–1694 (2011).">20</a></sup>, which will be a particular impediment for genomic studies for which high-quality DNA from related taxa is not accessible. Sequence capture may also introduce biases toward the capture of low-complexity, highly repetitive genomic regions, as well as an excess of fragments from the mitochondrial genome<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e626" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR9" id="ref-link-section-d28261e629" title="Carpenter, M. L. et al. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. Am. J. Hum. Genet. 93, 852–864 (2013).">9</a>,<a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR21" id="ref-link-section-d28261e632" title="Samuels, D. C. et al. Finding the lost treasures in exome sequencing data. Trends Genet. 29, 593–599 (2013).">21</a></sup>.</p><p>We have developed a method that makes noninvasive population genomics economically and practically feasible for the first time, by exploiting natural, evolutionarily ancient differences in CpG-methylation densities between vertebrate and bacterial genomes to enrich the host genome from feces. This method, which we call FecalSeq, uses methyl-CpG-binding domain (MBD) proteins to selectively bind and isolate DNA with high CpG-methylation density. Modified after techniques to enrich the microbiome from vertebrate samples<sup><a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR22" id="ref-link-section-d28261e639" title="Feehery, G. R. et al. A method for selectively enriching microbial DNA from contaminating vertebrate host DNA. PLoS One 8, e76096 (2013).">22</a></sup>, our method employs a bait protein created by genetically fusing the human methyl-CpG binding domain protein 2 (MBD2) to the Fc tail of human IgG1. The resulting MBD2-Fc protein is then bound by a paramagnetic Protein A immunoprecipitation bead to create a complex that selectively binds double-stranded DNA with 5-methyl CpG dinucleotides. Because vertebrate DNA contains a high frequency of methylated CpGs<sup><a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR23" id="ref-link-section-d28261e643" title="Hendrich, B. &amp; Tweedie, S. The methyl-CpG binding domain and the evolving role of DNA methylation in animals. Trends Genet. 19, 269–277 (2003).">23</a>,<a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR24" id="ref-link-section-d28261e646" title="Jabbari, K. &amp; Bernardi, G. Cytosine methylation and CpG, TpG (CpA) and TpA frequencies. Gene 333, 143–149 (2004).">24</a></sup> while bacterial DNA does not<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR25" id="ref-link-section-d28261e650" title="Fang, G. et al. Genome-wide mapping of methylated adenine residues in pathogenic Escherichia coli using single-molecule real-time sequencing. Nat. Biotechnol. 30, 1232–1239 (2012).">25</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR26" id="ref-link-section-d28261e653" title="Murray, I. A. et al. The methylomes of six bacteria. Nucleic Acids Res. 40, 11450–11462 (2012).">26</a></sup>, this MBD bait complex selectively binds host DNA (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-018-20427-9#Fig1">1</a>). This enrichment method is inexpensive and, crucially, captures target DNA without modification, thereby enabling downstream library preparation techniques including complexity reduction-based sequencing methods such as RADseq, which we validate in this study by preparing and sequencing double-digest RADseq libraries<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR27" id="ref-link-section-d28261e660" title="Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species. PLoS One 7, e37135 (2012).">27</a></sup>. Because of these properties, our method is well-suited for population genomic studies requiring high sequencing coverage, including those of nonmodel organisms for which few resources (e.g., high-quality samples or reference genomes) exist.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Figure 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-label="image" href="/articles/s41598-018-20427-9/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure1" aria-describedby="figure-1-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>Overview of the FecalSeq method.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41598-018-20427-9/figures/1" rel="nofollow"><span>Full size image</span></a></div></figure></div></div></div></section><section aria-labelledby="Sec2"><div class="c-article-section js-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content js-collapsible-section" id="Sec2-content"><p>Our enrichment approach captures eukaryotic DNA using a methylated CpG binding domain protein fused to the Fc fragment of human IgG (MBD2-Fc) to selectively target sequences with high CpG methylation density<sup><a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR22" id="ref-link-section-d28261e690" title="Feehery, G. R. et al. A method for selectively enriching microbial DNA from contaminating vertebrate host DNA. PLoS One 8, e76096 (2013).">22</a></sup>.</p><p>To evaluate our approach, we enriched DNA extractions from the feces of 6 captive and 46 wild baboons, which we then used to prepare and sequence ddRADseq libraries. We also prepared ddRADseq libraries from blood-derived genomic DNA of all six captive baboons to facilitate controlled (same-individual) comparisons of blood and fecal libraries. All libraries were sequenced using Illumina sequencing.</p><p>Quantitative PCR estimates of starting host DNA proportions in fecal DNA extracts ranged widely, but were substantially lower in samples obtained from the wild (captive samples: mean 5.3%, range &lt;0.01–17.4%; wild samples: mean 0.6%, range &lt;0.01–4.9%; Supplemental Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S2</a>).</p><p>Based on two pilot libraries constructed from MBD-enriched fecal DNA, we found that there was large variation in the proportion of reads mapping to the baboon reference genome (mean 24.8%, range 0.7–81.2%; Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>; Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>), with the read mapping proportion correlating with starting host DNA proportions as estimated via qPCR (library A: <i>r</i><sup>2</sup> = 0.7338; <i>p</i> = 0.03; library B: <i>r</i><sup>2</sup> = 0.9127, <i>p</i> &lt; 0.01). Endogenous DNA proportions on average increased 13-fold as estimated via comparison of pre-enrichment host proportion (from qPCR) and post-enrichment proportion of reads mapped (range 4.4–29.6; two samples removed due to starting proportions too low to quantify).</p><p>While some samples in our pilot libraries had high host DNA proportions following enrichment, these samples tended to already have high host DNA proportions prior to enrichment. Host DNA proportions following enrichment in the pilot libraries averaged only 4%, for instance, when samples with starting host DNA proportions greater than 1% were excluded. Because wild fecal DNA samples in our dataset on average started with less than 1% host DNA, we undertook a series of protocol optimization experiments to maximize the enrichment of these “low-quality” samples (Supplemental Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S4</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S7</a>).</p><p>Using a revised protocol based on our optimization experiments (Supplemental Protocol), we created and sequenced a third library from MBD-enriched fecal DNA. After noting substantial improvements in enrichment, we finally sequenced a fourth library with MBD-enriched fecal DNA from 40 wild baboons.</p><p>Despite having similar or even lower starting host DNA proportions, read mapping proportions in the third library were substantially higher than the prior two (mean 49.1%, range 8.9–75.3%; Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>; Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>). Endogenous DNA proportions on average increased 318-fold (range 4.3–2632.2; one sample removed due to starting proportion too low to quantify).</p><p>The fourth library consisting entirely of fecal DNA from wild animals had the lowest starting concentrations of host DNA (mean 0.3%, range &lt; 0.01–3.1%). Following enrichment, however, host DNA proportions were nonetheless higher than our pilot libraries (mean 28.8%, range 1.5–73.6%; Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>; Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>). Endogenous DNA proportions on average increased 195-fold (range 23.7–486.9).</p><p>Overall, the revised protocol produced substantially higher enrichment, measured as fold increases in the proportion of host DNA, particularly for samples with very low starting proportions of host DNA (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-018-20427-9#Fig2">2</a>). While we sometimes were forced to use multiple rounds of extraction, thereby introducing variation in starting host proportions across same-individual trials, the revised protocol nonetheless exhibited robust improvement in read mapping proportions even when starting host proportions were substantially lower.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Figure 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-label="image" href="/articles/s41598-018-20427-9/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure2" aria-describedby="figure-2-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig2_HTML.png"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Comparison of the enrichment magnitude using the manufacturer protocol and the revised protocol. (<b>A</b>) Violin plots with the mean depicted show that the revised protocol results in substantially higher fold enrichment by approximately one order of magnitude. (<b>B</b>) A scatter plot shows that the revised protocol is particularly effective for samples with low starting quantities of host DNA. While some samples still had relatively small percentages of reads mapping to the baboon reference genome, these generally also exhibited the highest fold increases.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41598-018-20427-9/figures/2" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>The distribution of blood- and fecal-derived reads did not differ significantly in the length of RADtags, the GC percentage, or the local CpG density, defined as the number of CpG sites in a region ± 5,000 bp from the boundaries of a RADtag (Wilcoxon rank sum tests, <i>p</i> &gt; 0.99 for all three tests; Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S2</a>).</p><p>MBD binding may in principle select for genomic regions with relatively high CpG-methylation density, leading to dropout of other loci. Assessment of the concordance between blood- and feces-derived reads from the same individual was complicated by the correlation in ddRADseq between total reads and expected RADtags recovered and thereby SNPs discovered: a given RADtag is sequenced at a frequency inversely proportional to the deviation of its length from the mean of the size selection. Thus, we had to discern between dropout due to coverage-related stochasticity inherent in ddRADseq<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR27" id="ref-link-section-d28261e804" title="Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species. PLoS One 7, e37135 (2012).">27</a></sup> and that due to MBD enrichment. To perform this comparison, we computed the proportion of unique alleles between blood- and feces-derived RADtags from the same individual. For this test, we controlled for variation in sequencing coverage by randomly sampling reads as necessary in order to equalize total coverage among same-individual samples. Allelic dropout due to MBD enrichment would result in a higher proportion of alleles unique to blood-derived libraries relative to feces-derived libraries. We did not find a significant discrepancy (multi-sample-called SNPs: mean proportion unique alleles in blood = 2.3%, mean proportion unique alleles in feces = 2.3%; Wilcoxon signed rank test, <i>p</i> = 0.97; Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-018-20427-9#Fig3">3A</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Figure 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-label="image" href="/articles/s41598-018-20427-9/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure3" aria-describedby="figure-3-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig3_HTML.png"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Concordance between blood- and feces-derived genotyping data from the same individuals. Colors symbolize the six captive individuals included in our study. Within these individuals, we did not find significant differences in (<b>A</b>) the proportion of unique alleles or (<b>B</b>) inbreeding coefficients from blood- and feces-derived libraries. The multidimensional scaling plot of identity-by-state shows (<b>C</b>) population structuring concordant with the known ancestry of animals (Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>). Distances between feces- and blood-derived sets of genotypes from the same individual are minimal, indicating that noise added by the enrichment method is dwarfed by the population structure signal in this baboon population dataset.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-category="article body" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41598-018-20427-9/figures/3" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>Dropout of entire RADtags is easily detectable given a reference genome or sufficient samples for comparison; dropout of a single allele at heterozygous sites is a more insidious potential bias. Allelic dropout due to MBD enrichment would result in a decrease in heterozygosity in MBD-enriched fecal libraries. Inbreeding coefficients (<i>F</i>) computed from same-individual RADtags exhibited in some cases higher values for feces-derived samples (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-018-20427-9#Fig3">3B</a>). This difference, however, was not statistically significant (mean <i>F</i><sub>blood</sub> = 0.63; mean <i>F</i><sub>feces</sub> = 0.71; Wilcoxon signed rank test, <i>p</i> = 0.47), indicating low allelic dropout attributable to the MBD enrichment. For this test, we also controlled for variation in sequencing coverage as described above.</p><p>As investigations of population structure are one potential application of our method, we visualized the wild and captive baboons’ identity-by-state via multidimensional scaling (MDS) using PLINK<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR28" id="ref-link-section-d28261e868" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).">28</a>,<a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR29" id="ref-link-section-d28261e871" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 559 (2015).">29</a></sup>, and confirmed that individuals clustered by their known species or ancestry and that blood- and feces-derived reads from the same individual were close together in the MDS space (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-018-20427-9#Fig3">3C</a>). The results of this “sanity check” are unsurprising, as variance in samples encapsulated by the first MDS components is expected to reflect population and species membership.</p><p>Stringent filtration of SNP sets, as would be implemented in a standard population genetic study, reduced the apparent biases attributable to fecal enrichment, measured both as total SNPs with a significant association with sample type (unfiltered: 25,079 out of 591,726, or 4.2%; filtered: 13 out of 7,202, or 0.2%) as well as total SNPs with significant missingness assessed via a chi-square test (unfiltered: 69,753 out of 550,224, or 12.7%; filtered: 0 out of 5,602, or 0%). Though more work is needed to quantify more exactly the extent and causal factors that lead to missingness, many population genetic analyses are robust to the low level of dropout our analyses reveal in addition to that which is inherent in the RADseq family of techniques<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR30" id="ref-link-section-d28261e881" title="Gautier, M. et al. The effect of RAD allele dropout on the estimation of genetic variation within and between populations. Mol. Ecol. 22, 3165–3178 (2013).">30</a></sup>.</p></div></div></section><section aria-labelledby="Sec3"><div class="c-article-section js-article-section" id="Sec3-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec3">Discussion</h2><div class="c-article-section__content js-collapsible-section" id="Sec3-content"><p>Our methylation-based capture method achieves substantial enrichment of host DNA from fecal samples. Using our revised protocol developed through experimentation, we produced a mean 195-fold enrichment on our final library consisting entirely of fecal DNA obtained noninvasively under remote field conditions, with most samples nearly a decade old. A mean 28.8% of reads mapped to the baboon genome, despite starting with only a mean 0.34% of host DNA. Using fecal and blood DNA obtained from captive animals, we further demonstrate that feces-derived genotyping data following our method are concordant with corresponding data obtained from blood.</p><p>Feces are among the most readily accessible sources of information on wild animals<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR1" id="ref-link-section-d28261e896" title="Kohn, M. H. &amp; Wayne, R. K. Facts from feces revisited. Trends Ecol. Evol. 12, 223–227 (1997).">1</a></sup>, and are particularly useful for population-level studies or studies of endangered or elusive species for which obtaining high-quality samples is difficult or undesirable. By exploiting methylation differences rather than sequence differences between host and bacterial DNA, FecalSeq is an enrichment strategy that requires neither prior genome sequence knowledge nor the use of high-quality DNA for preparation of capture baits. This results in enrichment which is both inexpensive and replicable. The enrichment procedure is also relatively rapid and uncomplicated. Using a 96-well plate, we performed two sequential rounds of enrichment on all forty samples in our final library within a day (see Supplemental Protocol).</p><p>Compared to comparable experiments using high-quality DNA samples such as blood, our enrichment method introduces extremely low added costs. After excluding shared costs such as DNA extraction, library preparation, and sequencing, major costs associated with our method are qPCR reagents for initial quality assessment of fecal DNA samples and enrichment reagents for capturing the host genome. qPCR reagents cost about $0.60 USD per reaction (or $1.20 per sample assuming samples are run in duplicate). For our enrichment protocol, the amount of reagents used will vary based on the starting proportion of host DNA in the sample (see Supplemental Protocol). Assuming fecal DNA samples on average contain 2.5% host DNA, a single enrichment kit will support a total of 240 enrichment reactions at $0.70 per sample. Based on our experience, most fecal DNA samples contain less than 2.5% host DNA and will therefore require less reagents, further lowering the cost per sample. Following enrichment, we purified DNA using homemade SPRI beads<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR31" id="ref-link-section-d28261e903" title="Rohland, N. &amp; Reich, D. Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. Genome Res. 22, 939–946 (2012).">31</a></sup> which add a very low cost per sample (about $0.10 per sample).</p><p>Altogether, compared to high-quality DNA experiments, the marginal cost of our method is approximately an additional $2 USD per sample factoring in duplicate qPCR reactions, enrichment, and SPRI bead cleanup. These costs represent a substantial decrease relative to enrichment methods based on oligonucleotide-based sequence capture<sup><a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR5" id="ref-link-section-d28261e910" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a>,<a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e913" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup>, which at present represents the only other published strategy for enriching fecal DNA. Snyder-Mackler <i>et al</i>.<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e920" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup> present the most cost-effective sequence-capture-based strategy to date at a cost of $60 per sample, or $29 per sample when using a multiplexed capture strategy on 10 samples at a time. Overall, our strategy therefore decreases the cost of enrichment by about one order of magnitude.</p><p>Importantly, FecalSeq is to our knowledge the first genomic-scale fecal DNA enrichment method that is compatible with most downstream library preparation methods for massively parallel sequencing. Through our use of ddRADseq, we demonstrate that our method facilitates low-cost high-capacity genotyping of wild populations without introducing significant bias. Further, because ddRADseq is customizable<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR27" id="ref-link-section-d28261e928" title="Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species. PLoS One 7, e37135 (2012).">27</a></sup>, there is substantial flexibility for researchers to optimize the number of samples and the fraction of the genome sequenced for particular research questions. This is not possible for libraries prepared using targeted sequence capture, which are therefore currently limited mainly to low-coverage analyses at the population level<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e932" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup>. Transcription of sequence capture baits from reduced-representation libraries may potentially help address this problem<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR14" id="ref-link-section-d28261e936" title="Suchan, T. et al. Hybridization capture using RAD probes (hyRAD), a new tool for performing genomic analyses on museum collection specimens. PLoS One 11, e0151651 (2016).">14</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR15" id="ref-link-section-d28261e936_1" title="Ali, O. A. et al. RAD Capture (Rapture): flexible and efficient sequence-based genotyping. Genetics 202, 389–400 (2016).">15</a>,<a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR16" id="ref-link-section-d28261e939" title="Hoffberg, S. L. et al. RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data. Mol. Ecol. Resour. 16, 1264–1278 (2016).">16</a></sup>, but its efficacy for fecal DNA has yet to be demonstrated.</p><p>Although ideally suited to taxa with a reference genome, genotyping via double-digest RADseq is possible for species that lack a closely related reference genome. We aligned our sequencing reads to the baboon reference genome for this study, but our approach is likely also applicable to species without a reference genome. A reference genome from a more phylogenetically distant species can be used, although reads from divergent regions will fail to map. If a nearby genome is not available, an additional pre-screening step would be necessary, in which exogenous reads are filtered out through comparison to the nearest available genome, before proceeding to clustering and variant identification as per normal reference-free ddRADseq methods. Although these caveats will be mitigated as more genomes become available, the proximity of study individuals to an available reference genome is an important consideration when deciding if ddRADseq methods, including ddRADseq following our enrichment method, are suitable to address the research goals.</p><p>We robustly found that sequencing efficiency (percentage of reads assigned to target genome) of MBD-enriched fecal DNA libraries correlates strongly with starting proportions of host DNA, echoing findings using other capture methods<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e949" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a></sup>. Future attention should therefore be directed towards fecal sample collection, storage, and extraction methods that maximize the selective recovery of host nuclear DNA<sup><a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR32" id="ref-link-section-d28261e953" title="Ramón-Laca, A., Soriano, L., Gleeson, D. &amp; Godoy, J. A. A simple and effective method for obtaining mammal DNA from faeces. Wildlife Biol. 21, 195–203 (2015).">32</a></sup>. While we demonstrated effective genotyping of samples with often very low starting proportions of host DNA (the vast majority &lt; 0.5%), future studies may consider pre-screening extracted DNA samples using qPCR to select for samples with high starting proportions of host DNA.</p><p>We found that, for unknown reasons, starting host DNA concentration for fecal samples collected in captive conditions was higher than that of fecal samples collected from the wild, despite consistent sample collection and processing protocols. We speculate that these differences may be due to either the freshness of samples collected in captivity or to dietary differences between captivity and the wild. Because our wild fecal samples were collected under common biological field conditions (e.g., tropical heat and humidity, lack of refrigeration), our study underscores the need to validate enrichment methods for wild animals using samples from the wild.</p><p>Low starting proportions of host DNA present a challenge not only because they result in lower sequencing efficiency, but also because they correlate with low absolute quantities of DNA belonging to the host organism. In some cases, particularly in samples collected from wild animals under field conditions, starting proportions of host DNA were so low that only approximately 0.1 ng of target DNA was available in a 1 μg fecal DNA extract. Given the large genome sizes of baboons (approximately 3 Gb) and many other vertebrates, substantial allelic dropout is expected in these cases. Significantly, this challenge cannot be fully addressed by this or any other enrichment method and remains an important consideration for researchers working with feces. It can be minimized, however, by optimizing the enrichment procedures to maximize the recovery of target DNA present in a fecal DNA sample, as well as by increasing the total amount of starting fecal DNA.</p><p>Because MBD enrichment partitions DNA based on CpG-methylation density, FecalSeq does not enrich hypomethylated host mitochondrial DNA<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR33" id="ref-link-section-d28261e966" title="Rebelo, A. P., Williams, S. L. &amp; Moraes, C. T. In vivo methylation of mtDNA reveals the dynamics of protein-mtDNA interactions. Nucleic Acids Res. 37, 6701–6715 (2009).">33</a></sup>. While this may be undesirable for studies requiring the matrilineally inherited marker, it also precludes the disproportionately high representation of mitochondrial DNA that is typical in libraries prepared using the targeted sequence capture approach<sup><a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR5" id="ref-link-section-d28261e970" title="Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. Mol. Ecol. 19, 5332–5344 (2010).">5</a>,<a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR6" id="ref-link-section-d28261e973" title="Snyder-Mackler, N. et al. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. Genetics 203, 699–714 (2016).">6</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR9" id="ref-link-section-d28261e976" title="Carpenter, M. L. et al. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. Am. J. Hum. Genet. 93, 852–864 (2013).">9</a>,<a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR21" id="ref-link-section-d28261e979" title="Samuels, D. C. et al. Finding the lost treasures in exome sequencing data. Trends Genet. 29, 593–599 (2013).">21</a></sup>. FecalSeq may, however, co-enrich nuclear DNA from exogenous eukaryotes such as from plant or animal digesta. Care should therefore be taken to minimize the presence of exogenous eukaryotic tissues or cells, although the degree to which this is a problem in practice is currently unknown. As cell-wall-bound plant cells may be more likely to pass through the digestive tract intact, extraction methods that minimize lysis of cell walls should be preferred. We speculate that prey DNA from carnivorous animals may be more difficult to partition from host DNA. While our comparisons of inbreeding coefficients, unique alleles, and locus-by-locus statistical tests from matched blood and fecal samples revealed no significant effect of sample type and therefore enrichment, our small sample size limits our power to detect differences with small effect size. Subsequent studies with larger sample sizes may detect a significant effect of enrichment, but one which may be negligible or correctable for some applications.</p><p>Since PCR amplification of DNA from feces was first achieved in the 1990s<sup><a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR3" id="ref-link-section-d28261e987" title="Höss, M., Kohn, M., Pääbo, S., Knauer, F. &amp; Schröder, W. Excrement analysis by PCR. Nature 359, 199 (1992).">3</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR34" id="ref-link-section-d28261e990" title="Constable, J. J., Packer, C., Collins, D. A. &amp; Pusey, A. E. Nuclear DNA from primate dung. Nature 373, 393–393 (1995).">34</a>,<a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR35" id="ref-link-section-d28261e993" title="Gerloff, U. et al. Amplification of hypervariable simple sequence repeats (microsatellites) from excremental DNA of wild living bonobos (Pan paniscus). Mol. Ecol. 4, 515–518 (1995).">35</a></sup>, noninvasive genetic studies have revolutionized our understanding of the evolution, ecology, and behavior of nonmodel organisms. By facilitating low-cost genomic-scale sequencing from feces, our method connects a community of field researchers with the benefits of massively parallel sequencing, ushering noninvasive organism studies into the genomic age.</p></div></div></section><section aria-labelledby="Sec4"><div class="c-article-section js-article-section" id="Sec4-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec4">Methods</h2><div class="c-article-section__content js-collapsible-section" id="Sec4-content"><h3 class="c-article__sub-heading">Samples</h3><p>Blood and fecal samples were collected from six captive baboons (genus <i>Papio</i>) housed at the Southwest National Primate Research Center (SNPRC) at the Texas Biomedical Research Institute. The individuals were of either <i>P</i>. <i>anubis</i> or hybrid ancestry (Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>). All six baboons were fed a diet manufactured by Purina LabDiet (“Monkey Diet 15%”) containing 15% minimum crude protein, 4% minimum crude fat, and 10% maximum crude fiber. In separate sedation events, blood and feces were collected from the same individual who was isolated for the duration of the sedation. Following centrifugation, the buffy coat was isolated from blood samples and stored at −80 °C. 2 ml of feces were also collected into 8 ml tubes containing 4 ml of RNALater (Ambion).</p><p>In addition, we collected or obtained fecal samples from 46 wild baboons in Zambia. Samples were collected between 2006 and 2015 from the Luangwa Valley, the Lower Zambezi National Park, Choma, or Kafue National Park and are of <i>P</i>. <i>kindae</i> × <i>P</i>. <i>cynocephalus</i>, <i>P</i>. <i>ursinus griseipes</i>, or <i>P</i>. <i>kindae</i> × <i>P</i>. <i>ursinus griseipes</i> ancestry (Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S1</a>). As with the SNPRC samples, 2 ml of feces were collected into 8 ml tubes containing 4 ml of RNALater. In contrast to the SNPRC samples, however, these samples were collected noninvasively from unhabituated animals in remote field conditions. Samples therefore could not be attributed to particular animals, although samples were selected to avoid duplication using either field observations or geographic distance. Following collection, samples were stored without refrigeration for 1–6 months before being frozen at −20 °C for long-term storage.</p><p>All procedures involving live animals were carried out in accordance with relevant guidelines and regulations. Experimental procedures at SNPRC were conducted with approval by the Institutional Animal Care and Use Committee of the Texas Biomedical Research Institute (protocol #1403 PC 0). Sedation and blood draws were performed under the supervision of a veterinarian and animals were returned immediately to their enclosures following recovery. Sample collection in Zambia was conducted with approval by the Animal Studies Committee of Washington University (assurance #A-3381–01) and following local laws and regulations in Zambia.</p><p>Buffy coat extractions were performed using the QIAamp DNA Blood Mini Kit (Qiagen), following manufacturer’s instructions. Fecal extractions were performed using the QIAamp DNA Stool Mini Kit (Qiagen) following manufacturer’s instructions for optimizing host DNA yields. DNA concentration and yield were measured using a Qubit dsDNA BR Assay (Life Technologies). In some cases, multiple DNA extractions from the same individuals were necessary when DNA was depleted over the course of this study.</p><p>We estimated the proportion of host DNA for each fecal DNA extraction using quantitative PCR (qPCR) by comparing estimates of host DNA concentration obtained by qPCR to estimates of total fecal DNA concentration obtained by Qubit. Amplification was conducted using universal mammalian <i>MYCBP</i> primers<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR36" id="ref-link-section-d28261e1071" title="Morin, P. A., Chambers, K. E. &amp; Boesch, C. &amp; Vigilant, L. Quantitative polymerase chain reaction analysis of DNA from noninvasive samples for accurate microsatellite genotyping of wild chimpanzees (Pan troglodytes verus). Mol. Ecol. 10, 1835–1844 (2001).">36</a></sup> and evaluated against a standard curve constructed from the liver DNA of an individual baboon. Samples and standards were run in duplicate alongside positive and negative controls (see Supplemental Protocol for full details).</p><h3 class="c-article__sub-heading">DNA enrichment</h3><p>DNA was enriched using the NEBNext Microbiome DNA Enrichment Kit (New England Biolabs)<sup><a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR22" id="ref-link-section-d28261e1083" title="Feehery, G. R. et al. A method for selectively enriching microbial DNA from contaminating vertebrate host DNA. PLoS One 8, e76096 (2013).">22</a></sup>.</p><p>MBD2-Fc-bound magnetic beads were prepared according to manufacturer instructions in batches ranging from 40 to 160 μl. For each <i>n</i> μl batch, we prebound 0.1 × <i>n</i> μl MBD2-Fc protein to <i>n</i> μl protein A magnetic beads by incubating the mixture with rotation for 10 min at room temperature. The bound MBD2-Fc magnetic beads were then collected by magnet and washed twice with 1 ml ice-cold 1x bind/wash buffer before being resuspended in <i>n</i> μl ice-cold 1x bind/wash buffer.</p><p>As a pilot experiment, we prepared two successive libraries, library A and library B, following manufacturer’s instructions for capturing methylated host DNA, with minor protocol modifications incorporated for the second pilot library (library B). Library A included MBD-enriched fecal DNA from 4 SNPRC baboons and 2 Luangwa Valley baboons, as well as blood DNA from the same SNPRC baboons. Library B included MBD-enriched fecal DNA from 4 SNPRC baboons (with two repeats from library A), 4 Kafue National Park baboons, and 2 Luangwa Valley baboons, as well as blood DNA from 2 SNPRC baboons. For each fecal DNA sample, we combined 1–2 μg of extracted fecal DNA with 160 μl of prepared protein-bound beads and a variable volume of ice-cold 5x bind/wash buffer for maintaining 1x concentration of bind/wash buffer. After combining beads and DNA, we incubated the mixture at room temperature with rotation for 15 min. DNA and MBD2-Fc-bound magnetic beads were then collected by magnet and the supernatant removed. For samples in library A, we washed the collected beads with 1 ml of ice-cold 1x bind/wash buffer. For samples in library B, we conducted three expanded wash steps to maximize the removal of unbound DNA. For each wash in library B, we added 1 ml of ice-cold 1x bind/wash buffer and mixed the beads on a rotating mixer for three minutes at room temperature before collecting the beads by magnet and removing the supernatant. Following the final wash, we resuspended and incubated the beads at 65 °C with 150 μL of 1x TE buffer and 15 μL of Proteinase K for 20 min with occasional mixing. The eluted DNA was then separated by magnet, purified with 1.5x homemade SPRI bead mixture<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR31" id="ref-link-section-d28261e1105" title="Rohland, N. &amp; Reich, D. Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. Genome Res. 22, 939–946 (2012).">31</a></sup>, and quantified using a Qubit dsDNA HS Assay (Life Technologies).</p><p>Our pilot sequencing results from libraries A and B revealed large variation in the percentage of reads mapping to the baboon genome, with mapping percentages ranging from 1.1% to 79.3%, with much of the variation correlating with the proportion of host DNA in the unenriched fecal DNA sample (Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>). To expand the utility of the enrichment protocol to all fecal DNA samples, we conducted a series of capture experiments designed to optimize the enrichment of host DNA from “low-quality” samples (i.e., samples with low proportions of host DNA). For these experiments, we artificially simulated fecal DNA by combining high-quality baboon liver or blood genomic DNA (liver: SNPRC ID #19334; blood: SNPRC ID #14068 or #25567) with <i>E</i>. <i>coli</i> DNA (K12 or ATCC 11303 strains) at controlled proportions. The resulting post-enrichment proportion of baboon and <i>E</i>. <i>coli</i> DNA was evaluated by qPCR in two analyses using (1) universal mammalian <i>MYCBP</i><sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR36" id="ref-link-section-d28261e1130" title="Morin, P. A., Chambers, K. E. &amp; Boesch, C. &amp; Vigilant, L. Quantitative polymerase chain reaction analysis of DNA from noninvasive samples for accurate microsatellite genotyping of wild chimpanzees (Pan troglodytes verus). Mol. Ecol. 10, 1835–1844 (2001).">36</a></sup> and (2) universal bacterial 16 S rRNA (16 S)<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR37" id="ref-link-section-d28261e1134" title="Corless, C. E. et al. Contamination and sensitivity issues with a real-time universal 16S rRNA PCR. J. Clin. Microbiol. 38, 1747–1752 (2000).">37</a></sup> primers along with standards created from the same respective organisms (experiments and results are described in detail in Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S2</a>).</p><p>Based on these capture optimization experiments, we prepared subsequent libraries using a version of the protocol incorporating modifications demonstrated to improve enrichment. Despite preferentially binding CpG-methylated DNA, the MBD2-Fc bait complex nevertheless bound a fraction of nonmethylated DNA. We therefore aimed to minimize both the absolute and relative amount of nonmethylated DNA binding to the bead complex. In our tests, the amount and fraction of nonmethylated bound DNA was highest when the ratio of the MBD2-Fc magnetic bead complex to total DNA was high, suggesting a surplus of MBD2 binding sites given the relatively small fraction of CpG-methylated DNA. We therefore reduced the ratio of prepared MBD2-Fc-bound magnetic beads to total DNA by tuning the amount of beads to the estimated amount of CpG-methylated host DNA. Because the amount of CpG-methylated host DNA in feces is extremely low, this modification also greatly decreased the cost of the reagents. Through our optimization experiments, we found that incorporating an additional wash step reduced the amount of contaminating nonmethylated DNA captured. Finally, we developed a method for serial enrichment of the samples (repeating the enrichment protocol), which substantially improved results. Our initial serial enrichment experiments failed to recover DNA, likely due to the use of proteinase K (combined with TE buffer) in the manufacturer’s elution protocol. Hypothesizing that incomplete removal of proteinase K, even following bead cleanup, interfered with the enrichment protocol, we instead eluted DNA using a high (2 M) NaCl concentration, resulting in successful serial enrichment. These changes, along with a full modified protocol, are detailed in the Supplemental Protocol.</p><p>For our next two libraries, libraries C and D, we added a much smaller volume of prepared MBD2-Fc-bound magnetic beads (1–22 μl) based on the estimated proportion of starting host DNA, kept the capture reaction volume consistent at a relatively low 40 μl (concentrating samples as needed using a SPRI bead cleanup), added an extra wash step in which samples were resuspended in 100 μl of 1x bind/wash buffer then incubated at room temperature for 3 minutes with rotation, and eluted samples in 100 μl 2 M NaCl. For four fecal DNA samples in library C and all of library D, we serially enriched the samples by repeating the capture reaction with 30 μl of MBD-enriched DNA (post SPRI-bead cleanup). Library C included fecal DNA from 5 SNPRC baboons, 2 Kafue National Park baboons, and 1 Luangwa Valley baboon. Library D contained fecal DNA from 6 Lower Zambezi National Park baboons, 4 Choma baboons, and 30 Kafue National Park baboons. We prepared a final library, library E, from independently extracted blood DNA from five SNPRC baboons in order to quantify the stochasticity associated with independent library preparation from independent extracts. The composition of libraries A-E are described in detail in Supplemental Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S2</a>-<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>.</p><h3 class="c-article__sub-heading">Library preparation and sequencing</h3><p>Library preparation followed standard double-digest restriction site-associated DNA sequencing (ddRADseq) procedures<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR27" id="ref-link-section-d28261e1162" title="Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species. PLoS One 7, e37135 (2012).">27</a></sup> with modifications to accommodate low input as described below.</p><p>For all samples, including blood DNA and MBD-enriched fecal DNA, we digested DNA with <i>Sph</i>I and <i>Mlu</i>CI (New England Biolabs), following a ratio of 1 unit of each enzyme per 20 ng of DNA. Enzymes were diluted up to 10× using compatible diluents (New England Biolabs) to facilitate pipetting of small quantities, using an excess of enzyme if necessary to avoid pipetting less than 1 μl of the diluted enzyme mix. As the total amount of post-enrichment fecal DNA is by nature low, we adjusted adapter concentrations in the ligation reaction to ~0.1 μM for barcoded P1 and ~3 μM for P2, which correspond to excesses of adapters between 1–2 orders of magnitude. Since adapter-ligated samples are multiplexed into pools in equimolar amounts, we made efforts to combine samples with similar concentrations and enrichment when known. We used the BluePippin (Sage Science) with a 1.5% agarose gel cassette for automated size selection of pooled individuals, with a target of 300 bp (including adapters) and extraction of a “tight” collection range (±39 bp). For PCR amplification, we ran all reactions in quadruplicate to minimize PCR biases and attempted to limit the number of PCR cycles. As the concentration of post-size-selection pools was below the limits of detection without loss of a considerable fraction of the sample, estimation of the required number of PCR cycles was difficult. We therefore iteratively quantified products post-PCR and added cycles as necessary. The total number of PCR cycles per pool is reported in Supplemental Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S3</a>, but was usually 24. Finally, libraries were sequenced using either Illumina MiSeq (libraries A-C; 2 × 150 paired-end) or Illumina HiSeq. 2500 (library D; 2 × 100 paired-end) sequencing with 30% spike in of PhiX control DNA.</p><h3 class="c-article__sub-heading">Analysis</h3><p>We demultiplexed reads by sample and mapped them to the baboon reference genome (Panu 2.0; Baylor College of Medicine Human Genome Sequencing Center) using BWA with default parameters and the BWA-ALN algorithm<sup><a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR38" id="ref-link-section-d28261e1186" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009).">38</a></sup>. For every pair of blood and fecal samples from the same individual, we downsampled mapped reads to create new pairs with equal coverage in order to control for biases due to differences in sequencing depth. After realignment around indels, we identified variants using GATK UnifiedGenotyper<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR39" id="ref-link-section-d28261e1190" title="DePristo, M. A. et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat. Genet. 43, 491–498 (2011).">39</a></sup>, in parallel analyses (1) calling variants in all samples at once and (2) processing each sample in isolation to avoid biasing variant calls from other samples at the expense of accuracy. Homozygous sites matching the reference genome were listed as missing when variants were inferred in single individuals. Variants were filtered with GATK VariantFiltration (filters: QD &lt; 2.0, MQ &lt; 40.0, FS &gt; 60.0, HaplotypeScore &gt; 13.0, MQRankSum &lt; −12.5, ReadPosRankSum &lt; −8.0) and indels were excluded.</p><p>We digested the baboon reference genome <i>in silico</i>, tallied reads within each predicted RADtag, and gathered the following information about each region: length, GC percentage, and CpG count in region ± 5 kb. We also calculated read depth in these simulated RADtags. Distributions of blood and fecal RADtags’ length, GC percentage, and local CpG density (Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-018-20427-9#MOESM1">S2</a>) were compared using Wilcoxon rank sum tests and visually inspected for possible gross distortion due to widespread dropout.</p><p>If the fecal enrichment procedure caused widespread allelic dropout, the proportion of alleles unique to the blood samples would be higher than that to the fecal sample. We tallied these unique alleles by using VCFtools<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR40" id="ref-link-section-d28261e1206" title="Danecek, P. et al. The variant call format and VCFtools. Bioinformatics 27, 2156–2158 (2011).">40</a></sup> to compute discordance on a per-site basis “--diff-site-discordance” as well as a discordance matrix “--diff-discordance-matrix”, and parsed the results to compute unique SNP percentage in paired blood and fecal samples. We tested for an excess of unique SNPs in blood with a Wilcoxon signed rank test.</p><p>To quantify loss of heterozygosity due to allelic dropout, we computed the method-of-moments inbreeding coefficient, <i>F</i> for all blood-feces pairs with equalized coverage, using both the individually called and multi-sample called SNP sets. <i>F</i> was calculated using the “--het” argument in PLINK<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR28" id="ref-link-section-d28261e1219" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).">28</a>,<a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR29" id="ref-link-section-d28261e1222" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 559 (2015).">29</a></sup>. The presence of dropout is expected to inflate <i>F</i>. We tested for differences in paired samples’ estimates of <i>F</i> via a Wilcoxon signed rank test. The dataset is not filtered for missingness, so sequencing errors inferred to be true variants may inflate heterozygosity estimates, thus deflating <i>F</i>.</p><p>To create a stringently filtered dataset with high genotyping rate, we filtered the multi-sample called SNPs in PLINK<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR28" id="ref-link-section-d28261e1240" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).">28</a>,<a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-018-20427-9#ref-CR29" id="ref-link-section-d28261e1243" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 559 (2015).">29</a></sup>, retaining only those genotyped in at least 90% of samples and removing samples with genotypes at fewer than 10% of sites. This filtered set was further pruned for linkage disequilibrium by sliding a window of 50 SNPs across the chromosome and removing one random SNP in each pair with <i>r</i><sup>2</sup> &gt; 0.5. Using all samples, we performed multidimensional scaling to visualize identity by state (IBS). Using just the samples that were part of the same-individual blood-feces pairs, we then performed an association test and missingness chi-square test to detect allele frequencies or missingness that correlated with sample type. We did the same with the unfiltered dataset as well. Though we had few pairs of fecal samples from the same individual, we computed distance between pairs of samples from the same individual using the stringently filtered dataset to compare distance between and within sample types via a Wilcoxon signed rank test.</p></div></div></section>
<section aria-labelledby="data-availability"><div class="c-article-section js-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data Availability</h2><div class="c-article-section__content js-collapsible-section" id="data-availability-content">
<p>All raw sequencing reads are publicly accessible through the NCBI Sequence Read Archive (BioProject #369456, BioSample accessions #SAMN06286370 - SAMN06286448). All code generated for this project is released under a GNU v3.0 license, archived on Zenodo (doi:10.5281/zenodo.848299), and can be accessed at <a href="https://github.com/bergeycm/RAD-faex">https://github.com/bergeycm/RAD-faex</a>.</p>
</div></div></section><section aria-labelledby="additional-information"><div class="c-article-section js-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content js-collapsible-section" id="additional-information-content"><p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section aria-labelledby="change-history"><div class="c-article-section js-article-section" id="change-history-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="change-history">Change history</h2><div class="c-article-section__content js-collapsible-section" id="change-history-content"><ul class="c-article-change-list"><li class="c-article-change-list__item" id="chg1"><ins datetime="2018-12-14"><h3 class="c-article-change-list__heading">14 December 2018</h3><div class="c-article-change-list__details"><p>
Editors’ Note: The editors have become aware of unusual aspects to the ‘Extract fecal DNA’ illustration in figure 1. We are investigating, and appropriate editorial action will be taken once the matter is resolved.
</p></div></ins></li><li class="c-article-change-list__item" id="chg2"><ins datetime="2018-12-19"><h3 class="c-article-change-list__heading">19 December 2018</h3><div class="c-article-change-list__details"><p>
In the original version of this Article, there were unusual aspects to the ‘Extract fecal DNA’ illustration in figure 1. These features have been removed.
</p></div></ins></li></ul></div></div></section><section aria-labelledby="Bib1"><div class="c-article-section js-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content js-collapsible-section" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">1.</span><p class="c-article-references__text" id="ref-CR1" itemprop="headline">Kohn, M. H. &amp; Wayne, R. K. Facts from feces revisited. <i>Trends Ecol. Evol.</i> <b>12</b>, 223–227 (1997).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 1 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ASTN%3A280%3ADC%252BC3M7itFKksg%253D%253D">CAS</a></li><li><a aria-label="View reference 1" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS0169-5347%2897%2901050-1">Article</a></li><li><a aria-label="Search for reference 1 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Facts%20from%20feces%20revisited&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=12&amp;pages=223-227&amp;publication_year=1997&amp;author=Kohn%2CMH&amp;author=Wayne%2CRK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">2.</span><p class="c-article-references__text" id="ref-CR2" itemprop="headline">Putman, R. J. Facts from faeces. <i>Mamm. Rev.</i> <b>14</b>, 79–97 (1984).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 2" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-2907.1984.tb00341.x">Article</a></li><li><a aria-label="Search for reference 2 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Facts%20from%20faeces&amp;journal=Mamm.%20Rev.&amp;volume=14&amp;pages=79-97&amp;publication_year=1984&amp;author=Putman%2CRJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">3.</span><p class="c-article-references__text" id="ref-CR3" itemprop="headline">Höss, M., Kohn, M., Pääbo, S., Knauer, F. &amp; Schröder, W. Excrement analysis by PCR. <i>Nature</i> <b>359</b>, 199 (1992).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 3 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1992Natur.359..199H">ADS</a></li><li><a aria-label="View reference 3" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2F359199a0">Article</a></li><li><a aria-label="Search for reference 3 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Excrement%20analysis%20by%20PCR&amp;journal=Nature&amp;volume=359&amp;publication_year=1992&amp;author=H%C3%B6ss%2CM&amp;author=Kohn%2CM&amp;author=P%C3%A4%C3%A4bo%2CS&amp;author=Knauer%2CF&amp;author=Schr%C3%B6der%2CW">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">4.</span><p class="c-article-references__text" id="ref-CR4" itemprop="headline">Qin, J. <i>et al</i>. A human gut microbial gene catalogue established by metagenomic sequencing. <i>Nature</i> <b>464</b>, 59–65 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 4 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D">CAS</a></li><li><a aria-label="View reference 4" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnature08821">Article</a></li><li><a aria-label="Search for reference 4 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;journal=Nature&amp;volume=464&amp;pages=59-65&amp;publication_year=2010&amp;author=Qin%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">5.</span><p class="c-article-references__text" id="ref-CR5" itemprop="headline">Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. <i>Mol. Ecol.</i> <b>19</b>, 5332–5344 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 5 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlagsLY%253D">CAS</a></li><li><a aria-label="View reference 5" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-294X.2010.04888.x">Article</a></li><li><a aria-label="Search for reference 5 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic-scale%20capture%20and%20sequencing%20of%20endogenous%20DNA%20from%20feces&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=5332-5344&amp;publication_year=2010&amp;author=Perry%2CGH&amp;author=Marioni%2CJC&amp;author=Melsted%2CP&amp;author=Gilad%2CY">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">6.</span><p class="c-article-references__text" id="ref-CR6" itemprop="headline">Snyder-Mackler, N. <i>et al</i>. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. <i>Genetics</i> <b>203</b>, 699–714 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 6 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitFSit77I">CAS</a></li><li><a aria-label="View reference 6" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1534%2Fgenetics.116.187492">Article</a></li><li><a aria-label="Search for reference 6 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20genome-wide%20sequencing%20and%20low%20coverage%20pedigree%20analysis%20from%20non-invasively%20collected%20samples&amp;journal=Genetics&amp;volume=203&amp;pages=699-714&amp;publication_year=2016&amp;author=Snyder-Mackler%2CN">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">7.</span><p class="c-article-references__text" id="ref-CR7" itemprop="headline">Whitney, D. <i>et al</i>. Enhanced retrieval of DNA from human fecal samples results in improved performance of colorectal cancer screening test. <i>J. Mol. Diagn.</i> <b>6</b>, 386–395 (2004).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 7 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhtVOhtbjI">CAS</a></li><li><a aria-label="View reference 7" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS1525-1578%2810%2960536-3">Article</a></li><li><a aria-label="Search for reference 7 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20retrieval%20of%20DNA%20from%20human%20fecal%20samples%20results%20in%20improved%20performance%20of%20colorectal%20cancer%20screening%20test&amp;journal=J.%20Mol.%20Diagn.&amp;volume=6&amp;pages=386-395&amp;publication_year=2004&amp;author=Whitney%2CD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">8.</span><p class="c-article-references__text" id="ref-CR8" itemprop="headline">Melnikov, A. <i>et al</i>. Hybrid selection for sequencing pathogen genomes from clinical samples. <i>Genome Biol.</i> <b>12</b>, R73 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 8 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtF2gu7jK">CAS</a></li><li><a aria-label="View reference 8" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1186%2Fgb-2011-12-8-r73">Article</a></li><li><a aria-label="Search for reference 8 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hybrid%20selection%20for%20sequencing%20pathogen%20genomes%20from%20clinical%20samples&amp;journal=Genome%20Biol.&amp;volume=12&amp;publication_year=2011&amp;author=Melnikov%2CA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">9.</span><p class="c-article-references__text" id="ref-CR9" itemprop="headline">Carpenter, M. L. <i>et al</i>. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. <i>Am. J. Hum. Genet.</i> <b>93</b>, 852–864 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 9 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhs12ltbbF">CAS</a></li><li><a aria-label="View reference 9" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.ajhg.2013.10.002">Article</a></li><li><a aria-label="Search for reference 9 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Pulling%20out%20the%201%25%3A%20whole-genome%20capture%20for%20the%20targeted%20enrichment%20of%20ancient%20DNA%20sequencing%20libraries&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=93&amp;pages=852-864&amp;publication_year=2013&amp;author=Carpenter%2CML">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">10.</span><p class="c-article-references__text" id="ref-CR10" itemprop="headline">Albert, T. J. <i>et al</i>. Direct selection of human genomic loci by microarray hybridization. <i>Nat. Methods</i> <b>4</b>, 903–905 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 10 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXht1aqtbfL">CAS</a></li><li><a aria-label="View reference 10" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth1111">Article</a></li><li><a aria-label="Search for reference 10 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-905&amp;publication_year=2007&amp;author=Albert%2CTJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">11.</span><p class="c-article-references__text" id="ref-CR11" itemprop="headline">Okou, D. T. <i>et al</i>. Microarray-based genomic selection for high-throughput resequencing. <i>Nat. Methods</i> <b>4</b>, 907–909 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 11 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXht1aqur7N">CAS</a></li><li><a aria-label="View reference 11" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth1109">Article</a></li><li><a aria-label="Search for reference 11 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Microarray-based%20genomic%20selection%20for%20high-throughput%20resequencing&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=907-909&amp;publication_year=2007&amp;author=Okou%2CDT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">12.</span><p class="c-article-references__text" id="ref-CR12" itemprop="headline">Gnirke, A. <i>et al</i>. Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. <i>Nat. Biotechnol.</i> <b>27</b>, 182–189 (2009).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 12 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXht1ehs7o%253D">CAS</a></li><li><a aria-label="View reference 12" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.1523">Article</a></li><li><a aria-label="Search for reference 12 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=182-189&amp;publication_year=2009&amp;author=Gnirke%2CA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">13.</span><p class="c-article-references__text" id="ref-CR13" itemprop="headline">Ng, S. B. <i>et al</i>. Targeted capture and massively parallel sequencing of 12 human exomes. <i>Nature</i> <b>461</b>, 272–276 (2009).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 13 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.461..272N">ADS</a></li><li><a aria-label="View reference 13 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXpvFCktLg%253D">CAS</a></li><li><a aria-label="View reference 13" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnature08250">Article</a></li><li><a aria-label="Search for reference 13 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20capture%20and%20massively%20parallel%20sequencing%20of%2012%20human%20exomes&amp;journal=Nature&amp;volume=461&amp;pages=272-276&amp;publication_year=2009&amp;author=Ng%2CSB">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">14.</span><p class="c-article-references__text" id="ref-CR14" itemprop="headline">Suchan, T. <i>et al</i>. Hybridization capture using RAD probes (hyRAD), a new tool for performing genomic analyses on museum collection specimens. <i>PLoS One</i> <b>11</b>, e0151651 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 14" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0151651">Article</a></li><li><a aria-label="Search for reference 14 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hybridization%20capture%20using%20RAD%20probes%20%28hyRAD%29%2C%20a%20new%20tool%20for%20performing%20genomic%20analyses%20on%20museum%20collection%20specimens&amp;journal=PLoS%20One&amp;volume=11&amp;publication_year=2016&amp;author=Suchan%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">15.</span><p class="c-article-references__text" id="ref-CR15" itemprop="headline">Ali, O. A. <i>et al</i>. RAD Capture (Rapture): flexible and efficient sequence-based genotyping. <i>Genetics</i> <b>202</b>, 389–400 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 15 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitVGlu7%252FM">CAS</a></li><li><a aria-label="View reference 15" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1534%2Fgenetics.115.183665">Article</a></li><li><a aria-label="Search for reference 15 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=RAD%20Capture%20%28Rapture%29%3A%20flexible%20and%20efficient%20sequence-based%20genotyping&amp;journal=Genetics&amp;volume=202&amp;pages=389-400&amp;publication_year=2016&amp;author=Ali%2COA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">16.</span><p class="c-article-references__text" id="ref-CR16" itemprop="headline">Hoffberg, S. L. <i>et al</i>. RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data. <i>Mol. Ecol. Resour.</i> <b>16</b>, 1264–1278 (2016).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 16 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtlOmsrbK">CAS</a></li><li><a aria-label="View reference 16" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2F1755-0998.12566">Article</a></li><li><a aria-label="Search for reference 16 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=RADcap%3A%20sequence%20capture%20of%20dual-digest%20RADseq%20libraries%20with%20identifiable%20duplicates%20and%20reduced%20missing%20data&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=16&amp;pages=1264-1278&amp;publication_year=2016&amp;author=Hoffberg%2CSL">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">17.</span><p class="c-article-references__text" id="ref-CR17" itemprop="headline">Maricic, T., Whitten, M. &amp; Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. <i>PLoS One</i> <b>5</b>, e14004 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 17 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PLoSO...514004M">ADS</a></li><li><a aria-label="View reference 17" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0014004">Article</a></li><li><a aria-label="Search for reference 17 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20DNA%20sequence%20capture%20of%20mitochondrial%20genomes%20using%20PCR%20products&amp;journal=PLoS%20One&amp;volume=5&amp;publication_year=2010&amp;author=Maricic%2CT&amp;author=Whitten%2CM&amp;author=P%C3%A4%C3%A4bo%2CS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">18.</span><p class="c-article-references__text" id="ref-CR18" itemprop="headline">Mason, V. C., Li, G., Helgen, K. M. &amp; Murphy, W. J. Efficient cross-species capture hybridization and next-generation sequencing of mitochondrial genomes from noninvasively sampled museum specimens. <i>Genome Res.</i> <b>21</b>, 1695–1704 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 18 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlSru7%252FM">CAS</a></li><li><a aria-label="View reference 18" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1101%2Fgr.120196.111">Article</a></li><li><a aria-label="Search for reference 18 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20cross-species%20capture%20hybridization%20and%20next-generation%20sequencing%20of%20mitochondrial%20genomes%20from%20noninvasively%20sampled%20museum%20specimens&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=1695-1704&amp;publication_year=2011&amp;author=Mason%2CVC&amp;author=Li%2CG&amp;author=Helgen%2CKM&amp;author=Murphy%2CWJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">19.</span><p class="c-article-references__text" id="ref-CR19" itemprop="headline">Bi, K. <i>et al</i>. Unlocking the vault: next-generation museum population genomics. <i>Mol. Ecol.</i> <b>22</b>, 6018–6032 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 19 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhvVyjsb7I">CAS</a></li><li><a aria-label="View reference 19" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fmec.12516">Article</a></li><li><a aria-label="Search for reference 19 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Unlocking%20the%20vault%3A%20next-generation%20museum%20population%20genomics&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=6018-6032&amp;publication_year=2013&amp;author=Bi%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">20.</span><p class="c-article-references__text" id="ref-CR20" itemprop="headline">George, R. D. <i>et al</i>. Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection. <i>Genome Res.</i> <b>21</b>, 1686–1694 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 20 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlSru77F">CAS</a></li><li><a aria-label="View reference 20" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1101%2Fgr.121327.111">Article</a></li><li><a aria-label="Search for reference 20 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Trans%20genomic%20capture%20and%20sequencing%20of%20primate%20exomes%20reveals%20new%20targets%20of%20positive%20selection&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=1686-1694&amp;publication_year=2011&amp;author=George%2CRD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">21.</span><p class="c-article-references__text" id="ref-CR21" itemprop="headline">Samuels, D. C. <i>et al</i>. Finding the lost treasures in exome sequencing data. <i>Trends Genet.</i> <b>29</b>, 593–599 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 21 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhsV2msb%252FF">CAS</a></li><li><a aria-label="View reference 21" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.tig.2013.07.006">Article</a></li><li><a aria-label="Search for reference 21 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Finding%20the%20lost%20treasures%20in%20exome%20sequencing%20data&amp;journal=Trends%20Genet.&amp;volume=29&amp;pages=593-599&amp;publication_year=2013&amp;author=Samuels%2CDC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">22.</span><p class="c-article-references__text" id="ref-CR22" itemprop="headline">Feehery, G. R. <i>et al</i>. A method for selectively enriching microbial DNA from contaminating vertebrate host DNA. <i>PLoS One</i> <b>8</b>, e76096 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 22 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...876096F">ADS</a></li><li><a aria-label="View reference 22 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhslWksrbO">CAS</a></li><li><a aria-label="View reference 22" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0076096">Article</a></li><li><a aria-label="Search for reference 22 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20selectively%20enriching%20microbial%20DNA%20from%20contaminating%20vertebrate%20host%20DNA&amp;journal=PLoS%20One&amp;volume=8&amp;publication_year=2013&amp;author=Feehery%2CGR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">23.</span><p class="c-article-references__text" id="ref-CR23" itemprop="headline">Hendrich, B. &amp; Tweedie, S. The methyl-CpG binding domain and the evolving role of DNA methylation in animals. <i>Trends Genet.</i> <b>19</b>, 269–277 (2003).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 23 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXjtFeqt7w%253D">CAS</a></li><li><a aria-label="View reference 23" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS0168-9525%2803%2900080-5">Article</a></li><li><a aria-label="Search for reference 23 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20methyl-CpG%20binding%20domain%20and%20the%20evolving%20role%20of%20DNA%20methylation%20in%20animals&amp;journal=Trends%20Genet.&amp;volume=19&amp;pages=269-277&amp;publication_year=2003&amp;author=Hendrich%2CB&amp;author=Tweedie%2CS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">24.</span><p class="c-article-references__text" id="ref-CR24" itemprop="headline">Jabbari, K. &amp; Bernardi, G. Cytosine methylation and CpG, TpG (CpA) and TpA frequencies. <i>Gene</i> <b>333</b>, 143–149 (2004).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 24 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXksFOmtrc%253D">CAS</a></li><li><a aria-label="View reference 24" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.gene.2004.02.043">Article</a></li><li><a aria-label="Search for reference 24 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Cytosine%20methylation%20and%20CpG%2C%20TpG%20%28CpA%29%20and%20TpA%20frequencies&amp;journal=Gene&amp;volume=333&amp;pages=143-149&amp;publication_year=2004&amp;author=Jabbari%2CK&amp;author=Bernardi%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">25.</span><p class="c-article-references__text" id="ref-CR25" itemprop="headline">Fang, G. <i>et al</i>. Genome-wide mapping of methylated adenine residues in pathogenic Escherichia coli using single-molecule real-time sequencing. <i>Nat. Biotechnol.</i> <b>30</b>, 1232–1239 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 25 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xhs1elsLbE">CAS</a></li><li><a aria-label="View reference 25" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.2432">Article</a></li><li><a aria-label="Search for reference 25 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20mapping%20of%20methylated%20adenine%20residues%20in%20pathogenic%20Escherichia%20coli%20using%20single-molecule%20real-time%20sequencing&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=1232-1239&amp;publication_year=2012&amp;author=Fang%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">26.</span><p class="c-article-references__text" id="ref-CR26" itemprop="headline">Murray, I. A. <i>et al</i>. The methylomes of six bacteria. <i>Nucleic Acids Res.</i> <b>40</b>, 11450–11462 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 26 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XhvVCitLfP">CAS</a></li><li><a aria-label="View reference 26" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgks891">Article</a></li><li><a aria-label="Search for reference 26 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20methylomes%20of%20six%20bacteria&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=11450-11462&amp;publication_year=2012&amp;author=Murray%2CIA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">27.</span><p class="c-article-references__text" id="ref-CR27" itemprop="headline">Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for <i>de novo</i> SNP discovery and genotyping in model and non-model species. <i>PLoS One</i> <b>7</b>, e37135 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 27 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PLoSO...737135P">ADS</a></li><li><a aria-label="View reference 27 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XosVeksrY%253D">CAS</a></li><li><a aria-label="View reference 27" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0037135">Article</a></li><li><a aria-label="Search for reference 27 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Double%20digest%20RADseq%3A%20an%20inexpensive%20method%20for%20de%20novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species&amp;journal=PLoS%20One&amp;volume=7&amp;publication_year=2012&amp;author=Peterson%2CBK&amp;author=Weber%2CJN&amp;author=Kay%2CEH&amp;author=Fisher%2CHS&amp;author=Hoekstra%2CHE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">28.</span><p class="c-article-references__text" id="ref-CR28" itemprop="headline">Purcell, S. <i>et al</i>. PLINK: a tool set for whole-genome association and population-based linkage analyses. <i>Am. J. Hum. Genet.</i> <b>81</b>, 559–575 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 28 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtVSqurrL">CAS</a></li><li><a aria-label="View reference 28" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1086%2F519795">Article</a></li><li><a aria-label="Search for reference 28 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=81&amp;pages=559-575&amp;publication_year=2007&amp;author=Purcell%2CS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">29.</span><p class="c-article-references__text" id="ref-CR29" itemprop="headline">Chang, C. C. <i>et al</i>. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>Gigascience</i> <b>4</b>, 559 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 29" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1186%2Fs13742-015-0047-8">Article</a></li><li><a aria-label="Search for reference 29 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=Gigascience&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">30.</span><p class="c-article-references__text" id="ref-CR30" itemprop="headline">Gautier, M. <i>et al</i>. The effect of RAD allele dropout on the estimation of genetic variation within and between populations. <i>Mol. Ecol.</i> <b>22</b>, 3165–3178 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 30 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXos12msr4%253D">CAS</a></li><li><a aria-label="View reference 30" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fmec.12089">Article</a></li><li><a aria-label="Search for reference 30 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effect%20of%20RAD%20allele%20dropout%20on%20the%20estimation%20of%20genetic%20variation%20within%20and%20between%20populations&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=3165-3178&amp;publication_year=2013&amp;author=Gautier%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">31.</span><p class="c-article-references__text" id="ref-CR31" itemprop="headline">Rohland, N. &amp; Reich, D. Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. <i>Genome Res.</i> <b>22</b>, 939–946 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 31 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xms1antbk%253D">CAS</a></li><li><a aria-label="View reference 31" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1101%2Fgr.128124.111">Article</a></li><li><a aria-label="Search for reference 31 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Cost-effective%2C%20high-throughput%20DNA%20sequencing%20libraries%20for%20multiplexed%20target%20capture&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=939-946&amp;publication_year=2012&amp;author=Rohland%2CN&amp;author=Reich%2CD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">32.</span><p class="c-article-references__text" id="ref-CR32" itemprop="headline">Ramón-Laca, A., Soriano, L., Gleeson, D. &amp; Godoy, J. A. A simple and effective method for obtaining mammal DNA from faeces. <i>Wildlife Biol.</i> <b>21</b>, 195–203 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 32" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.2981%2Fwlb.00096">Article</a></li><li><a aria-label="Search for reference 32 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20and%20effective%20method%20for%20obtaining%20mammal%20DNA%20from%20faeces&amp;journal=Wildlife%20Biol.&amp;volume=21&amp;pages=195-203&amp;publication_year=2015&amp;author=Ram%C3%B3n-Laca%2CA&amp;author=Soriano%2CL&amp;author=Gleeson%2CD&amp;author=Godoy%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">33.</span><p class="c-article-references__text" id="ref-CR33" itemprop="headline">Rebelo, A. P., Williams, S. L. &amp; Moraes, C. T. <i>In vivo</i> methylation of mtDNA reveals the dynamics of protein-mtDNA interactions. <i>Nucleic Acids Res.</i> <b>37</b>, 6701–6715 (2009).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 33 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhsVajt7%252FK">CAS</a></li><li><a aria-label="View reference 33" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkp727">Article</a></li><li><a aria-label="Search for reference 33 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20methylation%20of%20mtDNA%20reveals%20the%20dynamics%20of%20protein-mtDNA%20interactions&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=6701-6715&amp;publication_year=2009&amp;author=Rebelo%2CAP&amp;author=Williams%2CSL&amp;author=Moraes%2CCT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">34.</span><p class="c-article-references__text" id="ref-CR34" itemprop="headline">Constable, J. J., Packer, C., Collins, D. A. &amp; Pusey, A. E. Nuclear DNA from primate dung. <i>Nature</i> <b>373</b>, 393–393 (1995).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 34 on ADS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1995Natur.373..393C">ADS</a></li><li><a aria-label="View reference 34 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXjsFCjsr4%253D">CAS</a></li><li><a aria-label="View reference 34" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2F373393a0">Article</a></li><li><a aria-label="Search for reference 34 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclear%20DNA%20from%20primate%20dung&amp;journal=Nature&amp;volume=373&amp;pages=393-393&amp;publication_year=1995&amp;author=Constable%2CJJ&amp;author=Packer%2CC&amp;author=Collins%2CDA&amp;author=Pusey%2CAE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">35.</span><p class="c-article-references__text" id="ref-CR35" itemprop="headline">Gerloff, U. <i>et al</i>. Amplification of hypervariable simple sequence repeats (microsatellites) from excremental DNA of wild living bonobos (<i>Pan paniscus</i>). <i>Mol. Ecol.</i> <b>4</b>, 515–518 (1995).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 35 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXosFertLo%253D">CAS</a></li><li><a aria-label="View reference 35" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-294X.1995.tb00247.x">Article</a></li><li><a aria-label="Search for reference 35 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Amplification%20of%20hypervariable%20simple%20sequence%20repeats%20%28microsatellites%29%20from%20excremental%20DNA%20of%20wild%20living%20bonobos%20%28Pan%20paniscus%29&amp;journal=Mol.%20Ecol.&amp;volume=4&amp;pages=515-518&amp;publication_year=1995&amp;author=Gerloff%2CU">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">36.</span><p class="c-article-references__text" id="ref-CR36" itemprop="headline">Morin, P. A., Chambers, K. E. &amp; Boesch, C. &amp; Vigilant, L. Quantitative polymerase chain reaction analysis of DNA from noninvasive samples for accurate microsatellite genotyping of wild chimpanzees (<i>Pan troglodytes verus</i>). <i>Mol. Ecol.</i> <b>10</b>, 1835–1844 (2001).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 36 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXlvVynsLg%253D">CAS</a></li><li><a aria-label="View reference 36" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1046%2Fj.0962-1083.2001.01308.x">Article</a></li><li><a aria-label="Search for reference 36 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=%26%20Vigilant%2C%20L.%20Quantitative%20polymerase%20chain%20reaction%20analysis%20of%20DNA%20from%20noninvasive%20samples%20for%20accurate%20microsatellite%20genotyping%20of%20wild%20chimpanzees%20%28Pan%20troglodytes%20verus%29&amp;journal=Mol.%20Ecol.&amp;volume=10&amp;pages=1835-1844&amp;publication_year=2001&amp;author=Morin%2CPA&amp;author=Chambers%2CKE&amp;author=Boesch%2CC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">37.</span><p class="c-article-references__text" id="ref-CR37" itemprop="headline">Corless, C. E. <i>et al</i>. Contamination and sensitivity issues with a real-time universal 16S rRNA PCR. <i>J. Clin. Microbiol.</i> <b>38</b>, 1747–1752 (2000).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 37 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXjsFegs78%253D">CAS</a></li><li><a aria-label="View reference 37 on PubMed" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10790092" rel="nofollow">PubMed</a></li><li><a aria-label="View reference 37 on PubMed Central" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC86577" rel="nofollow">PubMed Central</a></li><li><a aria-label="Search for reference 37 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Contamination%20and%20sensitivity%20issues%20with%20a%20real-time%20universal%2016S%20rRNA%20PCR&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=38&amp;pages=1747-1752&amp;publication_year=2000&amp;author=Corless%2CCE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">38.</span><p class="c-article-references__text" id="ref-CR38" itemprop="headline">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 38 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXot1Cjtbo%253D">CAS</a></li><li><a aria-label="View reference 38" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324">Article</a></li><li><a aria-label="Search for reference 38 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">39.</span><p class="c-article-references__text" id="ref-CR39" itemprop="headline">DePristo, M. A. <i>et al</i>. A framework for variation discovery and genotyping using next-generation DNA sequencing data. <i>Nat. Genet.</i> <b>43</b>, 491–498 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 39 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXksFWguro%253D">CAS</a></li><li><a aria-label="View reference 39" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fng.806">Article</a></li><li><a aria-label="Search for reference 39 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=491-498&amp;publication_year=2011&amp;author=DePristo%2CMA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">40.</span><p class="c-article-references__text" id="ref-CR40" itemprop="headline">Danecek, P. <i>et al</i>. The variant call format and VCFtools. <i>Bioinformatics</i> <b>27</b>, 2156–2158 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a aria-label="View reference 40 on CAS" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXptFCqt7w%253D">CAS</a></li><li><a aria-label="View reference 40" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr330">Article</a></li><li><a aria-label="Search for reference 40 on Google Scholar" data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20variant%20call%20format%20and%20VCFtools&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-2158&amp;publication_year=2011&amp;author=Danecek%2CP">
                        Google Scholar</a></li></ul></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-category="article body" data-track-label="link" href="/articles/s41598-018-20427-9-references.ris">
                    Download references
                </a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section js-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content js-collapsible-section" id="Ack1-content"><p>We thank Clifford Jolly and Jane Phillips-Conroy for contributing fecal samples collected in 2006–2007 from Zambia. We also thank the Zambia Wildlife Authority (now the Department of National Parks &amp; Wildlife) and the University of Zambia for granting permission and providing support for fieldwork. We thank Erbay Yigit, Andrew Burrell, and Todd Disotell for helpful conversations. This study was funded by the National Science Foundation (BCS 1341018, BCS 1260816, BCS 1029302, SMA 1338524), the Leakey Foundation, the Wenner-Gren Foundation, the National Geographic Society, and the NYU University Research Challenge Fund. The Genome Technology Center at NYU is supported by NIH/NCATS UL1 TR00038 and NIH/NCI P30 CA016087. K.L.C. is supported by NSF fellowship DGE 1143954.</p></div></div></section><section aria-labelledby="author-information"><div class="c-article-section js-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content js-collapsible-section" id="author-information-content"><h3 class="c-article-author-information__subtitle visually-hidden" id="author-notes">Author notes</h3><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>Kenneth L. Chiou and Christina M. Bergey contributed equally to this work.</p></li></ol><h3 class="c-article-author-information__subtitle" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><h4 class="c-article-author-affiliation__address">Department of Anthropology, Washington University, St. Louis, MO, 63130, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kenneth L. Chiou</li></ul></li><li id="Aff2"><h4 class="c-article-author-affiliation__address">Department of Psychology, University of Washington, Seattle, WA, 98195, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Kenneth L. Chiou</li></ul></li><li id="Aff3"><h4 class="c-article-author-affiliation__address">Department of Anthropology, New York University, New York, NY, 10003, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Christina M. Bergey</li></ul></li><li id="Aff4"><h4 class="c-article-author-affiliation__address">New York Consortium in Evolutionary Primatology, New York, NY, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Christina M. Bergey</li></ul></li><li id="Aff5"><h4 class="c-article-author-affiliation__address">Department of Anthropology, Pennsylvania State University, University Park, PA, 16802, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Christina M. Bergey</li></ul></li><li id="Aff6"><h4 class="c-article-author-affiliation__address">Department of Biology, Pennsylvania State University, University Park, PA, 16802, USA</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Christina M. Bergey</li></ul></li></ol><div class="js-hide u-hide-print" data-test="author-info"><h3 class="c-article-author-information__subtitle">Authors</h3><ol class="c-article-author-authors-search"><li id="auth-1"><h3 class="c-article-author-authors-search__title js-search-name">Search for Kenneth L. Chiou in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kenneth L.+Chiou">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kenneth L.+Chiou%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li><li id="auth-2"><h3 class="c-article-author-authors-search__title js-search-name">Search for Christina M. Bergey in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christina M.+Bergey">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christina M.+Bergey%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li></ol></div><h3 class="c-article-author-information__subtitle" id="contributions">Contributions</h3><p>K.L.C. and C.M.B. conceived the project and wrote the paper. K.L.C. collected samples and led the labwork. C.M.B. led the analysis.</p>
<h3 class="c-article__sub-heading">Competing Interests</h3>
<p>New England Biolabs, the commercial vendor of the enrichment reagents, supplied three enrichment kits (retail value of $210 each) and 30 μg of purified K-12 <i>E</i>. <i>coli</i> genomic DNA at no cost for use in this project. K.L.C. and C.M.B. are not affiliated with New England Biolabs and declare no other competing interests.</p>
<h3 class="c-article-author-information__subtitle" id="corresponding-author">Corresponding authors</h3><p>Correspondence to
                <a href="/articles/s41598-018-20427-9/email/correspondent/c1/new" id="corresp-c1" rel="nofollow">Kenneth L. Chiou</a> or <a href="/articles/s41598-018-20427-9/email/correspondent/c2/new" id="corresp-c2" rel="nofollow">Christina M. Bergey</a>.</p></div></div></section><section aria-labelledby="Sec9"><div class="c-article-section js-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Electronic supplementary material</h2><div class="c-article-section__content js-collapsible-section" id="Sec9-content"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title"><a class="print-link" data-supp-info-image="" data-track="click" data-track-action="view supplementary info" data-track-category="article body" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_MOESM1_ESM.pdf">Supplemental Information</a></h3></div></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section js-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content js-collapsible-section" id="rightslink-content"><div class="c-article-license">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" itemprop="license" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
</div><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-category="article body" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?imprint=Nature&amp;oa=CC%20BY&amp;title=Methylation-based%20enrichment%20facilitates%20low-cost%2C%20noninvasive%20genomic%20scale%20sequencing%20of%20populations%20from%20feces&amp;author=Kenneth%20L.%20Chiou%20et%20al&amp;contentID=10.1038%2Fs41598-018-20427-9&amp;publication=Scientific%20Reports&amp;publicationDate=2018-01-31&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section js-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content js-collapsible-section" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column"><a data-crossmark="10.1038/s41598-018-20427-9" data-track="click" data-track-action="Click Crossmark" data-track-category="article body" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41598-018-20427-9" rel="noopener" target="_blank"><img alt="Verify currency and authenticity via CrossMark" height="81" src="https://www.nature.com/articles/s41598-018-20427-9/data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><h4>Received</h4><p class="c-bibliographic-information__value"><time datetime="2017-09-04">04 September 2017</time></p></li><li class="c-bibliographic-information__list-item"><h4>Accepted</h4><p class="c-bibliographic-information__value"><time datetime="2018-01-18">18 January 2018</time></p></li><li class="c-bibliographic-information__list-item"><h4>Published</h4><p class="c-bibliographic-information__value"><time datetime="2018-01-31">31 January 2018</time></p></li><li class="c-bibliographic-information__list-item"><h4><abbr title="Digital Object Identifier">DOI</abbr></h4><p class="c-bibliographic-information__value"><a data-track="click" data-track-action="view doi" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038/s41598-018-20427-9" itemprop="sameAs">https://doi.org/10.1038/s41598-018-20427-9</a></p></li></ul><div data-component="share-box"><div class="c-article-share-box"><h3 class="c-article-share-box__title">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-category="article body" data-track-label="button" id="get-share-url">Get shareable link</button> <div aria-hidden="true" class="js-no-share-url-container u-display-none"><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div> <div aria-hidden="true" class="js-share-url-container u-display-none"><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-category="article body" data-track-label="button" id="share-url"> </p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-category="article body" data-track-label="button" id="copy-share-url">Copy to clipboard</button><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">Provided by the Springer Nature SharedIt content-sharing initiative</p></div> </div></div><div data-component="article-info-list"><h3 class="c-article__sub-heading">Subjects</h3><ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" href="/subjects/genomics" itemprop="about">Genomics</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" href="/subjects/population-genetics" itemprop="about">Population genetics</a></li>
</ul></div></div></div></div></div></section>
<section aria-labelledby="article-comments"><div class="c-article-section js-article-section" id="article-comments-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-comments">Comments</h2><div class="c-article-section__content js-collapsible-section" id="article-comments-content"><p>By submitting a comment you agree to abide by our <a href="/info/tandc.html">Terms</a> and <a href="/info/community-guidelines.html">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p><div id="inject-comments">
<div class="pb20" data-disqus-identifier="10.1038/s41598-018-20427-9" data-disqus-key="ZZDdHrSs8UQRH0LAFbG4KmcMfQdvUwYDEyYczGZhznVSuj9QDfXFgWxFA1tz53jo" data-disqus-s3="e30= 11be1c39f815a7db51ec4b92ca97f48aa508937c 1562666113" data-disqus-shortname="srep" data-disqus-sso-button="//www.nature.com/static/images/buttons/disqus-sso-login-button.png" data-disqus-sso-icon="//www.nature.com/static/images/favicons/nature/favicon.ico" data-disqus-sso-logout="https://www.nature.com/foxtrot/svc/logout?target=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-20427-9" data-disqus-sso-name="Nature" data-disqus-sso-url="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=http%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-20427-9%2Fdisqus" data-disqus-url="https://www.nature.com/articles/s41598-018-20427-9" data-object="disqus" id="disqus_thread">
</div>
</div></div></div></section>
<iframe allowtransparency="true" frameborder="0" id="job-client" name="job-client" scrolling="no" src="https://recommended.springernature.com/v2296/generated/job-client.html?source=https://www.nature.com" style="width: 100%; height: 0px; bottom: 0px; right: 0px; z-index: 100000; padding: 0px; margin-top: 28px;" title="Personalised recommendations"></iframe>
</div>
</div>
<div class="c-article-extras u-hide-print" data-container-type="reading-companion" data-track-component="reading companion" role="complementary">
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" style="width: 312px;"><ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-tab-target="sections" id="tab-sections" role="tab">Sections</button></li><li role="presentation"><button aria-controls="tabpanel-figures" aria-selected="false" class="c-reading-companion__tab" data-tab-target="figures" id="tab-figures" role="tab" tabindex="-1">Figures</button></li><li role="presentation"><button aria-controls="tabpanel-references" aria-selected="false" class="c-reading-companion__tab" data-tab-target="references" id="tab-references" role="tab" tabindex="-1">References</button></li></ul>
<div aria-labelledby="tab-sections" class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" tabindex="0"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__sections-list"><li class="c-reading-companion__section-item" id="rc-sec-Abs1"><a data-track="click" data-track-action="section anchor" data-track-label="link:Abstract" href="#Abs1">Abstract</a></li><li class="c-reading-companion__section-item" id="rc-sec-Sec1"><a data-track="click" data-track-action="section anchor" data-track-label="link:Introduction" href="#Sec1">Introduction</a></li><li class="c-reading-companion__section-item" id="rc-sec-Sec2"><a data-track="click" data-track-action="section anchor" data-track-label="link:Results" href="#Sec2">Results</a></li><li class="c-reading-companion__section-item" id="rc-sec-Sec3"><a data-track="click" data-track-action="section anchor" data-track-label="link:Discussion" href="#Sec3">Discussion</a></li><li class="c-reading-companion__section-item" id="rc-sec-Sec4"><a data-track="click" data-track-action="section anchor" data-track-label="link:Methods" href="#Sec4">Methods</a></li><li class="c-reading-companion__section-item" id="rc-sec-data-availability"><a data-track="click" data-track-action="section anchor" data-track-label="link:Data Availability" href="#data-availability">Data Availability</a></li><li class="c-reading-companion__section-item" id="rc-sec-additional-information"><a data-track="click" data-track-action="section anchor" data-track-label="link:Additional information" href="#additional-information">Additional information</a></li><li class="c-reading-companion__section-item" id="rc-sec-change-history"><a data-track="click" data-track-action="section anchor" data-track-label="link:Change history" href="#change-history">Change history</a></li><li class="c-reading-companion__section-item" id="rc-sec-Bib1"><a data-track="click" data-track-action="section anchor" data-track-label="link:References" href="#Bib1">References</a></li><li class="c-reading-companion__section-item" id="rc-sec-Ack1"><a data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements" href="#Ack1">Acknowledgements</a></li><li class="c-reading-companion__section-item" id="rc-sec-author-information"><a data-track="click" data-track-action="section anchor" data-track-label="link:Author information" href="#author-information">Author information</a></li><li class="c-reading-companion__section-item" id="rc-sec-Sec9"><a data-track="click" data-track-action="section anchor" data-track-label="link:Electronic supplementary material" href="#Sec9">Electronic supplementary material</a></li><li class="c-reading-companion__section-item" id="rc-sec-rightslink"><a data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions" href="#rightslink">Rights and permissions</a></li><li class="c-reading-companion__section-item" id="rc-sec-article-info"><a data-track="click" data-track-action="section anchor" data-track-label="link:About this article" href="#article-info">About this article</a></li><li class="c-reading-companion__section-item" id="rc-sec-article-comments"><a data-track="click" data-track-action="section anchor" data-track-label="link:Comments" href="#article-comments">Comments</a></li></ul></div>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-google-query-id="CNKf1_rIp-MCFVAo4AodjE4MGQ" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41598-018-20427-9;doi=10.1038/s41598-018-20427-9;techmeta=22,23,45;subjmeta=208,212,457,631;kwrd=Genomics,Population genetics" data-gpt-unitpath="/285/scientific_reports/article" id="div-gpt-ad-right-2">
<div id="google_ads_iframe_/270604982/nature/srep/articles_1__container__" style="border: 0pt none;"><iframe data-google-container-id="1" frameborder="0" height="250" id="google_ads_iframe_/270604982/nature/srep/articles_1" marginheight="0" marginwidth="0" name="google_ads_iframe_/270604982/nature/srep/articles_1" scrolling="no" srcdoc="" style="border: 0px; vertical-align: bottom;" title="3rd party ad content" width="300"></iframe></div></div>
</div>
<div aria-labelledby="tab-figures" class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig1">Figure 1</b></figcaption><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure1" aria-describedby="figure-1-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig1_HTML.png"/></source></picture><p class="c-reading-companion__figure-links"><a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig1">View in article</a><a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www.nature.com/articles/s41598-018-20427-9/figures/1">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig2">Figure 2</b></figcaption><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure2" aria-describedby="figure-2-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig2_HTML.png"/></source></picture><p class="c-reading-companion__figure-links"><a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig2">View in article</a><a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www.nature.com/articles/s41598-018-20427-9/figures/2">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig3">Figure 3</b></figcaption><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure3" aria-describedby="figure-3-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-20427-9/MediaObjects/41598_2018_20427_Fig3_HTML.png"/></source></picture><p class="c-reading-companion__figure-links"><a data-track="click" data-track-action="figure anchor" data-track-label="link" href="#Fig3">View in article</a><a class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" href="https://www.nature.com/articles/s41598-018-20427-9/figures/3">Full size image</a></p></figure></li></ul></div></div>
<div aria-labelledby="tab-references" class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR1">Kohn, M. H. &amp; Wayne, R. K. Facts from feces revisited. <i>Trends Ecol. Evol.</i> <b>12</b>, 223–227 (1997).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ASTN%3A280%3ADC%252BC3M7itFKksg%253D%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0169-5347%2897%2901050-1">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Facts%20from%20feces%20revisited&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=12&amp;pages=223-227&amp;publication_year=1997&amp;author=Kohn%2CMH&amp;author=Wayne%2CRK">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR2">Putman, R. J. Facts from faeces. <i>Mamm. Rev.</i> <b>14</b>, 79–97 (1984).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-2907.1984.tb00341.x">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Facts%20from%20faeces&amp;journal=Mamm.%20Rev.&amp;volume=14&amp;pages=79-97&amp;publication_year=1984&amp;author=Putman%2CRJ">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR3">Höss, M., Kohn, M., Pääbo, S., Knauer, F. &amp; Schröder, W. Excrement analysis by PCR. <i>Nature</i> <b>359</b>, 199 (1992).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1992Natur.359..199H">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2F359199a0">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Excrement%20analysis%20by%20PCR&amp;journal=Nature&amp;volume=359&amp;publication_year=1992&amp;author=H%C3%B6ss%2CM&amp;author=Kohn%2CM&amp;author=P%C3%A4%C3%A4bo%2CS&amp;author=Knauer%2CF&amp;author=Schr%C3%B6der%2CW">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR4">Qin, J. <i>et al</i>. A human gut microbial gene catalogue established by metagenomic sequencing. <i>Nature</i> <b>464</b>, 59–65 (2010).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXislahsLc%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature08821">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;journal=Nature&amp;volume=464&amp;pages=59-65&amp;publication_year=2010&amp;author=Qin%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR5">Perry, G. H., Marioni, J. C., Melsted, P. &amp; Gilad, Y. Genomic-scale capture and sequencing of endogenous DNA from feces. <i>Mol. Ecol.</i> <b>19</b>, 5332–5344 (2010).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlagsLY%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-294X.2010.04888.x">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic-scale%20capture%20and%20sequencing%20of%20endogenous%20DNA%20from%20feces&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=5332-5344&amp;publication_year=2010&amp;author=Perry%2CGH&amp;author=Marioni%2CJC&amp;author=Melsted%2CP&amp;author=Gilad%2CY">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR6">Snyder-Mackler, N. <i>et al</i>. Efficient genome-wide sequencing and low coverage pedigree analysis from non-invasively collected samples. <i>Genetics</i> <b>203</b>, 699–714 (2016).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitFSit77I">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1534%2Fgenetics.116.187492">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20genome-wide%20sequencing%20and%20low%20coverage%20pedigree%20analysis%20from%20non-invasively%20collected%20samples&amp;journal=Genetics&amp;volume=203&amp;pages=699-714&amp;publication_year=2016&amp;author=Snyder-Mackler%2CN">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR7">Whitney, D. <i>et al</i>. Enhanced retrieval of DNA from human fecal samples results in improved performance of colorectal cancer screening test. <i>J. Mol. Diagn.</i> <b>6</b>, 386–395 (2004).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhtVOhtbjI">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS1525-1578%2810%2960536-3">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20retrieval%20of%20DNA%20from%20human%20fecal%20samples%20results%20in%20improved%20performance%20of%20colorectal%20cancer%20screening%20test&amp;journal=J.%20Mol.%20Diagn.&amp;volume=6&amp;pages=386-395&amp;publication_year=2004&amp;author=Whitney%2CD">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR8">Melnikov, A. <i>et al</i>. Hybrid selection for sequencing pathogen genomes from clinical samples. <i>Genome Biol.</i> <b>12</b>, R73 (2011).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtF2gu7jK">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fgb-2011-12-8-r73">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hybrid%20selection%20for%20sequencing%20pathogen%20genomes%20from%20clinical%20samples&amp;journal=Genome%20Biol.&amp;volume=12&amp;publication_year=2011&amp;author=Melnikov%2CA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR9">Carpenter, M. L. <i>et al</i>. Pulling out the 1%: whole-genome capture for the targeted enrichment of ancient DNA sequencing libraries. <i>Am. J. Hum. Genet.</i> <b>93</b>, 852–864 (2013).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhs12ltbbF">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.ajhg.2013.10.002">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Pulling%20out%20the%201%25%3A%20whole-genome%20capture%20for%20the%20targeted%20enrichment%20of%20ancient%20DNA%20sequencing%20libraries&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=93&amp;pages=852-864&amp;publication_year=2013&amp;author=Carpenter%2CML">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR10">Albert, T. J. <i>et al</i>. Direct selection of human genomic loci by microarray hybridization. <i>Nat. Methods</i> <b>4</b>, 903–905 (2007).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXht1aqtbfL">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth1111">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-905&amp;publication_year=2007&amp;author=Albert%2CTJ">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR11">Okou, D. T. <i>et al</i>. Microarray-based genomic selection for high-throughput resequencing. <i>Nat. Methods</i> <b>4</b>, 907–909 (2007).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXht1aqur7N">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth1109">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Microarray-based%20genomic%20selection%20for%20high-throughput%20resequencing&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=907-909&amp;publication_year=2007&amp;author=Okou%2CDT">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR12">Gnirke, A. <i>et al</i>. Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. <i>Nat. Biotechnol.</i> <b>27</b>, 182–189 (2009).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXht1ehs7o%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.1523">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=182-189&amp;publication_year=2009&amp;author=Gnirke%2CA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR13">Ng, S. B. <i>et al</i>. Targeted capture and massively parallel sequencing of 12 human exomes. <i>Nature</i> <b>461</b>, 272–276 (2009).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.461..272N">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXpvFCktLg%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature08250">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20capture%20and%20massively%20parallel%20sequencing%20of%2012%20human%20exomes&amp;journal=Nature&amp;volume=461&amp;pages=272-276&amp;publication_year=2009&amp;author=Ng%2CSB">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR14">Suchan, T. <i>et al</i>. Hybridization capture using RAD probes (hyRAD), a new tool for performing genomic analyses on museum collection specimens. <i>PLoS One</i> <b>11</b>, e0151651 (2016).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0151651">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Hybridization%20capture%20using%20RAD%20probes%20%28hyRAD%29%2C%20a%20new%20tool%20for%20performing%20genomic%20analyses%20on%20museum%20collection%20specimens&amp;journal=PLoS%20One&amp;volume=11&amp;publication_year=2016&amp;author=Suchan%2CT">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR15">Ali, O. A. <i>et al</i>. RAD Capture (Rapture): flexible and efficient sequence-based genotyping. <i>Genetics</i> <b>202</b>, 389–400 (2016).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitVGlu7%252FM">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1534%2Fgenetics.115.183665">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=RAD%20Capture%20%28Rapture%29%3A%20flexible%20and%20efficient%20sequence-based%20genotyping&amp;journal=Genetics&amp;volume=202&amp;pages=389-400&amp;publication_year=2016&amp;author=Ali%2COA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR16">Hoffberg, S. L. <i>et al</i>. RADcap: sequence capture of dual-digest RADseq libraries with identifiable duplicates and reduced missing data. <i>Mol. Ecol. Resour.</i> <b>16</b>, 1264–1278 (2016).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtlOmsrbK">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2F1755-0998.12566">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=RADcap%3A%20sequence%20capture%20of%20dual-digest%20RADseq%20libraries%20with%20identifiable%20duplicates%20and%20reduced%20missing%20data&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=16&amp;pages=1264-1278&amp;publication_year=2016&amp;author=Hoffberg%2CSL">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR17">Maricic, T., Whitten, M. &amp; Pääbo, S. Multiplexed DNA sequence capture of mitochondrial genomes using PCR products. <i>PLoS One</i> <b>5</b>, e14004 (2010).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PLoSO...514004M">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0014004">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20DNA%20sequence%20capture%20of%20mitochondrial%20genomes%20using%20PCR%20products&amp;journal=PLoS%20One&amp;volume=5&amp;publication_year=2010&amp;author=Maricic%2CT&amp;author=Whitten%2CM&amp;author=P%C3%A4%C3%A4bo%2CS">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR18">Mason, V. C., Li, G., Helgen, K. M. &amp; Murphy, W. J. Efficient cross-species capture hybridization and next-generation sequencing of mitochondrial genomes from noninvasively sampled museum specimens. <i>Genome Res.</i> <b>21</b>, 1695–1704 (2011).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlSru7%252FM">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgr.120196.111">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20cross-species%20capture%20hybridization%20and%20next-generation%20sequencing%20of%20mitochondrial%20genomes%20from%20noninvasively%20sampled%20museum%20specimens&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=1695-1704&amp;publication_year=2011&amp;author=Mason%2CVC&amp;author=Li%2CG&amp;author=Helgen%2CKM&amp;author=Murphy%2CWJ">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR19">Bi, K. <i>et al</i>. Unlocking the vault: next-generation museum population genomics. <i>Mol. Ecol.</i> <b>22</b>, 6018–6032 (2013).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhvVyjsb7I">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fmec.12516">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Unlocking%20the%20vault%3A%20next-generation%20museum%20population%20genomics&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=6018-6032&amp;publication_year=2013&amp;author=Bi%2CK">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR20">George, R. D. <i>et al</i>. Trans genomic capture and sequencing of primate exomes reveals new targets of positive selection. <i>Genome Res.</i> <b>21</b>, 1686–1694 (2011).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhtlSru77F">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgr.121327.111">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Trans%20genomic%20capture%20and%20sequencing%20of%20primate%20exomes%20reveals%20new%20targets%20of%20positive%20selection&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=1686-1694&amp;publication_year=2011&amp;author=George%2CRD">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR21">Samuels, D. C. <i>et al</i>. Finding the lost treasures in exome sequencing data. <i>Trends Genet.</i> <b>29</b>, 593–599 (2013).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhsV2msb%252FF">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.tig.2013.07.006">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Finding%20the%20lost%20treasures%20in%20exome%20sequencing%20data&amp;journal=Trends%20Genet.&amp;volume=29&amp;pages=593-599&amp;publication_year=2013&amp;author=Samuels%2CDC">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR22">Feehery, G. R. <i>et al</i>. A method for selectively enriching microbial DNA from contaminating vertebrate host DNA. <i>PLoS One</i> <b>8</b>, e76096 (2013).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...876096F">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhslWksrbO">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0076096">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20selectively%20enriching%20microbial%20DNA%20from%20contaminating%20vertebrate%20host%20DNA&amp;journal=PLoS%20One&amp;volume=8&amp;publication_year=2013&amp;author=Feehery%2CGR">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR23">Hendrich, B. &amp; Tweedie, S. The methyl-CpG binding domain and the evolving role of DNA methylation in animals. <i>Trends Genet.</i> <b>19</b>, 269–277 (2003).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXjtFeqt7w%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0168-9525%2803%2900080-5">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20methyl-CpG%20binding%20domain%20and%20the%20evolving%20role%20of%20DNA%20methylation%20in%20animals&amp;journal=Trends%20Genet.&amp;volume=19&amp;pages=269-277&amp;publication_year=2003&amp;author=Hendrich%2CB&amp;author=Tweedie%2CS">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR24">Jabbari, K. &amp; Bernardi, G. Cytosine methylation and CpG, TpG (CpA) and TpA frequencies. <i>Gene</i> <b>333</b>, 143–149 (2004).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXksFOmtrc%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.gene.2004.02.043">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Cytosine%20methylation%20and%20CpG%2C%20TpG%20%28CpA%29%20and%20TpA%20frequencies&amp;journal=Gene&amp;volume=333&amp;pages=143-149&amp;publication_year=2004&amp;author=Jabbari%2CK&amp;author=Bernardi%2CG">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR25">Fang, G. <i>et al</i>. Genome-wide mapping of methylated adenine residues in pathogenic Escherichia coli using single-molecule real-time sequencing. <i>Nat. Biotechnol.</i> <b>30</b>, 1232–1239 (2012).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xhs1elsLbE">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.2432">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20mapping%20of%20methylated%20adenine%20residues%20in%20pathogenic%20Escherichia%20coli%20using%20single-molecule%20real-time%20sequencing&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=1232-1239&amp;publication_year=2012&amp;author=Fang%2CG">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR26">Murray, I. A. <i>et al</i>. The methylomes of six bacteria. <i>Nucleic Acids Res.</i> <b>40</b>, 11450–11462 (2012).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XhvVCitLfP">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgks891">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20methylomes%20of%20six%20bacteria&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=11450-11462&amp;publication_year=2012&amp;author=Murray%2CIA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR27">Peterson, B. K., Weber, J. N., Kay, E. H., Fisher, H. S. &amp; Hoekstra, H. E. Double digest RADseq: an inexpensive method for <i>de novo</i> SNP discovery and genotyping in model and non-model species. <i>PLoS One</i> <b>7</b>, e37135 (2012).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012PLoSO...737135P">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XosVeksrY%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0037135">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Double%20digest%20RADseq%3A%20an%20inexpensive%20method%20for%20de%20novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species&amp;journal=PLoS%20One&amp;volume=7&amp;publication_year=2012&amp;author=Peterson%2CBK&amp;author=Weber%2CJN&amp;author=Kay%2CEH&amp;author=Fisher%2CHS&amp;author=Hoekstra%2CHE">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR28">Purcell, S. <i>et al</i>. PLINK: a tool set for whole-genome association and population-based linkage analyses. <i>Am. J. Hum. Genet.</i> <b>81</b>, 559–575 (2007).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtVSqurrL">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1086%2F519795">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=81&amp;pages=559-575&amp;publication_year=2007&amp;author=Purcell%2CS">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR29">Chang, C. C. <i>et al</i>. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>Gigascience</i> <b>4</b>, 559 (2015).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs13742-015-0047-8">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=Gigascience&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR30">Gautier, M. <i>et al</i>. The effect of RAD allele dropout on the estimation of genetic variation within and between populations. <i>Mol. Ecol.</i> <b>22</b>, 3165–3178 (2013).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXos12msr4%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fmec.12089">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effect%20of%20RAD%20allele%20dropout%20on%20the%20estimation%20of%20genetic%20variation%20within%20and%20between%20populations&amp;journal=Mol.%20Ecol.&amp;volume=22&amp;pages=3165-3178&amp;publication_year=2013&amp;author=Gautier%2CM">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR31">Rohland, N. &amp; Reich, D. Cost-effective, high-throughput DNA sequencing libraries for multiplexed target capture. <i>Genome Res.</i> <b>22</b>, 939–946 (2012).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xms1antbk%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1101%2Fgr.128124.111">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Cost-effective%2C%20high-throughput%20DNA%20sequencing%20libraries%20for%20multiplexed%20target%20capture&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=939-946&amp;publication_year=2012&amp;author=Rohland%2CN&amp;author=Reich%2CD">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR32">Ramón-Laca, A., Soriano, L., Gleeson, D. &amp; Godoy, J. A. A simple and effective method for obtaining mammal DNA from faeces. <i>Wildlife Biol.</i> <b>21</b>, 195–203 (2015).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.2981%2Fwlb.00096">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20and%20effective%20method%20for%20obtaining%20mammal%20DNA%20from%20faeces&amp;journal=Wildlife%20Biol.&amp;volume=21&amp;pages=195-203&amp;publication_year=2015&amp;author=Ram%C3%B3n-Laca%2CA&amp;author=Soriano%2CL&amp;author=Gleeson%2CD&amp;author=Godoy%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR33">Rebelo, A. P., Williams, S. L. &amp; Moraes, C. T. <i>In vivo</i> methylation of mtDNA reveals the dynamics of protein-mtDNA interactions. <i>Nucleic Acids Res.</i> <b>37</b>, 6701–6715 (2009).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhsVajt7%252FK">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkp727">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20methylation%20of%20mtDNA%20reveals%20the%20dynamics%20of%20protein-mtDNA%20interactions&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=6701-6715&amp;publication_year=2009&amp;author=Rebelo%2CAP&amp;author=Williams%2CSL&amp;author=Moraes%2CCT">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR34">Constable, J. J., Packer, C., Collins, D. A. &amp; Pusey, A. E. Nuclear DNA from primate dung. <i>Nature</i> <b>373</b>, 393–393 (1995).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1995Natur.373..393C">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXjsFCjsr4%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2F373393a0">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclear%20DNA%20from%20primate%20dung&amp;journal=Nature&amp;volume=373&amp;pages=393-393&amp;publication_year=1995&amp;author=Constable%2CJJ&amp;author=Packer%2CC&amp;author=Collins%2CDA&amp;author=Pusey%2CAE">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR35">Gerloff, U. <i>et al</i>. Amplification of hypervariable simple sequence repeats (microsatellites) from excremental DNA of wild living bonobos (<i>Pan paniscus</i>). <i>Mol. Ecol.</i> <b>4</b>, 515–518 (1995).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXosFertLo%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-294X.1995.tb00247.x">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Amplification%20of%20hypervariable%20simple%20sequence%20repeats%20%28microsatellites%29%20from%20excremental%20DNA%20of%20wild%20living%20bonobos%20%28Pan%20paniscus%29&amp;journal=Mol.%20Ecol.&amp;volume=4&amp;pages=515-518&amp;publication_year=1995&amp;author=Gerloff%2CU">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR36">Morin, P. A., Chambers, K. E. &amp; Boesch, C. &amp; Vigilant, L. Quantitative polymerase chain reaction analysis of DNA from noninvasive samples for accurate microsatellite genotyping of wild chimpanzees (<i>Pan troglodytes verus</i>). <i>Mol. Ecol.</i> <b>10</b>, 1835–1844 (2001).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXlvVynsLg%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1046%2Fj.0962-1083.2001.01308.x">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=%26%20Vigilant%2C%20L.%20Quantitative%20polymerase%20chain%20reaction%20analysis%20of%20DNA%20from%20noninvasive%20samples%20for%20accurate%20microsatellite%20genotyping%20of%20wild%20chimpanzees%20%28Pan%20troglodytes%20verus%29&amp;journal=Mol.%20Ecol.&amp;volume=10&amp;pages=1835-1844&amp;publication_year=2001&amp;author=Morin%2CPA&amp;author=Chambers%2CKE&amp;author=Boesch%2CC">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR37">Corless, C. E. <i>et al</i>. Contamination and sensitivity issues with a real-time universal 16S rRNA PCR. <i>J. Clin. Microbiol.</i> <b>38</b>, 1747–1752 (2000).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXjsFegs78%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10790092">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC86577">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Contamination%20and%20sensitivity%20issues%20with%20a%20real-time%20universal%2016S%20rRNA%20PCR&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=38&amp;pages=1747-1752&amp;publication_year=2000&amp;author=Corless%2CCE">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR38">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXot1Cjtbo%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR39">DePristo, M. A. <i>et al</i>. A framework for variation discovery and genotyping using next-generation DNA sequencing data. <i>Nat. Genet.</i> <b>43</b>, 491–498 (2011).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXksFWguro%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fng.806">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=491-498&amp;publication_year=2011&amp;author=DePristo%2CMA">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR40">Danecek, P. <i>et al</i>. The variant call format and VCFtools. <i>Bioinformatics</i> <b>27</b>, 2156–2158 (2011).</p><ul class="c-reading-companion__reference-links"><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXptFCqt7w%253D">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr330">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20variant%20call%20format%20and%20VCFtools&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-2158&amp;publication_year=2011&amp;author=Danecek%2CP">
                        Google Scholar</a></li></ul></li></ol></div></div>
</div>
</div>
</div>
</article>
</div>
</div>
</div>
<footer>
<div class="content pl30 pr30 text14 text-gray-light">
</div>
<div class="container-footer mt40 cleared container-footer-full" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="content pt20 pr30 pl30">
<div class="grid grid-12 last clear-float cleared mb15 text-gray-light text13 flex-box flex-wrap-reverse">
<h2 aria-level="2" class="text13 strong emphasis ma0 sans-serif" itemprop="name">Scientific Reports</h2>
<p class="pt0 pr10 pb0 pl10 ma0"><abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">2045-2322</span> (online)</p>
</div>
</div>
<div class="grid grid-12 pl30 pr30 last" id="footer">
<h2 aria-level="2" class="hide">nature.com sitemap</h2>
</div>
<div class="pl30 pr30 content" data-track-component="footer">
<div class="grid grid-12 last clear-float cleared mb5 pt20 text-gray-light flex-wrap">
<div class="grid grid-3 mq875-grid-12 just-mq640-last just-mq875-last mb20"><img alt="Nature" class="js-svg" data-png="/static/images/natureresearch-logo.1f5acc607c.png" src="https://www.nature.com/articles//static/images/natureresearch-logo.2fe351a520.svg"/></div>
<ul class="grid grid-7 mq875-grid-8 mq640-grid-12 just-mq640-last clean-list mb10 mt10 text15 u-hide-print">
<li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a class="text-gray-light" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq480-last"><a class="text-gray-light" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a class="text-gray-light" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 last"><a class="text-gray-light" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
<ul class="grid grid-2 mq875-grid-4 mq640-grid-12 last clean-list mb10 mt4 u-hide-print">
<li class="grid grid-4 mq875-grid-3"><a class="text-gray-light" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/nature/"><img alt="Facebook" class="cleared text-footer-img" src="https://www.nature.com/articles//static/images/fb.49ba391f15.svg"/></a></li>
<li class="grid grid-4 mq875-grid-3"><a class="text-gray-light" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/nresearchnews?lang=en"><img alt="Twitter" class="cleared text-footer-img" src="https://www.nature.com/articles//static/images/twitter.a90dbeb03a.svg"/></a></li>
<li class="grid grid-4 mq875-grid-3 last"><a class="text-gray-light" data-track="click" data-track-action="youtube" data-track-label="link" href="https://www.youtube.com/channel/UCvCLdSgYdSTpWcOgEJgi-ng"><img alt="Youtube" class="cleared text-footer-img" src="https://www.nature.com/articles//static/images/youtube.6071f6012c.svg"/></a></li>
</ul>
</div>
<div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-bottom hide-print u-hide-print">
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
<h3 class="text-footer-heading">Discover content</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex/">Journals A-Z</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects/">Articles by subject</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nano" data-track-label="link" href="https://nano.nature.com/">Nano</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="protocol exchange" data-track-label="link" href="https://www.nature.com/protocolexchange/">Protocol Exchange</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last">
<h3 class="text-footer-heading">Publish with us</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="guide to authors" data-track-label="link" href="https://www.nature.com/authors/author_resources/index.html">Guide to Authors</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="guide to referees" data-track-label="link" href="https://www.nature.com/authors/peer_review/">Guide to Referees</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="editorial policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies/">Editorial policies</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="open access" data-track-label="link" href="http://www.nature.com/openresearch/publishing-with-npg/">Open access</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints/">Reprints &amp; permissions</a></li>
</ul>
</div>
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-last">
<h3 class="text-footer-heading">Researcher services</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/?utm_source=natureAuthors&amp;utm_medium=referral&amp;utm_campaign=natureAuthors">Language editing</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/?utm_source=natureAuthors&amp;utm_medium=referral&amp;utm_campaign=natureAuthors">Scientific editing</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature research academies" data-track-label="link" href="https://partnerships.nature.com/product/nature-research-academies/">Nature Research Academies</a></li>
</ul>
</div>
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-clear just-mq640-last full-size-last just-mq1200-last just-mq875-last just-mq480-last">
<h3 class="text-footer-heading">Libraries &amp; institutions</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="open research" data-track-label="link" href="http://www.nature.com/openresearch/about-open-access/information-for-institutions/">Open research</a></li>
</ul>
</div>
</div>
<div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-top hide-print u-hide-print">
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
<h3 class="text-footer-heading">Advertising &amp; partnerships</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/">Advertising</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/home/services/">Partnerships &amp; Services</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/home/resources/">Media kits</a></li>
<li class="pb4"><a class="text-gray-light" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded content</a></li>
</ul>
</div>
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq875-last">
<h3 class="text-footer-heading">Career development</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers">Nature Careers</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://www.nature.com/natureconferences/">Nature<span class="visually-hidden"> </span> Conferences</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature events" data-track-label="link" href="https://www.nature.com/natureevents/">Nature<span class="visually-hidden"> </span> events</a></li>
</ul>
</div>
<div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
<h3 class="text-footer-heading">Regional websites</h3>
<ul class="clean-list ma0 mb6 text15">
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp/">Nature Japan</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature korea" data-track-label="link" href="https://www.natureasia.com/ko-kr/">Nature Korea</a></li>
<li class="pb4"><a class="text-gray-light" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast/">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="container-footer-box cleared text14">
<div class="content">
<img alt="Springer Nature" class="cleared" src="https://www.nature.com/articles//static/images/sn-logo.4368f3b177.svg"/>
<p class="mb10 text-gray-light">© 2019 Springer Nature Publishing AG</p>
<ul class="ma0 clean-list grid grid-12 hide-print u-hide-print">
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a class="text-gray-light" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy.html">Privacy Policy</a></li>
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a class="text-gray-light" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies.html">Use of cookies</a></li>
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq875-last just-mq480-last"><a class="optanon-toggle-display text-gray-light" data-track="click" data-track-action="manage cookies" data-track-label="link" href="javascript:;">Manage cookies</a></li>
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a class="text-gray-light" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal_notice.html">Legal notice</a></li>
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a class="text-gray-light" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility_statement.html">Accessibility statement</a></li>
<li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 last"><a class="text-gray-light" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/tandc.html">Terms &amp; Conditions</a></li>
</ul>
</div>
</div>
</div>
</footer>
<script crossorigin="anonymous" src="https://cdn.ravenjs.com/3.12.0/raven.min.js"></script>
<script>
    if (Raven) {
        Raven.config('https://46e8b5cbb6a34c8ab1888c41c7f86c61@sentry.dc.springernature.pe/388').install();
    }
</script>
<noscript>
<img border="0" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41598-018-20427-9&amp;format=js&amp;last_modified=2018-12-19"></script>
<img alt="" class="visually-hidden" height="1" src="https://www.nature.com/articles//platform/track/article/s41598-018-20427-9" width="1"/>
<script id="" type="text/javascript">Bootstrapper=window.Bootstrapper||{};Bootstrapper.npg=Bootstrapper.npg||{};Bootstrapper.npg.utils=Bootstrapper.npg.utils||{};
Bootstrapper.npg.utils.Request=function(h,e){var f=function(){for(var a,b={},c=/([^&=]+)=?([^&]*)/g,d=h.location.search.substring(1);a=c.exec(d);)b[decodeURIComponent(a[1].replace(/\+/g," "))]=decodeURIComponent(a[2].replace(/\+/g," "));return b}(),g=function(){for(var a=document.cookie.split(/;\s*/),b,c=a.length,d={};c--;)b=a[c].split("\x3d"),d[b[0]]=b.slice(1).join("\x3d");return d}();return{get:function(a){return 1===arguments.length?f[a]||e:f},cookie:function(a){return 0===arguments.length?g:
1===arguments.length?g[a]||e:this.setCookie.apply(this,[].slice.call(arguments,0))},deleteCookies:function(a,b){for(var c=0;c<a.length;++c)this.deleteCookie(a[c],b)},deleteCookie:function(a,b){b=b||{};b.days=-1;this.setCookie(a,"",b)},setCookie:function(a,b,c){a=[a+"\x3d"+b];c=c||{};c.days&&(b=new Date,b.setTime(+b+864E5*c.days),a.push("expires\x3d"+b.toGMTString()));c.path&&a.push("path\x3d"+c.path);c.domain&&a.push("domain\x3d"+c.domain);document.cookie=a.join(";");return!0},deleteCookiesByPattern:function(a,
b){for(var c=new RegExp("(?:^|;|\\s)("+a+"[^\x3d]*)","g"),d=[],e;null!==(e=c.exec(document.cookie));)d.push(e[1]);this.deleteCookies(d,b)}}}(window);</script><script id="" type="text/javascript">Bootstrapper=window.Bootstrapper||{};Bootstrapper.npg=Bootstrapper.npg||{};Bootstrapper.npg.utils=Bootstrapper.npg.utils||{};Bootstrapper.npg.utils.Consent=function(){var d=unescape(Bootstrapper.npg.utils.Request.cookie("OptanonConsent")||""),b=(/groups=([^&]+)/.exec(d)||["",""])[1].split(",");return{STRICTLY_NECESSARY:1,PERFORMANCE:2,FUNCTIONAL:3,TARGETING:4,allow:function(c){if(c===this.STRICTLY_NECESSARY)return!0;for(var a=0;b[a];++a)if(b[a]===c+":0")return!1;return!0}}}();</script><script id="" type="text/javascript">(function(b,c){b.location.pathname.match(/^\/(nature_education|scitable|principles)(\/|$)/)||b.location.hostname.match("natureindex.com")||c.addEventListener("accessdetailsloaded",function(a){if(a=a.detail)b._pageMetaData=b._pageMetaData||{},b._pageMetaData.site_id=a.site_id&&a.site_id.join?a.site_id.join(";"):"",b._pageMetaData.user_id=a.registrant_id||"",b._pageMetaData.business_partner_id=a.business_partner_id&&a.business_partner_id.join?a.business_partner_id.join(";"):"";a=c.createElement("script");
a.className="kxct";a.setAttribute("data-id","Jn7TaPwq");a.setAttribute("data-timing","async");a.setAttribute("data-version","3.0");a.appendChild(c.createTextNode('window.Krux||((Krux\x3dfunction(){Krux.q.push(arguments)}).q\x3d[]);(function(){var k\x3ddocument.createElement("script");k.type\x3d"text/javascript";k.async\x3dtrue;var m,src\x3d(m\x3dlocation.href.match(/\\bkxsrc\x3d([^\x26]+)/))\x26\x26decodeURIComponent(m[1]);k.src \x3d /^https?:\\/\\/([a-z0-9_\\-\\.]+\\.)?krxd\\.net(:\\d{1,5})?\\//i.test(src) ? src : src \x3d\x3d\x3d "disable" ? "" :(location.protocol\x3d\x3d\x3d"https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid\x3dJn7TaPwq";var s\x3ddocument.getElementsByTagName("script")[0];s.parentNode.insertBefore(k,s);}());'));
c.head.appendChild(a);a=Bootstrapper.npg.utils.Consent;a=a.allow(a.TARGETING);Krux("consent:set",{dc:a,al:a,tg:a,cd:!1,sh:!1,re:!1},function(a,b){a&&console.error(a)})},!1)})(window,document);</script><script id="" type="text/javascript">document.addEventListener("accessdetailsloaded",function(a){a=a.detail;var b={event:"update-access-details"};if(a){var c=a.registrant_id||"",d=a.site_id&&a.site_id.join?a.site_id.join(";"):"",e=a.business_partner_id&&a.business_partner_id.join?a.business_partner_id.join(";"):"";b.user={};b.user.profile={};b.user.profile.profileInfo={profileID:c||null,bpid:e||null};b.user.segment={registeredUser:!!c};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy=
{};b.session.authentication.legacy.webtrendsSiteID=d||null}window.dataLayer.push(b)},!1);</script><script id="" type="text/javascript">(function(a){if("function"===typeof window.CustomEvent)return!1;var b=function(b,c){c=c||{};var d=document.createEvent("CustomEvent");d.initCustomEvent(b,c.bubbles||!1,c.cancelable||!1,c.detail||a);return d};b.prototype=window.Event.prototype;window.CustomEvent=b})();
var parse=function(a,b){try{return 200===a?JSON.parse(b):null}catch(e){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},base=function(a){return-1===a.indexOf("www")?"https://www.nature.com":""};
if(-1!==window.location.hostname.indexOf(".nature.com")){var transport=new XMLHttpRequest;transport.open("GET",base(window.location.hostname)+"/platform/login/details",!0);transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script id="" type="text/javascript">function splitKeys(b){var f=[],d="",h=[],g=b.split(";");for(b=0;b<g.length;++b){void 0!==l&&(d=l);var c=g[b].split("\x3d");var l=c[0];l!==d&&(0<d.length&&0<f.length&&h.push([d,f]),f=[]);if(2===c.length&&""!==c[0]&&""!==c[1]){var m=c[1].split(",");for(c=0;c<m.length;++c)f.push(m[c])}}0<l.length&&0<f.length&&h.push([l,f]);return h}
function getScript(b,f){var d=document.createElement("script"),h=document.getElementsByTagName("script")[0];d.async=1;d.onload=d.onreadystatechange=function(b,c){if(c||!d.readyState||/loaded|complete/.test(d.readyState))d.onload=d.onreadystatechange=null,d=void 0,c||f&&f()};d.src=b;h.parentNode.insertBefore(d,h)}
function splitSizes(b){var f=[];if(null!==b){var d=-1!==b.indexOf("|")?b.split("|"):b.split(",");for(b=0;b<d.length;++b){var h=d[b].split("x");var g=parseInt(h[0],10);var c=parseInt(h[1],10);2===h.length&&!isNaN(g)&&!isNaN(c)&&0<=g&&0<=c&&f.push([g,c])}}return f}function hasClass(b,f){var d,h=b.className.split(/\s+/);var g=0;for(d=h.length;g<d;++g)if(h[g]===f)return!0;return!1}
function getAdContainers(){if("function"===typeof document.getElementsByClassName)return document.getElementsByClassName("div-gpt-ad");var b,f=[],d=document.getElementsByTagName("div");for(b=0;d[b];++b)hasClass(d[b],"div-gpt-ad")&&f.push(d[b]);return f}function debounce(b,f){var d=null,h=null;return function(){var g=this,c=+new Date,l=arguments;d&&c<d+f?(clearTimeout(h),h=setTimeout(function(){d=c;b.apply(g,l)},f)):(d=c,b.apply(g,l))}}
function addResizeEvent(b){window.addEventListener?window.addEventListener("resize",b,!1):window.attachEvent("resize",b)}function addScrollEvent(b){window.addEventListener?window.addEventListener("scroll",b,!1):window.attachEvent("onscroll",b)}function removeScrollEvent(b){window.removeEventListener?window.removeEventListener("scroll",b,!1):window.detachEvent("scroll",b)}function serveAdsFor(b){return-1!==b.indexOf("/naturejobs")?!1:!0}
function initAds(b,f,d,h){var g=Bootstrapper.npg.utils,c=[],l,m=null;if("true"!==g.Request.get("hide_ads")&&serveAdsFor(b.location.pathname)){var r=function(a){var c=Math.max(f.clientHeight,b.innerHeight||0);return x(a,function(a){if(a.isOutOfPage||a.forceLoadOnInit)return!0;a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})},x=function(a,b){for(var e=a.length,c=[];e--;)b(a[e],e)&&(c.push(a[e].slot),a.splice(e,1));c.length&&googletag.pubads().refresh(c);return a},
q={"career feature":!0,"career news":!0,"career q\x26a":!0,"career brief":!0,"career column":!0,spotlight:!0,"career guide":!0,"technology feature":!0,"nature careers podcast":!0},y=function(){var a=b.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(1):null;if(a)return a;if(-1!==b.location.hostname.indexOf("guide.labanimal"))return a=b.location.pathname,a=-1!==a.indexOf("categ")?"products":-1!==a.indexOf("supplier")?"suppliers":"homepage","laban/guide.labanimal/"+a;a=b.location.pathname.replace(/^\/+/,"").split("/");
var c=b.location.hostname.split(".").slice(1).join(".");return a.length&&""!==a[0]?a[0]:c},z=function(a){var c="/270604982";0!==a.indexOf("/")&&(a="/"+a);0===a.indexOf("/285/")&&(a=a.replace(/^\/285\/[^\/]+/,c+"/nature/"+y()));a=a.replace("/nature/laban","/laban");a=a.replace("/nature/nature.com/index","/nature/nature/homepage");a=a.replace("/collections/collections","/collections");a=a.replace("/search/search_results","/nature/search");a=a.replace(/\/article$/,"/articles");a=a.replace(/\/nature\/authors\/.*/,
"/nature/nature/authors");-1!==b.location.hostname.indexOf("blogs")&&(a=a.replace(/\/nature\/.*/,"/nature/nature/blogs"));-1!==b.location.hostname.indexOf("natureindex")&&(a=a.replace(/\/nature\/.*/,"/nature/nature_index"),"/"===b.location.pathname&&(a+="/homepage"));window.dataLayer&&q[google_tag_manager["GTM-NWDMT9Q"].macro(2)]&&(a=a.replace(/\/articles$/,"/naturecareers"));a:{if(/^\/nature\/articles\/?$/.test(window.location.pathname)){var e=(/^.*?(?:\?|&)type=([^&]+)/.exec(window.location.search)||["",""])[1];if(q[e.replace(/-/g,
" ").replace(/ and /,"\x26")]){e=!0;break a}}e=!1}e&&(a=a.replace(/\/article-list$/,"/naturecareers"));(e=document.querySelector('meta[name\x3d"brandedcontent"]'))&&"true"===e.getAttribute("content")&&(a=c+"/nature/brandedcontent");return a},A={"728x90":{count:0,name:"LB"},"300x250":{count:0,name:"MPU"},"160x600":{count:0,name:"SKY"},"970x250":{count:0,name:"BB"},"2x2":{count:0,name:"NATIVE"},"300x100":{count:0,name:"REC"},"180x150":{count:0,name:"EVENTS"},"120x60":{count:0,name:"TILE"}},p=function(a,
b,e){for(var c=!1,d=0;a[d];++d)a[d][0]===b&&null!==e&&(a[d][1]=[e],c=!0);c||null===e||a.push([b,[e]]);return a},t=function(a,b,c){for(var e=0;a[e];++e)a[e][0]===b&&(a[e][0]=c);return a},B=function(a,c){var e=a;var k=d.dfppKeyValues&&g.Consent.allow(g.Consent.TARGETING)?";"+d.dfppKeyValues:"";var n=g.Request.get("test");n=n&&"ads"===n?";adtype\x3dtest":"";e=e.getAttribute("data-gpt-targeting");n&&-1===e.indexOf(n)&&(e+=n);e+=h.registrant_id?";logged\x3dy":";logged\x3dn";h.business_partner_id&&(e+=
";bpid\x3d"+h.business_partner_id.join(","));0===b.location.pathname.indexOf("/collections/")&&(n=document.querySelector("span.hero-title-inner"))&&(e+=";sponsored\x3d"+n.innerText.replace(/^\s+/,"").replace(/\s+$/,"").replace(/\s+/g,"_").replace(/\W+/g,""));k=e+k;k=t(splitKeys(k),"artid","articleid");k=t(k,"kw","search");e=a.getAttribute("data-gpt-sizes");e=(e=A[e])?e.name+ ++e.count:null;k=p(k,"pos",e);k=p(k,"tile",c+1);e=b.location.pathname.split("/");e=3===e.length&&"subjects"===e[1]?e[2]:null;
k=p(k,"subject",e);k=p(k,"article",window.dataLayer&&q[google_tag_manager["GTM-NWDMT9Q"].macro(3)]?"naturecareers":null);e=window.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(4):null;return k=p(k,"collectionID",e)},u=function(a){for(var c=[],e=0;a[e];++e){var d=a[e];(-1===d.className.indexOf("leaderboard")||728<b.innerWidth)&&c.push({divId:d.getAttribute("id"),adUnitPath:z(d.getAttribute("data-gpt-unitpath")),sizeArray:splitSizes(d.getAttribute("data-gpt-sizes")),targeting:B(d,e),isOutOfPage:hasClass(d,"out-of-page"),forceLoadOnInit:!1,
refreshed:!1})}return c},v=function(){googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(g.Consent.allow(g.Consent.TARGETING)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices();var a={};googletag.pubads().addEventListener("slotRenderEnded",function(b){var c=b.slot&&b.slot.getSlotElementId?b.slot.getSlotElementId():null,d;a[c]=!b.isEmpty;"ads"===g.Request.get("test")&&(a["div-gpt-ad-native-2"]||a["div-gpt-ad-native-1"])&&(d=document.querySelector(".c-paid-content__heading"))&&
(d.style.display="block");"div-gpt-ad-billboard-2"===c&&!1===b.isEmpty&&(d=document.getElementById(c),d.parentNode.parentNode.classList.add("pb40"),d.parentNode.parentNode.classList.remove("pb20"),d.parentNode.parentNode.classList.remove("hide"));c&&b.isEmpty&&(d=document.getElementById(c),d.parentNode.parentNode.classList.remove("pb20"),d.parentNode.classList.remove("ad-with-label"))})});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=c[a].isOutOfPage?googletag.defineOutOfPageSlot(c[a].adUnitPath,
c[a].divId).addService(googletag.pubads()):googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());for(var b=0,e=c[a].targeting.length;b<e;++b)if(2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]){if("pos"===c[a].targeting[b][0]&&0===c[a].targeting[b][1][0].indexOf("BB")){l=c[a].slot;var d=googletag.sizeMapping().addSize([970,250],[3,3]).addSize([770,100],[4,4]).addSize([0,0],[5,5]).build();l.defineSizeMapping(d);l.setCollapseEmptyDiv(!0,
!0)}c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}}catch(C){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)})},w=function(){m&&removeScrollEvent(m);googletag.cmd.push(function(){c=r(c)});m=debounce(function(){googletag.cmd.push(function(){c=r(c);c.length||(removeScrollEvent(m),m=null)})},250);addScrollEvent(m)};getScript("//www.googletagservices.com/tag/js/gpt.js",function(){b.googletag=b.googletag||
{};b.googletag.cmd=b.googletag.cmd||[];c=u(getAdContainers());v();w();var a=969<b.innerWidth,d=769<b.innerWidth&&!a,e=770>b.innerWidth,f=debounce(function(){var c=b.innerWidth;970<=c&&!a?(a=!0,e=d=!1,googletag.cmd.push(function(){googletag.pubads().refresh([l])})):770>c&&!e?(d=a=!1,e=!0,googletag.cmd.push(function(){googletag.pubads().refresh([l])})):970>c&&769<c&&!d&&(a=!1,d=!0,e=!1,googletag.cmd.push(function(){googletag.pubads().refresh([l])}))},250);addResizeEvent(f);document.addEventListener("refreshads",
function(){googletag.destroySlots();c=u(getAdContainers());v();w()},!1)})}}-1===window.location.hostname.indexOf("nature.com")&&("complete"===document.readyState||"loaded"===document.readyState||"interactive"===document.readyState?initAds(window,document.documentElement,window.Krux||{},{}):document.addEventListener("load",function(){initAds(window,document.documentElement,window.Krux||{},{})}));
document.addEventListener("accessdetailsloaded",function(b){initAds(window,document.documentElement,window.Krux||{},b.detail||{})},!1);</script>
<script id="" type="text/javascript">window.location.pathname.match(/^\/(nature_education|scitable|principles)(\/|$)/)||(window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]),function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+
Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""}());</script><script id="" type="text/javascript">var filterMeta=function(b){for(var c=[],a=0;b[a];++a)b[a].name&&c.push(b[a]);return c},translateMeta=function(b){var c={"WT.cg_n":"product_name","WT.site_id":"site_id","WT.site_id_name":"site_id_name","WT.registrant_id":"user_id"};return c[b]||b},getMeta=function(b){var c=filterMeta(document.getElementsByTagName("meta")||[]),a=document.getElementsByTagName("body")||[],f=function(){for(var a=c.length,d={},e;a--;)if(e=translateMeta(c[a].name))d[e]?("string"===typeof d[e]&&(d[e]=[d[e]]),d[e].push(c[a].content)):
d[e]=c[a].content||"";d.keywords&&(d.keywords=d.keywords.replace(/,/g,";"));b(d)};a.length?f():c.length&&f()};getMeta(function(b){window._pageMetaData=b});</script><script id="" type="text/javascript">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)&&"www.nature.com"===window.location.hostname&&"/"===window.location.pathname){_prum=[["id","58bd3858754054efc9152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)};</script><script id="" type="text/javascript">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)&&"www.nature.com"===window.location.hostname&&"/"===window.location.pathname){_prum=[["id","58bd3858754054efc9152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)};</script><script id="" type="text/javascript">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)){var path=/^\/subjects/;if("www.nature.com"===window.location.hostname&&window.location.pathname.match(path)){_prum=[["id","58bd36ee7540540ec8152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)}};</script><script id="" type="text/javascript">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="" type="text/javascript">var images=document.images,sendNielson=!0;if(-1<images.length){var i=0;for(i;i<images.length;i++)images[i].src&&-1<images[i].src.indexOf("secure-us.imrworldwide.com")&&(sendNielson=!1)}sendNielson&&function(){var a=new Image(1,1);a.onerror=a.onload=function(){a.onerror=a.onload=null};a.src=["//secure-us.imrworldwide.com/cgi-bin/m?ci\x3dus-504674h\x26cg\x3d0\x26cc\x3d1\x26si\x3d",escape(window.location.href),"\x26rp\x3d",escape(document.referrer),"\x26ts\x3dcompact\x26rnd\x3d",(new Date).getTime()].join("")}();</script><script id="" type="text/javascript">(function(){var a=document,b=a.getElementsByTagName("script")[0];s=a.createElement("script");s.type="text/javascript";s.async=!0;s.src="//cdn.cookielaw.org/consent/792a5a21-3d99-4432-8f70-957d389bf2b9.js";b.parentNode.insertBefore(s,b)})();function OptanonWrapper(){};</script><script id="" type="text/javascript">var c_id="14617931",_comscore=window._comscore=_comscore||[];_comscore.push({c1:"2",c2:c_id});(function(){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;var c="https:"==document.location.protocol?"https://sb":"http://b";a.src=c+".scorecardresearch.com/beacon.js";b.parentNode.insertBefore(a,b)})();</script><script id="" type="text/javascript">(function(){if(window.location.hostname.match(/^(.*-?www.)?nature.com/)){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;a.defer=!0;a.src="//get.s-onetag.com/c692e01a-b65d-4aa9-aff9-bee34f082a39/tag.min.js";b.parentNode.insertBefore(a,b)}})();</script><script id="" type="text/javascript">(function(){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;a.src="https://recommended.springernature.com/latest/entry-point.js";b.parentNode.insertBefore(a,b)})();</script><iframe allowtransparency="true" frameborder="0" id="uptodate-client" name="uptodate-client" scrolling="no" src="https://recommended.springernature.com/v2296/generated/client.html?source=https://www.nature.com" style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" title="Personalised recommendations"></iframe><script id="" type="text/javascript">(function(win,doc,undefined){var isArray=google_tag_manager["GTM-NWDMT9Q"].macro(86);var isObject=google_tag_manager["GTM-NWDMT9Q"].macro(87);var map=google_tag_manager["GTM-NWDMT9Q"].macro(89);var closestByAttribute=google_tag_manager["GTM-NWDMT9Q"].macro(91);var closest=google_tag_manager["GTM-NWDMT9Q"].macro(92);var enforceDataType=google_tag_manager["GTM-NWDMT9Q"].macro(94);var normaliseAnd=google_tag_manager["GTM-NWDMT9Q"].macro(95);var normaliseWhitespace=google_tag_manager["GTM-NWDMT9Q"].macro(96);var formatDate=google_tag_manager["GTM-NWDMT9Q"].macro(97);var createEventPayload=google_tag_manager["GTM-NWDMT9Q"].macro(102);var createEcommercePromotionEvent=google_tag_manager["GTM-NWDMT9Q"].macro(106);var createEcommerceProductEvent=
google_tag_manager["GTM-NWDMT9Q"].macro(108);var productAtIndex=google_tag_manager["GTM-NWDMT9Q"].macro(111);var sendEvent=google_tag_manager["GTM-NWDMT9Q"].macro(112);var eventHandler=google_tag_manager["GTM-NWDMT9Q"].macro(121);var init=function(){(function(products){if(products&&products.length){for(var i=0;products[i];++i){var product=productAtIndex(products,i);product.list="Search Results";win.ga("ec:addImpression",product)}sendEvent({action:"impression",category:"catalog",nonInteraction:true})}})(google_tag_manager["GTM-NWDMT9Q"].macro(122));$('[data-track\x3d"click"]').on("click",function(e){eventHandler(e.target,
"click")});$('[data-track\x3d"change"]').on("change",function(e){eventHandler(e.target,"change")});$('[data-track\x3d"download"]').on("click",function(e){eventHandler(e.target,"download")});$('form[data-track\x3d"submit"]').on("submit",function(e){eventHandler(e.target,"submit")});$('form[data-component\x3d"search-form"]').on("submit",function(){populateDimensions({dimension30:true})});$('form[data-track-action\x3d"search-body"] select').on("change",function(e){var target=e.target;var form=closest(target,
function(node){return lc(node.nodeName)==="form"});var subject=target.options[target.selectedIndex].value;var label=subject?"search-body-"+subject:"search-body";form.setAttribute("data-track-label",label)});$('[data-track\x3d"click-promotion"]').on("click",function(e){var target=closestByAttribute(e.target,"data-track","click-promotion");var payload=null;if(target){payload=createEcommercePromotionEvent(target);if(payload){win.ga("ec:addPromo",payload);win.ga("ec:setAction","promo_click");sendEvent({category:"Internal Promotions",
action:"click",label:payload.name})}}});$('[data-track\x3d"click-product"]').on("click",function(e){var target=closestByAttribute(e.target,"data-track","click-product");var payload=null;if(target){payload=createEcommerceProductEvent(target);if(payload){win.ga("ec:addProduct",payload);win.ga("ec:setAction","click",{list:"Search Results"});sendEvent({category:"UX",action:"click",label:"Results",value:payload.name})}}});$('a[href*\x3d"sci-hub"],a[href*\x3d"dx.doi.org"]').on("click",function(e){sendEvent({action:"Click Event",
category:"External Link",label:this.href.indexOf("sci-hub")!==-1?"sci-hub":"dx.doi.org"})})};var checkJq=setInterval(function(){if(typeof jQuery!="undefined"){clearInterval(checkJq);init()}},250)})(window,document);</script><script id="" type="text/javascript">var getClientId=function(){return window.ga&&window.ga.getAll&&window.ga.getAll().length?window.ga.getAll()[0].get("clientId"):null},clientId=getClientId();if(clientId){var dlUpdate={event:"ga-client-id-pushed-to-datalayer",gaClientId:clientId};window.dataLayer.push(dlUpdate)};</script><iframe aria-hidden="true" height="0" id="event-tracker" src="https://event-tracker.springernature.com" style="display:none; visibility:hidden" tabindex="-1" title="EventTracker" width="0"></iframe><div class="kxhead" data-id="Jn7TaPwq" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20636"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20637"></span><span class="kxtag kxinvisible" data-id="20638"></span><span class="kxtag kxinvisible" data-id="20641"><script>
(function() {

    var pageMetaData = window._pageMetaData;
    Krux('scrape', {
        'page_attr_nature_url_path_1': {
            'url_path': '1'
        },
        'page_attr_nature_url_path_2': {
            'url_path': '2'
        },
        'page_attr_nature_url_path_3': {
            'url_path': '3'
        },
        'page_attr_nature_product_name': {
            'js_global': '_pageMetaData.product_name'
        },
        'page_attr_nature_site_id': {
            'js_global': '_pageMetaData.site_id'
        },
        'page_attr_nature_site_id_name': {
            'js_global': '_pageMetaData.site_id_name'
        }
    });
    if (Krux('get', 'page_attr_nature_site_id') && Krux('get', 'page_attr_nature_site_id').length > 1) {
        Krux('set', 'user_attr_nature_site_account_user', 'true');
    } else {
        Krux('set', 'user_attr_nature_site_account_user', 'false');
    }

})();
</script></span><span class="kxtag kxinvisible" data-id="20647"><script>
(function() {

    Krux('scrape', {
        'page_attr_Citation_Title': {
            'js_global': '_pageMetaData.citation_title'
        },
        'page_attr_Citation_DOI': {
            'js_global': '_pageMetaData.citation_doi'
        },
        'page_attr_Citation_Publication_Date': {
            'js_global': '_pageMetaData.citation_publication_date'
        },
        'page_attr_Citation_Author': {
            'js_global': '_pageMetaData.citation_author'
        }
    });

    if (window._pageMetaData && window._pageMetaData["WT.cg_s"]) {
        Krux('set', 'page_attr_Article_Type', _pageMetaData["WT.cg_s"]);

    } else if (window._pageMetaData && window._pageMetaData["prism.section"]) {
        Krux('set', 'page_attr_Article_Type', _pageMetaData["prism.section"]);
    }

    if (window._pageMetaData && window._pageMetaData.access === 'No') {
        Krux('set', 'user_attr_Access_Denied', _pageMetaData.product_name);
    }

    if (window._pageMetaData && window._pageMetaData["WT.z_subject_term"]) {
        Krux('set', 'page_attr_Article_Keywords', _pageMetaData["WT.z_subject_term"]);
    } else if (window._pageMetaData && window._pageMetaData.keywords) {
        Krux('scrape', {
            'page_attr_Article_Keywords': {
                'js_global': '_pageMetaData.keywords'
            }
        });
    } else {
        Krux('scrape', {
            'page_attr_Article_Keywords': {
                meta_name: 'keywords'
            }
        });
    }

})();
</script></span><span class="kxtag kxinvisible" data-id="20648"><script>
(function() {

    // Using Meta DCSext.job_id to produce page attribute dcsext_job_id
    Krux('scrape', {
        'page_attr_Job_ID': {
            meta_name: 'DCSext.job_id'
        },
        'page_attr_Employer_ID': {
            meta_name: 'DCSext.employer_id'
        },
        'page_attr_Job_Employer': {
            meta_name: 'Job Employer'
        },
        'page_attr_Job_Type': {
            meta_name: 'Job Type'
        }
    });

    var job_location = Krux('scrape', {
        'page_attr_Job_Location': {
            meta_name: 'Job Location'
        }
    });
    // meta attribute Job Location is always in format "city, country"; client only wants to ingest country
    if (job_location && job_location.page_attr_Job_Location && job_location.page_attr_Job_Location.split(',')[1]) Krux('set', 'page_attr_Job_Location', job_location.page_attr_Job_Location.split(',')[1].trim());

})();
</script></span><span class="kxtag kxinvisible" data-id="20651"><script type="text/javascript">Krux('social.init');</script></span><span class="kxtag kxinvisible" data-id="20655"><script>
(function(){
	var domain, path1, path2, key1, key2;
	var href = window.location.href;
	// pull in domain name
	domain = Krux('get', 'domain');
	// pull in path 1 & 2 of domain
	path1 = Krux('scrape.url_path' , '1');
	path2 = Krux('scrape.url_path' , '2');
	// only if domain is 'www.naturejobs.com'
	if (domain.match(/www\.nature\.com/g)){
		// and if both url path 1 is 'naturejobs' and url path 2 is 'science'
		if (path1.match(/naturejobs/g) && path2.match(/science/g)){
			// scrape keywords as naturejobs attributes
			Krux('scrape',{
				'page_attr_naturejobs_what':{'url_param':'q'},
				'page_attr_naturejobs_where':{'url_param':'where'}
			});
		}
	}
})();
</script></span><span class="kxtag kxinvisible" data-id="36836"><script>
(function() {
    /* Selective Attribute DataLayer Library Tag */
    var _, allAttr, allowedList, attr, attributes, dataLayerIngester, dataObj,
        isAllowed, keepCase, libUtil, omitKeys, pageAttr, prefix, toSet, trim,
        userAttr, util, value,
        hasProp = {}.hasOwnProperty;
    _ = Krux('require:underscore');
    util = Krux('require:util');
    libUtil = Krux('require:util.library-tag');
    dataLayerIngester = Krux('require:scrape').ingestDataLayer;

    /* Safe copy of dataLayer object */
    dataObj = Krux('scrape.javascript', 'dataLayer');

    /* String trimming helper function */
    trim = function(attr) {
        return ("" + attr).replace(/^\s+|\s+$/g, '');
    };

    /* Attribute configs */
    pageAttr = _.map('content.authorization.status,content.category.legacy.webtrendsSubjectTerms,content.category.contentType,content.contentInfo.authors,content.article.doi,content.contentInfo.publishedAtString,content.contentInfo.title,content.category.journal.title,page.category.pageType,page.attributes.template,window.location.pathname,window.location.pathname,window.location.pathname,content.jobs.jobTitle,content.jobs.jobLocation,content.jobs.employer,content.category.journal.title,content.journal.title'.split(','), trim);
    userAttr = _.map('user.profile.profileInfo.bpid,segment.registeredUser,gaClientId'.split(','), trim);

    /* Create a array of attributes striping any empty strings */
    allAttr = _.without(pageAttr.concat(userAttr), '');

    /* Configuration settings */
    keepCase = 'false' === 'true';
    omitKeys = 'none'.split(',');

    /* Resolve Prefix */
    prefix = libUtil.resolvePrefix('text', 'new_dl_',
        'undefined');

    /* Function to varify if attribute should be used */
    isAllowed = function(value, whitelist) {
        var i, len, str, x;
        str = "" + value;
        if (!((value != null) && str.length > 0)) {
            return false;
        }
        for (i = 0, len = whitelist.length; i < len; i++) {
            x = whitelist[i];
            if (value.match(x) != null) {
                return true;
            }
        }
        return false;
    };

    /* Get a full list of attributes usting the dataLayer tool */
    attributes = dataLayerIngester(dataObj, {
        omitKeys: libUtil.removeFalsyStrings(omitKeys.concat(libUtil.EXCLUDE_KEYS_CONFIG)),
        omitValues: libUtil.EXCLUDE_VALUES_CONFIG,
        caseSensitive: keepCase,
        useFullPath: 'true' === 'true',
        useLastValue: 'false' === 'true',
        customDelimited: [/./],
        altDelimiter: ',',
        userKeys: _.map(userAttr, function(exp) {
            return new RegExp("(^|\\.)" + exp + "$");
        }),
        optimizeNames: true
    });

    /* Only set Attributes in the allowed list */
    allowedList = _.map(allAttr, function(name) {
        return new RegExp("(_attr_|_attr_" + prefix + "|\\.)" + (keepCase ?
            name : libUtil.normalizeAttrName(name, {
                removeDot: false
            })) + "$");
    });
    toSet = {};
    for (attr in attributes) {
        if (!hasProp.call(attributes, attr)) continue;
        value = attributes[attr];
        if (isAllowed(attr, allowedList)) {
            toSet[attr] = value;
        }
    }
    toSet = Krux('prefix:attr', toSet, prefix);
    Krux('set', toSet);
}).call();
</script></span><span class="kxtag kxinvisible" data-id="38361"><script>

(function() { 

	var pageMetaData = window._pageMetaData; 
	Krux('scrape', { 
		'page_attr_url_path_1': { 
			'url_path': '1' 
		}, 
		'page_attr_url_path_2': { 
			'url_path': '2' 
		}, 
			'page_attr_url_path_3': { 
				'url_path': '3'
		}, 
			'page_attr_product_name': { 
				'js_global': '_pageMetaData.product_name'
		}, 
			'page_attr_site_id': { 
				'js_global': '_pageMetaData.site_id'
		}, 
			'page_attr_site_id_name': { 
				'js_global': '_pageMetaData.site_id_name' 
		}, 
			'user_attr_user_id': { 'js_global': '_pageMetaData.user_id' 
		} 
	}); 
	if (Krux('get', 'page_attr_site_id') && Krux('get', 'page_attr_site_id').length > 1) 	{ 
		Krux('set', 'user_attr_site_account_user', 'true'); 
	} else { 
		Krux('set', 'user_attr_site_account_user', 'false'); 
	} 
})(); 
</script></span></div><script id="" type="text/javascript">!function(d,e,f,a,b,c){d.twq||(a=d.twq=function(){a.exe?a.exe.apply(a,arguments):a.queue.push(arguments)},a.version="1.1",a.queue=[],b=e.createElement(f),b.async=!0,b.src="//static.ads-twitter.com/uwt.js",c=e.getElementsByTagName(f)[0],c.parentNode.insertBefore(b,c))}(window,document,"script");twq("init","o1unr");twq("track","PageView");</script><script id="" type="text/javascript">-1==document.location.href.search("appspot.com")&&-1==document.referrer.search("appspot.com")&&!function(b,e,f,g,a,c,d){b.fbq||(a=b.fbq=function(){a.callMethod?a.callMethod.apply(a,arguments):a.queue.push(arguments)},b._fbq||(b._fbq=a),a.push=a,a.loaded=!0,a.version="2.0",a.queue=[],c=e.createElement(f),c.async=!0,c.src=g,d=e.getElementsByTagName(f)[0],d.parentNode.insertBefore(c,d))}(window,document,"script","https://connect.facebook.net/en_US/fbevents.js");fbq("init","1229240860577415");
fbq("track","PageView");var segs=google_tag_manager["GTM-NWDMT9Q"].macro(201);fbq("trackCustom","SciRep_inMarket",{t4fs93hlz:segs});</script>
<noscript><img height="1" src="https://www.facebook.com/tr?id=1229240860577415&amp;ev=PageView&amp;noscript=1" style="display:none" width="1"/></noscript></body><iframe id="kx-proxy-Jn7TaPwq" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#!kxcid=Jn7TaPwq&amp;kxt=https%3A%2F%2Fwww.nature.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>