<html class="no-js" lang="en"><head><meta class="foundation-mq-small"/><meta class="foundation-mq-small-only"/><meta class="foundation-mq-medium"/><meta class="foundation-mq-medium-only"/><meta class="foundation-mq-large"/><meta class="foundation-mq-large-only"/><meta class="foundation-mq-xlarge"/><meta class="foundation-mq-xlarge-only"/><meta class="foundation-mq-xxlarge"/><meta class="foundation-data-attribute-namespace"/><script src="https://www.googletagservices.com/activeview/js/current/osd.js?cb=%2Fr20100101"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_rendering_2019070801.js"></script><script async="" src="https://www.gstatic.com/recaptcha/api2/v1562567553145/recaptcha__en.js" type="text/javascript"></script><script src="https://v1.addthis.com/live/red_lojson/300lo.json?si=5d2adbc3bf2dbef4&amp;bkl=0&amp;bl=1&amp;pdt=8434&amp;sid=5d2adbc3bf2dbef4&amp;pub=xa-5265518246c10183&amp;rev=v8.4.11-wp&amp;ln=en&amp;pc=men&amp;cb=0&amp;ab=-&amp;dp=academic.oup.com&amp;fp=femsec%2Farticle%2F54%2F1%2F21%2F565169&amp;fr=&amp;of=0&amp;pd=0&amp;irt=0&amp;vcl=0&amp;md=0&amp;ct=1&amp;tct=0&amp;abt=0&amp;cdn=0&amp;pi=1&amp;rb=0&amp;gen=100&amp;chr=UTF-8&amp;colc=1563089859648&amp;jsl=1&amp;uvs=5d2adbc3602c1478000&amp;skipb=1&amp;callback=addthis.cbs.vhp1_72983141462944250" type="text/javascript"></script><script src="https://v1.addthisedge.com/live/boost/xa-5265518246c10183/_ate.track.config_resp" type="text/javascript"></script>
<title>Effects of the inoculant strain Pseudomonas putida KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community | FEMS Microbiology Ecology | Oxford Academic</title>
<script async="" src="https://www.google-analytics.com/analytics.js" type="text/javascript"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-W6DD7HV"></script><script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="/UI/app/js/jquery-2.2.4.js">\x3C/script>')</script>
<meta content="width=device-width, initial-scale=1, maximum-scale=1" name="viewport"/>
<meta charset="utf-8"/>
<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
<meta content="IE=Edge" http-equiv="X-UA-Compatible"/>
<link href="https://academic.oup.com/UI/app/img/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="https://academic.oup.com/UI/app/img/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="https://academic.oup.com/UI/app/img/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png"/>
<link href="https://academic.oup.com/UI/app/img/manifest.json" rel="manifest"/>
<link color="#5bbad5" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" rel="mask-icon"/>
<link href="https://academic.oup.com/UI/app/img/favicon.ico" rel="shortcut icon"/>
<meta content="/UI/app/img/browserconfig.xml" name="msapplication-config"/>
<meta content="#002f65" name="theme-color"/>
<link href="//oup.silverchair-cdn.com/UI/app/css/style.min.css" rel="stylesheet" type="text/css"/>
<link href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/jquery.fancybox-1.3.4_1.css" rel="stylesheet" type="text/css"/>
<link href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/jscrollpane.css" rel="stylesheet" type="text/css"/>
<link href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/theme.default.css" rel="stylesheet" type="text/css"/>
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" rel="stylesheet" type="text/css"/>
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css"/>
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/femsec/Site-1178322233.css" rel="stylesheet" type="text/css"/>
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>
<script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'femsec';
            App.SiteURL = 'academic.oup.com/femsec';
        </script>
<link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css"/>
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/femsec/article/54/1/21/565169","@type":"ScholarlyArticle","name":"Effects of the inoculant strain  Pseudomonas putida  KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community","about":["polymerase chain reaction","dna","genes","naphthalenes","pseudomonas putida","rna, ribosomal, 16s","soil","community"],"datePublished":"2005-09-01","isPartOf":{"@id":"https://academic.oup.com/femsec/issue/54/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2005-09-01","isPartOf":{"@id":"https://academic.oup.com/femsec","@type":"Periodical","name":"FEMS Microbiology Ecology","issn":["1574-6941"]}},"url":"http://dx.doi.org/10.1016/j.femsec.2005.02.005","keywords":["polymerase chain reaction","dna","genes","naphthalenes","pseudomonas putida","rna, ribosomal, 16s","soil","community","<italic>Pseudomonas putida</italic> KT2442 (pNF142)","Naphthalene degradation","Denaturing gradient gel electrophoresis","Dioxygenase genes"],"inLanguage":"en","copyrightHolder":"Federation of European Microbiological Societies","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/femsec/article/54/1/21/565169","author":[{"name":"Gomes, Newton C.M.","affiliation":"Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany","@type":"Person"},{"name":"Kosheleva, Irina A.","affiliation":"Laboratory of Plasmid Biology, Skryabin Institute of Biochemistry and Physiology of Microorganisms, Russian Academy of Sciences, Pushchino, Moscow region 142290, Russia","@type":"Person"},{"name":"Abraham, Wolf-Rainer","affiliation":"Department of Microbiology, GBF-National Research Centre for Biotechnology, 38124 Braunschweig, Germany","@type":"Person"},{"name":"Smalla, Kornelia","affiliation":"Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany","@type":"Person"}],"description":"Abstract.  The naphthalene-degrading activity of a Pseudomonas sp. strain isolated from a creosote-contaminated soil was shown to be encoded by the IncP9 plasmi","pageStart":"21","pageEnd":"33","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/Issue/54/1/1/m_cover.gif?Expires=1626161848&Signature=KW22UbYf3qHhWZwZSvy8DC1XolrA0-qIYmg~BwaUXKpZc5rmc~mUJ6vVMTkH0tyHUBqIcZMdJ8YkIyKAu2wI0-SFc043zbzAR3Nxu0OMdLNd2lAtz02gCdK0MxIwiaDxCq1SbH3VcdytVzABdDO42OCVqm4dPl-BF6Wj~RkonZiJTwzA~lrEw4i8yHslnhjZbAg4Sd6SQClaePge8dGssrcCW~zrFOmk65eAMskEhlChgtHY0u7496f4Fo5tCnak-NfXxC80JoiXtgahk~E0W7kaOvuckaD8LDGWjOwXH6jrqCtiaVaQGvAc9lmjE5DtEKIphwW5ry1wqpp1EvugOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Effects of the inoculant strain  Pseudomonas putida  KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/Issue/54/1/1/m_cover.gif?Expires=1626161848&Signature=KW22UbYf3qHhWZwZSvy8DC1XolrA0-qIYmg~BwaUXKpZc5rmc~mUJ6vVMTkH0tyHUBqIcZMdJ8YkIyKAu2wI0-SFc043zbzAR3Nxu0OMdLNd2lAtz02gCdK0MxIwiaDxCq1SbH3VcdytVzABdDO42OCVqm4dPl-BF6Wj~RkonZiJTwzA~lrEw4i8yHslnhjZbAg4Sd6SQClaePge8dGssrcCW~zrFOmk65eAMskEhlChgtHY0u7496f4Fo5tCnak-NfXxC80JoiXtgahk~E0W7kaOvuckaD8LDGWjOwXH6jrqCtiaVaQGvAc9lmjE5DtEKIphwW5ry1wqpp1EvugOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
<meta content="article" property="og:type"/>
<meta content="Effects of the inoculant strain  Pseudomonas putida  KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community" property="og:title"/>
<meta content="Abstract.  The naphthalene-degrading activity of a Pseudomonas sp. strain isolated from a creosote-contaminated soil was shown to be encoded by the IncP9 plasmi" property="og:description"/>
<meta content="https://academic.oup.com/femsec/article/54/1/21/565169" property="og:url"/>
<meta content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/Issue/54/1/1/m_cover.gif?Expires=1626161848&amp;Signature=KW22UbYf3qHhWZwZSvy8DC1XolrA0-qIYmg~BwaUXKpZc5rmc~mUJ6vVMTkH0tyHUBqIcZMdJ8YkIyKAu2wI0-SFc043zbzAR3Nxu0OMdLNd2lAtz02gCdK0MxIwiaDxCq1SbH3VcdytVzABdDO42OCVqm4dPl-BF6Wj~RkonZiJTwzA~lrEw4i8yHslnhjZbAg4Sd6SQClaePge8dGssrcCW~zrFOmk65eAMskEhlChgtHY0u7496f4Fo5tCnak-NfXxC80JoiXtgahk~E0W7kaOvuckaD8LDGWjOwXH6jrqCtiaVaQGvAc9lmjE5DtEKIphwW5ry1wqpp1EvugOQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" property="og:image"/>
<meta content="2005-09-01" property="og:updated_time"/>
<meta content="OUP Academic" property="og:site_name"/>
<meta content="Gomes, Newton C.M." name="citation_author"/><meta content="Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany" name="citation_author_institution"/><meta content="Kosheleva, Irina A." name="citation_author"/><meta content="Laboratory of Plasmid Biology, Skryabin Institute of Biochemistry and Physiology of Microorganisms, Russian Academy of Sciences, Pushchino, Moscow region 142290, Russia" name="citation_author_institution"/><meta content="Abraham, Wolf-Rainer" name="citation_author"/><meta content="Department of Microbiology, GBF-National Research Centre for Biotechnology, 38124 Braunschweig, Germany" name="citation_author_institution"/><meta content="Smalla, Kornelia" name="citation_author"/><meta content="Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany" name="citation_author_institution"/><meta content="Effects of the inoculant strain Pseudomonas putida KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community" name="citation_title"/><meta content="21" name="citation_firstpage"/><meta content="33" name="citation_lastpage"/><meta content="10.1016/j.femsec.2005.02.005" name="citation_doi"/><meta content="FEMS Microbiology Ecology" name="citation_journal_title"/><meta content="FEMS Microbiol Ecol" name="citation_journal_abbrev"/><meta content="54" name="citation_volume"/><meta content="1" name="citation_issue"/><meta content="2005/09/01" name="citation_publication_date"/><meta content="0168-6496" name="citation_issn"/><meta content="Narnia" name="citation_publisher"/><meta content="citation_title=The need for DGGE and TGGE in microbial ecology; citation_author=Muyzer  G.; citation_author=Smalla  K.;  citation_journal_title=Antonie van Leeuwenhoek;  citation_year=1998;  citation_volume=73;  citation_pages=127-141; " name="citation_reference"/><meta content="citation_title=In situ, real-time catabolic gene expression: extraction and characterization of naphthalene dioxygenase mRNA transcripts from groundwater; citation_author=Wilson  M.S.; citation_author=Bakermans  C.; citation_author=Madsen  E.L.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1999;  citation_volume=65;  citation_pages=80-87; " name="citation_reference"/><meta content="citation_title=Psychrotolerant bacteria isolated from arctic soil that degrade polychlorinated biphenyls at low temperatures; citation_author=Master  E.R.; citation_author=Mohn  W.W.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1998;  citation_volume=64;  citation_pages=4823-4829; " name="citation_reference"/><meta content="citation_title=PCR primers for detection and characterisation of IncP-9 plasmids; citation_author=Krasowiak  R.; citation_author=Smalla  K.; citation_author=Sokolov  S.; citation_author=Kosheleva  I.; citation_author=Sevastyanovich  Y.; citation_author=Titok  M.; citation_author=Thomas  C.M.;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=2002;  citation_volume=42;  citation_pages=217-225; " name="citation_reference"/><meta content="citation_title=Simultaneous recovery of RNA and DNA from soils and sediments; citation_author=Hurt  R.A.; citation_author=Qiu  X.; citation_author=Wu  L.; citation_author=Roh  Y.; citation_author=Palumbo  A.V.; citation_author=Tiedje  J.M.; citation_author=Zhou  J.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2001;  citation_volume=67;  citation_pages=4495-4503; " name="citation_reference"/><meta content="citation_title=An optimized protocol for simultaneous extraction of DNA and RNA from soils; citation_author=Costa  R.; citation_author=Gomes  N.C.M.; citation_author=Milling  A.; citation_author=Smalla  K.;  citation_journal_title=Braz. J. Microbiol;  citation_year=2004;  citation_volume=35;  citation_pages=230-234; " name="citation_reference"/><meta content="citation_title=Analysis of catabolic genes for naphthalene and phenanthrene degradation in contaminated New Zealand soils; citation_author=Lloyd-Jones  G.; citation_author=Laurie  A.D.; citation_author=Hunter  D.W.F.; citation_author=Fraser  R.;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=1999;  citation_volume=29;  citation_pages=69-79; " name="citation_reference"/><meta content="citation_title=DNA recovery and PCR quantification of catechol 2,3-dioxygenase genes from different soil types; citation_author=Wikström  P.; citation_author=Wiklund  A.; citation_author=Andersson  A.C.; citation_author=Forsman  M.;  citation_journal_title=J. Biotechnol;  citation_year=1996;  citation_volume=52;  citation_pages=107-120; " name="citation_reference"/><meta content="citation_author=Sambrook  J.; citation_author=Fritsch  E.F.; citation_author=Maniatis  T.; citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY;  citation_title=Molecular cloning: a laboratory manual;  citation_year=1989; " name="citation_reference"/><meta content="citation_title=2,4-Dichlorophenoxyacetic acid degrading bacteria contain mosaics of catabolic genes; citation_author=Fulthorpe  R.R.; citation_author=McGowan  C.; citation_author=Maltseva  O.V.; citation_author=Holben  W.E.; citation_author=Tiedje  J.M.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1995;  citation_volume=61;  citation_pages=3274-3281; " name="citation_reference"/><meta content="citation_title=Analysis of actinomycete communities by specific amplification of 16S rRNA and gel electrophoretic separation in denaturing gradients; citation_author=Heuer  H.; citation_author=Krsek  M.; citation_author=Smalla  K.; citation_author=Wellington  E.M.H.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1997;  citation_volume=63;  citation_pages=3233-3241; " name="citation_reference"/><meta content="citation_title=Double-gradient DGGE for optimized detection of DNA point mutations; citation_author=Cremonesi  L.; citation_author=Firpo  S.; citation_author=Ferrari  M.; citation_author=Righetti  P.G.; citation_author=Gelfi  C.;  citation_journal_title=Biotechniques;  citation_year=1997;  citation_volume=22;  citation_pages=326-330; " name="citation_reference"/><meta content="citation_title=Bacterial community profiling using DGGE or TGGE analysis; citation_author=Heuer  H.; citation_author=Wieland  G.; citation_author=Schönfeld  J.; citation_author=Schönwälder  S.; citation_author=Gomes  N.C.M.; citation_author=Smalla  K.; citation_author=Rouchelle  P.; citation_publisher=Horizon Scientific Press, Wymondham, UK;  citation_title=Environmental molecular microbiology: protocols and applications;  citation_year=2001;  citation_pages=177-190; " name="citation_reference"/><meta content="citation_title=A new approach to utilize PCR-single-strand-conformation polymorphism for 16S rRNA gene-based microbial community analysis; citation_author=Schwieger  F.; citation_author=Tebbe  C.C.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1998;  citation_volume=64;  citation_pages=4870-4876; " name="citation_reference"/><meta content="citation_title=Bulk and rhizosphere soil bacterial communities studied by denaturing gradient gel electrophoresis: plant-dependent enrichment and seasonal shifts revealed; citation_author=Smalla  K.; citation_author=Wieland  G.; citation_author=Buchner  A.; citation_author=Zock  A.; citation_author=Parzy  J.; citation_author=Kaiser  S.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2001;  citation_volume=67;  citation_pages=4742-4751; " name="citation_reference"/><meta content="citation_title=Bacterial diversity of the rhizosphere of maize (Zea mays) grown in tropical soil studied by temperature gradient gel electrophoresis; citation_author=Gomes  N.C.M.; citation_author=Heuer  H.; citation_author=Schönfeld  J.; citation_author=Costa  R.; citation_author=Hagler-Mendonça  L.; citation_author=Smalla  K.;  citation_journal_title=Plant Soil;  citation_year=2001;  citation_volume=232;  citation_pages=167-180; " name="citation_reference"/><meta content="citation_title=Manure enhances plasmid mobilization and survival of Pseudomonas putida introduced into field soil; citation_author=Götz  A.; citation_author=Smalla  K.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1997;  citation_volume=63;  citation_pages=1980-1986; " name="citation_reference"/><meta content="citation_title=A bioluminescent derivative of Pseudomonas putida KT2440 for deliberate release into the environment; citation_author=Ramos  C.; citation_author=Molina  L.; citation_author=Mølbak  L.; citation_author=Ramos  J.L.; citation_author=Molin  S.;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=2000;  citation_volume=34;  citation_pages=91-102; " name="citation_reference"/><meta content="citation_title=Survival of Pseudomonas putida KT2440 in soil and in the rhizosphere of plants under greenhouse and environmental conditions; citation_author=Molina  L.; citation_author=Ramos  C.; citation_author=Duque  E.; citation_author=Ronchel  M.C.; citation_author=García  M.; citation_author=Wyke  L.; citation_author=Ramos  J.L.;  citation_journal_title=Soil Biol. Biochem;  citation_year=2000;  citation_volume=32;  citation_pages=315-321; " name="citation_reference"/><meta content="citation_title=Combined use of 16S Ribosomal DNA and 16S rRNA to study the bacterial community of polychlorinated biphenyl-polluted soil; citation_author=Nogales  B.; citation_author=Moore  E.R.B.; citation_author=Llobet-Brossa  E.; citation_author=Rossello-Mora  R.; citation_author=Amann  R.; citation_author=Timmis  N.K.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2001;  citation_volume=67;  citation_pages=1874-1884; " name="citation_reference"/><meta content="citation_title=Application of denaturing gradient gel electrophoresis and temperature gradient gel electrophoresis for studying soil microbial communities; citation_author=Heuer  H.; citation_author=Smalla  K.; citation_author=Elsas  J.D. van; citation_author=Wellington  E.M.H.; citation_author=Trevors  J.T.; citation_publisher=Marcel Dekker, New York;  citation_title=Modern soil microbiology;  citation_year=1997;  citation_pages=353-373; " name="citation_reference"/><meta content="citation_title=Pilot-scale evaluation of bioaugmentation for in-situ remediation of a carbon tetrachloride contaminated aquifer; citation_author=Dybas  M.J.; citation_author=Barcelona  M.; citation_author=Bezborodnikov  S.; citation_author=Davies  S.; citation_author=Forney  L.; citation_author=Heuer  H.; citation_author=Kawka  O.; citation_author=Mayotte  T.; citation_author=L  Smalla K.; citation_author=Sneathen  M.; citation_author=Tiedje  J.; citation_author=Voice  T.; citation_author=Wiggert  D.C.; citation_author=Witt  M.E.; citation_author=Criddle  C.S.;  citation_journal_title=Environ. Sci. Technol;  citation_year=1998;  citation_volume=32;  citation_pages=3598-3611; " name="citation_reference"/><meta content="citation_title=Establishment of introduced antagonistic bacteria in the rhizosphere of transgenic potatoes and their effect on the bacterial community; citation_author=Lottmann  J.; citation_author=Heuer  H.; citation_author=Vries  J.; citation_author=Mahn  A.; citation_author=Düring  K.; citation_author=Wackernagel  W.; citation_author=Smalla  K.; citation_author=Berg  G.;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=2000;  citation_volume=33;  citation_pages=41-49; " name="citation_reference"/><meta content="citation_title=The phn genes of Burkholderia sp. strain RP007 constitute a divergent gene cluster for polycyclic aromatic hydrocarbon catabolism; citation_author=Laurie  A.D.; citation_author=Lloyd-Jones  G.;  citation_journal_title=J. Bacteriol;  citation_year=1999;  citation_volume=181;  citation_pages=531-540; " name="citation_reference"/><meta content="citation_title=Molecular detection and diversity of polycyclic aromatic hydrocarbon-degrading bacteria isolated from geographically diverse sites; citation_author=Widada  J.; citation_author=Nojiri  H.; citation_author=Kasuga  K.; citation_author=Yoshida  T.; citation_author=Habe  H.; citation_author=Omori  T.;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2002;  citation_volume=58;  citation_pages=202-209; " name="citation_reference"/><meta content="citation_title=Horizontal transfer of phnAc dioxygenase genes within one of two phenotypically and genotypically distinctive naphthalene-degrading guilds from adjacent soil environments; citation_author=Wilson  M.S.; citation_author=Herrick  J.B.; citation_author=Che Ok Jeon  Hinman D.E.; citation_author=Madsen  E.L.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2003;  citation_volume=69;  citation_pages=2172-2181; " name="citation_reference"/><meta content="citation_title=Quantification of phnAc and nahAc in contaminated New Zealand soils by competitive PCR; citation_author=Laurie  A.D.; citation_author=Lloyd-Jones  G.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2000;  citation_volume=66;  citation_pages=1814-1817; " name="citation_reference"/><meta content="citation_title=Bioavailability of hydrophobic organic contaminants in soils: fundamental concepts and techniques for analysis; citation_author=Semple  K.T.; citation_author=Morriss  A.W.J.; citation_author=Paton  G.I.;  citation_journal_title=Eur. J. Soil Sci;  citation_year=2003;  citation_volume=54;  citation_pages=809-818; " name="citation_reference"/><meta content="citation_title=Molecular analysis of surfactant-driven microbial population shifts in hydrocarbon-contaminated soil; citation_author=Colores  G.M.; citation_author=Macur  R.E.; citation_author=Ward  D.M.; citation_author=Inskeep  W.P.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2000;  citation_volume=66;  citation_pages=2959-2964; " name="citation_reference"/><meta content="citation_title=Isolation of adherent polycyclic aromatic hydrocarbon (PAH)-degrading bacteria using PAH-sorbing carriers; citation_author=Bastiaens  L.; citation_author=Springael  D.; citation_author=Wattiau  P.; citation_author=Harms  H.; citation_author=de Wachter  R.; citation_author=Verachtert  H.; citation_author=Diels  L.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2000;  citation_volume=66;  citation_pages=1834-1843; " name="citation_reference"/><meta content="citation_title=Microcosm enrichment of biphenyl-degrading microbial communities from soils and sediments; citation_author=Wagner-Döbler  I.; citation_author=Bennasar  A.; citation_author=Vancanneyt  M.; citation_author=Strömpl  C.; citation_author=Brümmer  I.; citation_author=Eichner  C.; citation_author=Grammel  I.; citation_author=Moore  E.R.B.;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1998;  citation_volume=64;  citation_pages=3014-3022; " name="citation_reference"/><meta content="" name="citation_fulltext_world_readable"/><meta content="https://academic.oup.com/femsec/article-pdf/54/1/21/18100124/54-1-21.pdf" name="citation_pdf_url"/><meta content="Abstract.  The naphthalene-degrading activity of a Pseudomonas sp. strain isolated from a creosote-contaminated soil was shown to be encoded by the IncP9 plasmi" name="description"/> <link as="script" href="https://adservice.google.ie/adsid/integrator.js?domain=academic.oup.com" rel="preload"/><script src="https://adservice.google.ie/adsid/integrator.js?domain=academic.oup.com" type="text/javascript"></script><link as="script" href="https://adservice.google.com/adsid/integrator.js?domain=academic.oup.com" rel="preload"/><script src="https://adservice.google.com/adsid/integrator.js?domain=academic.oup.com" type="text/javascript"></script><script async="" src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_2019070801.js"></script><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-c09531d6f252ca72ffdadae2d9597c14.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link href="https://tpc.googlesyndication.com/safeframe/1-0-35/html/container.html" rel="prefetch"/><style></style><link href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-c0776af83c8e94e4e2c2ba1716193f9a.css" id="altmetric-embed-css" rel="stylesheet" type="text/css"/><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head><body class="off-canvas pg_Article pg_article" data-sitename="femsmicrobiologyecology" data-sitestyletemplate="Journal" style="cursor: default;" theme-femsec=""><div id="MathJax_Message" style="display: none;"></div><div><div id="cookiepolicy_div"><a href="https://academic.oup.com/#cookiepolicy_data" id="cookiepolicy_link"></a><div id="cookiepolicy_parent" style="display:none"><div id="cookiepolicy_data"></div></div></div></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
<div id="hypothesis-container">
</div>
</div>
<meta content="https://academic.oup.com/femsec/article-xml/54/1/21/565169" name="citation_xml_url"/>
<link href="https://academic.oup.com/femsec/article/54/1/21/565169" rel="canonical"/>
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
<script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/femsec_Behind_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/femsec_Behind_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/femsec_Behind_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/femsec_Behind_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "femsec/article/54/1/21/565169");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
<script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
<noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
<script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&siteid=femsec&authentication_method=IP&authzrequired=false&account_id=2740896&account_list=2740896,2765033,2798346,3680184,4639887&authnips=140.203.154.11&doi=10.1016/j.femsec.2005.02.005';
    </script>
<noscript>
<iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" style="display:none;visibility:hidden" width="0"></iframe>
</noscript>
<a class="skipnav" href="https://academic.oup.com/#main">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5383"/><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"/><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"/><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"/>
<div class="master-container js-master-container">
<section class="master-header row js-master-header vt-site-page-header">
<div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
<div class="ad-banner js-ad-banner-header">
<div class="widget widget-AdBlock widget-instance-HeaderAd">
<div data-google-query-id="CN3F4L7zs-MCFeVYFQgdzq4DWw" id="adBlockHeader" style=" ">
<div id="google_ads_iframe_/116097782/femsec_Behind_Ad1_0__container__" style="border: 0pt none;"><iframe data-google-container-id="1" data-load-complete="true" frameborder="0" height="90" id="google_ads_iframe_/116097782/femsec_Behind_Ad1_0" marginheight="0" marginwidth="0" name="google_ads_iframe_/116097782/femsec_Behind_Ad1_0" scrolling="no" srcdoc="" style="border: 0px; vertical-align: bottom;" title="3rd party ad content" width="728"></iframe></div></div>
</div>
</div>
<div class="oup-header">
<div class="center-inner-row">
<div class="oup-header-logo">
<a href="https://academic.oup.com/"><img alt="Oxford University Press" class="oup-header-image" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg"/></a>
</div>
<ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
<button class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle" href="">
<i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
</button>
</li>
<li class="oup-header-menu-item mobile">
<button class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle" href="">
<i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
</button>
</li>
<li class="oup-header-menu-item mobile">
<button class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle" href="">
<i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
</button>
</li>
<li class="oup-header-menu-item desktop">
<a class="dropdown-toggle institution" href="https://academic.oup.com/">James Hardiman Library, NUI Galway <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
<div class="dropdown-panel dropdown-panel-institution">
<div class="dropdown-menu">
<ul>
<li class="dropdown-institution-item">
<a href="http://www.library.nuigalway.ie">
James Hardiman Library, NUI Galway
</a> </li>
</ul>
</div>
</div>
</li>
<li class="oup-header-menu-item desktop">
<a class="dropdown-toggle signin" href="https://academic.oup.com/" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
</a>
<div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
<div class="spinner"></div>
</div>
</li>
<li class="oup-header-menu-item desktop"><a class="register" href="https://academic.oup.com/my-account/register?siteId=5383&amp;returnUrl=%2ffemsec%2farticle%2f54%2f1%2f21%2f565169">Register</a></li>
<!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->
</ul>
</div>
</div>
<div class="dropdown-panel-wrap">
<div class="dropdown-panel mobile-search-dropdown">
<div class="mobile-search-inner-wrap">
<div class="navbar-search">
<div class="mobile-microsite-search" id="MobileMicrositeSearch">
<label class="screenreader-text" for="mobile-navbar-search-filter">Navbar Search Filter</label>
<select class="navbar-search-filter" id="mobile-navbar-search-filter">
<optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  FEMS Microbiology Ecology</option>
</optgroup>
<optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  FEMS Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option> </optgroup>
</select>
<label class="screenreader-text" for="MobileMicrositeSearchTerm">Mobile Microsite Search Term</label>
<input class="mobile-search-input mobile-microsite-search-term" id="MobileMicrositeSearchTerm" maxlength="255" placeholder="Search" type="text"/>
<span class="mobile-microsite-search-icon mobile-search-submit icon-menu_search" id="MobileMicrositeSearchIcon"></span>
</div>
</div>
</div>
</div>
<div class="dropdown-panel mobile-account-dropdown">
<ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
<a class="nav-link mobile-account-signin dummy-link" href="https://academic.oup.com/" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
</a>
<ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
<div class="spinner"></div>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0">
<a class="nav-link mobile-account-institution dummy-link">
                            James Hardiman Library, NUI Galway
                        </a>
<i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">
<a href="http://www.library.nuigalway.ie">
James Hardiman Library, NUI Galway
</a> </li>
<li class="dropdown-institution-signout">
</li>
</ul>
</li>
<li class="site-menu-item">
<a class="register nav-link" href="https://academic.oup.com/my-account/register?siteId=5383&amp;returnUrl=%2ffemsec%2farticle%2f54%2f1%2f21%2f565169">Register</a>
</li>
</ul>
</div>
<div class="dropdown-panel mobile-nav-dropdown">
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286333">
<a class="nav-link" href="https://academic.oup.com/femsec/issue">Issues</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-100">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/8">Volume 95, Issue 8, August 2019 (In Progress)</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-101">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/7">Volume 95, Issue 7, July 2019 </a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-102">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/8">Browse all</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286329">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">More Content</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286334">
<a class="nav-link" href="https://academic.oup.com/femsec/advance-articles">Advance articles</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286335">
<a class="nav-link" href="https://academic.oup.com/femsec/search-results?page=1&amp;f_OUPSeries=Editor%27s+Choice">Editor's Choice</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286336">
<a class="nav-link" href="https://academic.oup.com/fems-journals/pages/prizes ">Prizes</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286337">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/thematic_issues">Thematic Issues</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286338">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/virtual_issues">Virtual Special Issues</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286330">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">FEMS Journals</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286339">
<a class="nav-link" href="https://academic.oup.com/fems-journals">FEMS Journals</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286340">
<a class="nav-link" href="https://academic.oup.com/femsle">FEMS Microbiology Letters</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286341">
<a class="nav-link" href="https://academic.oup.com/femsre">FEMS Microbiology Reviews</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286342">
<a class="nav-link" href="https://academic.oup.com/femsyr">FEMS Yeast Research</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286343">
<a class="nav-link" href="https://academic.oup.com/femspd">Pathogens and Disease</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286331">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">Submit</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286344">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/instructions_for_authors">Author Guidelines</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286345">
<a class="nav-link" href="http://mc.manuscriptcentral.com/femsec">Submission Site</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286346">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/instructions_for_authors#COPYRIGHT AND LICENCE INCLUDING OPEN ACCESS">Open Access</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286347">
<a class="nav-link" href="https://academic.oup.com/fems-journals/pages/policies">Policies</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286348">
<a class="nav-link" href="https://academic.oup.com/femsec/subscribe">Purchase</a>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286332">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">About</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286349">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/About">About FEMS Microbiology Ecology</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286350">
<a class="nav-link" href="http://fems-microbiology.org/about/">About the Federation of European Microbiological Societies</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286351">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/Editorial_Board">Editorial Board</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286352">
<a class="nav-link" href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286353">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/Policies">Policies</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286354">
<a class="nav-link" href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b">Self-Archiving Policy</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286355">
<a class="nav-link" href="http://fems-microbiology.org/contact/">Contact Us</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286356">
<a class="nav-link" href="https://academic.oup.com/my-account/email-alerts">Alerts</a>
</li>
</ul>
</li>
</ul>
</div>
</div>
<div class="journal-header journal-bg">
<div class="center-inner-row">
<a class="journal-logo-container" href="https://academic.oup.com/femsec">
<img alt="FEMS Microbiology Ecology" class="journal-logo" id="logo-FEMSMicrobiologyEcology" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/femsec/femsec_title168171755.svg"/>
</a>
<div class="society-logo-block">
<div class="society-block-inner-wrap">
<a class="society-logo-container" href="http://www.fems-microbiology.org/" target="">
<img alt="Federation of European Microbiological Societies" class="society-logo" id="logo-FederationofEuropeanMicrobiologicalSocieties" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/femsec/femsec_h1211110869.svg"/>
</a>
</div>
</div>
</div>
</div>
<div class="navbar">
<div class="center-inner-row">
<nav class="navbar-menu">
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286333">
<a class="nav-link" href="https://academic.oup.com/femsec/issue">Issues</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-100">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/8">Volume 95, Issue 8, August 2019 (In Progress)</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-101">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/7">Volume 95, Issue 7, July 2019 </a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-102">
<a class="nav-link" href="https://academic.oup.com/femsec/issue/95/8">Browse all</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286329">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">More Content</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286334">
<a class="nav-link" href="https://academic.oup.com/femsec/advance-articles">Advance articles</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286335">
<a class="nav-link" href="https://academic.oup.com/femsec/search-results?page=1&amp;f_OUPSeries=Editor%27s+Choice">Editor's Choice</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286336">
<a class="nav-link" href="https://academic.oup.com/fems-journals/pages/prizes ">Prizes</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286337">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/thematic_issues">Thematic Issues</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286338">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/virtual_issues">Virtual Special Issues</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286330">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">FEMS Journals</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286339">
<a class="nav-link" href="https://academic.oup.com/fems-journals">FEMS Journals</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286340">
<a class="nav-link" href="https://academic.oup.com/femsle">FEMS Microbiology Letters</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286341">
<a class="nav-link" href="https://academic.oup.com/femsre">FEMS Microbiology Reviews</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286342">
<a class="nav-link" href="https://academic.oup.com/femsyr">FEMS Yeast Research</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286343">
<a class="nav-link" href="https://academic.oup.com/femspd">Pathogens and Disease</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286331">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">Submit</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286344">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/instructions_for_authors">Author Guidelines</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286345">
<a class="nav-link" href="http://mc.manuscriptcentral.com/femsec">Submission Site</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286346">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/instructions_for_authors#COPYRIGHT AND LICENCE INCLUDING OPEN ACCESS">Open Access</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286347">
<a class="nav-link" href="https://academic.oup.com/fems-journals/pages/policies">Policies</a>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286348">
<a class="nav-link" href="https://academic.oup.com/femsec/subscribe">Purchase</a>
</li>
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1286332">
<a class="nav-link dummy-link" href="https://academic.oup.com/femsec/#">About</a>
<i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286349">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/About">About FEMS Microbiology Ecology</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286350">
<a class="nav-link" href="http://fems-microbiology.org/about/">About the Federation of European Microbiological Societies</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286351">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/Editorial_Board">Editorial Board</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286352">
<a class="nav-link" href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286353">
<a class="nav-link" href="https://academic.oup.com/femsec/pages/Policies">Policies</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286354">
<a class="nav-link" href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b">Self-Archiving Policy</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286355">
<a class="nav-link" href="http://fems-microbiology.org/contact/">Contact Us</a>
</li>
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1286356">
<a class="nav-link" href="https://academic.oup.com/my-account/email-alerts">Alerts</a>
</li>
</ul>
</li>
</ul>
</nav>
<div class="navbar-search-container">
<div class="navbar-search-close js_close-navsearch">Close</div>
<div class="navbar-search">
<div class="microsite-search" id="MicrositeSearch">
<label class="screenreader-text" for="navbar-search-filter">search filter</label>
<select class="navbar-search-filter" id="navbar-search-filter">
<optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" selected="selected" value="">All  FEMS Microbiology Ecology</option>
</optgroup>
<optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  FEMS Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option> </optgroup>
</select>
<label class="screenreader-text" for="MicrositeSearchTerm">search input</label>
<input class="navbar-search-input microsite-search-term" id="MicrositeSearchTerm" maxlength="255" title="search input" type="text"/>
<span class="microsite-search-icon navbar-search-submit icon-menu_search" id="MicrositeSearchIcon"></span>
</div>
</div>
<div class="navbar-search-advanced"><a class="advanced-search js-advanced-search" href="https://academic.oup.com/femsec/advanced-search?page=1&amp;fl_SiteID=5383&amp;SearchSourceType=1">Advanced Search</a></div>
</div>
<div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
</div>
</div>
<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"/>
<input id="hfSolrJournalID" name="searchScope" type="hidden" value=""/>
<input id="hfParentSiteName" name="parentSiteName" type="hidden" value="FEMS Journals"/>
<input id="hfparentSiteUrl" name="parentSiteUrl" type="hidden" value="academic.oup.com/fems-journals"/>
<input id="hfsiteID" name="siteID" type="hidden" value="5383"/>
<input id="hfparentSiteID" name="parentSiteID" type="hidden" value="5543"/>
<input id="hfSolrJournalName" type="hidden" value=""/>
<input id="hfSolrMaxAllowSearchChar" type="hidden" value="100"/>
<input id="hfJournalShortName" type="hidden" value=""/>
<input id="hfSearchPlaceholder" type="hidden" value=""/>
<input id="hfGlobalSearchSiteURL" name="hfGlobalSearchSiteURL" type="hidden" value=""/>
<input id="hfjournalSiteScope" name="journalSiteScope" type="hidden" value="Journals"/>
<input id="hfparentSiteScope" name="parentSiteScope" type="hidden" value="Parent"/>
<input id="hfdefaultSearchURL" name="defaultSearchURL" type="hidden" value="search-results?page=1&amp;q="/>
<input id="hfIssueSearch" name="issueSearch" type="hidden" value="&amp;fl_IssueID=22646"/>
<input id="hfIssueSiteScope" name="issueSiteScope" type="hidden" value="Issue"/>
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="/> <input id="hfSiteURL" name="hfSearchSiteURL" type="hidden" value="academic.oup.com/femsec"/>
<script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="femsec"/>
</div>
</section>
<div class="content-main js-main ui-base" id="main">
<section class="master-main row">
<div class="center-inner-row no-overflow">
<div class="page-column-wrap">
<div class="page-column page-column--left js-left-nav-col" id="InfoColumn">
<div class="mobile-content-topbar hide">
<button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
</div>
<div class="info-inner-wrap js-left-nav" style="top: 0px;">
<button class="toggle-left-col__close btn-as-icon icon-general-close">
<span class="screenreader-text">Close mobile search navigation</span>
</button>
<div class="responsive-nav-title">Article Navigation</div>
<div class="info-widget-wrap">
<div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
<div class="article-info-wrap clearfix" id="issueInfo-OUP_IssueInfo_Article">
<i class="icon-general-close mobile-nav-btn nav-open"></i>
<div class="article-issue-img">
<a href="https://academic.oup.com/femsec/issue/54/1"><img alt="Issue Cover" class="fb-featured-image" id="issueImage" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/Issue/54/1/1/m_cover.gif?Expires=1563183447&amp;Signature=aQx9S-DvzfG9fl2ITrd-aRBmYGry5DeZzC9IU-jax9uJERTnRdvoE6mtbTue~7mzeGndmmsuwj23C~TG3M31femzHUEFIrPIUlQw97wU1uVbqhLI4Rbxx21NtCHhjr~5-TcBsG-v2eSQoE3DaxCkk-1HHJ5Y8q3f3esFMvghxiEEfEy8kIKVpEH6~nH7u4yw~K-u2RqjUhwhYdOgk4wGCVqzCKaD6Y4vw79-aqjKDMCTd5Jco2pPyyRK6DxaynLvKCjoLOkudgqc103e6SvOLQ7gIZe4etPO1hO2~ci5qVqRC9elOo14c0OZYllQJi-J2hEKxGnsfAvT4iSibeFigg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/></a>
</div>
<div class="article-issue-info">
<div class="volume-issue__wrap">
<a href="https://academic.oup.com/femsec/issue/54/1">
<div class="volume trailing-comma">Volume 54</div>
<div class="issue">Issue 1</div>
</a>
</div>
<div class="ii-pub-date">
September 2005        </div>
</div>
</div>
</div>
<div class="content-nav">
<div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="90360658">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360658">Abstract</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360660">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360660">1 Introduction</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360664">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360664">2 Materials and methods</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360697">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360697">3 Results</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360720">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360720">4 Discussion</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360730">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360730">Acknowledgements</a>
</div>
</li>
<li class="section-jump-link head-1" link-destination="90360732">
<div class="section-jump-link__link-wrap">
<a class="js-jumplink scrollTo" href="https://academic.oup.com/#90360732">References</a>
</div>
</li>
</ul>
</div>
</div>
<div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
<ul class="inline-list">
<li class="prev arrow">
<a href="https://academic.oup.com/femsec/femsec/article/54/1/13/564304">&lt; Previous</a>
</li>
<li class="next arrow">
<a href="https://academic.oup.com/femsec/article/54/1/21/565169/femsec/article/54/1/35/565313">Next &gt;</a>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div class="page-column page-column--center" id="ContentColumn">
<div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
<div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
<button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
</div>
</div>
<div class="content-inner-wrap">
<div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
<div class="module-widget article-top-widget">
<div class="widget-items">
<h1 class="wi-article-title article-title-main">Effects of the inoculant strain <em>Pseudomonas putida</em> KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community
<i class="icon-availability_free" title="Free"></i> </h1>
<div class="wi-authors">
<div class="al-authors-list">
<span class="al-author-name-more">
<span class="al-author-info-wrap arrow-up" style="">
<div class="info-card-author" id="authorInfo_OUP_ArticleTop_Info_Widget">
<div class="name-role-wrap">
<div class="info-card-name">Newton C.M. Gomes</div>
</div>
<div class="info-card-affilitation">
<div class="aff">Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany</div>
</div>
<div class="info-card-search-label">
            Search for other works by this author on:
        </div>
<div class="info-card-search info-card-search-oxford">
<a href="https://academic.oup.com/femsec/search-results?f_Authors=Newton+C.M.+Gomes">Oxford Academic</a>
</div>
<div class="info-card-search info-card-search-pubmed">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gomes N">PubMed</a>
</div>
<div class="info-card-search info-card-search-google">
<a href='http://scholar.google.com/scholar?q=author:"Gomes Newton C.M."'>Google Scholar</a>
</div>
</div>
</span>
<a class="linked-name" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Newton C.M. Gomes</a>
</span>
<span class="al-author-name-more">
<span class="al-author-info-wrap arrow-up" style="">
<div class="info-card-author" id="authorInfo_OUP_ArticleTop_Info_Widget">
<div class="name-role-wrap">
<div class="info-card-name">Irina A. Kosheleva</div>
</div>
<div class="info-card-affilitation">
<div class="aff">Laboratory of Plasmid Biology, Skryabin Institute of Biochemistry and Physiology of Microorganisms, Russian Academy of Sciences, Pushchino, Moscow region 142290, Russia</div>
</div>
<div class="info-card-search-label">
            Search for other works by this author on:
        </div>
<div class="info-card-search info-card-search-oxford">
<a href="https://academic.oup.com/femsec/search-results?f_Authors=Irina+A.+Kosheleva">Oxford Academic</a>
</div>
<div class="info-card-search info-card-search-pubmed">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosheleva I">PubMed</a>
</div>
<div class="info-card-search info-card-search-google">
<a href='http://scholar.google.com/scholar?q=author:"Kosheleva Irina A."'>Google Scholar</a>
</div>
</div>
</span>
<a class="linked-name" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Irina A. Kosheleva</a>
</span>
<span class="al-author-name-more">
<span class="al-author-info-wrap arrow-up" style="">
<div class="info-card-author" id="authorInfo_OUP_ArticleTop_Info_Widget">
<div class="name-role-wrap">
<div class="info-card-name">Wolf-Rainer Abraham</div>
</div>
<div class="info-card-affilitation">
<div class="aff">Department of Microbiology, GBF-National Research Centre for Biotechnology, 38124 Braunschweig, Germany</div>
</div>
<div class="info-card-search-label">
            Search for other works by this author on:
        </div>
<div class="info-card-search info-card-search-oxford">
<a href="https://academic.oup.com/femsec/article/54/1/21/565169/femsec/search-results?f_Authors=Wolf-Rainer+Abraham">Oxford Academic</a>
</div>
<div class="info-card-search info-card-search-pubmed">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abraham W">PubMed</a>
</div>
<div class="info-card-search info-card-search-google">
<a href='http://scholar.google.com/scholar?q=author:"Abraham Wolf-Rainer"'>Google Scholar</a>
</div>
</div>
</span>
<a class="linked-name" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Wolf-Rainer Abraham</a>
</span>
<span class="al-author-name-more">
<span class="al-author-info-wrap arrow-up" style="">
<div class="info-card-author" id="authorInfo_OUP_ArticleTop_Info_Widget">
<div class="name-role-wrap">
<div class="info-card-name">Kornelia Smalla</div>
</div>
<div class="info-card-affilitation">
<div class="aff">Federal Biological Research Centre for Agriculture and Forestry (BBA), 38104 Braunschweig, Germany</div>
</div>
<div class="info-card-note">
<a class="link link-ref link-reveal xref-default" data-open="cor1" reveal-id="cor1"></a>
</div>
<div class="info-author-correspondence">
<div class="label">*</div>Corresponding author. Tel.: +49 531 299 3814; fax: +49 531 299 3013, <em>E-mail address</em>: <a href="https://academic.oup.com/femsec/article/54/1/21/mailto:k.smalla@bba.de" target="">k.smalla@bba.de</a>
</div>
<div class="info-card-search-label">
            Search for other works by this author on:
        </div>
<div class="info-card-search info-card-search-oxford">
<a href="https://academic.oup.com/femsec/search-results?f_Authors=Kornelia+Smalla">Oxford Academic</a>
</div>
<div class="info-card-search info-card-search-pubmed">
<a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Smalla K">PubMed</a>
</div>
<div class="info-card-search info-card-search-google">
<a href='http://scholar.google.com/scholar?q=author:"Smalla Kornelia"'>Google Scholar</a>
</div>
</div>
</span>
<a class="linked-name" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Kornelia Smalla<i class="icon-general-mail"></i></a>
</span>
</div>
</div>
<div class="pub-history-wrap clearfix">
<div class="pub-history-row clearfix">
<div class="ww-citation-primary"><em>FEMS Microbiology Ecology</em>, Volume 54, Issue 1, September 2005, Pages 21–33, <a href="https://doi.org/10.1016/j.femsec.2005.02.005">https://doi.org/10.1016/j.femsec.2005.02.005</a></div>
</div>
<div class="pub-history-row clearfix">
<div class="ww-citation-date-wrap">
<div class="citation-label">Published:</div>
<div class="citation-date">01 September 2005</div>
</div>
<div class="ww-citation-history-wrap">
<div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
</div>
</div>
</div>
<div class="ww-history">
<div class="history-entry">
<div class="wi-state">Received:</div>
<div class="wi-date">20 October 2004</div>
</div>
<div class="history-entry">
<div class="wi-state">Revision Received:</div>
<div class="wi-date">10 February 2005</div>
</div>
<div class="history-entry">
<div class="wi-state">Accepted:</div>
<div class="wi-date">15 February 2005</div>
</div>
<div class="ww-history-close"><a>Close</a></div>
</div>
</div>
</div>
</div>
<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
<div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
</div>
<div class="article-body">
<div class="toolbar-wrap js-toolbar-wrap">
<div class="toolbar-inner-wrap">
<ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
<a class="al-link pdf article-pdfLink" data-article-id="565169" href="https://academic.oup.com/femsec/article/54/1/femsec/article-pdf/54/1/21/18100124/54-1-21.pdf">
<img alt="pdf" src="https://academic.oup.com/UI/app/svg/pdf.svg"/><span class="pdf-link-text">PDF</span>
</a>
</li>
<li class="toolbar-item item-link item-split-view">
<a class="split-view js-split-view st-split-view" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;" target="">
<i class="icon-menu_split"></i>
                                    Split View
                                </a>
</li>
<li class="toolbar-item item-with-dropdown item-views">
<a aria-haspopup="true" class="drop-trigger" data-dropdown="FilterDrop" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">
<i class="icon-menu_views"></i>
<div class="toolbar-label">
<div class="toolbar-text">Views</div>
<i class="icon-general-arrow-filled-down arrow-icon"></i>
</div>
</a>
<ul aria-label="submenu" class="f-dropdown" data-dropdown-content="" id="ViewsDrop">
<div class="arrow-up" style="display: block;"></div>
<li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
<a href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><span>Article contents</span></a>
</li>
<li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
<a href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><span>Figures &amp; tables</span></a>
</li>
</ul>
</li>
<li class="toolbar-item">
<div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
<a data-reveal="" data-reveal-id="getCitation" href="https://academic.oup.com/#"><i class="icon-read-more"></i><span>Cite</span></a>
<div class="reveal-modal" data-reveal="" id="getCitation">
<div class="modal-title">Citation</div>
<p>Newton C.M. Gomes, Irina A. Kosheleva, Wolf-Rainer Abraham, Kornelia Smalla,  Effects of the inoculant strain <em>Pseudomonas putida</em> KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community, <em>FEMS Microbiology Ecology</em>, Volume 54, Issue 1, September 2005, Pages 21–33, <a href="https://doi.org/10.1016/j.femsec.2005.02.005">https://doi.org/10.1016/j.femsec.2005.02.005</a></p>
<p class="citation-label">Download citation file:</p>
<ul>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
<li><a href="https://academic.oup.com/Citation/Download?resourceId=565169&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
</ul>
<hr/>
<p>© 2019 Oxford University Press</p>
<a class="close-reveal-modal" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
</div>
</li>
<li class="toolbar-item item-tools">
<div class="widget widget-ToolboxPermissions widget-instance-">
<div class="module-widget">
<a class="" href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1574-6941&amp;title=Effects%20of%20the%20inoculant%20strain%20%26lt%3Bem%26gt%3BPseudomonas%20putida%26lt%3B%2Fem%26gt%3B%20KT2442%20(pNF142)%20and%20of%20naphthalene%20contamination%20on%20the%20soil%20bacterial%20community&amp;publicationDate=2005-09-01&amp;volumeNum=54&amp;issueNum=1&amp;author=Gomes%2C%20Newton%20C.M.%3B%20Kosheleva%2C%20Irina%20A.&amp;startPage=21&amp;endPage=33&amp;contentId=10.1016%2Fj.femsec.2005.02.005&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" target="">
<i class="icon-menu_permissions">
<span class="screenreader-text">Permissions Icon</span>
</i>
            Permissions
        </a>
</div>
</div>
</li>
<li class="toolbar-item item-with-dropdown item-share last">
<a aria-haspopup="true" class="drop-trigger" data-dropdown="ShareDrop" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">
<i class="icon-menu_share"></i>
<div class="toolbar-label">
<div class="toolbar-text">Share</div>
<i class="icon-general-arrow-filled-down arrow-icon"></i>
</div>
</a>
<ul aria-label="submenu" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" id="ShareDrop">
<div class="arrow-up"></div>
<li><a class="addthis_button_email at300b" href="https://academic.oup.com/#" target="" title="Email"><span>Email</span></a></li>
<li><a class="addthis_button_twitter at300b" href="https://academic.oup.com/#" title="Twitter"><span>Twitter</span></a></li>
<li><a class="addthis_button_facebook at300b" href="https://academic.oup.com/#" title="Facebook"><span>Facebook</span></a></li>
<li><a class="addthis_button_compact at300m" href="https://academic.oup.com/#"><span>More</span></a></li>
<div class="atclear"></div></ul>
</li>
</ul>
<div class="toolbar-search">
<div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
<div class="dropdown-panel-wrap">
<div class="dropdown-panel mobile-search-dropdown">
<div class="mobile-search-inner-wrap">
<div class="navbar-search">
<div class="mobile-microsite-search" id="MobileMicrositeSearch">
<label class="screenreader-text" for="mobile-navbar-search-filter">Navbar Search Filter</label>
<select class="navbar-search-filter" id="mobile-navbar-search-filter">
<optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  FEMS Microbiology Ecology</option>
</optgroup>
<optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  FEMS Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option> </optgroup>
</select>
<label class="screenreader-text" for="MobileMicrositeSearchTerm">Mobile Microsite Search Term</label>
<input class="mobile-search-input mobile-microsite-search-term" id="MobileMicrositeSearchTerm" maxlength="255" placeholder="Search" type="text"/>
<span class="mobile-microsite-search-icon mobile-search-submit icon-menu_search" id="MobileMicrositeSearchIcon"></span>
</div>
</div>
</div>
</div>
<div class="dropdown-panel mobile-account-dropdown">
<ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
<a class="nav-link mobile-account-signin dummy-link" href="https://academic.oup.com/" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
</a>
<ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
<div class="spinner"></div>
</li>
</ul>
</li>
<li class="site-menu-item site-menu-lvl-0">
<a class="nav-link mobile-account-institution dummy-link">
                            James Hardiman Library, NUI Galway
                        </a>
<i class="mobile-nav-arrow icon-general_arrow-down"></i>
<ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">
<a href="http://www.library.nuigalway.ie">
James Hardiman Library, NUI Galway
</a> </li>
<li class="dropdown-institution-signout">
</li>
</ul>
</li>
<li class="site-menu-item">
<a class="register nav-link" href="https://academic.oup.com/my-account/register?siteId=5383&amp;returnUrl=%2ffemsec%2farticle%2f54%2f1%2f21%2f565169">Register</a>
</li>
</ul>
</div>
<div class="dropdown-panel mobile-nav-dropdown">
</div>
</div>
<div class="navbar">
<div class="center-inner-row">
<nav class="navbar-menu">
</nav>
<div class="navbar-search-container">
<div class="navbar-search-close js_close-navsearch">Close</div>
<div class="navbar-search">
<div class="microsite-search" id="MicrositeSearch">
<label class="screenreader-text" for="navbar-search-filter">search filter</label>
<select class="navbar-search-filter" id="navbar-search-filter">
<optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" selected="selected" value="">All  FEMS Microbiology Ecology</option>
</optgroup>
<optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  FEMS Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option> </optgroup>
</select>
<label class="screenreader-text" for="MicrositeSearchTerm">search input</label>
<input class="navbar-search-input microsite-search-term" id="MicrositeSearchTerm" maxlength="255" title="search input" type="text"/>
<span class="microsite-search-icon navbar-search-submit icon-menu_search" id="MicrositeSearchIcon"></span>
</div>
</div>
<div class="navbar-search-advanced"><a class="advanced-search js-advanced-search" href="https://academic.oup.com/femsec/advanced-search?page=1&amp;fl_SiteID=5383&amp;SearchSourceType=1">Advanced Search</a></div>
</div>
<div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
</div>
</div>
<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"/>
<input id="routename" name="RouteName" type="hidden" value="femsec"/>
</div>
</div>
</div>
</div>
<div class="content active" id="ContentTab">
<div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
<div class="module-widget">
<div class="widget-items" data-widgetname="ArticleFulltext">
<h2 class="abstract-title" id="90360658" scrollto-destination="90360658">Abstract</h2>
<section class="abstract"><p>The naphthalene-degrading activity of a <em>Pseudomonas</em> sp. strain isolated from a creosote-contaminated soil was shown to be encoded by the IncP9 plasmid pNF142 by transfer to <em>Pseudomonas putida</em> KT2442. The effects of the inoculant strain KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community were studied in microcosms with the following treatments: (I) soil, (II) soil with naphthalene, (III) soil with naphthalene and inoculated with KT2442 (pNF142). The inoculant became the dominant bacterial population in treatment (III) as evidenced by cultivation and denaturing gradient gel electrophoresis (DGGE) analysis. The bacterial DGGE profiles revealed drastically reduced complexity due to the numerical dominance of the inoculant. However, group-specific fingerprints (<em>β</em>-proteobacteria, actinobacteria) that excluded KT2442 (pNF142) showed less severe changes in the bacterial community patterns. A major effect of naphthalene on the soil bacterial community was observed in treatment (II) after 21 days. Two dominant bands appeared whose sequences showed the highest similarity to those of <em>Burkholderia</em> sp. RP007 and <em>Nocardia vinaceae</em> based on 16S rRNA gene sequencing. These bands were less intense in treatment (III). The increased abundance of RP007-like populations due to naphthalene contamination was also confirmed by PCR amplification of the <em>phnAc</em> gene. The <em>nahAc</em> and <em>nahH</em> genes were detected in DNA and cDNA only in treatment III. Although the inoculant strain KT2442 (pNF142) showed good survival and expression of genes involved in naphthalene degradation, this study suggests that KT2442 (pNF142) suppressed the enrichment of indigenous naphthalene degraders.</p></section> <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" data-keyword='"Pseudomonas putida KT2442 (pNF142)"' href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><em>Pseudomonas putida</em> KT2442 (pNF142)</a>, <a class="kwd-part kwd-main" data-keyword='"Naphthalene degradation"' href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Naphthalene degradation</a>, <a class="kwd-part kwd-main" data-keyword='"Denaturing gradient gel electrophoresis"' href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Denaturing gradient gel electrophoresis</a>, <a class="kwd-part kwd-main" data-keyword='"Dioxygenase genes"' href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Dioxygenase genes</a></div><div class="related-topic-tags">
<div class="related-topic-tag-label">Topic:</div>
<ul class="related-topic-tag-list">
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=polymerase chain reaction"><span>polymerase chain reaction</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=dna"><span>dna</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=genes"><span>genes</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=naphthalenes"><span>naphthalenes</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=pseudomonas putida"><span>pseudomonas putida</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=rna, ribosomal, 16s"><span>rna, ribosomal, 16s</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=soil"><span>soil</span></a>
</li>
<li>
<a href="https://academic.oup.com/femsec/search-results?f_SemanticFilterTopics=community"><span>community</span></a>
</li>
</ul>
</div><div class="article-metadata-tocSections">
<div class="article-metadata-tocSections-title">Issue Section:</div>
<a href="https://academic.oup.com/femsec/search-results?f_TocHeadingTitle=Articles">Articles</a>
</div></div>
<h2 class="section-title" data-legacy-id="ss1" id="90360660" scrollto-destination="90360660">1 Introduction</h2>
<p>The exploitation of microbes for cleaning contaminated sites is still one of the big challenges to environmental microbiologists. The efficient and reliable use of microbes carrying degradative pathways often suffers from a lack of information on their survival and metabolic activity under different environmental conditions, their potential effects on the indigenous microbial community and knowledge of the bioavailability of the pollutants. Because microbial degradation processes under environmental conditions result from enzyme, cell and community-based activities, a better understanding of these processes requires a polyphasic approach. DNA- and RNA-based methods enable researchers not only to follow the fate and activity of inoculant strains but also to evaluate potential shifts in the composition of indigenous bacteria during bioremediation processes. These shifts might result directly from the introduction of inoculant strains or consortia, or indirectly from changes in the concentration of pollutants, their degradation metabolites and their bioavailability. Recently developed molecular fingerprinting techniques based on the analysis of ribosomal RNA fragments amplified from cDNA or RNA are particularly suited to follow changes in the relative abundance of microbial populations during bioremediation processes or in response to the exposure to changing concentrations of pollutants [<a class="link link-ref link-reveal xref-bibr" data-open="b1" reveal-id="b1">1</a>]. The detection of mRNA transcripts of degradative genes, although still a methodological challenge, offers the possibility of measuring in situ activity. This approach has recently been successfully applied to follow in situ, real #-time gene expression by characterising naphthalene dioxygenase mRNA transcripts from groundwater [<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>].</p><p>Because the success of bioremediation processes facilitated by microbial inoculant strains does not only depend on the ability of the inoculant strain to actively colonise the site but also on the expression of the degradative genes, most studies focussed on these aspects. The major aim of this study was to assess the response of indigenous soil bacteria to naphthalene either in the presence or absence of the inoculant strain KT2442 (pNF142).</p><p>The naphthalene-degrading activity of <em>Pseudomonas</em> sp. NF142 isolated from creosote-contaminated soil samples collected from a chemical plant in the Nizhnii Novgorod region (Russia) was shown to be encoded by the plasmid pNF142 by transferring it to <em>Pseudomonas putida</em> KT2442. Soil microcosm experiments were performed to provide information on the ability of <em>P. putida</em> KT2442 (pNF142) to survive in naphthalene-contaminated soil and to study the effect of its inoculation and of artificial naphthalene contamination on the soil bacterial communities. Soil samples were taken at five time points and analysed by selective cultivation and DNA/RNA-based methods. Community-level responses were mainly followed by analysing 16S rRNA gene fragments PCR-amplified from soil DNA or RNA by means of denaturing gradient gel electrophoresis (DGGE). Functional analysis focussed on the presence and expression of dioxygenase genes followed by PCR amplification from soil DNA or cDNA.</p> <h2 class="section-title" data-legacy-id="ss2" id="90360664" scrollto-destination="90360664">2 Materials and methods</h2>
<h3 class="section-title" data-legacy-id="ss2-1" id="90360665" scrollto-destination="90360665">2.1 Isolation of the naphthalene-degrading strain Pseudomonas sp. NF142 and transfer of plasmid pNF142 to P. putida KT2442</h3>
<p>The naphthalene-degrading isolate NF142 was obtained after selective enrichment with naphthalene from a creosote-contaminated soil collected from a territory of the chemical plant Metoxil in the Nizhnii Novgorod region in 2001. Partial 16S rRNA gene sequencing of strain NF142 indicated that its 16S rRNA gene was identical with that of <em>Pseudomonas</em> sp. Cam-1 (AF098464), a polychlorinated biphenyl-degrading strain from arctic soil [<a class="link link-ref link-reveal xref-bibr" data-open="b3" reveal-id="b3">3</a>]. The naphthalene-degrading activity of <em>Pseudomonas</em> sp. NF142 was shown to be carried on plasmid pNF142 by transferring it to <em>P. putida</em> KT2442. Plasmid pNF142 was assigned to the IncP9-δ group by PCR as described by Krasowiak et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b4" reveal-id="b4">4</a>].</p> <h3 class="section-title" data-legacy-id="ss2-2" id="90360667" scrollto-destination="90360667">2.2 Setup of microcosms</h3>
<p>The soil used in the experiments was composed of a sieved (&lt;2 mm) mixture of sand (20%, v/v) and commercially available potting soil (clay substrate of Klasmann-Deilmann GmbH –<a class="link link-uri" href="http://www.klasmann-deilmann.de/" target="">http://www.klasmann-deilmann.de/</a> with the following characteristics (w/v): 160–260 mg l<sup>−1</sup> N, 180–280 mg l<sup>−1</sup> P<sub>2</sub>O<sub>5</sub>, 200–350 mg l<sup>−1</sup> K<sub>2</sub>O, 80–150 mg l<sup>−1</sup> MgO, organic matter 125 g l<sup>−1</sup> and pH 5.5–6.0). Three different soil treatments were analysed: (I) control soil, (II) soil spiked with naphthalene (2 mg g<sup>−1</sup>), and (III) soil spiked with naphthalene (2 mg g<sup>−1</sup>) and inoculated with KT2442 (pNF142). The soil contamination was done by thoroughly mixing naphthalene crystals (2 mg g<sup>−1</sup>) into the soil. Before inoculation KT2442 (pNF142) was grown in Luria-Bertani (LB) broth for 24 h and then re-suspended in sterile saline (0.85%). The soil inoculation was performed by adding 1 ml of cell suspension per each 10 g of soil followed by careful mixing to a resulting concentration of 1 × 10<sup>8</sup> cfu g<sup>−1</sup> of dry soil. The same volume of sterile saline was added to non-inoculated soils to standardise the final moisture content (34%) in each microcosm. The microcosms consisted of Falcon tubes of 50 ml containing 20 g (wet weight) of each soil. The flasks were kept partially open to avoid oxygen deprivation. Three entire microcosms were sacrificed per treatment after 1 h (referred to as day 0) as well as 1, 3, 11 and 21 days post-inoculation (dpi) for analysis. Five grams of soil were re-suspended in 45 ml sterile saline and treated mechanically by means of a Stomacher laboratory blender at maximum speed for 2 min. The remaining soil samples were kept frozen at −80 °C until processed for DNA/RNA extraction and naphthalene analysis by high-performance liquid chromatography (HPLC).</p> <h3 class="section-title" data-legacy-id="ss2-3" id="90360669" scrollto-destination="90360669">2.3 HPLC analysis</h3>
<p>The concentration of naphthalene in the soil microcosms from each time interval was determined after hexane extraction via HPLC analysis (Waters HPLC dual gradients system, equipped with diode array and fluorescence detectors), fitted with a Varian ChromSpher 3PAH column (100 × 4.6 mm, 3 μm). Acetonitrile and water were used as the mobile phase (acetonitrile:water 40:60 to 100:0 in 20 min, then 5 min acetonitrile).</p> <h3 class="section-title" data-legacy-id="ss2-4" id="90360671" scrollto-destination="90360671">2.4 Cfu counts of P. putida KT2442 (pNF142) in naphthalene-contaminated soil</h3>
<p>Serial dilutions of the cell suspension obtained by Stomacher treatment were plated onto King’ s B agar supplemented with rifampicin at 50 μg ml<sup>−1</sup>. The cfu counts were determined after 48 h incubation at 28 °C.</p> <h3 class="section-title" data-legacy-id="ss2-5" id="90360673" scrollto-destination="90360673">2.5 Extraction of RNA and DNA</h3>
<p>Two-ml Eppendorf tubes were filled with 0.5 g of microcosm soil sample and 0.2 g of baked sterile glass beads and kept on ice until simultaneous DNA and RNA extraction as described by Hurt et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b5" reveal-id="b5">5</a>] with the modifications made by Costa et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b6" reveal-id="b6">6</a>]. The separation of RNA from DNA was performed using the Qiagen<sup>®</sup> RNA/DNA Mini Kit – 25 (Qiagen GmbH) according to the manufacturer’ s protocol. The kits UltraClean™ 15 (Mobio) and the RNeasy Mini Kit (Qiagen GmbH) were used to purify the recovered DNA and RNA, respectively, according to the manufacturer’ s instructions.</p> <h3 class="section-title" data-legacy-id="ss2-6" id="90360675" scrollto-destination="90360675">2.6 Preparation of cDNA from soil RNA</h3>
<p>The reverse transcriptase (RT) reaction mixtures (20 μl) containing 50 mM Tris–HCl (pH 8.3), 75 mM KCl, 3 mM MgCl<sub>2</sub>, 10 mM dithiothreitol, deoxynucleoside triphosphates, each at a concentration of 0.5 mM, 250 ng of random primers, and 50 U of Superscript II RNase H-reverse transcriptase (Life Technologies) were used to reverse transcribe purified RNA extracted from soil samples. The reactions were performed for 50 min at 42 °C, followed by incubation at 70 °C for 15 min to inactivate the reverse transcriptase. RT products were kept frozen at −20 °C until use. The suitability of the extracted RNA for RT-PCR amplification was checked by performing RT-PCR control experiments with 16S rRNA. To check any contamination with DNA in the RNA extracts for RT-PCR, controls were prepared consisting of PCR mixtures that contained RNA and were not previously subjected to RT reaction.</p> <h3 class="section-title" data-legacy-id="ss2-7" id="90360677" scrollto-destination="90360677">2.7 Naphthalene dioxygenase (nahAc, phnAc) and catechol 2,3-dioxygenase (nahH) PCR</h3>
<p>PCR amplification of the genes <em>nahAc</em>, <em>phnAc</em> and <em>nahH</em> were performed with primer sets described by Wilson et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>], Lloyd-Jones et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] and Wikström et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>], respectively (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>). The reaction mixtures contained: 1 μl template DNA (approx. 20 ng) or 2 μl of the cDNA reaction mixture, Stoffel buffer II (Applied Biosystems), 0.2 mM dNTPs, 3.75 mM MgCl<sub>2</sub>, 0.25 μg bovine serum albumin, 100 nM of the forward primer, 100 nM of reverse primer and 1 U AmpliTaq GoldTM Polymerase (Applied Biosystem) in each 25 μl reaction. A touchdown PCR programme was used for amplification of <em>nahAC</em> gene homologues as described by Wilson et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>]. For <em>phnAc</em> amplification, an initial 5 min denaturation at 94 °C was followed by 35 thermal cycles of 45 s at 94 °C, 30 s at 65 °C and 60 s at 72 °C, followed by a final extension step at 72 °C for 4 min according to Lloyd-Jones et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>]. The <em>nahH</em> PCR consisted of an initial 5 min of denaturation at 94 °C followed by 35 cycles of 30 s at 94 °C, 30 s at 52 °C and 45 s at 72 °C, followed by a final extension step at 72 °C for 4 min.</p> <a id="90360679" scrollto-destination="90360679"></a>
<div class="table-modal" content-id="t1" style="display: block;"><div class="table-wrap"><div class="table-wrap-title" data-id="t1" id="t1"><div class="label">1</div><div class="caption"><p>Primers used in PCR experiments</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>Primer<sup>a</sup> </td><td>Target </td><td>Sequence (5′ 3′) </td><td>Reference </td></tr></thead><tbody><tr><td>F984-GC<sup>b</sup> </td><td>Bacteria – 16S rRNA </td><td>AACGCGAAGAACCTTAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1378 </td><td>Bacteria – 16S rRNA </td><td>CGGTGTGTACAAGGCCCGGGAACG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1492 </td><td>Bacteria – 16S rRNA </td><td>TACGG(C/T)TACCTTGTTACGACTT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>F948β </td><td>β-proteobacteria – 16S rRNA </td><td>CGCACAAGCGGTGGATGA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>] </td></tr><tr><td>F243HGC </td><td>Actinobacteria – 16S rRNA </td><td>GGATGAGCCCGCGGCCTA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>Ac114F </td><td>NDO<sup>c</sup>–<em>nahAc</em> </td><td>CTGGC(T/A)(T/A)TT(T/C)CTCAC(T/C)CAT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>Ac596R </td><td>NDO –<em>nahAc</em> </td><td>C(G/A)GGTG(C/T)CTTCCAGTTG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>P8073 </td><td>NDO –<em>phnAc</em> </td><td>TTCGAGCTGGAATGTGAGC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>P9047 </td><td>NDO –<em>phnAc</em> </td><td>AATAACCGGCGATTCCAAAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>23DOF </td><td>Catechol 2,3-dioxygenase </td><td>ATGGAT(AGT)T(AGT)ATGGG(AGT)TTCAAGGT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr><tr><td>23DOR </td><td>Catechol 2,3-dioxygenase </td><td>AC(AGT)GTCA(AGT)GAA(AGT)CG(AGT)TCGTTGAG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr></tbody></table></div><div class="table-modal" content-id="t1"><table><thead align="" name="thead"><tr><td>Primer<sup>a</sup> </td><td>Target </td><td>Sequence (5′ 3′) </td><td>Reference </td></tr></thead><tbody><tr><td>F984-GC<sup>b</sup> </td><td>Bacteria – 16S rRNA </td><td>AACGCGAAGAACCTTAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1378 </td><td>Bacteria – 16S rRNA </td><td>CGGTGTGTACAAGGCCCGGGAACG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1492 </td><td>Bacteria – 16S rRNA </td><td>TACGG(C/T)TACCTTGTTACGACTT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>F948β </td><td>β-proteobacteria – 16S rRNA </td><td>CGCACAAGCGGTGGATGA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>] </td></tr><tr><td>F243HGC </td><td>Actinobacteria – 16S rRNA </td><td>GGATGAGCCCGCGGCCTA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>Ac114F </td><td>NDO<sup>c</sup>–<em>nahAc</em> </td><td>CTGGC(T/A)(T/A)TT(T/C)CTCAC(T/C)CAT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>Ac596R </td><td>NDO –<em>nahAc</em> </td><td>C(G/A)GGTG(C/T)CTTCCAGTTG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>P8073 </td><td>NDO –<em>phnAc</em> </td><td>TTCGAGCTGGAATGTGAGC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>P9047 </td><td>NDO –<em>phnAc</em> </td><td>AATAACCGGCGATTCCAAAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>23DOF </td><td>Catechol 2,3-dioxygenase </td><td>ATGGAT(AGT)T(AGT)ATGGG(AGT)TTCAAGGT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr><tr><td>23DOR </td><td>Catechol 2,3-dioxygenase </td><td>AC(AGT)GTCA(AGT)GAA(AGT)CG(AGT)TCGTTGAG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="t1_note48"><div class="fn"><p><sup>a</sup>F, forward primer; R, reverse primer.</p></div></div><div content-id="t1_note49"><div class="fn"><p><sup>b</sup>GC clamp [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>].</p></div></div><div content-id="t1_note50"><div class="fn"><p><sup>c</sup>NDO, naphthalene dioxygenase.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360679" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div class="table-wrap-title" data-id="t1" id="t1"><div class="label">1</div><div class="caption"><p>Primers used in PCR experiments</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>Primer<sup>a</sup> </td><td>Target </td><td>Sequence (5′ 3′) </td><td>Reference </td></tr></thead><tbody><tr><td>F984-GC<sup>b</sup> </td><td>Bacteria – 16S rRNA </td><td>AACGCGAAGAACCTTAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1378 </td><td>Bacteria – 16S rRNA </td><td>CGGTGTGTACAAGGCCCGGGAACG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1492 </td><td>Bacteria – 16S rRNA </td><td>TACGG(C/T)TACCTTGTTACGACTT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>F948β </td><td>β-proteobacteria – 16S rRNA </td><td>CGCACAAGCGGTGGATGA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>] </td></tr><tr><td>F243HGC </td><td>Actinobacteria – 16S rRNA </td><td>GGATGAGCCCGCGGCCTA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>Ac114F </td><td>NDO<sup>c</sup>–<em>nahAc</em> </td><td>CTGGC(T/A)(T/A)TT(T/C)CTCAC(T/C)CAT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>Ac596R </td><td>NDO –<em>nahAc</em> </td><td>C(G/A)GGTG(C/T)CTTCCAGTTG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>P8073 </td><td>NDO –<em>phnAc</em> </td><td>TTCGAGCTGGAATGTGAGC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>P9047 </td><td>NDO –<em>phnAc</em> </td><td>AATAACCGGCGATTCCAAAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>23DOF </td><td>Catechol 2,3-dioxygenase </td><td>ATGGAT(AGT)T(AGT)ATGGG(AGT)TTCAAGGT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr><tr><td>23DOR </td><td>Catechol 2,3-dioxygenase </td><td>AC(AGT)GTCA(AGT)GAA(AGT)CG(AGT)TCGTTGAG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr></tbody></table></div><div class="table-modal" content-id="t1"><table><thead align="" name="thead"><tr><td>Primer<sup>a</sup> </td><td>Target </td><td>Sequence (5′ 3′) </td><td>Reference </td></tr></thead><tbody><tr><td>F984-GC<sup>b</sup> </td><td>Bacteria – 16S rRNA </td><td>AACGCGAAGAACCTTAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1378 </td><td>Bacteria – 16S rRNA </td><td>CGGTGTGTACAAGGCCCGGGAACG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>R1492 </td><td>Bacteria – 16S rRNA </td><td>TACGG(C/T)TACCTTGTTACGACTT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>F948β </td><td>β-proteobacteria – 16S rRNA </td><td>CGCACAAGCGGTGGATGA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>] </td></tr><tr><td>F243HGC </td><td>Actinobacteria – 16S rRNA </td><td>GGATGAGCCCGCGGCCTA </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] </td></tr><tr><td>Ac114F </td><td>NDO<sup>c</sup>–<em>nahAc</em> </td><td>CTGGC(T/A)(T/A)TT(T/C)CTCAC(T/C)CAT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>Ac596R </td><td>NDO –<em>nahAc</em> </td><td>C(G/A)GGTG(C/T)CTTCCAGTTG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] </td></tr><tr><td>P8073 </td><td>NDO –<em>phnAc</em> </td><td>TTCGAGCTGGAATGTGAGC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>P9047 </td><td>NDO –<em>phnAc</em> </td><td>AATAACCGGCGATTCCAAAC </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>] </td></tr><tr><td>23DOF </td><td>Catechol 2,3-dioxygenase </td><td>ATGGAT(AGT)T(AGT)ATGGG(AGT)TTCAAGGT </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr><tr><td>23DOR </td><td>Catechol 2,3-dioxygenase </td><td>AC(AGT)GTCA(AGT)GAA(AGT)CG(AGT)TCGTTGAG </td><td>[<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>] </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="t1_note48"><div class="fn"><p><sup>a</sup>F, forward primer; R, reverse primer.</p></div></div><div content-id="t1_note49"><div class="fn"><p><sup>b</sup>GC clamp [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>].</p></div></div><div content-id="t1_note50"><div class="fn"><p><sup>c</sup>NDO, naphthalene dioxygenase.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360679" target="_blank">View Large</a></div></div> <h3 class="section-title" data-legacy-id="ss2-8" id="90360680" scrollto-destination="90360680">2.8 Southern blotting hybridisation</h3>
<p>Probes were generated from specific <em>nahAc</em> and <em>nahH</em> PCR of <em>P. putida</em> KT2442 (pNF142) by labelling the PCR products, excised from the agarose gel, using DIG-labelled dUTP as recommended by the manufacturer (Roche). Southern blotting was done according to Sambrook et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b9" reveal-id="b9">9</a>]. PCR products were electrophoresed on agarose gels and subsequently transferred onto HYBOND N nylon membranes (Amersham-Pharmacia Biotech). Hybridisation was performed under conditions of medium stringency following the protocol published by Fulthorpe et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b10" reveal-id="b10">10</a>]. Hybridised probes were detected by using a DIG luminescent detection kit (Roche) as specified by the manufacturer and exposure to X-ray film (Roche).</p> <h3 class="section-title" data-legacy-id="ss2-9" id="90360682" scrollto-destination="90360682">2.9 PCR amplification of 16S rRNA gene fragments for DGGE analysis</h3>
<p>Primers F984-GC and R1378 described by Heuer et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>) were used for amplification of bacterial 16S rRNA gene fragments (473 bp). The PCR mixture was: 1 μl template DNA (ca. 20 ng), Stoffel buffer (Applied Biosystems), 0.2 mM dNTPs, 3.75 mM MgCl<sub>2</sub>, 4% (w/v) acetamide, 100 nM F984-GC, 100 nM R1378, and 1 U <em>Taq</em> DNA polymerase (Stoffel fragment, Applied Biosystems) in each 25 μl reaction. After 5 min of denaturation at 94 °C, 35 thermal cycles of 1 min at 94 °C, 1 min at 53 °C, and 2 min at 72 °C, the PCR was finished by an extension step at 72 °C for 10 min.</p> <h3 class="section-title" data-legacy-id="ss2-10" id="90360684" scrollto-destination="90360684">2.10 Group-specific PCR of 16S rRNA gene fragments</h3>
<p>The forward primers for β-proteobacteria F948β and actinobacteria F243HGC were used in combination with the reverse universal bacterial primers R1494 and R1378, respectively (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>). The reaction mixture was: 1 μl template DNA (ca. 20 ng), Stoffel buffer II (Applied Biosystems), 0.2 mM dNTPs, 3.75 mM MgCl<sub>2</sub>, 100 nM of the forward group-specific primer, 100 nM of reverse bacterial primer and 1 U AmpliTaq Gold™ Polymerase (Applied Biosystem) in each 25 μl reaction. After 5 min of denaturation at 94 °C, 20 thermal cycles of 1 min at 94 °C, 2 min at 64 °C (β-proteobacteria) or 1 min at 63 °C (actinobacteria) and 2 min at 72 °C, the PCR was followed by a final extension step at 72 °C for 10 min. Bovine serum albumin (0.1 μg μl<sup>−1</sup>) was added to all group-specific PCRs. The amplicons obtained with group-specific primers without GC clamp were diluted 1:20 and used as template for a second PCR with bacterial primers F984-GC and R1378 [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>] (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>).</p> <h3 class="section-title" data-legacy-id="ss2-11" id="90360686" scrollto-destination="90360686">2.11 DGGE analysis</h3>
<p>DGGE of the amplified rRNA gene sequences was performed using the Dcode System (Universal Mutation Detection System, Bio-Rad). Amplified bacterial 16S rRNA gene fragments were applied to a double gradient polyacrylamide gel containing 6–9% acrylamide [<a class="link link-ref link-reveal xref-bibr" data-open="b12" reveal-id="b12">12</a>] with a gradient of 26–59% of denaturant. The run was performed in 1 × Tris–acetate–EDTA buffer at 58 °C at a constant voltage of 220 V for 6.5 h. The DGGE gels were silver-stained according to Heuer et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>].</p> <h3 class="section-title" data-legacy-id="ss2-12" id="90360688" scrollto-destination="90360688">2.12 Cloning and sequencing</h3>
<p>Selected DGGE bands were excised with a scalpel and transferred into 1.5 ml Eppendorf tubes containing 500 μl of destaining solution (6 mM K<sub>3</sub>Fe(CN]<sub>6</sub>) and incubated at 37 °C in a heating block (Thermomixer, Eppendorf) for 10 min. The destaining solution was discarded and the gel slices washed twice with MilliQ water at 37 °C for 5 min each time. The isolation of DNA fragments from destained gel slices was performed as described by Schwieger and Tebbe [<a class="link link-ref link-reveal xref-bibr" data-open="b14" reveal-id="b14">14</a>]. The recovered DNA was re-amplified and the PCR product was again subjected to DGGE analysis to ensure correct electrophoretic mobility within the gels. The amplification of 16S rRNA gene fragments from excised DGGE bands, cloning of the PCR products and clone screening were performed as described by Smalla et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b15" reveal-id="b15">15</a>].</p><p>Cloning of PCR products amplified with primers targeting the <em>phnAc</em> gene was performed by using the pGEM-T Easy Vector System I (Promega Corporation) according to the manufacturer’ s instructions. The <em>phnAc</em>-PCR was performed again directly from the selected white colonies, and the PCR products were analysed by agarose gel electrophoresis to confirm the presence of the insert in the transformants. PCR products were digested with <em>Msp</em>I (New England Biolabs) and analysed in 4% agarose gels (4% NuSieve<sup>®</sup> 3:1 agarose, Cambrex). Clones representing each group of <em>Msp</em>I restriction enzyme patterns were sequenced.</p><p>Cloned DNA fragments were sequenced with the standard primers SP6 and T7 (IIT GmbH). 16S rRNA gene sequences were aligned and analysed with ARB software (Department of Microbiology, Technical University of Munich, München, Germany (<a class="link link-uri" href="http://www.arb-home.de" target="">http://www.arb-home.de</a>]). The <em>phnAc</em> sequences were aligned and the similarities were determined by using the unweighted pair group method using arithmetic averages (UPGMA) included in the GeneCompar software (version 1.3, Applied Maths).</p> <h3 class="section-title" data-legacy-id="ss2-13" id="90360692" scrollto-destination="90360692">2.13 Cluster analysis</h3>
<p>Analysis of bacterial community profiles in the soil microcosms was performed with the software package Gelcompar 4.0 program (Applied Maths). Background was subtracted by using a rolling disk method with an intensity of 10 (relative units), and the lanes were normalised. A dendrogram was constructed by the Pearson correlation index for each pair of lanes within a gel and cluster analysis by UPGMA.</p> <h3 class="section-title" data-legacy-id="ss2-14" id="90360694" scrollto-destination="90360694">2.14 Nucleotide sequence accession numbers</h3>
<p>The 16S rRNA gene nucleotide sequence accession numbers are listed in <a class="link link-reveal link-table xref-fig" data-open="t3" reveal-id="t3">Table 3</a>. The <em>phnAc</em> sequences are available under GenBank Accession Nos. AY540615–AY540620.</p> <a id="90360696" scrollto-destination="90360696"></a>
<div class="table-modal" content-id="t3" style="display: block;"><div class="table-wrap"><div class="table-wrap-title" data-id="t3" id="t3"><div class="label">3</div><div class="caption"><p>Results of partial sequence analysis and tentative phylogenetic affiliations of bands</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>DGGE band </td><td>Sequenced clone and accession number<sup>a</sup> </td><td>Most closely related bacterial 16S rRNA gene sequence </td><td>% Identity </td><td>Accession number<sup>b</sup> </td></tr></thead><tbody><tr><td>A </td><td>A1/A2 – AY376449<sup>c</sup> </td><td><em>Burkholderia</em> sp. RP007 </td><td>99.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A3 – AY376448 </td><td><em>Burkholderia</em> sp. RP007 </td><td>98.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A4 – AY376447 </td><td><em>Burkholderia</em> sp. RP007 </td><td>97.0% </td><td>AF061872 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>B1/B2/B3/B4 – AY376450 </td><td><em>Nocardia vinacea</em> </td><td>98.9% </td><td>AB024312 </td></tr></tbody></table></div><div class="table-modal" content-id="t3"><table><thead align="" name="thead"><tr><td>DGGE band </td><td>Sequenced clone and accession number<sup>a</sup> </td><td>Most closely related bacterial 16S rRNA gene sequence </td><td>% Identity </td><td>Accession number<sup>b</sup> </td></tr></thead><tbody><tr><td>A </td><td>A1/A2 – AY376449<sup>c</sup> </td><td><em>Burkholderia</em> sp. RP007 </td><td>99.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A3 – AY376448 </td><td><em>Burkholderia</em> sp. RP007 </td><td>98.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A4 – AY376447 </td><td><em>Burkholderia</em> sp. RP007 </td><td>97.0% </td><td>AF061872 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>B1/B2/B3/B4 – AY376450 </td><td><em>Nocardia vinacea</em> </td><td>98.9% </td><td>AB024312 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="t3_note30"><div class="fn"><p><sup>a</sup>GenBank accession number of the sequence from excised bands.</p></div></div><div content-id="t3_note31"><div class="fn"><p><sup>b</sup>GenBank sequence accession number of most closely related bacterial sequence.</p></div></div><div content-id="t3_note32"><div class="fn"><p><sup>c</sup>Sequences with 100% of similarity. Only one sequence was submitted to the GenBank.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360696" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div class="table-wrap-title" data-id="t3" id="t3"><div class="label">3</div><div class="caption"><p>Results of partial sequence analysis and tentative phylogenetic affiliations of bands</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>DGGE band </td><td>Sequenced clone and accession number<sup>a</sup> </td><td>Most closely related bacterial 16S rRNA gene sequence </td><td>% Identity </td><td>Accession number<sup>b</sup> </td></tr></thead><tbody><tr><td>A </td><td>A1/A2 – AY376449<sup>c</sup> </td><td><em>Burkholderia</em> sp. RP007 </td><td>99.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A3 – AY376448 </td><td><em>Burkholderia</em> sp. RP007 </td><td>98.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A4 – AY376447 </td><td><em>Burkholderia</em> sp. RP007 </td><td>97.0% </td><td>AF061872 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>B1/B2/B3/B4 – AY376450 </td><td><em>Nocardia vinacea</em> </td><td>98.9% </td><td>AB024312 </td></tr></tbody></table></div><div class="table-modal" content-id="t3"><table><thead align="" name="thead"><tr><td>DGGE band </td><td>Sequenced clone and accession number<sup>a</sup> </td><td>Most closely related bacterial 16S rRNA gene sequence </td><td>% Identity </td><td>Accession number<sup>b</sup> </td></tr></thead><tbody><tr><td>A </td><td>A1/A2 – AY376449<sup>c</sup> </td><td><em>Burkholderia</em> sp. RP007 </td><td>99.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A3 – AY376448 </td><td><em>Burkholderia</em> sp. RP007 </td><td>98.1% </td><td>AF061872 </td></tr><tr><td> </td><td>A4 – AY376447 </td><td><em>Burkholderia</em> sp. RP007 </td><td>97.0% </td><td>AF061872 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>B </td><td>B1/B2/B3/B4 – AY376450 </td><td><em>Nocardia vinacea</em> </td><td>98.9% </td><td>AB024312 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="t3_note30"><div class="fn"><p><sup>a</sup>GenBank accession number of the sequence from excised bands.</p></div></div><div content-id="t3_note31"><div class="fn"><p><sup>b</sup>GenBank sequence accession number of most closely related bacterial sequence.</p></div></div><div content-id="t3_note32"><div class="fn"><p><sup>c</sup>Sequences with 100% of similarity. Only one sequence was submitted to the GenBank.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360696" target="_blank">View Large</a></div></div> <h2 class="section-title" data-legacy-id="ss3" id="90360697" scrollto-destination="90360697">3 Results</h2>
<h3 class="section-title" data-legacy-id="ss3-1" id="90360698" scrollto-destination="90360698">3.1 Survival of P. putida KT2442 (pNF142) in naphthalene-contaminated soil</h3>
<p>Survival of KT2442 (pNF142) in bulk soil artificially contaminated with naphthalene was followed by plating of serial dilutions of soil samples onto King’ s B supplemented with rifampicin. During the first few days the cfu numbers of KT2442 (pNF142) increased by more than 1 order of magnitude reaching a maximal titre of 1.7 × 10<sup>9</sup> cfu g<sup>−1</sup> of soil 3 dpi (see <a class="link link-reveal link-table xref-fig" data-open="t2" reveal-id="t2">Table 2</a>). At the end of the experiment a slightly lower titre than that determined at 0 dpi was found. The total nucleic acids (DNA/RNA) recovered from the soil samples were of sufficient purity for successful reverse transcription and PCR amplification. The DGGE fingerprints of the bacterial populations obtained from DNA and RNA samples showed little variation between replicates suggesting good reproducibility of the nucleic acid extraction protocol. DGGE analysis of 16S rRNA gene fragments amplified from total community DNA showed a remarkable numerical dominance of KT2442 (pNF142) in treatment (III) at 3 dpi (see <a class="link link-reveal link-table xref-fig" data-open="f1" reveal-id="f1">Fig. 1</a>). In contrast to the community patterns obtained for treatment (II) in which approximately 30 bands were detected, the bacterial patterns of treatment (III) showed a drastically reduced number of bands, and the bands corresponding to those of KT2442 (pNF142) were mainly visible. DNA- and RNA-based bacterial patterns of treatment (III) revealed that the inoculated strain remained a numerically dominant population during the time course of the experiment and, as indicated by the RNA-based profiles, seemed to be metabolically active (data not shown).</p> <a id="90360700" scrollto-destination="90360700"></a>
<div class="table-modal" content-id="t2" style="display: block;"><div class="table-wrap"><div class="table-wrap-title" data-id="t2" id="t2"><div class="label">2</div><div class="caption"><p>Cfu counts of <em>P. putida</em> KT2442 (pNF142) per gram (dry weight) of soil amended with naphthalene</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>Days </td><td>CFU (×10<sup>8</sup>) </td></tr></thead><tbody><tr><td>0 </td><td>1.1 ± 0.07 </td></tr><tr><td>1 </td><td>3.3 ± 0.28 </td></tr><tr><td>3 </td><td>17.1 ± 0.91 </td></tr><tr><td>11 </td><td>2.2 ± 0.50 </td></tr><tr><td>21 </td><td>0.8 ± 0.42 </td></tr></tbody></table></div><div class="table-modal" content-id="t2"><table><thead align="" name="thead"><tr><td>Days </td><td>CFU (×10<sup>8</sup>) </td></tr></thead><tbody><tr><td>0 </td><td>1.1 ± 0.07 </td></tr><tr><td>1 </td><td>3.3 ± 0.28 </td></tr><tr><td>3 </td><td>17.1 ± 0.91 </td></tr><tr><td>11 </td><td>2.2 ± 0.50 </td></tr><tr><td>21 </td><td>0.8 ± 0.42 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360700" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div class="table-wrap-title" data-id="t2" id="t2"><div class="label">2</div><div class="caption"><p>Cfu counts of <em>P. putida</em> KT2442 (pNF142) per gram (dry weight) of soil amended with naphthalene</p></div> </div><div class="table-overflow"><table><thead align="" name="thead"><tr><td>Days </td><td>CFU (×10<sup>8</sup>) </td></tr></thead><tbody><tr><td>0 </td><td>1.1 ± 0.07 </td></tr><tr><td>1 </td><td>3.3 ± 0.28 </td></tr><tr><td>3 </td><td>17.1 ± 0.91 </td></tr><tr><td>11 </td><td>2.2 ± 0.50 </td></tr><tr><td>21 </td><td>0.8 ± 0.42 </td></tr></tbody></table></div><div class="table-modal" content-id="t2"><table><thead align="" name="thead"><tr><td>Days </td><td>CFU (×10<sup>8</sup>) </td></tr></thead><tbody><tr><td>0 </td><td>1.1 ± 0.07 </td></tr><tr><td>1 </td><td>3.3 ± 0.28 </td></tr><tr><td>3 </td><td>17.1 ± 0.91 </td></tr><tr><td>11 </td><td>2.2 ± 0.50 </td></tr><tr><td>21 </td><td>0.8 ± 0.42 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/90360700" target="_blank">View Large</a></div></div> <a id="90360701" scrollto-destination="90360701"></a>
<div class="fig fig-section" data-id="f1" id="f1"><div class="label fig-label">1</div><div class="graphic-wrap"><img alt="PCR-DGGE analyses of bacterial populations in soil samples amended with naphthalene with no inoculation and after inoculation of P. putida KT2442 (pNF142) (3 dpi). Bacterial standard (ST) [13], P. putida KT2442 (pNF142) (KT)." class="content-image" path-from-xml="FEM_21_f1.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f1.jpeg?Expires=1563183447&amp;Signature=KTnMyqL8BIJt7hCPAEAl~WmTKE8tJv4gHm-4AKeUerug21oXWF80pdMKcWBhZ0QkmMzcahdlyQEv6HcZpxaEJqYXyOvCaNt04JS0p6ox5bQK~2RdKV3bu7A~x19AxdSOjKnPOwddK1K5D6SzAlzfMEsQPF9RXv9aE9IsIuwXbIbg3SzzijbE5CvdrHIIWkY7jtVrilKl7SGGNL0LLSVpmKGqaws7LgPWJfVLKOcaZLbfHTIxb8RQtFu50SR6pBRJ~BdTTJbOlCQiGsR-Z31JJGCJFyYj8mI6fvO78FEm3Zc0L11oehzP3gFvVGHZtMoN3NdJDOm56NMElKyRhwdHbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360701/FEM_21_f1.jpeg" path-from-xml="FEM_21_f1.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f1.jpeg?Expires=1563183447&amp;Signature=o0pj5sQjghdwH26sv35sn9GPoO5PjRUvEBTXlmwQI0MxJlwg5ggFWFe4nZPm22QrI6r90TAnBZN9ULJe~pXmmLWx8fz0tZ1bDfxferbD11YdhZAl4xhSGioDODDbcLlEKpMHV3A8Ys9kvvux~nwhXBmK5G~JKk2r0xOpFy1xG~UksGCxqx6z4VfzIDSo5N4rZDa6wzqQ2QVZzzjDDhyL8KQFhlNDn3JP5sqCerHN0o0N9AdKzq8nwx2kkT-KbYV13Hz8DHaaypvR2vR0DhKyAiWwf-21LA6MgLxHlocDu7gMSt8bH-Ogdj5NnU6Z8zUtxDmMnMk9cUAI~imU4DT9Eg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360701&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f1.jpeg" section="90360701" target="">Download slide</a></div></div><div class="caption fig-caption"><p>PCR-DGGE analyses of bacterial populations in soil samples amended with naphthalene with no inoculation and after inoculation of <em>P. putida</em> KT2442 (pNF142) (3 dpi). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>], <em>P. putida</em> KT2442 (pNF142) (KT).</p></div></div><div class="fig fig-modal reveal-modal" content-id="f1" style="display: block;"><div class="label fig-label">1</div><div class="graphic-wrap"><img alt="PCR-DGGE analyses of bacterial populations in soil samples amended with naphthalene with no inoculation and after inoculation of P. putida KT2442 (pNF142) (3 dpi). Bacterial standard (ST) [13], P. putida KT2442 (pNF142) (KT)." class="content-image" path-from-xml="FEM_21_f1.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f1.jpeg?Expires=1563183447&amp;Signature=KTnMyqL8BIJt7hCPAEAl~WmTKE8tJv4gHm-4AKeUerug21oXWF80pdMKcWBhZ0QkmMzcahdlyQEv6HcZpxaEJqYXyOvCaNt04JS0p6ox5bQK~2RdKV3bu7A~x19AxdSOjKnPOwddK1K5D6SzAlzfMEsQPF9RXv9aE9IsIuwXbIbg3SzzijbE5CvdrHIIWkY7jtVrilKl7SGGNL0LLSVpmKGqaws7LgPWJfVLKOcaZLbfHTIxb8RQtFu50SR6pBRJ~BdTTJbOlCQiGsR-Z31JJGCJFyYj8mI6fvO78FEm3Zc0L11oehzP3gFvVGHZtMoN3NdJDOm56NMElKyRhwdHbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360701/FEM_21_f1.jpeg" path-from-xml="FEM_21_f1.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f1.jpeg?Expires=1563183447&amp;Signature=o0pj5sQjghdwH26sv35sn9GPoO5PjRUvEBTXlmwQI0MxJlwg5ggFWFe4nZPm22QrI6r90TAnBZN9ULJe~pXmmLWx8fz0tZ1bDfxferbD11YdhZAl4xhSGioDODDbcLlEKpMHV3A8Ys9kvvux~nwhXBmK5G~JKk2r0xOpFy1xG~UksGCxqx6z4VfzIDSo5N4rZDa6wzqQ2QVZzzjDDhyL8KQFhlNDn3JP5sqCerHN0o0N9AdKzq8nwx2kkT-KbYV13Hz8DHaaypvR2vR0DhKyAiWwf-21LA6MgLxHlocDu7gMSt8bH-Ogdj5NnU6Z8zUtxDmMnMk9cUAI~imU4DT9Eg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360701&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f1.jpeg" section="90360701" target="">Download slide</a></div></div><div class="caption fig-caption"><p>PCR-DGGE analyses of bacterial populations in soil samples amended with naphthalene with no inoculation and after inoculation of <em>P. putida</em> KT2442 (pNF142) (3 dpi). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>], <em>P. putida</em> KT2442 (pNF142) (KT).</p></div></div> <h3 class="section-title" data-legacy-id="ss3-2" id="90360702" scrollto-destination="90360702">3.2 Indigenous soil bacterial communities affected by inoculation of KT2442 (pNF142) in naphthalene-contaminated soil</h3>
<p>Due to the numerical dominance of strain KT2442 (pNF142), it was actually impossible to evaluate by DGGE whether or not inoculation affected the composition of the indigenous bacterial community. Therefore, group-specific primers which allow amplification of the actinobacterial or β-proteobacterial 16S rRNA gene fragments before amplification of the GC-clamped PCR-fragment in a second, nested PCR step [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>,<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>] were applied. Since the inoculant strain belongs to the γ-proteobacteria, its 16S rRNA gene fragments were not amplified by the group-specific primers used. Comparison of the group-specific patterns observed for naphthalene-amended soil with or without inoculation of KT2442 (pNF142) revealed rather similar patterns for samples taken at day 0 or 3 dpi (data of the β-proteobacterial patterns in <a class="link link-reveal link-table xref-fig" data-open="f2" reveal-id="f2">Fig. 2</a>(a); actinobacterial patterns in <a class="link link-reveal link-table xref-fig" data-open="f3" reveal-id="f3">Fig. 3</a>(a)). Patterns differed clearly from the other samples only for samples of treatment (II) taken 21 dpi, mainly due to the appearance of a strong band in each of the replicates, and formed a separate cluster when Pearson correlation indices were analysed by UPGMA. While all replicates of the β-proteobacterial patterns (<a class="link link-reveal link-table xref-fig" data-open="f2" reveal-id="f2">Fig. 2</a>(b)) shared more than 80% similarity, the replicates of treatment II (a–c) and replicate b of treatment III of 21 dpi showed only 60% similarity with the main cluster. A similar trend was seen in the UPGMA analysis of the actinobacterial patterns (<a class="link link-reveal link-table xref-fig" data-open="f3" reveal-id="f3">Fig. 3</a>(b)). All replicates of treatment II (a–c) taken 21 dpi formed a separate cluster with less than 60% similarity with the main cluster. The patterns of replicates of the main cluster were highly similar to each other (more than 80%) except for two replicates of treatment III taken 21 dpi.</p> <a id="90360704" scrollto-destination="90360704"></a>
<div class="fig fig-section" data-id="f2" id="f2"><div class="label fig-label">2</div><div class="graphic-wrap"><img alt="Analysis of β-proteobacterial populations in soil samples amended with naphthalene with no inoculation (II) and with inoculation (III) of P. putida KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of β-proteobacterial populations. The relative band position of the band corresponding to Burkholderia sp. is shown (A). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the β-proteobacterial DGGE profiles. The differences between profiles are indicated by percentage similarity." class="content-image" path-from-xml="FEM_21_f2.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f2.jpeg?Expires=1563183447&amp;Signature=0eybLKi9L018fWoB5nkqaq5wvN-6PM4m7WiuNrkyKaM6LB725doIVnhMYHcHVBncrfyPKeAUJfteQZj4coiozxUEby-CNApCUreAJyeSQ~~puvpX1KU0MehGkQQb8~dTJiTTVvuD-~dIgdl-1p~AtG9~RAzASlPcv6uDc70yHHr7OOvOD2ENV0ewZytlNj17apDI5B2hsmVnwK0RJviBg9usfXqBrrPo6kxrjx3Ez0vcnU6mNBJy6Zlmqr9aW6qmblOSU2cFWup~AdWj-9jQJcBFoVeIz-RTrXrDWoBi0I0Uvkjh-Z~CNhNqE6HMxdxmdVT06zwwmD8hX0XtlDNnxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360704/FEM_21_f2.jpeg" path-from-xml="FEM_21_f2.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f2.jpeg?Expires=1563183447&amp;Signature=0nYCBvE5C4aGqstc3y~vtxgGAnGrQZKGvl9hmgFXXUdJi-7Oya-mlVE7XCK1ycwoK97pZlQpNdTUnaAgVjHfgmdSq4H-ojSN2qEwbkJvzDb3KYcfKN~9Xu4swVzXhm9cxopRwqJzyfgU0ZkZaEerfbpcGEEsSKJVyR6V-SfaBod6HX5CFhaR9Gqb0he3Xh7U9ctqkHaFxiX0SB6oXZ0Wiio~UtKLMREVGfQNBUnF70bf~7oxLDLoLIQGt5ecs29BNdTISihbNcUFGU5SSkEhWBDRvYJ~iQNDOQVokfC0pUl~49PbTxi4VCmDR-QviFyURvBtJDbq6I7D~Sf3DD-s1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360704&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f2.jpeg" section="90360704" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of β-proteobacterial populations in soil samples amended with naphthalene with no inoculation (II) and with inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of β-proteobacterial populations. The relative band position of the band corresponding to <em>Burkholderia</em> sp. is shown (A). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the β-proteobacterial DGGE profiles. The differences between profiles are indicated by percentage similarity.</p></div></div><div class="fig fig-modal reveal-modal" content-id="f2" style="display: block;"><div class="label fig-label">2</div><div class="graphic-wrap"><img alt="Analysis of β-proteobacterial populations in soil samples amended with naphthalene with no inoculation (II) and with inoculation (III) of P. putida KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of β-proteobacterial populations. The relative band position of the band corresponding to Burkholderia sp. is shown (A). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the β-proteobacterial DGGE profiles. The differences between profiles are indicated by percentage similarity." class="content-image" path-from-xml="FEM_21_f2.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f2.jpeg?Expires=1563183447&amp;Signature=0eybLKi9L018fWoB5nkqaq5wvN-6PM4m7WiuNrkyKaM6LB725doIVnhMYHcHVBncrfyPKeAUJfteQZj4coiozxUEby-CNApCUreAJyeSQ~~puvpX1KU0MehGkQQb8~dTJiTTVvuD-~dIgdl-1p~AtG9~RAzASlPcv6uDc70yHHr7OOvOD2ENV0ewZytlNj17apDI5B2hsmVnwK0RJviBg9usfXqBrrPo6kxrjx3Ez0vcnU6mNBJy6Zlmqr9aW6qmblOSU2cFWup~AdWj-9jQJcBFoVeIz-RTrXrDWoBi0I0Uvkjh-Z~CNhNqE6HMxdxmdVT06zwwmD8hX0XtlDNnxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360704/FEM_21_f2.jpeg" path-from-xml="FEM_21_f2.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f2.jpeg?Expires=1563183447&amp;Signature=0nYCBvE5C4aGqstc3y~vtxgGAnGrQZKGvl9hmgFXXUdJi-7Oya-mlVE7XCK1ycwoK97pZlQpNdTUnaAgVjHfgmdSq4H-ojSN2qEwbkJvzDb3KYcfKN~9Xu4swVzXhm9cxopRwqJzyfgU0ZkZaEerfbpcGEEsSKJVyR6V-SfaBod6HX5CFhaR9Gqb0he3Xh7U9ctqkHaFxiX0SB6oXZ0Wiio~UtKLMREVGfQNBUnF70bf~7oxLDLoLIQGt5ecs29BNdTISihbNcUFGU5SSkEhWBDRvYJ~iQNDOQVokfC0pUl~49PbTxi4VCmDR-QviFyURvBtJDbq6I7D~Sf3DD-s1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360704&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f2.jpeg" section="90360704" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of β-proteobacterial populations in soil samples amended with naphthalene with no inoculation (II) and with inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of β-proteobacterial populations. The relative band position of the band corresponding to <em>Burkholderia</em> sp. is shown (A). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the β-proteobacterial DGGE profiles. The differences between profiles are indicated by percentage similarity.</p></div></div> <a id="90360705" scrollto-destination="90360705"></a>
<div class="fig fig-section" data-id="f3" id="f3"><div class="label fig-label">3</div><div class="graphic-wrap"><img alt="Analysis of actinobacterial populations in soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of P. putida KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of actinobacterial populations. The relative position of the band corresponding to Nocardia vinaceae is shown (B). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the actinobacterial DGGE profiles. The differences between the profiles are indicated by percentage of similarity." class="content-image" path-from-xml="FEM_21_f3.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f3.jpeg?Expires=1563183447&amp;Signature=bbup6xEaWZhcESAfqhF704VoQCEWGoVGpZ1njadLnyJNNCWWRvuJLMF8X5uY-TJp0kCPA8Z~y5SgVr5mqCmd94UmfB8-O~n5AThI3CqDKN5AEaAPdqI873ej84o6yjkahKTEq6zmL7OMxqGZNJw8jlyOby3d2h5jkfktPEJmUeNKm9LNxYrpT49ruFucD41966k7m8yHZOTANhh-VzNd-emHpvbZ2Y1dvxRXX-bEV9NvmYR92PyH53UzViYFIipKTZ7j7VLqO0cOBjB7VHFNRK6QhH~O6fubPnlnxLMNq~A0yZ5L3LRS7JBxAzGlqojV3C98vkYrekobZIy6FkATvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360705/FEM_21_f3.jpeg" path-from-xml="FEM_21_f3.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f3.jpeg?Expires=1563183447&amp;Signature=UanVTSMc~wcBavJb7tZTMFltRhyuWHAh5mWT1p9kPwUnPH2p3EfVVMxNnpsAK4DaY-HAPfE8vcrkm5Z2Bt9bpRxKlO4H-EjXBqhYp91plTTOt~nvhffnHNIEtX5Enkk0-oWfkHb0Vm8FUXvRjQzOyYgEP5hVKe-m-~aEsbzZlCQssDwZn1nwQG4lxGHVdyTFALvPlq0LTkaU7-Axxcx9ZQymbCGlHNrTPX7vKYbIG8~H5IrrVmXsnb3H27WIA3bAj~ACTxUhkOL4Lpg1eS~2-RvIGKkseJsljAroZubLZqjxRlZprrUOeyQjEKm38fSeFKDW7t8biiGep59ZHirgqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360705&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f3.jpeg" section="90360705" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of actinobacterial populations in soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of actinobacterial populations. The relative position of the band corresponding to <em>Nocardia vinaceae</em> is shown (B). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the actinobacterial DGGE profiles. The differences between the profiles are indicated by percentage of similarity.</p></div></div><div class="fig fig-modal reveal-modal" content-id="f3" style="display: block;"><div class="label fig-label">3</div><div class="graphic-wrap"><img alt="Analysis of actinobacterial populations in soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of P. putida KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of actinobacterial populations. The relative position of the band corresponding to Nocardia vinaceae is shown (B). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the actinobacterial DGGE profiles. The differences between the profiles are indicated by percentage of similarity." class="content-image" path-from-xml="FEM_21_f3.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f3.jpeg?Expires=1563183447&amp;Signature=bbup6xEaWZhcESAfqhF704VoQCEWGoVGpZ1njadLnyJNNCWWRvuJLMF8X5uY-TJp0kCPA8Z~y5SgVr5mqCmd94UmfB8-O~n5AThI3CqDKN5AEaAPdqI873ej84o6yjkahKTEq6zmL7OMxqGZNJw8jlyOby3d2h5jkfktPEJmUeNKm9LNxYrpT49ruFucD41966k7m8yHZOTANhh-VzNd-emHpvbZ2Y1dvxRXX-bEV9NvmYR92PyH53UzViYFIipKTZ7j7VLqO0cOBjB7VHFNRK6QhH~O6fubPnlnxLMNq~A0yZ5L3LRS7JBxAzGlqojV3C98vkYrekobZIy6FkATvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360705/FEM_21_f3.jpeg" path-from-xml="FEM_21_f3.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f3.jpeg?Expires=1563183447&amp;Signature=UanVTSMc~wcBavJb7tZTMFltRhyuWHAh5mWT1p9kPwUnPH2p3EfVVMxNnpsAK4DaY-HAPfE8vcrkm5Z2Bt9bpRxKlO4H-EjXBqhYp91plTTOt~nvhffnHNIEtX5Enkk0-oWfkHb0Vm8FUXvRjQzOyYgEP5hVKe-m-~aEsbzZlCQssDwZn1nwQG4lxGHVdyTFALvPlq0LTkaU7-Axxcx9ZQymbCGlHNrTPX7vKYbIG8~H5IrrVmXsnb3H27WIA3bAj~ACTxUhkOL4Lpg1eS~2-RvIGKkseJsljAroZubLZqjxRlZprrUOeyQjEKm38fSeFKDW7t8biiGep59ZHirgqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360705&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f3.jpeg" section="90360705" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of actinobacterial populations in soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (0, 3 and 21 dpi). (a) DGGE community fingerprints of actinobacterial populations. The relative position of the band corresponding to <em>Nocardia vinaceae</em> is shown (B). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the actinobacterial DGGE profiles. The differences between the profiles are indicated by percentage of similarity.</p></div></div> <h3 class="section-title" data-legacy-id="ss3-3" id="90360706" scrollto-destination="90360706">3.3 Effect of naphthalene on the soil bacterial community</h3>
<p>To evaluate the effect of the naphthalene contamination on the structural composition of the soil bacterial community, 16S rRNA gene fragments amplified from total DNA and cDNA of soil samples taken at 0, 3 and 21 dpi from treatments I and II were analysed by DGGE (DNA: <a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">Fig. 4</a>(a); cDNA: data not shown). Three independent replicates per treatment and per sampling time analysed were run in blocks next to each other.</p> <a id="90360708" scrollto-destination="90360708"></a>
<div class="fig fig-section" data-id="f4" id="f4"><div class="label fig-label">4</div><div class="graphic-wrap"><img alt="Analysis of bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and those artificially contaminated with naphthalene (II). (a) DGGE community fingerprints of bacterial populations. The band position of populations enhanced by naphthalene contamination is shown (A and B). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the bacterial DGGE profiles. Differences between the profiles are indicated by percentage of similarity." class="content-image" path-from-xml="FEM_21_f4.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f4.jpeg?Expires=1563183447&amp;Signature=TFlebwRz6mtLEplQZgbGDNjODyahXzsej-S4RCPoLHpcjqSvwcZvvfY3h5gROPIt-b-tbtXi22VW1L1RjyNX0TS2IVrWajhjnDqmLDDay-t7Zp2Fwi-59b1ZwC8cQ8r0f--LFTGkp2FjS1HdkSgOhc6VOsvrs3WwUq4uGoWjQ71rg22PSUigDkacBGlHxEYrBUYzanPgto5yTfKzBbtTjVDFOvhk3P06Wm5J4Wj2qioPwlHFbF0ImHrGO8tDkbGIvQYYsc7ayzb-MTQ4jz3WXZJ32Ux6-8v6Och~c-B-dmfIyIvF7Goe8SJgcy4xOmsEZDhZIogrpU-~2kPhukxAiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360708/FEM_21_f4.jpeg" path-from-xml="FEM_21_f4.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f4.jpeg?Expires=1563183447&amp;Signature=DMGN1MiLa-olDlr3LKZG00oeWFRp8QuN0QDkAAe4jRePYMaeLzQWYxO2rFNmBjkf-4qzCsES0gx-SaC6n160KMOzyuuFDksN4Qn~oISRUTgkcv0BLxYbVgpfKSI4dSaRb3Y~spwEYqcxazdvTWIVdUXESQdn2UQdsNqDsaXeG4P0fP38D38jSllMPgTSidIbndwebrGxr79PsTyiVwkzsh9Run88AcnNpfKY7L3KbloDtpuWX-FDZfqI5MbBo4SrEuBHlPqtkHU7SFzc3~NXdhJ-15SsNJGMkwaJnEgedRS2k4qOHpLOJRIvn-lsbtJLLyJuOGo7XGAX2JF4yRjDdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360708&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f4.jpeg" section="90360708" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and those artificially contaminated with naphthalene (II). (a) DGGE community fingerprints of bacterial populations. The band position of populations enhanced by naphthalene contamination is shown (A and B). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the bacterial DGGE profiles. Differences between the profiles are indicated by percentage of similarity.</p></div></div><div class="fig fig-modal reveal-modal" content-id="f4" style="display: block;"><div class="label fig-label">4</div><div class="graphic-wrap"><img alt="Analysis of bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and those artificially contaminated with naphthalene (II). (a) DGGE community fingerprints of bacterial populations. The band position of populations enhanced by naphthalene contamination is shown (A and B). Bacterial standard (ST) [13]. (b) Dendrogram constructed with the bacterial DGGE profiles. Differences between the profiles are indicated by percentage of similarity." class="content-image" path-from-xml="FEM_21_f4.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f4.jpeg?Expires=1563183447&amp;Signature=TFlebwRz6mtLEplQZgbGDNjODyahXzsej-S4RCPoLHpcjqSvwcZvvfY3h5gROPIt-b-tbtXi22VW1L1RjyNX0TS2IVrWajhjnDqmLDDay-t7Zp2Fwi-59b1ZwC8cQ8r0f--LFTGkp2FjS1HdkSgOhc6VOsvrs3WwUq4uGoWjQ71rg22PSUigDkacBGlHxEYrBUYzanPgto5yTfKzBbtTjVDFOvhk3P06Wm5J4Wj2qioPwlHFbF0ImHrGO8tDkbGIvQYYsc7ayzb-MTQ4jz3WXZJ32Ux6-8v6Och~c-B-dmfIyIvF7Goe8SJgcy4xOmsEZDhZIogrpU-~2kPhukxAiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360708/FEM_21_f4.jpeg" path-from-xml="FEM_21_f4.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f4.jpeg?Expires=1563183447&amp;Signature=DMGN1MiLa-olDlr3LKZG00oeWFRp8QuN0QDkAAe4jRePYMaeLzQWYxO2rFNmBjkf-4qzCsES0gx-SaC6n160KMOzyuuFDksN4Qn~oISRUTgkcv0BLxYbVgpfKSI4dSaRb3Y~spwEYqcxazdvTWIVdUXESQdn2UQdsNqDsaXeG4P0fP38D38jSllMPgTSidIbndwebrGxr79PsTyiVwkzsh9Run88AcnNpfKY7L3KbloDtpuWX-FDZfqI5MbBo4SrEuBHlPqtkHU7SFzc3~NXdhJ-15SsNJGMkwaJnEgedRS2k4qOHpLOJRIvn-lsbtJLLyJuOGo7XGAX2JF4yRjDdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360708&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f4.jpeg" section="90360708" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and those artificially contaminated with naphthalene (II). (a) DGGE community fingerprints of bacterial populations. The band position of populations enhanced by naphthalene contamination is shown (A and B). Bacterial standard (ST) [<a class="link link-ref link-reveal xref-bibr" data-open="b13" reveal-id="b13">13</a>]. (b) Dendrogram constructed with the bacterial DGGE profiles. Differences between the profiles are indicated by percentage of similarity.</p></div></div><p>The comparison of DNA-based DGGE profiles of samples taken at 0 dpi was only possible for the more intense bands because PCR amplification of DNA from naphthalene-amended soils seemed to be inhibited, in particular for replicate C (<a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">Fig. 4</a>(a)). No differences were seen for the dominant bands. Comparison of DGGE profiles of samples taken 3 dpi also allowed the conclusion that the structural composition of the bacterial community seemed not to be affected by amendment with naphthalene. However, this picture changed 21 days after the start of the experiment. While the profiles of all other samples shared more than 70% similarity, the profiles of samples from naphthalene-amended soil taken 21 dpi were strikingly different and less than 40% similar to the other profiles (<a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">Fig. 4</a>(b)). The same trend was found for RNA-based profiles (data not shown). Two new dominant bands, A and B, appeared in all three replicates of DNA- and RNA-based patterns from soils amended with naphthalene 21 dpi (<a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">Figs. 4(a)</a> and <a class="link link-reveal link-table xref-fig" data-open="f5" reveal-id="f5">5</a>).</p> <a id="90360710" scrollto-destination="90360710"></a>
<div class="fig fig-section" data-id="f5" id="f5"><div class="label fig-label">5</div><div class="graphic-wrap"><img alt="PCR-DGGE analysis of 16S ribosomal RNA gene fragments amplified from DNA and RNA (cDNA) from bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and artificially contaminated with naphthalene (II). Concomitantly, arrows indicate bands present in the DNA or RNA profiles which cannot be seen or have a reduced intensity when both profiles are compared with each other." class="content-image" path-from-xml="FEM_21_f5.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f5.jpeg?Expires=1563183447&amp;Signature=hLYF1kNfp324SqUM3uWT19C~~RtKKUSS1CaUz7yGgPEw17TmtoQ~x-R1HwpBCX-7ulC566k3cYhm3nUdaPcCe3aGVkC-~SQGeiOPON5lsN1P28bG78SZ9kYe70awiwABYW8KURcB7hrDriehpKrcAkyIKBCxHXFaFelCcwggm3YkoVHls12EJMENa6GYQOjfedRiOdmPqfWS7mGLIVggyGBi576FvbD-fbPuyfBvUPbwfNH0oWCAqNS8J8nXp2g4wOb5Kn5jTQB21utLhpO-vjqyVn70DbHHqHBaWBCvAnBPTtLT1crxlyuSHOJlvPG-7MX~tfqEwrAtC8w3Jx4H6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360710/FEM_21_f5.jpeg" path-from-xml="FEM_21_f5.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f5.jpeg?Expires=1563183447&amp;Signature=e5UCkKKqfbxE-1oF8hxzPeZNHC3kdBEzjLYz1BT78q~JuYaFvE8nbrIpJajOgVo4E6fRkXUtCyqHaIzJfqN6Xqek6ipHJgNC6zn7SVZfx42OOGX4VX8q0Q3~rMgMu~nVnul78Lw6qmXZg0WeF9PKtSmyL0fhHQu7W5hrnsVfdjh0nAxxeaAfL121ti60dc3Oha4JadV6P3mYTe~plS-8tYIPKRjo3eOGp4k64hJIhvP~BF8Zp2bMYzneNdkAZF8Fu1Y9fp7I2X~ETg10RZw7YDCHEOOjXxuJLc~Bj8scrf120SeFXscPNj33Hc~mbGRQlAEjz5T5qkCPlAnkxvaSCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360710&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f5.jpeg" section="90360710" target="">Download slide</a></div></div><div class="caption fig-caption"><p>PCR-DGGE analysis of 16S ribosomal RNA gene fragments amplified from DNA and RNA (cDNA) from bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and artificially contaminated with naphthalene (II). Concomitantly, arrows indicate bands present in the DNA or RNA profiles which cannot be seen or have a reduced intensity when both profiles are compared with each other.</p></div></div><div class="fig fig-modal reveal-modal" content-id="f5" style="display: block;"><div class="label fig-label">5</div><div class="graphic-wrap"><img alt="PCR-DGGE analysis of 16S ribosomal RNA gene fragments amplified from DNA and RNA (cDNA) from bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and artificially contaminated with naphthalene (II). Concomitantly, arrows indicate bands present in the DNA or RNA profiles which cannot be seen or have a reduced intensity when both profiles are compared with each other." class="content-image" path-from-xml="FEM_21_f5.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f5.jpeg?Expires=1563183447&amp;Signature=hLYF1kNfp324SqUM3uWT19C~~RtKKUSS1CaUz7yGgPEw17TmtoQ~x-R1HwpBCX-7ulC566k3cYhm3nUdaPcCe3aGVkC-~SQGeiOPON5lsN1P28bG78SZ9kYe70awiwABYW8KURcB7hrDriehpKrcAkyIKBCxHXFaFelCcwggm3YkoVHls12EJMENa6GYQOjfedRiOdmPqfWS7mGLIVggyGBi576FvbD-fbPuyfBvUPbwfNH0oWCAqNS8J8nXp2g4wOb5Kn5jTQB21utLhpO-vjqyVn70DbHHqHBaWBCvAnBPTtLT1crxlyuSHOJlvPG-7MX~tfqEwrAtC8w3Jx4H6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360710/FEM_21_f5.jpeg" path-from-xml="FEM_21_f5.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f5.jpeg?Expires=1563183447&amp;Signature=e5UCkKKqfbxE-1oF8hxzPeZNHC3kdBEzjLYz1BT78q~JuYaFvE8nbrIpJajOgVo4E6fRkXUtCyqHaIzJfqN6Xqek6ipHJgNC6zn7SVZfx42OOGX4VX8q0Q3~rMgMu~nVnul78Lw6qmXZg0WeF9PKtSmyL0fhHQu7W5hrnsVfdjh0nAxxeaAfL121ti60dc3Oha4JadV6P3mYTe~plS-8tYIPKRjo3eOGp4k64hJIhvP~BF8Zp2bMYzneNdkAZF8Fu1Y9fp7I2X~ETg10RZw7YDCHEOOjXxuJLc~Bj8scrf120SeFXscPNj33Hc~mbGRQlAEjz5T5qkCPlAnkxvaSCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360710&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f5.jpeg" section="90360710" target="">Download slide</a></div></div><div class="caption fig-caption"><p>PCR-DGGE analysis of 16S ribosomal RNA gene fragments amplified from DNA and RNA (cDNA) from bacterial populations of non-inoculated soil microcosms comparing samples from microcosms not contaminated (I) and artificially contaminated with naphthalene (II). Concomitantly, arrows indicate bands present in the DNA or RNA profiles which cannot be seen or have a reduced intensity when both profiles are compared with each other.</p></div></div><p>These bands were excised and re-amplified. The re-amplified DNA from band A resulted in one dominant band. The corresponding GC-less PCR product was cloned and four clones were sequenced (<a class="link link-reveal link-table xref-fig" data-open="t3" reveal-id="t3">Table 3</a>). The sequences of all four clones derived from band A in the patterns of soil amended with naphthalene were most similar to <em>Burkholderia</em> sp. RP007. While two of the clones had approximately 99% similarity to the RP007 sequences of the other clones had 98% and 97% similarity. The re-amplified DNA of band B from the bacterial DGGE patterns resulted in several bands when checked by DGGE and thus band B was extracted from the actinobacterial patterns. The sequences of the four clones derived from band B were identical and showed 98.85% similarity to <em>Nocardia vinacea</em>. The DGGE patterns of actinobacterial communities revealed that the <em>Nocardia</em> sp. population appearing in naphthalene-contaminated soil 21 dpi was hardly visible in the soil without naphthalene (data not shown). In contrast, a band with the identical electrophoretic mobility to <em>Burkholderia</em> sp. was detectable in the control soil (I) but only after enrichment for β-proteobacterial 16S rRNA gene fragments (data not shown). This population was obviously enriched in the presence of naphthalene because this band was not detectable in the bacterial patterns of the control soil (I) at day 21 (<a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">Fig. 4</a>(a)). Overall, the DNA and RNA-based patterns were remarkably similar as exemplified in <a class="link link-reveal link-table xref-fig" data-open="f5" reveal-id="f5">Fig. 5</a> for treatment II. When comparing DNA and RNA-based patterns, differences in the relative band intensity were observed only for a few bands. In the upper part of the gel (low denaturant concentrations) some bands were detected in the DNA-based patterns which were not detected or much fainter in the RNA-based patterns.</p> <h3 class="section-title" data-legacy-id="ss3-4" id="90360712" scrollto-destination="90360712">3.4 Detection of naphthalene-degrading genes and their expression in soil</h3>
<p>Published primer systems targeting two genes coding for key enzymes in the naphthalene degradation pathway, the <em>nahAc</em> gene encoding the naphthalene-1,2-dioxygenase large subunit [<a class="link link-ref link-reveal xref-bibr" data-open="b2" reveal-id="b2">2</a>] and the <em>nahH</em> gene coding for the catechol-2,3-dioxygenase [<a class="link link-ref link-reveal xref-bibr" data-open="b8" reveal-id="b8">8</a>], were used to monitor the presence of the gene (DNA) and its expression (RNA). Both genes are localised on plasmid pNF142. Using the primer pair Ac114F and Ac596R (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>) the <em>nahAc</em> gene was detected in DNA from all soil samples inoculated with KT2442 (pNF142) (III) but not in soils from treatments (I) and (II). The <em>nahAc</em> gene could also be amplified from reverse transcribed RNA extracted from soil samples of treatment (III), indicating that this gene was expressed under soil conditions at all time points analysed (<a class="link link-reveal link-table xref-fig" data-open="f6" reveal-id="f6">Fig. 6</a>). The <em>nahH</em> gene was amplified from DNA of all soil samples of treatment (III) by means of primers 23DOF and 23DOR (<a class="link link-reveal link-table xref-fig" data-open="t1" reveal-id="t1">Table 1</a>) but a weak signal was also obtained from one sample of the treatment (II) at 11 dpi (data not shown). However, PCR amplicons were obtained from reverse transcribed RNA only for samples taken 3 dpi with KT2442 (pNF142) but not from cDNA of samples taken after 0 or 21 dpi (data not shown).</p> <a id="90360714" scrollto-destination="90360714"></a>
<div class="fig fig-section" data-id="f6" id="f6"><div class="label fig-label">6</div><div class="graphic-wrap"><img alt="Southern blot hybridisation to nahAc of PCR products from DNA or RNA (cDNA) extracted from soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of P. putida KT2442 (pNF142) (KT) (0, 3 and 21 dpi). DIG-labelled DNA molecular weight marker VI (Roche)." class="content-image" path-from-xml="FEM_21_f6.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f6.jpeg?Expires=1563183447&amp;Signature=riiCPlJ8YLDtsKxHihXdGudiZkCjkDvN3RmdmNcIASb-z5d324-87T-gnntuSdNCMvLACx9qcZ4dJ-iZ9QHnz7-r2Nj23-pkTbKtxOdaStggB8xiK7yQ-ON4rN8GXxQxNcSruB7qGd~d6d6nxeGLpIjH7Y~PUHsnIwsvL0~3hKmOtRYWKSEDO63ZWFMxku8k3RvI31UXuVxtb5NHCTVgeghtni372powghJb1ZThtlun-yrMv~P4CN9WpiA2PhYsTFtSWrPUQZtpbXpIb1g~v6UhdFZen1whgrciwA0XtrxbZ2GVVmwxHK~65z0~QIGhPLtSJvJcQ5OE8qo5jPfUkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360714/FEM_21_f6.jpeg" path-from-xml="FEM_21_f6.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f6.jpeg?Expires=1563183447&amp;Signature=djBPZzZDaTD3UvVxCjLFXdBRVac9pEfNNAZdbbTQ7617WZBmEPShAH2fDm5yDAoaEM1N7PEC9s909gzP1Ponz4YHFj693n4AVVdmcVeJylG1ZnyVBHTcuPL3RsDT92sWImyDxShuwg2xzkB5e4Bma4LZLZjP~ytGKS-gg7HNb6AJzwkJ3XeM4ZWv4UrYwm8i0TcDyhnyZFU41MzapZakXmL4gELlfM6VBFlTjdDlZXSU9dNQThxAXU2Aatcg-IC-E1kXeUokCmBJsCvyh2sUmhFTnIimKYtyyjyerP1oW~050zXYh7T~bP6Hg1KwmKelwJE5Ydo1jiZuU~pMG3YjAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360714&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f6.jpeg" section="90360714" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Southern blot hybridisation to <em>nahAc</em> of PCR products from DNA or RNA (cDNA) extracted from soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (KT) (0, 3 and 21 dpi). DIG-labelled DNA molecular weight marker VI (Roche).</p></div></div><div class="fig fig-modal reveal-modal" content-id="f6" style="display: block;"><div class="label fig-label">6</div><div class="graphic-wrap"><img alt="Southern blot hybridisation to nahAc of PCR products from DNA or RNA (cDNA) extracted from soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of P. putida KT2442 (pNF142) (KT) (0, 3 and 21 dpi). DIG-labelled DNA molecular weight marker VI (Roche)." class="content-image" path-from-xml="FEM_21_f6.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f6.jpeg?Expires=1563183447&amp;Signature=riiCPlJ8YLDtsKxHihXdGudiZkCjkDvN3RmdmNcIASb-z5d324-87T-gnntuSdNCMvLACx9qcZ4dJ-iZ9QHnz7-r2Nj23-pkTbKtxOdaStggB8xiK7yQ-ON4rN8GXxQxNcSruB7qGd~d6d6nxeGLpIjH7Y~PUHsnIwsvL0~3hKmOtRYWKSEDO63ZWFMxku8k3RvI31UXuVxtb5NHCTVgeghtni372powghJb1ZThtlun-yrMv~P4CN9WpiA2PhYsTFtSWrPUQZtpbXpIb1g~v6UhdFZen1whgrciwA0XtrxbZ2GVVmwxHK~65z0~QIGhPLtSJvJcQ5OE8qo5jPfUkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360714/FEM_21_f6.jpeg" path-from-xml="FEM_21_f6.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f6.jpeg?Expires=1563183447&amp;Signature=djBPZzZDaTD3UvVxCjLFXdBRVac9pEfNNAZdbbTQ7617WZBmEPShAH2fDm5yDAoaEM1N7PEC9s909gzP1Ponz4YHFj693n4AVVdmcVeJylG1ZnyVBHTcuPL3RsDT92sWImyDxShuwg2xzkB5e4Bma4LZLZjP~ytGKS-gg7HNb6AJzwkJ3XeM4ZWv4UrYwm8i0TcDyhnyZFU41MzapZakXmL4gELlfM6VBFlTjdDlZXSU9dNQThxAXU2Aatcg-IC-E1kXeUokCmBJsCvyh2sUmhFTnIimKYtyyjyerP1oW~050zXYh7T~bP6Hg1KwmKelwJE5Ydo1jiZuU~pMG3YjAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360714&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f6.jpeg" section="90360714" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Southern blot hybridisation to <em>nahAc</em> of PCR products from DNA or RNA (cDNA) extracted from soil samples amended with naphthalene with no inoculation (II) and after inoculation (III) of <em>P. putida</em> KT2442 (pNF142) (KT) (0, 3 and 21 dpi). DIG-labelled DNA molecular weight marker VI (Roche).</p></div></div><p>Because the 16S rRNA gene sequence of one of the enriched populations in treatment (II) was almost identical to the sequence of <em>Burkholderia</em> sp. RP007, which carries the naphthalene dioxygenase gene <em>phnAc</em>, a primer system described by Lloyd-Jones et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>], targeting this gene, was used. Indeed, strong <em>phnAc</em> amplicons could be detected from DNA of all replicates of treatment (II) and from replicate b of treatment III 21 dpi (see <a class="link link-reveal link-table xref-fig" data-open="f7" reveal-id="f7">Fig. 7</a>). The detection of <em>phnAc</em> correlated with the appearance of the <em>Burkholderia</em> band in the bacterial DGGE patterns. However, the expression of this gene could not be detected in any sample from treatment II but curiously it was detected in replicate b of treatment (III) at 21 dpi (data not shown). The <em>phnAc</em> PCR product obtained from DNA of treatment (II) (mix of all three replicates) was cloned and sequenced. The <em>phnAc</em> gene was re-amplified from all 25 clones and PCR products were digested with <em>Msp</em>I. The restriction patterns obtained showed that most of the clones had identical patterns (18/24). The remaining clone fragments had one strong band missing or had additional bands (data not shown). However, sequencing showed that all sequences were highly related to the <em>phnAc</em> sequence of <em>Burkholderia</em> sp. RP007 (more than 99% similarity).</p> <a id="90360716" scrollto-destination="90360716"></a>
<div class="fig fig-section" data-id="f7" id="f7"><div class="label fig-label">7</div><div class="graphic-wrap"><img alt="Naphthalene dioxygenase like phnAc gene detection by PCR amplification using the primer sets developed for Burkholderia sp. strain RP007 [7]. The PCR products were obtained from DNA samples extracted from soil microcosms not contaminated (I), artificially contaminated with naphthalene with no inoculation (II) and microcosms artificially contaminated with naphthalene inoculated with P. putida KT2442 (pNF142) (III)." class="content-image" path-from-xml="FEM_21_f7.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f7.jpeg?Expires=1563183447&amp;Signature=L5w13f1YsbKhUbmFM42pYWFKT6VWNvMO8JzaOqIOtZz0yzDpnsctDCifWUTRFbfYIBq7ZoMR48LKrc1CS80~IpHcHHFzm0bqXDzfZSwCJ0BgsYPOh-ab45MGp9yzbtpO15hHzKUgIHhJEZOKwFZrphjvn9jxA5aqEYIajw2L6sxyvyEJvBHRrUKEut6iBiHeD7MaQTM89rSTa5Bocyp0XnqjQobk9nCdMjAHgDV1gKMC5qBKhaCghuof38hk06ZnDl5z378jWYSy1oOlBdHSzWWFDNCR-8F67fDiE3zRZRzR4G3AHcuZqt2etTNHr6PUtoJLal1oB-6DHO9tRTpXcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360716/FEM_21_f7.jpeg" path-from-xml="FEM_21_f7.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f7.jpeg?Expires=1563183447&amp;Signature=bG8qsIxsDKr9EK6~ihsi8PVHxG~8tARUGWRbnfOdJjRz56IIWU2fB1LFRW7B9~7pUCk8hiL950KHWwuGVgm2CR7XzuN2Ol~Srxm1KDqSbvcHPhFpTcMAxXoy0MmzAk4azw3m6BS2zdKFbXWZDwIBAuWsyNs2OfeBlN9XbjYIu4qrwdmUoKNjD6lCTddZ~Stqw9zjOdvERsYLF-k1p3VFitSNuzE6K8qaLBuOpjwupC~kvDnOd6PE-~4SgmQIbnJmbYHlFIoER8OUJ3aN9qXXQZo1zzb6q9GUIkxy6r9dl8V~N-Y1F0O3RW88rxT1EddI9GKwPdZcd4GjkwK8tiW51Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360716&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f7.jpeg" section="90360716" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Naphthalene dioxygenase like <em>phnAc</em> gene detection by PCR amplification using the primer sets developed for <em>Burkholderia</em> sp. strain RP007 [<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>]. The PCR products were obtained from DNA samples extracted from soil microcosms not contaminated (I), artificially contaminated with naphthalene with no inoculation (II) and microcosms artificially contaminated with naphthalene inoculated with <em>P. putida</em> KT2442 (pNF142) (III).</p></div></div><div class="fig fig-modal reveal-modal" content-id="f7" style="display: block;"><div class="label fig-label">7</div><div class="graphic-wrap"><img alt="Naphthalene dioxygenase like phnAc gene detection by PCR amplification using the primer sets developed for Burkholderia sp. strain RP007 [7]. The PCR products were obtained from DNA samples extracted from soil microcosms not contaminated (I), artificially contaminated with naphthalene with no inoculation (II) and microcosms artificially contaminated with naphthalene inoculated with P. putida KT2442 (pNF142) (III)." class="content-image" path-from-xml="FEM_21_f7.jpeg" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/m_FEM_21_f7.jpeg?Expires=1563183447&amp;Signature=L5w13f1YsbKhUbmFM42pYWFKT6VWNvMO8JzaOqIOtZz0yzDpnsctDCifWUTRFbfYIBq7ZoMR48LKrc1CS80~IpHcHHFzm0bqXDzfZSwCJ0BgsYPOh-ab45MGp9yzbtpO15hHzKUgIHhJEZOKwFZrphjvn9jxA5aqEYIajw2L6sxyvyEJvBHRrUKEut6iBiHeD7MaQTM89rSTa5Bocyp0XnqjQobk9nCdMjAHgDV1gKMC5qBKhaCghuof38hk06ZnDl5z378jWYSy1oOlBdHSzWWFDNCR-8F67fDiE3zRZRzR4G3AHcuZqt2etTNHr6PUtoJLal1oB-6DHO9tRTpXcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"/><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/90360716/FEM_21_f7.jpeg" path-from-xml="FEM_21_f7.jpeg" target="_blank">View large</a><a class="download-slide" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsec/54/1/10.1016_j.femsec.2005.02.005/1/FEM_21_f7.jpeg?Expires=1563183447&amp;Signature=bG8qsIxsDKr9EK6~ihsi8PVHxG~8tARUGWRbnfOdJjRz56IIWU2fB1LFRW7B9~7pUCk8hiL950KHWwuGVgm2CR7XzuN2Ol~Srxm1KDqSbvcHPhFpTcMAxXoy0MmzAk4azw3m6BS2zdKFbXWZDwIBAuWsyNs2OfeBlN9XbjYIu4qrwdmUoKNjD6lCTddZ~Stqw9zjOdvERsYLF-k1p3VFitSNuzE6K8qaLBuOpjwupC~kvDnOd6PE-~4SgmQIbnJmbYHlFIoER8OUJ3aN9qXXQZo1zzb6q9GUIkxy6r9dl8V~N-Y1F0O3RW88rxT1EddI9GKwPdZcd4GjkwK8tiW51Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=90360716&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5383" path-from-xml="FEM_21_f7.jpeg" section="90360716" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Naphthalene dioxygenase like <em>phnAc</em> gene detection by PCR amplification using the primer sets developed for <em>Burkholderia</em> sp. strain RP007 [<a class="link link-ref link-reveal xref-bibr" data-open="b7" reveal-id="b7">7</a>]. The PCR products were obtained from DNA samples extracted from soil microcosms not contaminated (I), artificially contaminated with naphthalene with no inoculation (II) and microcosms artificially contaminated with naphthalene inoculated with <em>P. putida</em> KT2442 (pNF142) (III).</p></div></div> <h3 class="section-title" data-legacy-id="ss3-5" id="90360717" scrollto-destination="90360717">3.5 Chemical analysis</h3>
<p>The addition of naphthalene as crystals to the soil led to rather large variations in naphthalene concentrations at the beginning of the experiments (195 ± 142 mg kg<sup>−1</sup> treatment II and 225.3 ± 156.7 mg kg<sup>−1</sup> treatment III) and after 3 days (335 ± 185.3 mg kg<sup>−1</sup> treatment II and 40 ± 37.3 mg kg<sup>−1</sup> treatment III). These variations were much smaller after 21 days (1 ± 1 mg kg<sup>−1</sup> treatment II and 68 ± 8.5 mg kg<sup>−1</sup> treatment III) probably due to sublimation of the substrate in the soil leading to a much more uniform distribution of the substrate in the soil pores.</p><p>Despite the low recovery rate and the variations of naphthalene in the samples during the first 3 days, very clear differences could be seen between the soil samples of treatments III and II. The data demonstrate that no further decrease in the naphthalene concentration occurred in treatment III because naphthalene concentrations at 21 dpi were similar to those at 3 dpi. One interesting exception from this observation is microcosm b of treatment III where no naphthalene could be found after 3 weeks (data not shown). This finding corroborates with the observation of a deviating microbial community structure in this sample after 3 weeks. In soils from treatment II hardly any reduction in substrate concentration could be found after 3 days. However, in clear contrast to treatment III, all three replicates of treatment II contained naphthalene concentrations near the detection limit of 1 mg kg<sup>−1</sup> soil at the last sampling point.</p> <h2 class="section-title" data-legacy-id="ss4" id="90360720" scrollto-destination="90360720">4 Discussion</h2>
<p>In contrast to previously published studies on the survival of inoculant strains, the emphasis of this study was on the microbial community response to naphthalene and to the inoculation of naphthalene-contaminated soil with the naphthalene-degrading strain <em>P. putida</em> KT2442 (pNF142). The ability of <em>P. putida</em> KT2442 (pNF142) to colonise bulk soil artificially contaminated with naphthalene and to express genes coding for key enzymes for naphthalene degradation was primarily assessed in a community context using DNA and RNA-based methods.</p><p>In the soil microcosm experiments performed in this study, the population density of the inoculant strain KT2442 (pNF142) increased during the first days by more than 1 order of magnitude. A similar increase in the population density of <em>P. putida</em> KT2442 carrying an IncPα plasmid was observed in manure-treated soil under field conditions [<a class="link link-ref link-reveal xref-bibr" data-open="b17" reveal-id="b17">17</a>]. This kind of increase was not observed in soil plots which were not treated with manure. Although in this study the survival of the inoculant strain was not followed in soil which was not amended with naphthalene, it is tempting to speculate that nutrient availability is a key factor for the survival of this strain in bulk soil. In contrast to the good survival in bulk soil observed in the presence of manure [<a class="link link-ref link-reveal xref-bibr" data-open="b17" reveal-id="b17">17</a>] or in the presence of high concentrations of naphthalene, rather poor survival was observed in pot experiments for <em>P. putida</em> KT2442 in bulk soil poor in organic matter [<a class="link link-ref link-reveal xref-bibr" data-open="b18" reveal-id="b18">18</a>]. Similarly, the cfu numbers of <em>P. putida</em> KT2442 introduced into non-vegetated field soil decreased constantly and the strain showed good colonisation of the rhizosphere when introduced as seed inoculum of corn and beans. The levels detected in the bulk soil were 1–2 orders of magnitude lower [<a class="link link-ref link-reveal xref-bibr" data-open="b19" reveal-id="b19">19</a>]. Due to the release of root exudates and rhizodeposits, the rhizosphere is considered to be a nutrient-rich environment compared to bulk soil and the availability of nutrients seems crucial for the colonisation ability of KT2442. In this study, KT2442 pNF142 showed good colonisation of soil, most likely due to the availability of a substrate degradable by the inoculant strain.</p><p>The DGGE analysis of 16S rRNA gene fragments amplified with bacterial primers from DNA or RNA showed only minor differences between the DNA- and RNA-based patterns for all samples analysed. This observation does not seem to be unusual for bulk soils. Nogales et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b20" reveal-id="b20">20</a>] found good correspondence of sequences in clone libraries derived from DNA or RNA extracted from contaminated soils. DGGE patterns are a display of the numerically dominant ribotypes [<a class="link link-ref link-reveal xref-bibr" data-open="b21" reveal-id="b21">21</a>]. Due to the numerical dominance of KT2442 (pNF142), the number of bands observed in the bacterial DGGE patterns was drastically reduced, suggesting tremendous changes in the bacterial community composition. Only after using group-specific primers which did not amplify the 16S rRNA gene of the inoculant strain did it become clear that at least the groups analysed were less severely affected by inoculation. Other studies have used DGGE analysis of 16S rRNA gene fragments amplified from DNA to follow inoculant strains and their effect on bacterial community composition [<a class="link link-ref link-reveal xref-bibr" data-open="b22" reveal-id="b22">22</a>,<a class="link link-ref link-reveal xref-bibr" data-open="b23" reveal-id="b23">23</a>]. While inoculant strains can only be detected in the community patterns when they belong to the numerically dominant populations, high abundance of the inoculant might disturb evaluation of potential effects on the composition of the bacterial community. This study showed that the use of group-specific primers can help overcome this problem. The use of group-specific primers to generate 16S rRNA gene fingerprints offers not only the advantage of reducing the complexity and detecting populations which are less abundant [<a class="link link-ref link-reveal xref-bibr" data-open="b11" reveal-id="b11">11</a>,<a class="link link-ref link-reveal xref-bibr" data-open="b16" reveal-id="b16">16</a>], but also of evaluation of the effects of inoculant strains on the structural composition of the bacterial community, which would otherwise have been masked by numerically dominant inoculant strains.</p><p>One finding of this study was that at least two bacterial populations were enriched from soil artificially contaminated with naphthalene which was not previously exposed to this pollutant. Twenty-one days after naphthalene contamination (II), two dominant bands appeared in all three replicates of the DGGE patterns which were not detected in patterns of non-contaminated soil samples (I). Based on sequence analysis, the enriched bacterial populations showed highest similarity with genera <em>Burkholderia</em> and <em>Nocardia</em>. Isolates belonging to these genera have previously been reported as naphthalene or phenanthrene degraders [<a class="link link-ref link-reveal xref-bibr" data-open="b24  b26" reveal-id="b24  b26">24–26</a>]. Interestingly, in a recent study by Wilson et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b26" reveal-id="b26">26</a>] most of the naphthalene-degrading soil isolates from a hillside adjacent to a seep belonged to different <em>Burkholderia</em> species, while all naphthalene-degrading bacteria isolated from sediments taken from the coal tar deposit showed highest similarity to different <em>Pseudomonas</em> species [<a class="link link-ref link-reveal xref-bibr" data-open="b26" reveal-id="b26">26</a>]. The sequence of band A (<a class="link link-reveal link-table xref-fig" data-open="f2" reveal-id="f2">Fig. 2</a> and <a class="link link-reveal link-table xref-fig" data-open="f4" reveal-id="f4">4</a>(a)) showed highest similarity to the 16S rRNA gene sequence of <em>Burkholderia</em> sp. RP007, which was isolated from a PAH-contaminated site in New Zealand [<a class="link link-ref link-reveal xref-bibr" data-open="b24" reveal-id="b24">24</a>]. It is unclear whether the slight differences in 16S rRNA gene sequences of the different clones sequenced result from 16S rRNA operon heterogeneities or from different but phylogenetically closely related populations. While in non-inoculated contaminated soil (II) a band co-migrating with the <em>Burkholderia</em> band could be detected in the β-proteobacterial fingerprints, a band corresponding to the enriched <em>Nocardia</em> population was barely detectable in the actinobacterial patterns. Thus it seems that this population was abundant at cell densities below the limit of detection. As in this study the enrichment of <em>Burkholderia</em> and <em>Nocardia</em> populations in naphthalene-contaminated soils was shown only by nucleic acid-based techniques, it remains speculative whether or not the enriched populations are indeed naphthalene degraders. However, it is most likely that they were enriched in response to the naphthalene contamination because these populations were either not observed or only present as weaker bands in the DGGE patterns in non-contaminated soil (treatment I). The relative abundance of the populations enriched in the presence of naphthalene was much lower when the soil was inoculated with KT2442 (pNF142) (treatment III; except replicate b). Furthermore, the intensity of band A correlates with degradation of naphthalene shown by HPLC analysis.</p><p>The <em>nahAc</em> and <em>nahH</em> genes were amplified from DNA extracted from soil samples of treatment III at all sampling times but not from soils which did not receive the inoculant strains. Obviously, <em>nahAc</em>- and <em>nahH</em>-containing bacterial populations were not enriched in soils contaminated with naphthalene (treatment II). This confirms findings by Laurie and Lloyd-Jones [<a class="link link-ref link-reveal xref-bibr" data-open="b27" reveal-id="b27">27</a>] who reported that the <em>Burkholderia</em>-derived naphthalene dioxygenase <em>phnAc</em> gene was present in different pristine soils from Siberia, the Antarctic and New Zealand but not the <em>nahAc</em> gene. Because the sequence of the enriched <em>Burkholderia</em> sp. populations showed highest similarity to the 16S rRNA gene sequence of <em>Burkholderia</em> sp. strain RP007, carrying a divergent set of PAH catabolic genes (<em>phn</em> genes), PCR amplification was used to test whether the <em>phnAc</em> gene was also enriched in our experiments. The detection of <em>phnAc</em> correlated with the appearance of the <em>Burkholderia</em> band in the bacterial DGGE patterns, supporting the idea that populations capable to degrade naphthalene were indeed enriched from soil with no previous history of naphthalene contamination.</p><p>By using PCR amplification of cDNA combined with Southern blot hybridisation we could furthermore demonstrate <em>nahAc</em> gene expression in soils of treatment III. In contrast, <em>nahH</em> expression was detected in one of the three replicates only 3 and 21 dpi, respectively. There may be several reasons for the fact that <em>nahH</em> gene expression was not detected at all sampling times and in particular not at 0 dpi. Gene expression of <em>nahH</em> might be induced after exposure to naphthalene, or mRNA might be less stable compared to the transcript of <em>nahAc</em>.</p><p>After 3 weeks two of the three microcosms (a, c) containing strain KT2442 (pNF142) run in parallel (treatment III) showed only moderate degradation of the hydrocarbon, while the third and all three microcosms amended only with naphthalene (treatment II) showed a naphthalene concentration below the detection limit. A comparison of the β-proteobacterial community patterns at 21 dpi showed that all microcosms with excellent naphthalene degradation fell into one separate cluster (IIa–c, IIIb) that shared only 60% similarity with the patterns of the remaining profiles.</p><p>The finding that a rapid initial decrease in the naphthalene concentration occurred in all three replicates of treatment III and the subsequent cessation of its removal might be explained by decreased bioavailability of naphthalene. Naphthalene is a hydrophobic organic contaminant and its bioavailability can be reduced due to its sequestration by the mineral and organic matter fraction of soil [<a class="link link-ref link-reveal xref-bibr" data-open="b28" reveal-id="b28">28</a>]. The longer the time of contact the lower is its bioavailability. In contrast to treatment III, the microbial community of treatment II showed a higher ability to reduce naphthalene concentrations near to 1 mg kg<sup>−1</sup> (21 dpi). The two populations related to bacteria belonging to the genus <em>Burkholderia</em> and <em>Nocardia</em> were enriched, and these may have specific physiological properties that could have contributed to enhanced bioavailability of naphthalene. The hydrophobic cell surfaces and the production of biosurfactants, which facilitate the degradation of less available PAH, have been described at least for members of the Nocardia–Rhodococcus–Mycobacterium group [<a class="link link-ref link-reveal xref-bibr" data-open="b29" reveal-id="b29">29</a>,<a class="link link-ref link-reveal xref-bibr" data-open="b30" reveal-id="b30">30</a>]. Wagner-Döbler et al. [<a class="link link-ref link-reveal xref-bibr" data-open="b31" reveal-id="b31">31</a>] showed that biphenyl-degrading bacteria could be enriched from soils and sediments of both polluted and non-polluted sites after 6 months. The majority of the biphenyl-degrading isolates belonged to the genus <em>Rhodococcus</em>.</p><p>A link between structural diversity and function could be made because the observed enrichment of a population with highest similarity of the 16S rRNA gene to <em>Burkholderia</em> sp. RP007 was confirmed by detection of a functional gene (<em>phnAc</em>). Although KT2442 (pNF142) showed excellent survival and expression of genes involved in naphthalene degradation, its introduction prevented enrichment of indigenous naphthalene degraders. Thus we suggest that in soils affected by PAH contamination (e.g. after an oil spill) degrading potential of the indigenous soil bacterial community in the first phase should be exploited before the need to introduce a degrading strain or microbial consortium in a second phase is considered.</p> <h2 class="backacknowledgements-title" id="90360730" scrollto-destination="90360730">Acknowledgements</h2>
<p>This work has been supported by a DAAD postdoctoral fellowship and by INTAS Grants 99-01487 and 01-2383.</p> <h2 class="backreferences-title" id="90360732" scrollto-destination="90360732">References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b1"><div class="label">[1]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Muyzer</div>
<div class="given-names">G.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div> (<div class="year">1998</div>) <div class="article-title">The need for DGGE and TGGE in microbial ecology</div>. <div class="source">Antonie van Leeuwenhoek</div> <div class="volume">73</div>, <div class="fpage">127</div>–<div class="lpage">141</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20need%20for%20DGGE%20and%20TGGE%20in%20microbial%20ecology&amp;author=G.%20Muyzer&amp;author=K.%20Smalla&amp;publication_year=1998&amp;journal=Antonie%20van%20Leeuwenhoek&amp;volume=73&amp;pages=127-141" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1000669317571" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1000669317571" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1023%2FA:1000669317571"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1023%2FA:1000669317571" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9602286" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:The%20need%20for%20DGGE%20and%20TGGE%20in%20microbial%20ecology&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b2"><div class="label">[2]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Wilson</div>
<div class="given-names">M.S.</div>
</div>
<div class="name">
<div class="surname">Bakermans</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Madsen</div>
<div class="given-names">E.L.</div>
</div>
</div> (<div class="year">1999</div>) <div class="article-title">In situ, real-time catabolic gene expression: extraction and characterization of naphthalene dioxygenase mRNA transcripts from groundwater</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">65</div>, <div class="fpage">80</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20situ%2C%20real-time%20catabolic%20gene%20expression%3A%20extraction%20and%20characterization%20of%20naphthalene%20dioxygenase%20mRNA%20transcripts%20from%20groundwater&amp;author=M.S.%20Wilson&amp;author=C.%20Bakermans&amp;author=E.L.%20Madsen&amp;publication_year=1999&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=65&amp;pages=80-87" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9872763" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:In%20situ%2C%20real-time%20catabolic%20gene%20expression%3A%20extraction%20and%20characterization%20of%20naphthalene%20dioxygenase%20mRNA%20transcripts%20from%20groundwater&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b3"><div class="label">[3]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Master</div>
<div class="given-names">E.R.</div>
</div>
<div class="name">
<div class="surname">Mohn</div>
<div class="given-names">W.W.</div>
</div>
</div> (<div class="year">1998</div>) <div class="article-title">Psychrotolerant bacteria isolated from arctic soil that degrade polychlorinated biphenyls at low temperatures</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">64</div>, <div class="fpage">4823</div>–<div class="lpage">4829</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Psychrotolerant%20bacteria%20isolated%20from%20arctic%20soil%20that%20degrade%20polychlorinated%20biphenyls%20at%20low%20temperatures&amp;author=E.R.%20Master&amp;author=W.W.%20Mohn&amp;publication_year=1998&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=64&amp;pages=4823-4829" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9835569" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Psychrotolerant%20bacteria%20isolated%20from%20arctic%20soil%20that%20degrade%20polychlorinated%20biphenyls%20at%20low%20temperatures&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b4"><div class="label">[4]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Krasowiak</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Sokolov</div>
<div class="given-names">S.</div>
</div>
<div class="name">
<div class="surname">Kosheleva</div>
<div class="given-names">I.</div>
</div>
<div class="name">
<div class="surname">Sevastyanovich</div>
<div class="given-names">Y.</div>
</div>
<div class="name">
<div class="surname">Titok</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Thomas</div>
<div class="given-names">C.M.</div>
</div>
</div> (<div class="year">2002</div>) <div class="article-title">PCR primers for detection and characterisation of IncP-9 plasmids</div>. <div class="source">FEMS Microbiol. Ecol</div> . <div class="volume">42</div>, <div class="fpage">217</div>–<div class="lpage">225</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR%20primers%20for%20detection%20and%20characterisation%20of%20IncP-9%20plasmids&amp;author=R.%20Krasowiak&amp;author=K.%20Smalla&amp;author=S.%20Sokolov&amp;author=I.%20Kosheleva&amp;author=Y.%20Sevastyanovich&amp;author=M.%20Titok&amp;author=C.M.%20Thomas&amp;publication_year=2002&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=42&amp;pages=217-225" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2002.42.issue-2" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2002.42.issue-2" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1111%2Ffem.2002.42.issue-2"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1111%2Ffem.2002.42.issue-2" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/19709281" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:PCR%20primers%20for%20detection%20and%20characterisation%20of%20IncP-9%20plasmids&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b5"><div class="label">[5]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Hurt</div>
<div class="given-names">R.A.</div>
</div>
<div class="name">
<div class="surname">Qiu</div>
<div class="given-names">X.</div>
</div>
<div class="name">
<div class="surname">Wu</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Roh</div>
<div class="given-names">Y.</div>
</div>
<div class="name">
<div class="surname">Palumbo</div>
<div class="given-names">A.V.</div>
</div>
<div class="name">
<div class="surname">Tiedje</div>
<div class="given-names">J.M.</div>
</div>
<div class="name">
<div class="surname">Zhou</div>
<div class="given-names">J.</div>
</div>
</div> (<div class="year">2001</div>) <div class="article-title">Simultaneous recovery of RNA and DNA from soils and sediments</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">67</div>, <div class="fpage">4495</div>–<div class="lpage">4503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simultaneous%20recovery%20of%20RNA%20and%20DNA%20from%20soils%20and%20sediments&amp;author=R.A.%20Hurt&amp;author=X.%20Qiu&amp;author=L.%20Wu&amp;author=Y.%20Roh&amp;author=A.V.%20Palumbo&amp;author=J.M.%20Tiedje&amp;author=J.%20Zhou&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=67&amp;pages=4495-4503" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.10.4495-4503.2001" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.10.4495-4503.2001" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.67.10.4495-4503.2001"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.67.10.4495-4503.2001" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/11571148" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Simultaneous%20recovery%20of%20RNA%20and%20DNA%20from%20soils%20and%20sediments&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b6"><div class="label">[6]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Costa</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Gomes</div>
<div class="given-names">N.C.M.</div>
</div>
<div class="name">
<div class="surname">Milling</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div> (<div class="year">2004</div>) <div class="article-title">An optimized protocol for simultaneous extraction of DNA and RNA from soils</div>. <div class="source">Braz. J. Microbiol</div> . <div class="volume">35</div>, <div class="fpage">230</div>–<div class="lpage">234</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20optimized%20protocol%20for%20simultaneous%20extraction%20of%20DNA%20and%20RNA%20from%20soils&amp;author=R.%20Costa&amp;author=N.C.M.%20Gomes&amp;author=A.%20Milling&amp;author=K.%20Smalla&amp;publication_year=2004&amp;journal=Braz.%20J.%20Microbiol&amp;volume=35&amp;pages=230-234" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1517-83822004000200011" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1517-83822004000200011" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1590%2FS1517-83822004000200011"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1590%2FS1517-83822004000200011" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:An%20optimized%20protocol%20for%20simultaneous%20extraction%20of%20DNA%20and%20RNA%20from%20soils&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b7"><div class="label">[7]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Lloyd-Jones</div>
<div class="given-names">G.</div>
</div>
<div class="name">
<div class="surname">Laurie</div>
<div class="given-names">A.D.</div>
</div>
<div class="name">
<div class="surname">Hunter</div>
<div class="given-names">D.W.F.</div>
</div>
<div class="name">
<div class="surname">Fraser</div>
<div class="given-names">R.</div>
</div>
</div> (<div class="year">1999</div>) <div class="article-title">Analysis of catabolic genes for naphthalene and phenanthrene degradation in contaminated New Zealand soils</div>. <div class="source">FEMS Microbiol. Ecol</div> . <div class="volume">29</div>, <div class="fpage">69</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20catabolic%20genes%20for%20naphthalene%20and%20phenanthrene%20degradation%20in%20contaminated%20New%20Zealand%20soils&amp;author=G.%20Lloyd-Jones&amp;author=A.D.%20Laurie&amp;author=D.W.F.%20Hunter&amp;author=R.%20Fraser&amp;publication_year=1999&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=29&amp;pages=69-79" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.1999.29.issue-1" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.1999.29.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1111%2Ffem.1999.29.issue-1"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1111%2Ffem.1999.29.issue-1" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Analysis%20of%20catabolic%20genes%20for%20naphthalene%20and%20phenanthrene%20degradation%20in%20contaminated%20New%20Zealand%20soils&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b8"><div class="label">[8]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Wikström</div>
<div class="given-names">P.</div>
</div>
<div class="name">
<div class="surname">Wiklund</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Andersson</div>
<div class="given-names">A.C.</div>
</div>
<div class="name">
<div class="surname">Forsman</div>
<div class="given-names">M.</div>
</div>
</div> (<div class="year">1996</div>) <div class="article-title">DNA recovery and PCR quantification of catechol 2,3-dioxygenase genes from different soil types</div>. <div class="source">J. Biotechnol</div> . <div class="volume">52</div>, <div class="fpage">107</div>–<div class="lpage">120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20recovery%20and%20PCR%20quantification%20of%20catechol%202%2C3-dioxygenase%20genes%20from%20different%20soil%20types&amp;author=P.%20Wikstr%C3%B6m&amp;author=A.%20Wiklund&amp;author=A.C.%20Andersson&amp;author=M.%20Forsman&amp;publication_year=1996&amp;journal=J.%20Biotechnol&amp;volume=52&amp;pages=107-120" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-1656(96)01635-5" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-1656(96)01635-5" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1016%2FS0168-1656(96)01635-5"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1016%2FS0168-1656(96)01635-5" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9084210" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:DNA%20recovery%20and%20PCR%20quantification%20of%20catechol%202%2C3-dioxygenase%20genes%20from%20different%20soil%20types&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b9"><div class="label">[9]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Sambrook</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Fritsch</div>
<div class="given-names">E.F.</div>
</div>
<div class="name">
<div class="surname">Maniatis</div>
<div class="given-names">T.</div>
</div>
</div>
<div class="source">Molecular cloning: a laboratory manual</div> 
<div class="year">1989</div>
<div class="publisher-name">Cold Spring Harbor Laboratory Press</div>
<div class="publisher-loc">Cold Spring Harbor, NY</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a href="https://www.google.com/search?q=Molecular%20cloning%3A%20a%20laboratory%20manual&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="">Google Preview</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Molecular%20cloning%3A%20a%20laboratory%20manual&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b10"><div class="label">[10]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Fulthorpe</div>
<div class="given-names">R.R.</div>
</div>
<div class="name">
<div class="surname">McGowan</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Maltseva</div>
<div class="given-names">O.V.</div>
</div>
<div class="name">
<div class="surname">Holben</div>
<div class="given-names">W.E.</div>
</div>
<div class="name">
<div class="surname">Tiedje</div>
<div class="given-names">J.M.</div>
</div>
</div> (<div class="year">1995</div>) <div class="article-title">2,4-Dichlorophenoxyacetic acid degrading bacteria contain mosaics of catabolic genes</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">61</div>, <div class="fpage">3274</div>–<div class="lpage">3281</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=2%2C4-Dichlorophenoxyacetic%20acid%20degrading%20bacteria%20contain%20mosaics%20of%20catabolic%20genes&amp;author=R.R.%20Fulthorpe&amp;author=C.%20McGowan&amp;author=O.V.%20Maltseva&amp;author=W.E.%20Holben&amp;author=J.M.%20Tiedje&amp;publication_year=1995&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=61&amp;pages=3274-3281" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/7574638" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:2%2C4-Dichlorophenoxyacetic%20acid%20degrading%20bacteria%20contain%20mosaics%20of%20catabolic%20genes&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b11"><div class="label">[11]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Krsek</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Wellington</div>
<div class="given-names">E.M.H.</div>
</div>
</div> (<div class="year">1997</div>) <div class="article-title">Analysis of actinomycete communities by specific amplification of 16S rRNA and gel electrophoretic separation in denaturing gradients</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">63</div>, <div class="fpage">3233</div>–<div class="lpage">3241</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20actinomycete%20communities%20by%20specific%20amplification%20of%2016S%20rRNA%20and%20gel%20electrophoretic%20separation%20in%20denaturing%20gradients&amp;author=H.%20Heuer&amp;author=M.%20Krsek&amp;author=K.%20Smalla&amp;author=E.M.H.%20Wellington&amp;publication_year=1997&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=63&amp;pages=3233-3241" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9251210" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Analysis%20of%20actinomycete%20communities%20by%20specific%20amplification%20of%2016S%20rRNA%20and%20gel%20electrophoretic%20separation%20in%20denaturing%20gradients&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b12"><div class="label">[12]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Cremonesi</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Firpo</div>
<div class="given-names">S.</div>
</div>
<div class="name">
<div class="surname">Ferrari</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Righetti</div>
<div class="given-names">P.G.</div>
</div>
<div class="name">
<div class="surname">Gelfi</div>
<div class="given-names">C.</div>
</div>
</div> (<div class="year">1997</div>) <div class="article-title">Double-gradient DGGE for optimized detection of DNA point mutations</div>. <div class="source">Biotechniques</div> <div class="volume">22</div>, <div class="fpage">326</div>–<div class="lpage">330</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double-gradient%20DGGE%20for%20optimized%20detection%20of%20DNA%20point%20mutations&amp;author=L.%20Cremonesi&amp;author=S.%20Firpo&amp;author=M.%20Ferrari&amp;author=P.G.%20Righetti&amp;author=C.%20Gelfi&amp;publication_year=1997&amp;journal=Biotechniques&amp;volume=22&amp;pages=326-330" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9043706" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Double-gradient%20DGGE%20for%20optimized%20detection%20of%20DNA%20point%20mutations&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b13"><div class="label">[13]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Wieland</div>
<div class="given-names">G.</div>
</div>
<div class="name">
<div class="surname">Schönfeld</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Schönwälder</div>
<div class="given-names">S.</div>
</div>
<div class="name">
<div class="surname">Gomes</div>
<div class="given-names">N.C.M.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div>
<div class="article-title">Bacterial community profiling using DGGE or TGGE analysis</div>
<div class="person-group">
<div class="name">
<div class="surname">Rouchelle</div>
<div class="given-names">P.</div>
</div>
</div>
<div class="source">Environmental molecular microbiology: protocols and applications</div> 
<div class="year">2001</div>
<div class="publisher-name">Horizon Scientific Press</div>
<div class="publisher-loc">Wymondham, UK</div>
<div class="fpage">177</div>
<div class="lpage">190</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a href="https://www.google.com/search?q=Environmental%20molecular%20microbiology%3A%20protocols%20and%20applications&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="">Google Preview</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Environmental%20molecular%20microbiology%3A%20protocols%20and%20applications&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b14"><div class="label">[14]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Schwieger</div>
<div class="given-names">F.</div>
</div>
<div class="name">
<div class="surname">Tebbe</div>
<div class="given-names">C.C.</div>
</div>
</div> (<div class="year">1998</div>) <div class="article-title">A new approach to utilize PCR-single-strand-conformation polymorphism for 16S rRNA gene-based microbial community analysis</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">64</div>, <div class="fpage">4870</div>–<div class="lpage">4876</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20approach%20to%20utilize%20PCR-single-strand-conformation%20polymorphism%20for%2016S%20rRNA%20gene-based%20microbial%20community%20analysis&amp;author=F.%20Schwieger&amp;author=C.C.%20Tebbe&amp;publication_year=1998&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=64&amp;pages=4870-4876" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9835576" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:A%20new%20approach%20to%20utilize%20PCR-single-strand-conformation%20polymorphism%20for%2016S%20rRNA%20gene-based%20microbial%20community%20analysis&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b15"><div class="label">[15]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Wieland</div>
<div class="given-names">G.</div>
</div>
<div class="name">
<div class="surname">Buchner</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Zock</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Parzy</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Kaiser</div>
<div class="given-names">S.</div>
</div>
</div> (<div class="year">2001</div>) <div class="article-title">Bulk and rhizosphere soil bacterial communities studied by denaturing gradient gel electrophoresis: plant-dependent enrichment and seasonal shifts revealed</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">67</div>, <div class="fpage">4742</div>–<div class="lpage">4751</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bulk%20and%20rhizosphere%20soil%20bacterial%20communities%20studied%20by%20denaturing%20gradient%20gel%20electrophoresis%3A%20plant-dependent%20enrichment%20and%20seasonal%20shifts%20revealed&amp;author=K.%20Smalla&amp;author=G.%20Wieland&amp;author=A.%20Buchner&amp;author=A.%20Zock&amp;author=J.%20Parzy&amp;author=S.%20Kaiser&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=67&amp;pages=4742-4751" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.10.4742-4751.2001" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.10.4742-4751.2001" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.67.10.4742-4751.2001"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.67.10.4742-4751.2001" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/11571180" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Bulk%20and%20rhizosphere%20soil%20bacterial%20communities%20studied%20by%20denaturing%20gradient%20gel%20electrophoresis%3A%20plant-dependent%20enrichment%20and%20seasonal%20shifts%20revealed&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b16"><div class="label">[16]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Gomes</div>
<div class="given-names">N.C.M.</div>
</div>
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Schönfeld</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Costa</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Hagler-Mendonça</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div> (<div class="year">2001</div>) <div class="article-title">Bacterial diversity of the rhizosphere of maize (<em>Zea mays</em>) grown in tropical soil studied by temperature gradient gel electrophoresis</div>. <div class="source">Plant Soil</div> <div class="volume">232</div>, <div class="fpage">167</div>–<div class="lpage">180</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20diversity%20of%20the%20rhizosphere%20of%20maize%20(Zea%20mays)%20grown%20in%20tropical%20soil%20studied%20by%20temperature%20gradient%20gel%20electrophoresis&amp;author=N.C.M.%20Gomes&amp;author=H.%20Heuer&amp;author=J.%20Sch%C3%B6nfeld&amp;author=R.%20Costa&amp;author=L.%20Hagler-Mendon%C3%A7a&amp;author=K.%20Smalla&amp;publication_year=2001&amp;journal=Plant%20Soil&amp;volume=232&amp;pages=167-180" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1010350406708" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1010350406708" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1023%2FA:1010350406708"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1023%2FA:1010350406708" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Bacterial%20diversity%20of%20the%20rhizosphere%20of%20maize%20(Zea%20mays)%20grown%20in%20tropical%20soil%20studied%20by%20temperature%20gradient%20gel%20electrophoresis&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b17"><div class="label">[17]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Götz</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div> (<div class="year">1997</div>) <div class="article-title">Manure enhances plasmid mobilization and survival of <em>Pseudomonas putida</em> introduced into field soil</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">63</div>, <div class="fpage">1980</div>–<div class="lpage">1986</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Manure%20enhances%20plasmid%20mobilization%20and%20survival%20of%20Pseudomonas%20putida%20introduced%20into%20field%20soil&amp;author=A.%20G%C3%B6tz&amp;author=K.%20Smalla&amp;publication_year=1997&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=63&amp;pages=1980-1986" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/16535608" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Manure%20enhances%20plasmid%20mobilization%20and%20survival%20of%20Pseudomonas%20putida%20introduced%20into%20field%20soil&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b18"><div class="label">[18]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Ramos</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Molina</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Mølbak</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Ramos</div>
<div class="given-names">J.L.</div>
</div>
<div class="name">
<div class="surname">Molin</div>
<div class="given-names">S.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">A bioluminescent derivative of <em>Pseudomonas putida</em> KT2440 for deliberate release into the environment</div>. <div class="source">FEMS Microbiol. Ecol</div> . <div class="volume">34</div>, <div class="fpage">91</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20bioluminescent%20derivative%20of%20Pseudomonas%20putida%20KT2440%20for%20deliberate%20release%20into%20the%20environment&amp;author=C.%20Ramos&amp;author=L.%20Molina&amp;author=L.%20M%C3%B8lbak&amp;author=J.L.%20Ramos&amp;author=S.%20Molin&amp;publication_year=2000&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=34&amp;pages=91-102" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2000.34.issue-2" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2000.34.issue-2" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1111%2Ffem.2000.34.issue-2"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1111%2Ffem.2000.34.issue-2" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/11102686" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:A%20bioluminescent%20derivative%20of%20Pseudomonas%20putida%20KT2440%20for%20deliberate%20release%20into%20the%20environment&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b19"><div class="label">[19]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Molina</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Ramos</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Duque</div>
<div class="given-names">E.</div>
</div>
<div class="name">
<div class="surname">Ronchel</div>
<div class="given-names">M.C.</div>
</div>
<div class="name">
<div class="surname">García</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Wyke</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Ramos</div>
<div class="given-names">J.L.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">Survival of <em>Pseudomonas putida</em> KT2440 in soil and in the rhizosphere of plants under greenhouse and environmental conditions</div>. <div class="source">Soil Biol. Biochem</div> . <div class="volume">32</div>, <div class="fpage">315</div>–<div class="lpage">321</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survival%20of%20Pseudomonas%20putida%20KT2440%20in%20soil%20and%20in%20the%20rhizosphere%20of%20plants%20under%20greenhouse%20and%20environmental%20conditions&amp;author=L.%20Molina&amp;author=C.%20Ramos&amp;author=E.%20Duque&amp;author=M.C.%20Ronchel&amp;author=M.%20Garc%C3%ADa&amp;author=L.%20Wyke&amp;author=J.L.%20Ramos&amp;publication_year=2000&amp;journal=Soil%20Biol.%20Biochem&amp;volume=32&amp;pages=315-321" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0038-0717(99)00156-X" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0038-0717(99)00156-X" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1016%2FS0038-0717(99)00156-X"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1016%2FS0038-0717(99)00156-X" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Survival%20of%20Pseudomonas%20putida%20KT2440%20in%20soil%20and%20in%20the%20rhizosphere%20of%20plants%20under%20greenhouse%20and%20environmental%20conditions&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b20"><div class="label">[20]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Nogales</div>
<div class="given-names">B.</div>
</div>
<div class="name">
<div class="surname">Moore</div>
<div class="given-names">E.R.B.</div>
</div>
<div class="name">
<div class="surname">Llobet-Brossa</div>
<div class="given-names">E.</div>
</div>
<div class="name">
<div class="surname">Rossello-Mora</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Amann</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Timmis</div>
<div class="given-names">N.K.</div>
</div>
</div> (<div class="year">2001</div>) <div class="article-title">Combined use of 16S Ribosomal DNA and 16S rRNA to study the bacterial community of polychlorinated biphenyl-polluted soil</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">67</div>, <div class="fpage">1874</div>–<div class="lpage">1884</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20use%20of%2016S%20Ribosomal%20DNA%20and%2016S%20rRNA%20to%20study%20the%20bacterial%20community%20of%20polychlorinated%20biphenyl-polluted%20soil&amp;author=B.%20Nogales&amp;author=E.R.B.%20Moore&amp;author=E.%20Llobet-Brossa&amp;author=R.%20Rossello-Mora&amp;author=R.%20Amann&amp;author=N.K.%20Timmis&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=67&amp;pages=1874-1884" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.4.1874-1884.2001" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.4.1874-1884.2001" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.67.4.1874-1884.2001"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.67.4.1874-1884.2001" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/11282645" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Combined%20use%20of%2016S%20Ribosomal%20DNA%20and%2016S%20rRNA%20to%20study%20the%20bacterial%20community%20of%20polychlorinated%20biphenyl-polluted%20soil&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b21"><div class="label">[21]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
</div>
<div class="article-title">Application of denaturing gradient gel electrophoresis and temperature gradient gel electrophoresis for studying soil microbial communities</div>
<div class="person-group">
<div class="name">
<div class="surname">Elsas</div>
<div class="given-names">J.D. van</div>
</div>
<div class="name">
<div class="surname">Wellington</div>
<div class="given-names">E.M.H.</div>
</div>
<div class="name">
<div class="surname">Trevors</div>
<div class="given-names">J.T.</div>
</div>
</div>
<div class="source">Modern soil microbiology</div> 
<div class="year">1997</div>
<div class="publisher-name">Marcel Dekker</div>
<div class="publisher-loc">New York</div>
<div class="fpage">353</div>
<div class="lpage">373</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a href="https://www.google.com/search?q=Modern%20soil%20microbiology&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="">Google Preview</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Modern%20soil%20microbiology&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b22"><div class="label">[22]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Dybas</div>
<div class="given-names">M.J.</div>
</div>
<div class="name">
<div class="surname">Barcelona</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Bezborodnikov</div>
<div class="given-names">S.</div>
</div>
<div class="name">
<div class="surname">Davies</div>
<div class="given-names">S.</div>
</div>
<div class="name">
<div class="surname">Forney</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Kawka</div>
<div class="given-names">O.</div>
</div>
<div class="name">
<div class="surname">Mayotte</div>
<div class="given-names">T.</div>
</div>
<div class="name">
<div class="surname">L</div>
<div class="given-names">Smalla K.</div>
</div>
<div class="name">
<div class="surname">Sneathen</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Tiedje</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Voice</div>
<div class="given-names">T.</div>
</div>
<div class="name">
<div class="surname">Wiggert</div>
<div class="given-names">D.C.</div>
</div>
<div class="name">
<div class="surname">Witt</div>
<div class="given-names">M.E.</div>
</div>
<div class="name">
<div class="surname">Criddle</div>
<div class="given-names">C.S.</div>
</div>
</div> (<div class="year">1998</div>) <div class="article-title">Pilot-scale evaluation of bioaugmentation for in-situ remediation of a carbon tetrachloride contaminated aquifer</div>. <div class="source">Environ. Sci. Technol</div> . <div class="volume">32</div>, <div class="fpage">3598</div>–<div class="lpage">3611</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pilot-scale%20evaluation%20of%20bioaugmentation%20for%20in-situ%20remediation%20of%20a%20carbon%20tetrachloride%20contaminated%20aquifer&amp;author=M.J.%20Dybas&amp;author=M.%20Barcelona&amp;author=S.%20Bezborodnikov&amp;author=S.%20Davies&amp;author=L.%20Forney&amp;author=H.%20Heuer&amp;author=O.%20Kawka&amp;author=T.%20Mayotte&amp;author=Smalla%20K.%20L&amp;author=M.%20Sneathen&amp;author=J.%20Tiedje&amp;author=T.%20Voice&amp;author=D.C.%20Wiggert&amp;author=M.E.%20Witt&amp;author=C.S.%20Criddle&amp;publication_year=1998&amp;journal=Environ.%20Sci.%20Technol&amp;volume=32&amp;pages=3598-3611" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/es980200z" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fes980200z" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1021%2Fes980200z"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1021%2Fes980200z" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Pilot-scale%20evaluation%20of%20bioaugmentation%20for%20in-situ%20remediation%20of%20a%20carbon%20tetrachloride%20contaminated%20aquifer&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b23"><div class="label">[23]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Lottmann</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Heuer</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Vries</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Mahn</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Düring</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Wackernagel</div>
<div class="given-names">W.</div>
</div>
<div class="name">
<div class="surname">Smalla</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Berg</div>
<div class="given-names">G.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">Establishment of introduced antagonistic bacteria in the rhizosphere of transgenic potatoes and their effect on the bacterial community</div>. <div class="source">FEMS Microbiol. Ecol</div> . <div class="volume">33</div>, <div class="fpage">41</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20introduced%20antagonistic%20bacteria%20in%20the%20rhizosphere%20of%20transgenic%20potatoes%20and%20their%20effect%20on%20the%20bacterial%20community&amp;author=J.%20Lottmann&amp;author=H.%20Heuer&amp;author=J.%20Vries&amp;author=A.%20Mahn&amp;author=K.%20D%C3%BCring&amp;author=W.%20Wackernagel&amp;author=K.%20Smalla&amp;author=G.%20Berg&amp;publication_year=2000&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=33&amp;pages=41-49" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2000.33.issue-1" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2000.33.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1111%2Ffem.2000.33.issue-1"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1111%2Ffem.2000.33.issue-1" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/10922502" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Establishment%20of%20introduced%20antagonistic%20bacteria%20in%20the%20rhizosphere%20of%20transgenic%20potatoes%20and%20their%20effect%20on%20the%20bacterial%20community&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b24"><div class="label">[24]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Laurie</div>
<div class="given-names">A.D.</div>
</div>
<div class="name">
<div class="surname">Lloyd-Jones</div>
<div class="given-names">G.</div>
</div>
</div> (<div class="year">1999</div>) <div class="article-title">The <em>phn</em> genes of <em>Burkholderia</em> sp. strain RP007 constitute a divergent gene cluster for polycyclic aromatic hydrocarbon catabolism</div>. <div class="source">J. Bacteriol</div> . <div class="volume">181</div>, <div class="fpage">531</div>–<div class="lpage">540</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20phn%20genes%20of%20Burkholderia%20sp.%20strain%20RP007%20constitute%20a%20divergent%20gene%20cluster%20for%20polycyclic%20aromatic%20hydrocarbon%20catabolism&amp;author=A.D.%20Laurie&amp;author=G.%20Lloyd-Jones&amp;publication_year=1999&amp;journal=J.%20Bacteriol&amp;volume=181&amp;pages=531-540" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9882667" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:The%20phn%20genes%20of%20Burkholderia%20sp.%20strain%20RP007%20constitute%20a%20divergent%20gene%20cluster%20for%20polycyclic%20aromatic%20hydrocarbon%20catabolism&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b25"><div class="label">[25]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Widada</div>
<div class="given-names">J.</div>
</div>
<div class="name">
<div class="surname">Nojiri</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Kasuga</div>
<div class="given-names">K.</div>
</div>
<div class="name">
<div class="surname">Yoshida</div>
<div class="given-names">T.</div>
</div>
<div class="name">
<div class="surname">Habe</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Omori</div>
<div class="given-names">T.</div>
</div>
</div> (<div class="year">2002</div>) <div class="article-title">Molecular detection and diversity of polycyclic aromatic hydrocarbon-degrading bacteria isolated from geographically diverse sites</div>. <div class="source">Appl. Microbiol. Biotechnol</div> . <div class="volume">58</div>, <div class="fpage">202</div>–<div class="lpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20detection%20and%20diversity%20of%20polycyclic%20aromatic%20hydrocarbon-degrading%20bacteria%20isolated%20from%20geographically%20diverse%20sites&amp;author=J.%20Widada&amp;author=H.%20Nojiri&amp;author=K.%20Kasuga&amp;author=T.%20Yoshida&amp;author=H.%20Habe&amp;author=T.%20Omori&amp;publication_year=2002&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=58&amp;pages=202-209" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-001-0880-9" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-001-0880-9" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00253-001-0880-9"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1007%2Fs00253-001-0880-9" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/11876413" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Molecular%20detection%20and%20diversity%20of%20polycyclic%20aromatic%20hydrocarbon-degrading%20bacteria%20isolated%20from%20geographically%20diverse%20sites&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b26"><div class="label">[26]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Wilson</div>
<div class="given-names">M.S.</div>
</div>
<div class="name">
<div class="surname">Herrick</div>
<div class="given-names">J.B.</div>
</div>
<div class="name">
<div class="surname">Che Ok Jeon</div>
<div class="given-names">Hinman D.E.</div>
</div>
<div class="name">
<div class="surname">Madsen</div>
<div class="given-names">E.L.</div>
</div>
</div> (<div class="year">2003</div>) <div class="article-title">Horizontal transfer of <em>phnAc</em> dioxygenase genes within one of two phenotypically and genotypically distinctive naphthalene-degrading guilds from adjacent soil environments</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">69</div>, <div class="fpage">2172</div>–<div class="lpage">2181</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20transfer%20of%20phnAc%20dioxygenase%20genes%20within%20one%20of%20two%20phenotypically%20and%20genotypically%20distinctive%20naphthalene-degrading%20guilds%20from%20adjacent%20soil%20environments&amp;author=M.S.%20Wilson&amp;author=J.B.%20Herrick&amp;author=Hinman%20D.E.%20Che%20Ok%20Jeon&amp;author=E.L.%20Madsen&amp;publication_year=2003&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=69&amp;pages=2172-2181" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.69.4.2172-2181.2003" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.69.4.2172-2181.2003" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.69.4.2172-2181.2003"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.69.4.2172-2181.2003" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/12676698" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Horizontal%20transfer%20of%20phnAc%20dioxygenase%20genes%20within%20one%20of%20two%20phenotypically%20and%20genotypically%20distinctive%20naphthalene-degrading%20guilds%20from%20adjacent%20soil%20environments&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b27"><div class="label">[27]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Laurie</div>
<div class="given-names">A.D.</div>
</div>
<div class="name">
<div class="surname">Lloyd-Jones</div>
<div class="given-names">G.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">Quantification of <em>phnAc</em> and <em>nahAc</em> in contaminated New Zealand soils by competitive PCR</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">66</div>, <div class="fpage">1814</div>–<div class="lpage">1817</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantification%20of%20phnAc%20and%20nahAc%20in%20contaminated%20New%20Zealand%20soils%20by%20competitive%20PCR&amp;author=A.D.%20Laurie&amp;author=G.%20Lloyd-Jones&amp;publication_year=2000&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=66&amp;pages=1814-1817" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.66.5.1814-1817.2000" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.66.5.1814-1817.2000" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.66.5.1814-1817.2000"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.66.5.1814-1817.2000" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/10788344" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Quantification%20of%20phnAc%20and%20nahAc%20in%20contaminated%20New%20Zealand%20soils%20by%20competitive%20PCR&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b28"><div class="label">[28]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Semple</div>
<div class="given-names">K.T.</div>
</div>
<div class="name">
<div class="surname">Morriss</div>
<div class="given-names">A.W.J.</div>
</div>
<div class="name">
<div class="surname">Paton</div>
<div class="given-names">G.I.</div>
</div>
</div> (<div class="year">2003</div>) <div class="article-title">Bioavailability of hydrophobic organic contaminants in soils: fundamental concepts and techniques for analysis</div>. <div class="source">Eur. J. Soil Sci</div> . <div class="volume">54</div>, <div class="fpage">809</div>–<div class="lpage">818</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioavailability%20of%20hydrophobic%20organic%20contaminants%20in%20soils%3A%20fundamental%20concepts%20and%20techniques%20for%20analysis&amp;author=K.T.%20Semple&amp;author=A.W.J.%20Morriss&amp;author=G.I.%20Paton&amp;publication_year=2003&amp;journal=Eur.%20J.%20Soil%20Sci&amp;volume=54&amp;pages=809-818" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1351-0754.2003.0564.x" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1351-0754.2003.0564.x" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1351-0754.2003.0564.x"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1046%2Fj.1351-0754.2003.0564.x" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Bioavailability%20of%20hydrophobic%20organic%20contaminants%20in%20soils%3A%20fundamental%20concepts%20and%20techniques%20for%20analysis&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b29"><div class="label">[29]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Colores</div>
<div class="given-names">G.M.</div>
</div>
<div class="name">
<div class="surname">Macur</div>
<div class="given-names">R.E.</div>
</div>
<div class="name">
<div class="surname">Ward</div>
<div class="given-names">D.M.</div>
</div>
<div class="name">
<div class="surname">Inskeep</div>
<div class="given-names">W.P.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">Molecular analysis of surfactant-driven microbial population shifts in hydrocarbon-contaminated soil</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">66</div>, <div class="fpage">2959</div>–<div class="lpage">2964</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20surfactant-driven%20microbial%20population%20shifts%20in%20hydrocarbon-contaminated%20soil&amp;author=G.M.%20Colores&amp;author=R.E.%20Macur&amp;author=D.M.%20Ward&amp;author=W.P.%20Inskeep&amp;publication_year=2000&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=66&amp;pages=2959-2964" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.66.7.2959-2964.2000" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.66.7.2959-2964.2000" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.66.7.2959-2964.2000"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.66.7.2959-2964.2000" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/10877792" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Molecular%20analysis%20of%20surfactant-driven%20microbial%20population%20shifts%20in%20hydrocarbon-contaminated%20soil&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b30"><div class="label">[30]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Bastiaens</div>
<div class="given-names">L.</div>
</div>
<div class="name">
<div class="surname">Springael</div>
<div class="given-names">D.</div>
</div>
<div class="name">
<div class="surname">Wattiau</div>
<div class="given-names">P.</div>
</div>
<div class="name">
<div class="surname">Harms</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">de Wachter</div>
<div class="given-names">R.</div>
</div>
<div class="name">
<div class="surname">Verachtert</div>
<div class="given-names">H.</div>
</div>
<div class="name">
<div class="surname">Diels</div>
<div class="given-names">L.</div>
</div>
</div> (<div class="year">2000</div>) <div class="article-title">Isolation of adherent polycyclic aromatic hydrocarbon (PAH)-degrading bacteria using PAH-sorbing carriers</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">66</div>, <div class="fpage">1834</div>–<div class="lpage">1843</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20adherent%20polycyclic%20aromatic%20hydrocarbon%20(PAH)-degrading%20bacteria%20using%20PAH-sorbing%20carriers&amp;author=L.%20Bastiaens&amp;author=D.%20Springael&amp;author=P.%20Wattiau&amp;author=H.%20Harms&amp;author=R.%20de%20Wachter&amp;author=H.%20Verachtert&amp;author=L.%20Diels&amp;publication_year=2000&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=66&amp;pages=1834-1843" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.66.5.1834-1843.2000" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.66.5.1834-1843.2000" target="">Search ADS</a></div><div class="xslopenurl" style=""><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.66.5.1834-1843.2000"> <a href="http://search.library.nuigalway.ie/openurl/353GAL_INST/353GAL_services_page?&amp;rft_id=info%3Adoi%2F10.1128%2FAEM.66.5.1834-1843.2000" target=""><img src="https://oup.silverchair-cdn.com/Images/OpenURLbutton/00058000_openurl_button_4662.gif"/></a> </span></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/10788347" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Isolation%20of%20adherent%20polycyclic%20aromatic%20hydrocarbon%20(PAH)-degrading%20bacteria%20using%20PAH-sorbing%20carriers&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div class="ref-content" role="b31"><div class="label">[31]</div><div class="citation mixed-citation">
<div class="person-group">
<div class="name">
<div class="surname">Wagner-Döbler</div>
<div class="given-names">I.</div>
</div>
<div class="name">
<div class="surname">Bennasar</div>
<div class="given-names">A.</div>
</div>
<div class="name">
<div class="surname">Vancanneyt</div>
<div class="given-names">M.</div>
</div>
<div class="name">
<div class="surname">Strömpl</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Brümmer</div>
<div class="given-names">I.</div>
</div>
<div class="name">
<div class="surname">Eichner</div>
<div class="given-names">C.</div>
</div>
<div class="name">
<div class="surname">Grammel</div>
<div class="given-names">I.</div>
</div>
<div class="name">
<div class="surname">Moore</div>
<div class="given-names">E.R.B.</div>
</div>
</div> (<div class="year">1998</div>) <div class="article-title">Microcosm enrichment of biphenyl-degrading microbial communities from soils and sediments</div>. <div class="source">Appl. Environ. Microbiol</div> . <div class="volume">64</div>, <div class="fpage">3014</div>–<div class="lpage">3022</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microcosm%20enrichment%20of%20biphenyl-degrading%20microbial%20communities%20from%20soils%20and%20sediments&amp;author=I.%20Wagner-D%C3%B6bler&amp;author=A.%20Bennasar&amp;author=M.%20Vancanneyt&amp;author=C.%20Str%C3%B6mpl&amp;author=I.%20Br%C3%BCmmer&amp;author=C.%20Eichner&amp;author=I.%20Grammel&amp;author=E.R.B.%20Moore&amp;publication_year=1998&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=64&amp;pages=3014-3022" target="">Google Scholar</a></div><div class="pub-id"><a class="link link-pub-id" href="http://www.ncbi.nlm.nih.gov/pubmed/9687466" target="">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a href="https://www.worldcat.org/search?q=ti:Microcosm%20enrichment%20of%20biphenyl-degrading%20microbial%20communities%20from%20soils%20and%20sediments&amp;qt=advanced&amp;dblist=638" target="">WorldCat</a></div> </div></div></div></div></div></div>
<div class="copyright copyright-statement">© 2005 Federation of European Microbiological Societies.</div>
<span data-userhasaccess="True" id="UserHasAccess"></span>
</div>
</div>
</div>
<div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
</div>
<div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
</div>
<div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
</div>
<div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
</div>
<div id="ContentTabFilteredView"></div>
<div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
<a class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=565169&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5383" id="lnkDownloadAllImages">Download all figures</a>
</div>
<div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
</div>
<div class="comments">
<div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
</div>
<div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
</div>
</div>
</div>
</div>
</div>
</div>
<div class="page-column page-column--right" id="Sidebar">
<div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
<div data-google-query-id="CN7F4L7zs-MCFeVYFQgdzq4DWw" id="adBlockMainBodyTop" style=" ">
<div id="google_ads_iframe_/116097782/femsec_Behind_Ad2_0__container__" style="border: 0pt none;"><iframe data-google-container-id="2" data-load-complete="true" frameborder="0" height="250" id="google_ads_iframe_/116097782/femsec_Behind_Ad2_0" marginheight="0" marginwidth="0" name="google_ads_iframe_/116097782/femsec_Behind_Ad2_0" scrolling="no" srcdoc="" style="border: 0px; vertical-align: bottom;" title="3rd party ad content" width="300"></iframe></div></div>
</div>
<div class="widget widget-dynamic" data-count="1">
<div class="widget-dynamic-inner-wrap">
<div class="widget widget-dynamic" data-count="9">
<div class="widget-dynamic-inner-wrap">
<div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
<div class="artmet-condensed-wrap clearfix">
<div class="artmet-condensed-stats clearfix">
<div class="artmet-item artmet-views">
<span class="artmet-number">732</span>
<span class="artmet-text">Views</span>
</div>
<div class="artmet-item artmet-citations">
<span class="artmet-number">
<a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000231548300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">76</a>
</span>
<span class="artmet-text">Citations</span>
</div>
<div class="artmet-item artmet-altmetric">
<div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
<div class="altmetric-embed" data-badge-type="donut" data-doi="10.1016/j.femsec.2005.02.005" data-hide-no-mentions="false" data-uuid="3540a6b0-fc1a-7914-05ab-64449c148612" id="altmetricEmbedId" runat="server"><a href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=3290718" style="display:inline-block;" target="">
<img alt="Article has an altmetric score of 3" height="64px" src="https://badges.altmetric.com/?size=128&amp;score=3&amp;types=wwwwwwww" style="border:0; margin:0; max-width: none;" width="64px"/>
</a></div>
<script src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js" type="text/javascript"></script>
</div>
</div>
</div>
<div class="artmet-modal-trigger-wrap clearfix">
<a class="artmet-modal-trigger" data-article-id="565169"><i class="icon-metrics"></i><span>View Metrics</span></a>
</div>
</div>
<div class="artmet-modal" id="MetricsModal">
<div class="artmet-modal-contents">
<a class="artmet-close-modal">×</a>
</div>
</div>
</div>
<div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
<div class="alertsWidget">
<h3>Email alerts</h3>
<div class="userAlert alertType-5">
<a aria-label="New issue alert" data-alerttype="5" data-userloggedin="False" role="link">New issue alert</a>
</div>
<div class="userAlert alertType-30">
<a aria-label="In progress issue alert" data-alerttype="30" data-userloggedin="False" role="link">In progress issue alert</a>
</div>
<div class="userAlert alertType-3">
<a aria-label="Advance article alerts" data-alerttype="3" data-userloggedin="False" role="link">Advance article alerts</a>
</div>
<div class="userAlert alertType-1">
<a aria-label="Article activity alert" data-alerttype="1" data-userloggedin="False" role="link">Article activity alert</a>
</div>
<div class="userAlert alertType-MarketingLink">
<a aria-label="Receive exclusive offers and updates from Oxford Academic" data-additionalurl="/my-account/communication-preferences" data-userloggedin="False" role="link">Receive exclusive offers and updates from Oxford Academic</a>
</div>
<div class="userAlertSignUpModal reveal-modal small" data-reveal="">
<div class="userAlertSignUp"></div>
<label class="screenreader-text" for="alert-signup-modal-close">Close</label>
<a class="close-reveal-modal" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;" id="alert-signup-modal-close" role="button">
<i class="icon-general-close"></i>
</a>
</div>
</div>
</div>
<div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
</div>
<div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
<h3 class="article-widget-head">More on this topic</h3>
<div class="widget-related-content">
<div class="article-title"><a href="https://academic.oup.com/femsec/femsec/article/80/3/534/441493">Symbiotic characterization and diversity of rhizobia associated with native and introduced acacias in arid and semi-arid regions in Algeria</a></div>
<div class="article-title"><a href="https://academic.oup.com/femsec/article/88/2/424/471269">Plant age and genotype affect the bacterial community composition in the tuber rhizosphere of field-grown sweet potato plants</a></div>
<div class="article-title"><a href="https://academic.oup.com/femsec/article/71/2/197/468164">Chitinase genes revealed and compared in bacterial isolates, DNA extracts and a metagenomic library from a phytopathogen-suppressive soil</a></div>
<div class="article-title"><a href="https://academic.oup.com/femsec/article/59/1/71/479963">Archaeal diversity in Icelandic hot springs</a></div>
</div>
</div>
<div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
<h3 class="relatedArticleIn-title">Related articles in</h3>
<div class="relatedArticleIn-content">
<ul>
<li>
<a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000231548300003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
</li>
<li>
<a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2ffemsec%2farticle%2f54%2f1%2f21%2f565169">Google Scholar</a>
</li>
</ul>
</div>
</div>
<div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
<div class="rail-widget_wrap vt-articles-cited-by">
<h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
<div class="article-cited-link-wrap web-of-science">
<a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000231548300003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (76)</a>
</div>
<div class="article-cited-link-wrap google-scholar-url">
<a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Ffemsec%2Farticle%2F54%2F1%2F21%2F565169" target="">Google Scholar</a>
</div>
<div class="article-cited-link-wrap crossref-citedby">
<a href="https://academic.oup.com/femsec/CrossRef-CitedBy/565169">CrossRef</a>
</div>
</div>
</div>
</div>
<div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
<ul class="articleListNewAndPopularCombinedView">
<li>
<a class="articleListNewAndPopular-mode active" data-mode="MostRecent" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Latest</a>
</li>
<li>
<a class="articleListNewAndPopular-mode" data-mode="MostRead" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Most Read</a>
</li>
<li>
<a class="articleListNewAndPopular-mode" data-mode="MostCited" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;">Most Cited</a>
</li>
</ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">
<div class="fb" id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
<div class="widget-layout widget-layout--vert">
<div class="widget-columns widget-col-1">
<div class="col">
<div class="widget-dynamic-entry">
<a href="https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz113/5531308?searchresult=1">
<div class="widget-dynamic-journal-title">
                                   Multiple factors drive the abundance and diversity of diazotrophic community in typical farmland soils of China
                               </div>
<div class="widget-dynamic-journal-image-synopsis">
<div class="widget-dynamic-journal-synopsis">
</div>
</div>
</a>
</div>
<div class="widget-dynamic-entry">
<a href="https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz114/5531307?searchresult=1">
<div class="widget-dynamic-journal-title">
                                   Primed to be strong, primed to be fast: modeling benefits of microbial stress responses
                               </div>
<div class="widget-dynamic-journal-image-synopsis">
<div class="widget-dynamic-journal-synopsis">
</div>
</div>
</a>
</div>
<div class="widget-dynamic-entry">
<a href="https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz096/5531306?searchresult=1">
<div class="widget-dynamic-journal-title">
                                   Microbial communities in a dynamic  In Vitro  model for the human ileum resemble the human ileal microbiota
                               </div>
<div class="widget-dynamic-journal-image-synopsis">
<div class="widget-dynamic-journal-synopsis">
</div>
</div>
</a>
</div>
<div class="widget-dynamic-entry">
<a href="https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz112/5530754?searchresult=1">
<div class="widget-dynamic-journal-title">
                                   The impact of mixtures of xylose and glucose on the microbial diversity and fermentative metabolism of sequencing-batch or continuous enrichment cultures
                               </div>
<div class="widget-dynamic-journal-image-synopsis">
<div class="widget-dynamic-journal-synopsis">
</div>
</div>
</a>
</div>
<div class="widget-dynamic-entry">
<a href="https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiz104/5530753?searchresult=1">
<div class="widget-dynamic-journal-title">
                                   Aerobic anoxygenic phototrophs are highly abundant in hypertrophic and polyhumic waters
                               </div>
<div class="widget-dynamic-journal-image-synopsis">
<div class="widget-dynamic-journal-synopsis">
</div>
</div>
</a>
</div>
</div>
</div>
</div></div>
</section>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
</section>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
</section>
</div>
</div>
</div>
</div>
</div> <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
<div data-google-query-id="CN_F4L7zs-MCFeVYFQgdzq4DWw" id="adBlockMainBodyBottom" style=" ">
<div id="google_ads_iframe_/116097782/femsec_Behind_Ad3_0__container__" style="border: 0pt none;"><iframe data-google-container-id="3" data-load-complete="true" frameborder="0" height="250" id="google_ads_iframe_/116097782/femsec_Behind_Ad3_0" marginheight="0" marginwidth="0" name="google_ads_iframe_/116097782/femsec_Behind_Ad3_0" scrolling="no" srcdoc="" style="border: 0px; vertical-align: bottom;" title="3rd party ad content" width="300"></iframe></div></div>
</div>
</div>
</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Effects of the inoculant strain Pseudomonas putida KT2442 (pNF142) and of naphthalene contamination on the soil bacterial community | FEMS Microbiology Ecology | Oxford Academic"/>
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29"/>
</div>
</section>
</div>
<div class="mobile-mask">
</div>
<section class="footer_wrap vt-site-footer">
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit" type="text/javascript"></script>
<script src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/UI/app/jsdist/article-page.min.js" type="text/javascript"></script>
<script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1016/j.femsec.2005.02.005");
            googletag.enableServices();
        });
    </script>
<div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
<div class="ad-banner ad-banner-footer">
<div class="widget widget-AdBlock widget-instance-FooterAd">
<div data-google-query-id="CODF4L7zs-MCFeVYFQgdzq4DWw" id="adBlockFooter" style=" ">
<div id="google_ads_iframe_/116097782/femsec_Behind_Ad4_0__container__" style="border: 0pt none;"><iframe data-google-container-id="4" data-load-complete="true" frameborder="0" height="90" id="google_ads_iframe_/116097782/femsec_Behind_Ad4_0" marginheight="0" marginwidth="0" name="google_ads_iframe_/116097782/femsec_Behind_Ad4_0" scrolling="no" srcdoc="" style="border: 0px; vertical-align: bottom;" title="3rd party ad content" width="728"></iframe></div>
</div>
</div>
</div>
<div class="journal-footer journal-bg">
<div class="center-inner-row">
<div class="journal-footer-menu">
<ul>
<li class="link-1">
<a href="https://academic.oup.com/femsec/pages/About">About FEMS Microbiology Ecology</a>
</li>
<li class="link-2">
<a href="https://academic.oup.com/femsec/pages/Editorial_Board">Editorial Board</a>
</li>
<li class="link-3">
<a href="https://academic.oup.com/femsec/pages/Policies">Policies</a>
</li>
<li class="link-4">
<a href="https://academic.oup.com/femsec/pages/General_Instructions">Author Guidelines</a>
</li>
<li class="link-5">
<a href="http://fems-microbiology.org/contact/">Contact Us</a>
</li>
</ul><ul> <li class="link-1">
<a href="https://www.facebook.com/FEMSmicro">Facebook</a>
</li>
<li class="link-2">
<a href="https://twitter.com/FEMSTweets">Twitter</a>
</li>
<li class="link-3">
<a href="https://www.linkedin.com/company/federation-of-european-microbiological-societies">LinkedIn</a>
</li>
<li class="link-4">
<a href="https://academic.oup.com/femsec/subscribe">Purchase</a>
</li>
<li class="link-5">
<a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
</li>
</ul><ul> <li class="link-1">
<a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
</li>
<li class="link-2">
<a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
</li>
</ul>
</div>
<div class="journal-footer-affiliations">
<a href="https://academic.oup.com/" target="">
<img alt="FEMS Microbiology Ecology" class="journal-footer-affiliations-logo" id="footer-logo-FEMSMicrobiologyEcology" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/femsec/femsec_f1-1662987418.svg"/>
</a>
</div>
<div class="journal-footer-colophon">
<ul>
<li>Online ISSN 1574-6941</li>
<li>Print ISSN 0168-6496</li>
<li>Copyright © 2019 Federation of European Microbiological Societies</li>
</ul>
</div>
</div>
</div>
</div>
<div class="oup-footer">
<div class="center-inner-row">
<div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
<a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
</li>
<li>
<a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
</li>
<li>
<a href="http://global.oup.com/jobs">Careers</a>
</li>
<li>
<a href="https://academic.oup.com/journals/pages/help">Help</a>
</li>
<li>
<a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
<li>
<a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
</li>
<li>
<a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
</li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 class="title footerheader" id="global-footer-customer services">Connect</h4>
<ul>
<li>
<a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
</li>
<li>
<a href="http://blog.oup.com/">OUPblog</a>
</li>
<li>
<a href="http://twitter.com/oupacademic">Twitter</a>
</li>
<li>
<a href="https://www.facebook.com/OUPAcademic">Facebook</a>
</li>
<li>
<a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
</li>
<li>
<a href="http://oupacademic.tumblr.com/">Tumblr</a>
</li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
<li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
<li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
<li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
<li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
<li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
<li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
<li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
<li><a href="https://www.epigeum.com/">Epigeum</a></li>
<li><a href="http://global.oup.com/">OUP Worldwide</a></li>
<li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img alt="Oxford University Press" class="journal-footer-logo" height="42" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" width="150"/>
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
<li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
<li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
<li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
<li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
<li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
<li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>
</div>
</div>
<div class="ss-ui-only">
</div>
</section>
</div><div id="_atssh" style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;"><iframe id="_atssh320" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.14953320293139893&amp;iit=1563089859638&amp;tmr=load%3D1563089859571%26core%3D1563089859628%26main%3D1563089859631%26ifr%3D1563089859643&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Ffemsec%2Farticle%2F54%2F1%2F21%2F565169&amp;href=https%3A%2F%2Facademic.oup.com%2Ffemsec%2Farticle%2F54%2F1%2F21%2F565169&amp;dt=Effects%20of%20the%20inoculant%20strain%20%20Pseudomonas%20putida%20%20KT2442%20(pNF142)%20and%20of%20naphthalene%20contamination%20on%20the%20soil%20bacterial%20community&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5d2adbc3bf2dbef4&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DEffects%2520of%2520the%2520inoculant%2520strain%2520%2520Pseudomonas%2520putida%2520%2520KT2442%2520(pNF142)%2520and%2520of%2520naphthalene%2520contamination%2520on%2520the%2520soil%2520bacterial%2520community%26description%3DAbstract.%2520%2520The%2520naphthalene-degrading%2520activity%2520of%2520a%2520Pseudomonas%2520sp.%2520strain%2520isolated%2520from%2520a%2520creosote-contaminated%2520soil%2520was%2520shown%2520to%2520be%2520encoded%2520by%2520the%2520IncP9%2520plasmi%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Ffemsec%252Farticle%252F54%252F1%252F21%252F565169%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Ffemsec%252FIssue%252F54%252F1%252F1%252Fm_cover.gif%253FExpires%253D1626161848%2526Signature%253DKW22UbYf3qHhWZwZSvy8DC1XolrA0-qIYmg~BwaUXKpZc5rmc~mUJ6vVMTkH0tyHUBqIcZMdJ8YkIyKAu2wI0-SFc043zbzAR3Nxu0OMdLNd2lAtz02gCdK0MxIwiaDxCq1SbH3VcdytVzABdDO42OCVqm4dPl-BF6Wj~RkonZiJTwzA~lrEw4i8yHslnhjZbAg4Sd6SQClaePge8dGssrcCW~zrFOmk65eAMskEhlChgtHY0u7496f4Fo5tCnak-NfXxC80JoiXtgahk~E0W7kaOvuckaD8LDGWjOwXH6jrqCtiaVaQGvAc9lmjE5DtEKIphwW5ry1wqpp1EvugOQ__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2005-09-01%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.4.11-wp&amp;ct=1&amp;xld=1&amp;xd=1" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px; left: 0px;" title="AddThis utility frame"></iframe></div><style id="service-icons-0"></style>
<div class="reveal-modal" data-reveal="" id="revealModal">
<div id="revealContent"></div>
<a class="close-reveal-modal" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
<div class="reveal-modal small" data-reveal="" id="NeedSubscription">
<div class="subscription-needed">
<h5>This Feature Is Available To Subscribers Only</h5>
<p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5383&amp;returnUrl=%2ffemsec%2farticle%2f54%2f1%2f21%2f565169">Create an Account</a></p>
</div>
<a class="close-reveal-modal" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
<div class="reveal-modal tiny" data-reveal="" id="noAccessReveal">
<p>This PDF is available to Subscribers Only</p>
<a class="hide-for-article-page" data-reveal="" id="articleLinkToPurchase"><span>View Article Abstract &amp; Purchase Options</span></a>
<div class="issue-purchase-modal">
<p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
</div>
<a class="close-reveal-modal" href="https://academic.oup.com/femsec/article/54/1/21/javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/UI/app/jsdist/footer.min.js" type="text/javascript"></script>
<script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
<noscript>
<img alt="Scholarly IQ" border="0" height="1" hspace="0" src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" vspace="0" width="1"/>
</noscript>
<div class="reveal-modal" data-reveal="math-jax-load" id="math-jax-modal" style="display: none;"></div><div id="oupcookiepolicy_fancybox-tmp"></div><div id="oupcookiepolicy_fancybox-loading"><div></div></div><div id="oupcookiepolicy_fancybox-overlay"></div><div class="oupcookiepolicy_fancybox-ie" id="oupcookiepolicy_fancybox-wrap" style=""><div id="oupcookiepolicy_fancybox-outer"><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-n"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-ne"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-e"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-se"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-s"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-sw"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-w"></div><div class="oupcookiepolicy_fancybox-bg" id="oupcookiepolicy_fancybox-bg-nw"></div><div id="oupcookiepolicy_fancybox-content"></div><a id=""></a><div id="oupcookiepolicy_fancybox-title"></div><a href="https://academic.oup.com/femsec/article/54/1/21/javascript:;" id="oupcookiepolicy_fancybox-left"><span class="fancy-ico" id="oupcookiepolicy_fancybox-left-ico"></span></a><a href="https://academic.oup.com/femsec/article/54/1/21/javascript:;" id="oupcookiepolicy_fancybox-right"><span class="fancy-ico" id="oupcookiepolicy_fancybox-right-ico"></span></a></div></div><script src="https://api.altmetric.com/v1/doi/10.1016/j.femsec.2005.02.005?callback=_altmetric.embed_callback&amp;domain=academic.oup.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=8-14" type="text/javascript"></script><iframe id="google_osd_static_frame_3803035524164" name="google_osd_static_frame" style="display: none; width: 0px; height: 0px;"></iframe></body></html>